{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "import numpy as np\n",
    "import pandas as pd\n",
    "import json\n",
    "from copy import deepcopy\n",
    "import re\n",
    "# -*- coding:utf-8 -*-"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 训练集"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "tags": []
   },
   "outputs": [],
   "source": [
    "anno_path = \"../5.Orgnize/文献标注—校对\"\n",
    "anno_l = [ele for ele in os.listdir(anno_path) if 'add' in ele]\n",
    "txt_path = \"../4.Label_prepare/paper_ld_old/\"\n",
    "\n",
    "train_set = {\"data\":[{'paragraphs':[], 'title':\"LungB\"}], \"version\":\"LungB\"}\n",
    "train_qa = []"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "tags": []
   },
   "outputs": [],
   "source": [
    "for anno in anno_l:\n",
    "    \n",
    "    print(anno)\n",
    "    df = pd.read_excel(\"%s/%s\" % (anno_path, anno), header=0) # 读取biao\n",
    "    df[\"Title\"] = 0 # 添加文献的信息用于后续的识别\n",
    "    df[\"Ab\"] = 0\n",
    "    df[\"Full\"] = 0\n",
    "    df.loc[df[\"是否是致病菌\"]==\"是\", \"是否是致病菌\"] = \"yes\"\n",
    "    df.loc[df[\"是否是致病菌\"]==\"不是\", \"是否是致病菌\"] = \"no\"\n",
    "    df.loc[df[\"是否是致病菌\"]==\"无法确定\", \"是否是致病菌\"] = \"not sure\"\n",
    "\n",
    "    for idx in df.index.tolist():\n",
    "        \n",
    "        txt = open(\"%s/%s/%s.txt\" % (txt_path, anno.split(\"_\")[0], df.loc[idx, \"ID\"])).readlines() # \n",
    "        df.loc[idx, \"Title\"] = txt[2]\n",
    "        df.loc[idx, \"Ab\"] = txt[3]\n",
    "        df.loc[idx, \"Full\"] = txt[2] + txt[3]\n",
    "        if df.loc[idx, \"是否是致病菌\"] in ['yes', 'no']:\n",
    "            im = (df.loc[idx, \"是否是致病菌\"] != 'yes')\n",
    "            qa = {'qas': [{\n",
    "                'id': '%s:%s' % (df.loc[idx, \"ID\"], df.loc[idx, \"物种名称\"]),\n",
    "                'question': 'Can %s cause Pneumonia?' % df.loc[idx, \"物种名称\"],\n",
    "                'is_impossible': im,\n",
    "                'answers': '%s' % df.loc[idx, \"是否是致病菌\"]}],\n",
    "                  'context': '%s' % df.loc[idx, \"Full\"]}\n",
    "            train_qa.append(qa)\n",
    "\n",
    "#     df.to_csv(\"input/train/%s.csv\" % anno.split(\"_\")[0], header=True, index=False)\n",
    "train_set['data'][0][\"paragraphs\"] = train_qa\n",
    "# json.dump(train_set, open(\"input/train_set.json\", \"w\"))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 验证"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    },
    "tags": []
   },
   "outputs": [],
   "source": [
    "import random\n",
    "from random import choice \n",
    "random.seed(10)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 57,
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    },
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[{'qas': [{'id': '23548110:Nocardia veterana', 'question': 'Can Nocardia veterana cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Nocardia veterana endogenous endophthalmitis in a cardiac transplant patient.\\nABSTRACT:\\n'}, {'qas': [{'id': '27995010:Tsukamurella tyrosinosolvens', 'question': 'Can Tsukamurella tyrosinosolvens cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Tsukamurella tyrosinosolvens bacteremia with coinfection of Mycobacterium bovis pneumonia: case report and literature review.\\nABSTRACT:INTRODUCTION: We describe an immunocompromised patient with Tsukamurella tyrosinosolvens bacteremia and coinfection of Mycobacterium bovis pneumonia. CASE DESCRIPTION: A 75-year-old male was admitted to our hospital complaining of persistent fever with general malaise. His medical history showed that he had diabetes mellitus (HbA1C 9.2%). A chest computed tomography (CT) showed left upper lung consolidation . Two sets of blood culture at admission finally showed Tsukamurella tyrosinosolvens. Moreover, three transbronchoscopy washing specimen cultures revealed Mycobacterium bovis. DISCUSSION AND EVALUATION: The organism Tsukamurella tyrosinosolvens was identified using conventional biochemical identification methods, PCR-restriction DNA fragment analysis, and 16S rRNA gene sequencing. The clinical mycobacterial isolates were identified to the species level by combining Polymerase Chain Reaction (PCR) with an oligonucleotide microarray to detect the M. bovis amplicons. \\n'}, {'qas': [{'id': '27388158:Ehrlichia ewingii', 'question': 'Can Ehrlichia ewingii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Interstitial pneumonia and pulmonary hypertension associated with suspected ehrlichiosis in a dog.\\nABSTRACT:\\n'}, {'qas': [{'id': '21461846:Nocardia carnea ', 'question': 'Can Nocardia carnea  cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Clinical and microbiological characteristics of infections caused by various Nocardia species in Taiwan: a multicenter study from 1998 to 2010.\\nABSTRACT:This multicenter study in Taiwan investigated the clinical presentations of various Nocardia species infections based on 16S rRNA sequence analysis. Patients with nocardiosis in four large medical centers from 1998 to 2010 were included. A total of 100 preserved nonduplicate isolates causing human infection were identified as Nocardia species. Sequencing analysis of 16S rRNA confirmed that 35 of 36 N. asteroides isolates identified by conventional tests were non-asteroides Nocardia species, and that two of 50 N. brasiliensis isolates had also been initially misidentified. N. brasiliensis (50%) was the most common pathogen, followed by N. cyriacigeorgica (18%). In addition, several rare pathogens were identified, including N. asiatica, N. rhamnosiphila, N. abscessus, N. transvalensis, N. elegans, and N. carnea. Primary cutaneous infection was the most common presentation, noted in 55 (55%) patients, while pulmonary infection presented in 26 (26%) patients. The crude mortality rate was 6.7% (6/89), and was lowest for primary cutaneous infection (2.2%) and highest for disseminated disease and pulmonary infection (16.7%). In conclusion, N. brasiliensis and N. cyriacigeorgica were the most common pathogens causing nocardiosis in Taiwan. Molecular methods for identifying Nocardia to the species level are mandatory for better understanding the epidemiology and clinical characteristics of patients with nocardiosis.'}, {'qas': [{'id': '32189223:Bordetella trematum', 'question': 'Can Bordetella trematum cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': \"TITLE:Isolation of Bordetella trematum from the respiratory tract of a patient with lung cancer: a case report.\\nABSTRACT:We report the case of isolation of Bordetella trematum from the respiratory tract of a patient with lung carcinoma. This gram-negative, opportunistic rod was firstly described in 1996. To date, only several strains of Bordetella trematum have been isolated and reported, mostly from skin and soft tissue infections. The patient was admitted to the ICU of the Pulmonary Department in incipient septic shock with respiratory failure. Intravenous fluid resuscitation and non-invasive ventilation were administered immediately. A broad spectrum antibiotic piperacillin/tazobactam was administered empirically after sampling of material for microbiological examination. The bronchoscopy showed a large cavern of decayed tumour invading into mediastinum. Both sample cultures showed significant quantities of gram-negative non-fermenting bacteria. The isolate was identified using MALDI-TOF MS as Bordetella trematum and the identification was confirmed using 16S ribosomal RNA sequencing. In the last few years, routine bacterial identification using MALDI-TOF MS has enabled correct discrimination of this species. Nevertheless, isolation of Bordetella trematum in clinical samples is still very uncommon, and it is appropriate to confirm the species identification via 16S ribosomal RNA sequencing. To our knowledge, this is the first case of B. trematum isolated from the human respiratory tract since its first description. The clinical significance of Bordetella trematum in the rapid deterioration of the patient's status remains unclear.\"}, {'qas': [{'id': '30027827:Bartonella henselae', 'question': 'Can Bartonella henselae cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Necrotizing interstitial pneumonia and suppurative myocarditis associated with Bartonella henselae infection in three Florida pumas.\\nABSTRACT:Three Florida pumas ( Puma concolor coryi) that had spent time in captivity prior to being released in the wild were found exhibiting respiratory signs and reluctance to move. All 3 pumas died shortly after immobilization, despite supportive veterinary care. Significant autopsy findings included necrotizing interstitial pneumonia, with pulmonary edema and hyaline membranes, and suppurative myocarditis. Organisms morphologically consistent with Bartonella henselae were identified in intravascular histiocytes in the lung of one of the pumas on histopathology and confirmed via transmission electron microscopy. B. henselae was detected in fresh lung tissue and confirmed by PCR and sequence analysis (16S-23S spacer region, pap31, and rpoB genes) from one of the affected pumas. In all affected pumas, B. henselae was detected by PCR in formalin-fixed, paraffin-embedded lung tissue, and positively staining organisms were identified in sections of lung by immunohistochemistry for B. henselae. In situ hybridization detected B. henselae DNA in lung tissue from 2 of 3 affected pumas. Our case series suggests that B. henselae can be associated with a fatal disease syndrome in Florida pumas. The cause of susceptibility to fatal disease associated with B. henselae infection in these pumas remains unknown.'}, {'qas': [{'id': '677536:Acholeplasma laidlawii', 'question': 'Can Acholeplasma laidlawii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Isolation of Mycoplasma arginini from swine and from a swine waste disposal system.\\nABSTRACT:Mycoplasma arginini was isolated from the pharynges and tonsils of 4 of 6 swine necropsied during an epizootic of respiratory tract diseases and lameness. The organism was also isolated from 2 of 16 asymptomatic swine examined in the same herd approximately 10 months after the epizootic. In addition, M arginini, Acholeplasma laidlawii, and other acholeplasmas were isolated from recycled effluent of an anaerobic lagoon used for disposal of waste from the swine herd. This observation represents the 1st report of a Mycoplasma species isolated from a lagoon or sewage system. The pathogenicity of a strain of M arginini isolated during the epizootic was examined by inoculating young swine intranasally, intratracheally, intravenously, and intraperitoneally. The agent colonized in the pharynges of 2 pigs inoculated intranasally, but did not produce overt signs of clinical disease. Although neutralizing antibodies to M arginini were not detected, 1 animal that had been infected naturally and 2 that had been inoculated intravenously did develop complement-fixing antibodies. The findings indicate that M arginini can colonize in the nasal and pharyngeal mucosae of swine, but that it is not highly pathogenic for these animals.'}, {'qas': [{'id': '25224198:Nocardia elegans', 'question': 'Can Nocardia elegans cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': \"TITLE:Lung Nocardia elegans infection diagnosed on matrix-assisted laser desorption ionization-time of flight mass spectrometry (MALDI-TOF MS).\\nABSTRACT:A 73-year-old man with adultonset Still's disease developed a high fever, coughing, dyspnea and bloody sputum and was therefore admitted to our hospital. Thoracic X-ray and CT scans revealed oval lesions in the bilateral lungs. A bacterial isolate from the sputum was identified to be Nocardia elegans (N. elegans) on comparative 16S rRNA gene sequencing and Matrix-Assisted Laser Desorption Ionization-Time of Flight Mass Spectrometry (MALDI-TOF MS). The patient recovered following treatment with imipenem/cilastatin and amikacin. To the best of our knowledge, this is the first case of nocardiosis caused by N. elegans identified on MALDI-TOF MS. \"}, {'qas': [{'id': '27242717:Pandoraea pnomenusa', 'question': 'Can Pandoraea pnomenusa cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Pandoraea pnomenusa Isolated from an Australian Patient with Cystic Fibrosis.\\nABSTRACT:Pandoraea species are considered as emerging pathogens in people with cystic fibrosis (CF). The contribution of these organisms to disease progression in CF patients is not fully understood owing in large measure to the scant reports in clinical and research literature describing their colonization of CF patients and their associated virulence determinants. In an effort to increase awareness and evidence for Pandoraea spp. infection in people with CF, and to stimulate research aimed at unraveling the pathogenic properties of Pandoraea, we report a case of a 26-year-old Australian (Tasmanian) man with CF who was chronically infected with Pandoraea pnomenusa for at least one year prior to his death from respiratory failure. In addition, we describe for the first time evidence suggesting that this bacterium is a facultative anaerobe and report on the availability of a whole genome sequence for this organism. To the best of our knowledge, this report represents only the second clinical case study of P. pnomenusa infection in the world, and the first in an Australian CF patient. '}, {'qas': [{'id': '32909234:Finegoldia magna', 'question': 'Can Finegoldia magna cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:A rare case of emphysematous pyelonephritis caused by Candida parapsilosis and Finegoldia magna complicated by medical care avoidance.\\nABSTRACT:Emphysematous pyelonephritis (EPN) is a necrotizing gas producing infection of the renal parenchyma that commonly occurs in patients with diabetes. EPN requires early diagnosis and treatment due to the possible life-threatening septic complications. We report a rare case of EPN caused by an unfavorable mixed infection of Candida parapsilosis and Finegoldia magna. To our knowledge, this is the first reported case of EPN caused by Finegoldia magna. A 62-year-old male with diabetes mellitus (DM) presented with abdominal pain, shortness of breath, and nausea in which a diagnosis of septic shock was made due to EPN. Our patient first noticed abdominal pain 3 weeks prior to hospital presentation; however, he avoided getting treatment due to a fear of contracting severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). This patient exhibited deterioration and expired after surgical intervention despite uneventful nephrectomy. This case suggests that medical care avoidance behaviors among patients could potentially complicate their clinical course.'}, {'qas': [{'id': '30978196:Alcaligenes aquatilis', 'question': 'Can Alcaligenes aquatilis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Genomic characterization of Kerstersia gyiorum SWMUKG01, an isolate from a patient with respiratory infection in China.\\nABSTRACT:\\n'}, {'qas': [{'id': '26615404:Ochrobactrum anthropi', 'question': 'Can Ochrobactrum anthropi cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Ochrobactrum endophyticum sp. nov., isolated from roots of Glycyrrhiza uralensis.\\nABSTRACT:A novel Gram-staining negative, motile, rod-shaped and aerobic bacterial strain, designated EGI 60010(T), was isolated from healthy roots of Glycyrrhiza uralensis F. collected from Yili County, Xinjiang Province, North-West China. The 16S rRNA gene sequence of strain EGI 60010(T) showed 97.2 % sequence similarities with Ochrobactrum anthropi ATCC 49188(T) and Ochrobactrum cytisi ESC1(T), and 97.1 % with Ochrobactrum lupini LUP21(T). The phylogenetic analysis based on 16S rRNA gene sequences showed that the new isolate clustered with members of the genera Ochrobactrum, and formed a distinct clade in the neighbour-joining tree. Q-10 was identified as the respiratory quinone for strain EGI 60010(T). The major fatty acids were summed feature 8 (C18:1 omega6c and/or C18:1 omega7c), C19:0 cyclo omega8c, summed feature 4 (C17:1 iso I/anteiso B) and C16:0. The polar lipids detected were diphosphatidylglycerol, phosphatidylethanolamine, phosphatidylmethylethanolamine, phosphatidylglycerol and phosphatidylcholine. The DNA G+C content of strain EGI 60010(T) was determined to be 60.4 mol%. The genomic DNA relatedness values determined between strain EGI 60010(T) and the closely related strains O. anthropi JCM 21032(T), O. cytisi CCTCC AB2014258(T) and O. lupini NBRC 102587(T) were 50.3, 50.0 and 41.6 %, respectively. Based on the results of the molecular studies supported by its differentiating phenotypic characteristics, strain EGI 60010(T) was considered to represent a novel species within the genus Ochrobactrum, for which the name Ochrobactrum endophyticum sp. nov., is proposed. The type strain is EGI 60010(T) (=CGMCC 1.15082(T) = KCTC 42485(T) = DSM 29930(T)).'}, {'qas': [{'id': '22639319:Erwinia milletiae', 'question': 'Can Erwinia milletiae cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:[A case of ventilator-associated pneumonia due to Pantoea agglomerans].\\nABSTRACT:Pantoea species, which are the members of Enterobacteriaceae family are gram-negative bacilli that are frequently found on many plants and in soil. They may lead to localized infections in healthy subjects and systemic infections in immunosuppressed patients. In this case report a ventilator-associated pneumoniae due to Pantoea agglomerans was presented. A 55 year-old male patient with chronic renal failure was hospitalized in intensive care unit following cardiopulmonary arrest. The patient developed fever (38.8 C), had pulmonary infiltrations in chest X-ray and leucocytosis. Treatment with piperacillintazobactam was initiated upon diagnosis of nosocomial pneumoniae. Bacterial growth from the deep tracheal aspirate of the patient was identified as P.agglomerans by VITEK2 automated system (bioMerieux, France) and the identification was confirmed by conventional microbiological methods. Since the strain was susceptible to the cephoperazon-sulbactam, tobramycin, tetracycline, gentamicin and levofloxacine, the treatment was changed to levofloxacine and cephoperazon-sulbactam and the patient improved. This case was presented to withdraw attention to rare opportunistic pathogens that may lead to nosocomial infections particularly in patients with underlying diseases.'}, {'qas': [{'id': '29017944:Yersinia ruckeri', 'question': 'Can Yersinia ruckeri cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Immunological responses and disease resistance of rainbow trout (Oncorhynchus mykiss) juveniles following dietary administration of stinging nettle (Urtica dioica).\\nABSTRACT:The present study investigated the effects of dietary supplementation of stinging nettle (Urtica dioica) on growth performance, skin mucus, immune response and disease resistance of rainbow trout (Oncorhynchus mykiss) fed with diets supplemented with U. dioica at 0, 1, 2 and 3%. After 8 weeks of feeding, the addition of U. dioica at 3% level resulted in improved weight gain, specific growth rate and feed conversion ratio significantly when compared to the other groups (P &lt; 0.05). Hematological responses including: hematocrit (Htc), hemoglobin (Hb), lymphocyte and neutrophil populations enhanced significantly in fish fed 3% of stinging nettle when measured after 4 weeks; while, total red blood cells, white blood, Htc, Hb, lymphocyte and neutrophil populations significantly increased after 8 weeks in the same group (P &lt; 0.05). Total serum protein and glucose contents increased significantly in fish fed stinging nettle at 3% when compared to the other groups after 8 weeks; however, triglycerides decreased significantly in the same group on the 4th and 8th week (P &lt; 0.05). Additionally, several immune parameters, namely, IgM, lysozyme, complement components C3 and C4, and respiratory burst of blood leukocytes significantly increased in the 3% fed group on the 4th week; while, after 8 weeks the immune responses enhanced in fish fed 2 and 3% diets (P &lt; 0.05). At the end of the feeding trial, mucus samples obtained from the fish fed stinging nettle supplementation exhibited improved antagonistic activities against several bacterial pathogens (Streptococcus iniae, Yersinia ruckeri, Vibrio anguillarum and Lactococcus garviae), skin mucus enzymes activities (alkaline phosphatase, lysozyme, protease and esterase) and protein levels in 2 and 3% groups with the highest being in case of 3% group when compared to the other groups (P &lt; 0.05). The cumulative mortality of rainbow trout subjected to Y. ruckeri infectious exhibited relatively low mortality levels in all supplemented groups with the lowest being in fish fed 3% stinging nettle. The present findings demonstrated that dietary administration of U. dioica enhanced growth and stimulated fish immunity; thus, enabling the fish to be more resistant against bacterial infections.'}, {'qas': [{'id': '29631017:Sulfurospirillum', 'question': 'Can Sulfurospirillum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Electron transport chains in organohalide-respiring bacteria and bioremediation implications.\\nABSTRACT:In situ remediation employing organohalide-respiring bacteria represents a promising solution for cleanup of persistent organohalide pollutants. The organohalide-respiring bacteria conserve energy by utilizing H2 or organic compounds as electron donors and organohalides as electron acceptors. Reductive dehalogenase (RDase), a terminal reductase of the electron transport chain in organohalide-respiring bacteria, is the key enzyme that catalyzes halogen removal. Accumulating experimental evidence thus far suggests that there are distinct models for respiratory electron transfer in organohalide-respirers of different lineages, e.g., Dehalococcoides, Dehalobacter, Desulfitobacterium and Sulfurospirillum. In this review, to connect the knowledge in organohalide-respiratory electron transport chains to bioremediation applications, we first comprehensively review molecular components and their organization, together with energetics of the organohalide-respiratory electron transport chains, as well as recent elucidation of intramolecular electron shuttling and halogen elimination mechanisms of RDases. We then highlight the implications of organohalide-respiratory electron transport chains in stimulated bioremediation. In addition, major challenges and further developments toward understanding the organohalide-respiratory electron transport chains and their bioremediation applications are identified and discussed.'}, {'qas': [{'id': '30994108:Filifactor', 'question': 'Can Filifactor cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:A Preliminary Study of Microbiota Diversity in Saliva and Bronchoalveolar Lavage Fluid from Patients with Primary Bronchogenic Carcinoma.\\nABSTRACT:BACKGROUND The present study aimed to evaluate the difference in microbiota diversity in the oral cavity and fluid bronchoalveolar lavage (BALF) of patients with lung cancer. MATERIAL AND METHODS Buccal (saliva) and lower respiratory tract BALF samples were collected from 51 patients with primary bronchogenic carcinoma and 15 healthy controls, and bacterial genomic DNA was extracted. High-throughput 16S rDNA amplicon sequencing was performed, and microbial diversity, composition, and functions of microbiota were analyzed by bioinformatics methods. \\n'}, {'qas': [{'id': '30315374:Cronobacter malonaticus', 'question': 'Can Cronobacter malonaticus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Characterization of the pathogenicity of clinical Cronobacter malonaticus strains based on the tissue culture investigations.\\nABSTRACT:Cronobacter malonaticus is a member of the genus Cronobacter which is considered an opportunistic pathogen. The significance of C. malonaticus has recently increased since it was documented to be involved in several serious neonatal infections. However, the virulence factors of C. malonaticus including their ability to adhere, invade and overcome host barriers have not been studied before. Unlike previous Cronobacter research, this study is mainly focused on C. malonaticus and is aimed to investigate its virulence characteristics that enable this species to cause adult and neonatal infections. Altogether, 20 strains were included in this study (19 clinical and one environmental strain). Our data showed that the clinical C. malonaticus has an ability to adhere and invade Caco-2, HBMEC, A549 and T24 cell lines. Moreover, the result showed that certain strains of C. malonaticus (including 1827 and 2018) were able to persist well in macrophages. However, ST7 strains 1827 and 2018 proved to be the most invasive strains among all used strains. The CDC strain 1569 (ST307) which was isolated from the blood of a fatal neonatal case showed also significant results in this study as it was able to invade all used human cells and survive and replicate within microphages. Finally, the findings of this study confirm the potential ability of C. malonaticus to cause serious infections in neonates or adults such as necrotising enterocolitis, meningitis, bacteraemia, pneumonia and urinary tract infection.'}, {'qas': [{'id': '28303934:Streptomyces avermitilis', 'question': 'Can Streptomyces avermitilis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Redox-sensing regulator Rex regulates aerobic metabolism, morphological differentiation, and avermectin production in Streptomyces avermitilis.\\nABSTRACT:The regulatory role of redox-sensing regulator Rex was investigated in Streptomyces avermitilis. Eleven genes/operons were demonstrated to be directly regulated by Rex; these genes/operons are involved in aerobic metabolism, morphological differentiation, and secondary metabolism. Rex represses transcription of target genes/operons by binding to Rex operator (ROP) sequences in the promoter regions. NADH reduces DNA-binding activity of Rex to target promoters, while NAD+ competitively binds to Rex and modulates its DNA-binding activity. Rex plays an essential regulatory role in aerobic metabolism by controlling expression of the respiratory genes atpIBEFHAGDC, cydA1B1CD, nuoA1-N1, rex-hemAC1DB, hppA, and ndh2. Rex also regulates morphological differentiation by repressing expression of wblE, which encodes a putative WhiB-family transcriptional regulator. A rex-deletion mutant (Drex) showed higher avermectin production than the wild-type strain ATCC31267, and was more tolerant of oxygen limitation conditions in regard to avermectin production.'}, {'qas': [{'id': '28761296:Nocardia takedensis', 'question': 'Can Nocardia takedensis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Primary Cutaneous Nocardiosis Caused by Nocardia takedensis.\\nABSTRACT:Nocardia species are aerobic, gram-positive, filamentous, partially acid-fast actinomycetes which are found worldwide in soil and decaying organic plant matter. When they infect human beings, they generally enter through the respiratory tract and then disseminate systemically. Rarely has a primary infection occurred as the result of direct inoculation. Isolation of Nocardia from clinical specimens and identification of species are difficult. But, with the introduction of new genetic technologies, reports of novel species of Nocardia have increased. We describe a case of cutaneous nocardiosis caused by Nocardia takedensis in an 87-year-old woman who was diagnosed by bacterial culture and 16S ribosomal RNA sequencing. N. takedensis has been described as a new species. This report describes the first clinical isolate of N. takedensis from a skin specimen in Korea.'}, {'qas': [{'id': '16512684:Mycoplasma penetrans', 'question': 'Can Mycoplasma penetrans cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Proteome of the bacterium Mycoplasma penetrans.\\nABSTRACT:A proteome map of Mycoplasma penetrans has been constructed using two-dimensional gel electrophoresis in combination with mass spectrometry (MS). Mycoplasma penetrans infects the urogenital and respiratory tracts of humans. A total of 207 spots were characterized with MS and, in comparing the experimental data with the DNA sequence-derived predictions, it was possible to assign these 207 spots to 153 genes. The Pro-Q Diamond phosphoprotein dye technology was used for the fluorescent detection of 26 phosphoproteins in the 4-7 pH range.'}, {'qas': [{'id': '6410947:Clostridium bifermentans', 'question': 'Can Clostridium bifermentans cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Necrotizing pneumonia and empyema caused by Bacillus cereus and Clostridium bifermentans.\\nABSTRACT:'}, {'qas': [{'id': '27388158:Ehrlichia canis', 'question': 'Can Ehrlichia canis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Interstitial pneumonia and pulmonary hypertension associated with suspected ehrlichiosis in a dog.\\nABSTRACT:\\n'}, {'qas': [{'id': '26904200:Achromobacter insuavis', 'question': 'Can Achromobacter insuavis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Distribution of innate efflux-mediated aminoglycoside resistance among different Achromobacter species.\\nABSTRACT:Achromobacter spp. are emerging respiratory pathogens in cystic fibrosis patients. Since 2013 the genus Achromobacter includes 15 species for which innate antibiotic resistance is unknown. Previously the AxyXY-OprZ efflux system has been described to confer aminoglycoside (AG) resistance in A. xylosoxidans. Nevertheless, some Achromobacter spp. strains are susceptible to AG. This study including 49 Achromobacter isolates reveals that AG resistance is correlated with different Achromobacter spp. It is noteworthy that the axyXY-oprZ operon is detected only in AG-resistant species, including the most frequently encountered in cystic fibrosis patients: A. xylosoxidans, A. ruhlandii, A. dolens and A. insuavis. '}, {'qas': [{'id': '14657137:Staphylococcus arlettae', 'question': 'Can Staphylococcus arlettae cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Staphylococcus nepalensis sp. nov., isolated from goats of the Himalayan region.\\nABSTRACT:Four coagulase-negative, novobiocin-resistant cocci, designated CW1(T), PM34, MM3 and RW78, were isolated from the respiratory tract of goats kept in the Himalayan region. The four isolates were assigned to a single species on the basis of almost identical biochemical and physiological traits, protein profiles obtained after SDS-PAGE and identical genomic fingerprints generated after enterobacterial repetitive intergenic consensus (ERIC)-PCR. Strain CW1(T) showed highest 16S rDNA sequence similarities to Staphylococcus cohnii subsp. urealyticus ATCC 49330(T), Staphylococcus saprophyticus subsp. saprophyticus ATCC 15305(T), S. cohnii subsp. cohnii ATCC 29974(T), Staphylococcus arlettae ATCC 43957(T), Staphylococcus gallinarum ATCC 35539(T), Staphylococcus succinus ATCC 700337(T) and Staphylococcus xylosus ATCC 29971(T) (99.0, 98.8, 98.8, 98.4, 98.2, 98.1 and 98.1 %, respectively), indicating its classification within the genus Staphylococcus. The polar lipid composition, fatty acid profiles, quinone systems and diagnostic cell-wall diamino acid were in agreement with the characteristics of the genus Staphylococcus. DNA-DNA hybridization with closely related Staphylococcus species suggested that strain CW1(T) represents an as-yet unrecognized species. Based on these results, a novel species of the genus Staphylococcus is described, Staphylococcus nepalensis sp. nov. The type strain is CW1(T) (=DSM 15150(T)=CCM 7045(T)) and the most dissimilar strain is PM34 (=DSM 15151=CCM 7046).'}, {'qas': [{'id': '29661119:Aeromonas sobria', 'question': 'Can Aeromonas sobria cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Combined Fatal Systemic Chlamydia sp. and Aeromonas sobria Infection in Juvenile Siamese Crocodiles ( Crocodylus siamensis).\\nABSTRACT:Chlamydial infections in crocodiles have been described in several countries and in several different species. These are typically associated with severe pharyngitis and conjunctivitis, with death occurring secondary to compromise of the upper respiratory tract due to obstruction of the trachea. A population of ranched Siamese crocodiles in central Thailand experienced an epizootic of sudden death in juvenile animals. The affected animals had fulminant systemic disease primarily involving the liver and spleen but also affecting the kidneys, heart, and the whole of the respiratory tract. Chlamydia sp. were noted in liver and spleen during histopathological examination and confirmed with transmission electron microscopy and polymerase chain reaction (PCR). The sequence of the PCR product suggested a novel Chlamydia sp. of Siamese crocodiles. Crocodile farming represents an important economy in several parts of the world. Epizootics, such as the one described in this manuscript in association with Chlamydia sp., can have devastating impact on the industry and represent a potential zoonosis of significant public health concern. This is the first report of Chlamydia sp. and Aeromonas sobria causing systemic disease in crocodiles as well as the first histopathological and ultrastructural description of Chlamydia infection in Siamese crocodiles.'}, {'qas': [{'id': '28566081:Streptococcus uberis', 'question': 'Can Streptococcus uberis cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': \"TITLE:[A rarely isolated bacterium in microbiology laboratories: Streptococcus uberis].\\nABSTRACT:Streptococcus uberis is a gram-positive bacterium that is mostly responsible for mastitis in cattle. The bacterium rarely has been associated with human infections. Conventional phenotyphic methods can be inadequate for the identification of S.uberis; and in microbiology laboratories S.uberis is confused with the other streptococci and enterococci isolates. Recently, molecular methods are recommended for the accurate identification of S.uberis isolates. The aim of this report is to present a lower respiratory tract infection case caused by S.uberis and the microbiological methods for identification of this bacterium. A 66-year-old male patient with squamous cell lung cancer who received radiotherapy was admitted in our hospital for the control. According to the chest X-Ray, patient was hospitalized with the prediagnosis of ''cavitary tumor, pulmonary abscess''. In the first day of the hospitalization, blood and sputum cultures were drawn. Blood culture was negative, however, Candida albicans was isolated in the sputum culture and it was estimated to be due to oral lesions. After two weeks from the hospitalization, sputum sample was taken from the patient since he had abnormal respiratory sounds and cough complaint. In the Gram stained smear of the sputum there were abundant leucocytes and gram-positive cocci, and S.uberis was isolated in both 5% sheep blood and chocolate agar media. Bacterial identification and antibiotic susceptibility tests were performed by VITEK 2 (Biomerieux, France) and also, the bacterium was identified by matrix assisted laser desorption/ionization time of flight mass spectrometry (MALDI-TOF MS) based VITEK MS system as S.uberis. The isolate was determined susceptible to ampicillin, erythromycin, clindamycin, levofloxacin, linezolid, penicillin, cefotaxime, ceftriaxone, tetracycline and vancomycin. 16S, 23S ribosomal RNA and 16S-23S intergenic spacer gene regions were amplified with specific primers and partial DNA sequence analysis of 16S rRNA polymerase chain reaction (PCR) products were performed by 3500xL Genetic Analyzer (Applied Biosystems, USA). According to the partial 16S rRNA gene sequencing results, bacterium was confirmed as S.uberis. This report makes a significant contribution to the number of case reports of human infections caused by S.uberis as the identification was performed by current microbiological methods in our case. In conclusion, S.uberis should be evaluated as an opportunistic pathogen among the immunosuppressed patients and in addition to phenotypic bacteriological methods, the other recent microbiological methods should also be utilized for the identification.\"}, {'qas': [{'id': '32897213:Porphyromonas intermedia', 'question': 'Can Porphyromonas intermedia cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:[Research progress of correlation between periodontal pathogens and systemic diseases].\\nABSTRACT:Periodontal pathogens are the main pathogenic factor of periodontitis. Periodontal pathogens have a large variety of virulence factors such as lipopolysaccharide, fimbriae and proteases, which enables the pathogens to infect periodontal tissues and stimulate the secretion of inflammatory cytokines, causing chronic systemic inflammation. Periodontal pathogens may invade multiple systems such as the circulatory system, immune system, respiratory system and digestive system to cause systematic diseases. Recent studies have shown that periodontal pathogens may have close relations with systemic diseases such as cardiovascular disease, diabetes, rheumatoid arthritis, and cancer. Among the periodontal pathogens, Porphyromonas gingivalis can be found in atherosclerotic plaques to impairing the function of the vascular endothelium; Porphyromonas gingivalis may also increase the level of inflammatory factors such as TNF-alpha to promote insulin resistance and diabetes. Many of the periodontal pathogens such as Porphyromonas gingivalis, Tannerella forsythia and Prevotella intermedia can be detected in the synovial fluid of rheumatoid arthritis patients, suggesting their involvement in the pathogenesis of rheumatoid arthritis. Fusobacterium nucleatum may cause alterations in the intestinal microbiome in mice and promote the occurrence of intestinal tumors. Herein we review the recent progresses in the relationship between periodontal pathogens and systemic diseases.'}, {'qas': [{'id': '19244433:Pigmentiphaga', 'question': 'Can Pigmentiphaga cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Pigmentiphaga litoralis sp. nov., a facultatively anaerobic bacterium isolated from a tidal flat sediment.\\nABSTRACT:A novel Gram-negative, facultatively anaerobic, non-sporulating, non-motile, catalase- and oxidase-positive, rod-shaped bacterium (strain JSM 061001(T)) was isolated from a tidal flat in the South China Sea, China. Growth occurred with 0-5 % (w/v) NaCl [optimum, 0.5-1 % (w/v) NaCl], at pH 5.0-10.0 (optimum, pH 7.0) and at 4-35 degrees C (optimum, 25-30 degrees C). The major cellular fatty acids were C(16 : 0), cyclo C(17 : 0), C(18 : 1)omega7c and C(16 : 1). Strain JSM 061001(T) contained ubiquinone Q-8 as the predominant respiratory quinone, and phosphatidylglycerol, phosphatidylethanolamine and an unidentified phospholipid as the polar lipids. The genomic DNA G+C content was 65.5 mol%. Phylogenetic analysis based on 16S rRNA gene sequences indicated that strain JSM 061001(T) belongs to the family Alcaligenaceae and was related most closely to the type strains of the two recognized species of the genus Pigmentiphaga. The three strains formed a robust cluster in the neighbour-joining, maximum-parsimony and maximum-likelihood phylogenetic trees. Levels of DNA-DNA relatedness between strain JSM 061001(T) and the type strains of Pigmentiphaga daeguensis and Pigmentiphaga kullae were 15.8 and 10.5 %, respectively. The combination of phylogenetic analysis, DNA-DNA hybridization data, phenotypic characteristics and chemotaxonomic differences supported the view that strain JSM 061001(T) represents a novel species of the genus Pigmentiphaga, for which the name Pigmentiphaga litoralis sp. nov. is proposed. The type strain is JSM 061001(T) (=CCTCC AA207034(T)=KCTC 22165(T)).'}, {'qas': [{'id': '3490016:Micrococcus roseus', 'question': 'Can Micrococcus roseus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Growth-enhancing effects of culture filtrates of sputum isolates on the L-forms of Haemophilus influenzae.\\nABSTRACT:The growth-enhancing effects of culture filtrates of respiratory pathogenic bacteria, including Haemophilus influenzae, as well as normal floral bacteria other than Neisseria perflava and Branhamella catarrhalis on L-forms of H. influenzae were examined in vitro, using five species of major respiratory pathogenic bacteria and seven species of normal floral bacteria commonly isolated from the sputum of patients with chronic respiratory tract infections. The growth-enhancing factor(s) was present in the filtrates prepared from the culture of respiratory pathogenic Streptococcus pneumoniae and Staphylococcus aureus, the effects of which were as potent as those of a culture filtrate of B. catarrhalis used as the positive control. The culture filtrates of respiratory pathogenic Pseudomonas aeruginosa and Klebsiella pneumoniae had weak growth-enhancing effects on H. influenzae L-forms. The culture filtrates of 21 strains of normal floral bacteria isolated from the sputum including alpha-hemolytic Streptococci, non-hemolytic Streptococci, Micrococcus roseus, coagulase-negative Staphylococci, and Neisseria spp. had growth-enhancing effects on the L-forms of H. influenzae. These data elucidate the significance of L-forms of H. influenzae in recurrent infections due to H. influenzae in patients with chronic respiratory tract infections.'}, {'qas': [{'id': '32084536:Edwardsiella tarda', 'question': 'Can Edwardsiella tarda cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Effect of cassic acid on immunity and immune-reproductive genes transcription in Clarias gariepinus against Edwardsiella tarda.\\nABSTRACT:The present study was investigated the dietary administration of cassic acid (CA) on growth, innate immunity, transcription profiles of estrogen and follicle-stimulating hormones as well as lysozyme enzyme determined in Clarias gariepinus against Edwardsiella tarda. The weight gain (WG), protein efficacy ratio (PER), and feed conversion ratio (FCR) were significantly improved in infected fish fed dietary administration with CA at 5 and 10 mg kg-1 diets. The survival is higher (96.7% and 98.3%) in the infected groups fed at 5 and 10 mg kg-1 CA diets. The red (RBC) and white (WBC) blood cells, hemoglobin (Hb), and packed cell volume (PCV) was found significantly high in the infected fish feeding at 5 and 10 mg kg-1 CA diets. Total protein and albumin were significantly increased with 5 and 10 mg kg-1 CA diets among weeks 1-4 while the globulin and albumin: globulin ratio increased of these diet only after week 2. The phagocytic and respiratory burst activities were enhanced statistically the infected fish fed at 10 mg kg-1 CA diet group whereas the production of superoxide anion (SOA) and nitric oxide (NO) were significantly increased at 5 and 10 mg kg-1 CA diets. The lymphocyte proliferation and myeloperoxidase (MPO) activity were significantly high the infected fish when fed at 5 and 10 mg kg-1 CA diets after 2nd week whereas the alternate complement activity (ACP), generation of reactive oxygen species (ROS), and lysozyme activity (Lyz) were observed at 5 and 10 mg kg-1 CA diets among weeks 1-4. The accumulative mortality was 10% in infected fish fed at 1 and 5 mg kg-1 CA diets whereas 15% mortality found with 10 mg kg-1 CA diet. The highest levels of estrogen receptor alpha (ERalpha) mRNA expression found in gonad while the highest levels of follicle stimulating hormone-beta subunit (FSH-beta) mRNA expression found in testis of the infected fish given at 5 and 10 mg kg-1 CA diets. The up-regulation of chick-type lysozyme (c-Lyz) mRNA was observed at 5 and 10 mg kg-1 CA diets after 2nd week while goose-type lysozyme (g-Lyz) mRNA was up-regulation amongst weeks 1-4 of these diets. The present study suggested that E. tarda challenged fish after feeding with 5 and 10 mg kg-1 CA diets did not affect growth and hemato-biochemical parameter, but it enhanced nonspecific immune system and improving ERalpha, FSH-beta, c-Lyz, and g-Lyz mRNA expression in C. gariepinus against E. tarda.'}, {'qas': [{'id': '32589701:Odoribacter', 'question': 'Can Odoribacter cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Microbiota-associated risk factors for C. difficile acquisition in hospitalized patients: A prospective, multicentric study.\\nABSTRACT:\\n'}, {'qas': [{'id': '28632113:Pusillimonas', 'question': 'Can Pusillimonas cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Paracandidimonas soli gen. nov., sp. nov., isolated from soil.\\nABSTRACT:A slightly yellow-pigmented, Gram-stain-negative, rod-shaped bacterium, strain IMT-305T, was isolated from soil in Alabama, USA. Phylogenetic analysis based on the nearly full-length 16S rRNA gene sequence placed the strain in between the genera Pusillimonas, Parapusillimonas and Candidimonas with highest 16S rRNA gene sequence similarity to the type strain of Parapusillimonas granuli (97.5 %) and Candidimonas nitroreducens (97.4 %). The genomic G+C content of strain IMT-305T was 63.9 mol%. The main cellular fatty acids were C18:1omega7c, C17:0 cyclo, C16:0 and C16:1omega7c/iso-C15:0 2-OH (detected as summed feature 3). The polyamine pattern of strain IMT-305T contained the major compound putrescine and the betaproteobacterial diagnostic 2-hydroxyputrescine and the major respiratory quinone was ubiquinone Q-8. Predominant polar lipids were diphosphatidylglycerol, phosphatidylethanolamine, phosphatidylglycerol, phosphatidylserine, an unidentified aminolipid, an unidentified aminophospholipid and an unidentified lipid lacking any functional group. Based on phylogenetic, chemotaxonomic and phenotypic analyses a novel species within a new genus, Paracandidimonas soli gen. nov., sp. nov., is proposed. The type strain of Paracandidimonas soli is IMT-305T (=DSM 100048T=CIP 110902T=LMG 28740T=CCM 8599T).'}, {'qas': [{'id': '32257046:Selenomonas sputigena', 'question': 'Can Selenomonas sputigena cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:The landscape of bacterial presence in tumor and adjacent normal tissue across 9 major cancer types using TCGA exome sequencing.\\nABSTRACT:Identification of microbial composition directly from tumor tissue permits studying the relationship between microbial changes and cancer pathogenesis. We interrogated bacterial presence in tumor and adjacent normal tissue strictly in pairs utilizing human whole exome sequencing to generate microbial profiles. Profiles were generated for 813 cases from stomach, liver, colon, rectal, lung, head & neck, cervical and bladder TCGA cohorts. Core microbiota examination revealed twelve taxa to be common across the nine cancer types at all classification levels. Paired analyses demonstrated significant differences in bacterial shifts between tumor and adjacent normal tissue across stomach, colon, lung squamous cell, and head & neck cohorts, whereas little or no differences were evident in liver, rectal, lung adenocarcinoma, cervical and bladder cancer cohorts in adjusted models. Helicobacter pylori in stomach and Bacteroides vulgatus in colon were found to be significantly higher in adjacent normal compared to tumor tissue after false discovery rate correction. Computational results were validated with tissue from an independent population by species-specific qPCR showing similar patterns of co-occurrence among Fusobacterium nucleatum and Selenomonas sputigena in gastric samples. This study demonstrates the ability to identify bacteria differential composition derived from human tissue whole exome sequences. Taken together our results suggest the microbial profiles shift with advanced disease and that the microbial composition of the adjacent tissue can be indicative of cancer stage disease progression.'}, {'qas': [{'id': '11707228:Ralstonia pickettii', 'question': 'Can Ralstonia pickettii cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:[Ralstonia pickettii pneumonia in an inmunocompetent adult].\\nABSTRACT:'}, {'qas': [{'id': '17108291:Staphylococcus gallinarum', 'question': 'Can Staphylococcus gallinarum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Microevolution of cytochrome bd oxidase in Staphylococci and its implication in resistance to respiratory toxins released by Pseudomonas.\\nABSTRACT:Pseudomonas aeruginosa and Staphylococcus aureus are opportunistic pathogens and frequently coinfect the lungs of cystic fibrosis patients. P. aeruginosa secretes an arsenal of small respiratory inhibitors, like pyocyanin, hydrogen cyanide, or quinoline N-oxides, that may act against the commensal flora as well as host cells. Here, we show that with respect to their susceptibility to these respiratory inhibitors, staphylococcal species can be divided into two groups: the sensitive group, comprised of pathogenic species such as S. aureus and S. epidermidis, and the resistant group, represented by nonpathogenic species such as S. carnosus, S. piscifermentans, and S. gallinarum. The resistance in the latter group of species was due to cydAB genes that encode a pyocyanin- and cyanide-insensitive cytochrome bd quinol oxidase. By exchanging cydB in S. aureus with the S. carnosus-specific cydB, we could demonstrate that CydB determines resistance. The resistant or sensitive phenotype was based on structural alterations in CydB, which is part of CydAB, the cytochrome bd quinol oxidase. CydB represents a prime example of both microevolution and the asymmetric pattern of evolutionary change.'}, {'qas': [{'id': '22854353:Pseudomonas alcaligenes', 'question': 'Can Pseudomonas alcaligenes cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Microbial contamination of a disinfectant-soaked unwoven cleaning cloth.\\nABSTRACT:In December 2009, a 76-year-old male patient developed pneumonia due to Burkholderia cepacia whilst in an intensive care unit at a Japanese university hospital. During the subsequent environmental investigation to find the source, B. cepacia with an identical DNA type was found in his denture storage solution. Open packets of unwoven rayon cloths soaked in 0.2% alkyldiaminoethylglycine hydrochloride, used for environmental cleaning, were shown to be contaminated with B. cepacia, Alcaligenes xylosoxidans, Pseudomonas fluorescens and Pseudomonas aeruginosa. B. cepacia of a different DNA type was found in five of 42 samples from sealed packets of cloths.'}, {'qas': [{'id': '16166681:Gordonia otitidis', 'question': 'Can Gordonia otitidis cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': \"TITLE:Gordonia otitidis sp. nov., isolated from a patient with external otitis.\\nABSTRACT:The taxonomic positions of two clinically isolated actinomycetes were established using a polyphasic approach. The two strains, IFM 10032(T), isolated from ear discharge of a 28-year-old Japanese female patient with external otitis, and IFM 10148, isolated from pleural fluid of a 60-year-old Japanese male patient with bronchitis, possessed meso-diaminopimelic acid as the diagnostic amino acid, MK-9(H(2)) as the predominant menaquinone and mycolic acids ranging from 58 to 64 carbons. The 16S rRNA gene sequences of the two strains were most closely related to those of Gordonia aichiensis, Gordonia sputi and 'Gordonia jacobaea'. Differences in several phenotypic characteristics together with genotypic distinctiveness distinguish strains IFM 10032(T) and IFM 10148 from these three species. DNA-DNA hybridization results and the combination of genotypic and phenotypic data showed that the two strains belong to a single species, and merit recognition of a novel species within the genus Gordonia. The name proposed for this taxon is Gordonia otitidis sp. nov.; the type strain is IFM 10032(T) (=DSM 44809(T)=JCM 12355(T)=NBRC 100426(T)).\"}, {'qas': [{'id': '28488020:Facklamia hominis', 'question': 'Can Facklamia hominis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:[Unusual complication following placement of an epidural catheter].\\nABSTRACT:We report an unusual complication following placement of a thoracic epidural catheter in a patient that had undergone surgery for pulmonary metastases of a malignant melanoma. The intra- and postoperative course was initially without complications. At 2 days, there was a small, conspicuous swelling above the site of epidural puncture, which was neither reddened nor painful. No neurological deficits were observed, and the patient explained that he had been aware of the swelling for more than 6 months; thus, the epidural catheter was not deemed to be responsible. After catheter removal on day 4, an apparently purulent fluid drained from the puncture site. Although the patient had not reported this in the initial medical examination, he now explained that this cutaneous process had been squeezed out by his wife several times before. We initially obtained an exudate by means of ultrasound-guided puncture, and two epidermal cysts were detected on subsequent magnetic resonance imaging (MRI). The epidural catheter had been placed through one of these epidermal cysts. After reassessing the medical files, we found the earlier reports of two positron emission tomographies in which two epidermoid cysts were described at the level of thoracic vertebra 5/6 and 6/7. Facklamia hominis was detected in the pure culture. To avoid bacterial infection, we treated the patient for 7 days with antibiotics. The clinical course was without further pathological findings.'}, {'qas': [{'id': '24997586:Capnocytophaga canimorsus', 'question': 'Can Capnocytophaga canimorsus cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Antibiotic treatment following a dog bite in an immunocompromized patient in order to prevent Capnocytophaga canimorsus infection: a case report.\\nABSTRACT:\\n'}, {'qas': [{'id': '21193474:Streptococcus cristatus', 'question': 'Can Streptococcus cristatus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Molecular characterization and phylogenetic analysis of quinolone resistance-determining regions (QRDRs) of gyrA, gyrB, parC and parE gene loci in viridans group streptococci isolated from adult patients with cystic fibrosis.\\nABSTRACT:OBJECTIVES: Ciprofloxacin is the most frequently used member of the fluoroquinolones during initial eradication therapy of Pseudomonas aeruginosa, as well as during acute pulmonary exacerbations. However, its long-term effect on the susceptibility of the commensal flora within the cystic fibrosis (CF) airways has not yet been examined. The aim of this study was therefore to examine the consequence of oral ciprofloxacin usage on the resistance of the commensal viridans group streptococci (VGS), in terms of MICs and mutational analysis of the quinolone resistance-determining regions (QRDRs). \\n'}, {'qas': [{'id': '25632695:Mycobacterium pinnipedii', 'question': 'Can Mycobacterium pinnipedii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Mycobacterium pinnipedii tuberculosis in a free-ranging Australian fur seal (Arctocephalus pusillus doriferus) in South Australia.\\nABSTRACT:This report describes the first case in South Australia, Australia, of Mycobacterium pinnipedii tuberculosis in a free-ranging Australian fur seal (Arctocephalus pusillus doriferus). Severe pyogranulomatous pleuropneumonia with intrahistocytic acid-fast beaded filamentous bacilli was seen on histology. M. pinnipedii was confirmed by full 24-loci mycobacterial interspersed repetitive-unit-variable-number tandem-repeat (MIRU-VNTR) typing. Spillover concerns for public health and cattle are discussed.'}, {'qas': [{'id': '29515530:Actinomyces naeslundii', 'question': 'Can Actinomyces naeslundii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Antibacterial Effect of Copaifera duckei Dwyer Oleoresin and Its Main Diterpenes against Oral Pathogens and Their Cytotoxic Effect.\\nABSTRACT:This study evaluates the antibacterial activity of the Copaifera duckei Dwyer oleoresin and two isolated compounds [eperu-8(20)-15,18-dioic acid and polyalthic acid] against bacteria involved in primary endodontic infections and dental caries and assesses the cytotoxic effect of these substances against a normal cell line. MIC and MBC assays pointed out the most promising metabolites for further studies on bactericidal kinetics, antibiofilm activity, and synergistic antibacterial action. The oleoresin and polyalthic acid but not eperu-8(20)-15,18-dioic provided encouraging MIC and MBC results at concentrations lower than 100 mug mL-1. The oleoresin and polyalthic acid activities depended on the evaluated strain. A bactericidal effect on Lactobacillus casei (ATCC 11578 and clinical isolate) emerged before 8 h of incubation. For all the tested bacteria, the oleoresin and polyalthic acid inhibited biofilm formation by at least 50%. The oleoresin and polyalthic acid gave the best activity against Actinomyces naeslundii (ATCC 19039) and L. casei (ATCC 11578), respectively. The synergistic assays combining the oleoresin or polyalthic acid with chlorhexidine did not afford interesting results. We examined the cytotoxicity of C. duckei oleoresin, eperu-8(20)-15,18-dioic acid, and polyalthic acid against Chinese hamster lung fibroblasts. The oleoresin and polyalthic acid were cytotoxic at concentrations above 78.1 mug mL-1, whereas eperu-8(20)-15,18-dioic displayed cytotoxicity at concentrations above 312.5 mug mL-1. In conclusion, the oleoresin and polyalthic acid are potential sources of antibacterial agents against bacteria involved in primary endodontic infections and dental caries in both the sessile and the planktonic modes at concentrations that do not cause cytotoxicity.'}, {'qas': [{'id': '26953488:Aggregatibacter aphrophilus', 'question': 'Can Aggregatibacter aphrophilus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:HACEK endocarditis: state-of-the-art.\\nABSTRACT:The HACEK group of bacteria - Haemophilus parainfluenzae, Aggregatibacter spp. (A. actinomycetemcomitans, A. aphrophilus, A. paraphrophilus, and A. segnis), Cardiobacterium spp. (C. hominis, C. valvarum), Eikenella corrodens, and Kingella spp. (K. kingae, K. denitrificans) - are fastidious gram-negative bacteria, part of the normal microbiota of oral and upper respiratory tract in humans. Although their pathogenicity is limited, they are responsible for 1-3% of all infective endocarditis. HACEK endocarditis mostly affect patients with underlying heart disease or prosthetic valves, and are characterized by an insidious course, with a mean diagnosis delay of 1 month (Haemophilus spp.) to 3 months (Aggregatibacter and Cardiobacterium spp.). The advent of continuously monitored blood culture systems with enriched media has erased the need for extended incubation for the diagnosis of HACEK endocarditis. Medical treatment relies on third-generation cephalosporin, with a favorable outcome in 80-90% of cases, with or without cardiac surgery. '}, {'qas': [{'id': '11064263:Bacillus thuringiensis', 'question': 'Can Bacillus thuringiensis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Super-infection by Bacillus thuringiensis H34 or 3a3b can lead to death in mice infected with the influenza A virus.\\nABSTRACT:Bacterial super-infections are the main cause of complication and mortality after influenza virus (IAV) infection. Since Bacillus thuringiensis (Bt) is considered non-pathogenic for humans and is widely sprayed in urban areas, the aim of this work was to evaluate the potential pathogenicity of a combined infection Bt-IAV in a mouse model of pneumonia. Bacteria used for super-infections were Bt serotype H34 isolated from human infection and the insecticidal strain 3a3b obtained from a commercial source. Virus strain was A/Scotland/20/74 (H3N2) adapted to BALB/c mice by serial lung passage. Combined infection with 4% of the viral lethal dose 50% (LD(50)) and 10(2) spores of Bt H34 killed 40% of the mice. Mortality rates increased up to 55% and 100% when combined infections were done with respectively 10(4) and 10(7) spores. The insecticidal strain Bt 3a3b was less pathogenic than Bt H34. A dose of 10(4) spores associated with 4% of IAV LD(50) killed 50% of the mice. This inoculum must be compared with the doses usually sprayed in agriculture: 10(11) spores m(-2). Total protection against super-infection was obtained when mice were treated with amantadine. Even if only a few cases of Bt human infection have been reported, these results suggest a possible risk for workers spraying Bt-based biopesticides during flu outbreaks.'}, {'qas': [{'id': '28712163:Bacteroides vulgatus', 'question': 'Can Bacteroides vulgatus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:Molecular Characterization Of Fecal Microbiota Of Healthy Chinese Tobacco Smoker Subjects In Shaanxi Province, Xi'an China.\\nABSTRACT:\\n\"}, {'qas': [{'id': '22073241:Micrococcus luteus', 'question': 'Can Micrococcus luteus cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:The role of speciation in positive Lowenstein-Jensen culture isolates from a high tuberculosis burden country.\\nABSTRACT:\\n'}, {'qas': [{'id': '32609431:Photorhabdus luminescens', 'question': 'Can Photorhabdus luminescens cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Making and breaking leupeptin protease inhibitors in pathogenic gammaproteobacteria.\\nABSTRACT:Leupeptin is a bacterial small molecule that is used worldwide as a protease inhibitor. However, its biosynthesis and genetic distribution remain unknown. Here, we identified a family of leupeptins in gammaproteobacterial pathogens, including Photorhabdus, Xenorhabdus, and Klebsiella species, amongst others. Through genetic, metabolomic, and heterologous expression analyses, we established their construction from discretely expressed ligases and accessory enzymes. In Photorhabdus species, a hypothetical protein required for colonizing nematode hosts was established as a new class of proteases. This enzyme cleaved the tripeptide aldehyde protease inhibitors, leading to the formation of \"pro-pyrazinones\" featuring a hetero-tricyclic architecture. In Klebsiella oxytoca, the pathway was enriched in clinical isolates associated with respiratory tract infections. Thus, the bacterial production and proteolytic degradation of leupeptins can be associated with animal colonization phenotypes.'}, {'qas': [{'id': '30275093:Clostridium butyricum', 'question': 'Can Clostridium butyricum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:Antimicrobial Susceptibility of 260 Clostridium botulinum Type A, B, Ba, and Bf Strains and a Neurotoxigenic Clostridium baratii Type F Strain Isolated from California Infant Botulism Patients.\\nABSTRACT:Infant botulism is an infectious intestinal toxemia that results from colonization of the infant large bowel by Clostridium botulinum (or rarely, by neurotoxigenic Clostridium baratii or Clostridium butyricum), with subsequent intraintestinal production and absorption of botulinum neurotoxin that then produces flaccid paralysis. The disease is often initially misdiagnosed as suspected sepsis or meningitis, diagnoses that require prompt empirical antimicrobial therapy. Antibiotics may also be needed to treat infectious complications of infant botulism, such as pneumonia or urinary tract infection. Clinical evidence suggests (see case report below) that broad-spectrum antibiotics that are eliminated by biliary excretion may cause progression of the patient's paralysis by lysing C. botulinum vegetative cells in the large bowel lumen, thereby increasing the amount of botulinum neurotoxin available for absorption. The purpose of this antimicrobial susceptibility study was to identify an antimicrobial agent with little or no activity against C. botulinum that could be used to treat infant botulism patients initially diagnosed with suspected sepsis or meningitis, or who acquired secondary infections, without lysing C. botulinum Testing of 12 antimicrobial agents indicated that almost all California infant botulism patient isolates are susceptible to most clinically utilized antibiotics and are also susceptible to newer antibiotics not previously tested against large numbers of C. botulinum patient isolates. No antibiotic with little or no activity against C. botulinum was identified. These findings reinforce the importance of promptly treating infant botulism patients with human botulism immune globulin (BIG-IV [BabyBIG]).\"}, {'qas': [{'id': '27099677:Enterococcus plantarum', 'question': 'Can Enterococcus plantarum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Efficacy of Locally Isolated Lactic Acid Bacteria Against Antibiotic-Resistant Uropathogens.\\nABSTRACT:\\n'}, {'qas': [{'id': '32673495:Pseudomonas pseudoalcaligenes', 'question': 'Can Pseudomonas pseudoalcaligenes cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:Evidence for Environmental-human Microbiota Transfer at a Manufacturing Facility with Novel Work-related Respiratory Disease.\\nABSTRACT:INTRODUCTION: Workers' exposure to metalworking fluid (MWF) has been associated with respiratory disease. As part of a public health investigation of a manufacturing facility, we performed paired environmental and human sampling to evaluate cross-pollination of microbes between environment and host and possible effects on lung pathology present among workers. \\n\"}, {'qas': [{'id': '32943860:Tannerella', 'question': 'Can Tannerella cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:Periodontal Status and Microbiologic Pathogens in Patients with Chronic Obstructive Pulmonary Disease and Periodontitis: A Case-Control Study.\\nABSTRACT:Purpose: To evaluate clinical periodontal status and microbiologic pathogens in patients with chronic obstructive pulmonary disease (COPD) and periodontitis. Patients and Methods: We conducted a case-control study of 60 periodontitis patients with COPD (case group) and 60 periodontitis patients with normal pulmonary function (control group). Their periodontal status and respiratory function were clinically examined. Real-time polymerase chain reaction assays were used to measure five dental pathogens and four respiratory pathogens in subgingival dental plaque. Spearman's rank correlation coefficients (r2) were calculated to assess correlations of pathogens. Principal component analysis (PCA) was employed to assess the similarity of bacterial diversity between the two groups. Logistic regression was performed to examine the associations of periodontal variables and pathogens with COPD risk. Results: COPD patients had fewer remaining teeth, higher plaque index (PLI), and more severe site percentages of clinical attachment level (CAL) than the controls. Although COPD patients tended to have relatively higher ranked means of Porphyromonas gingivalis, Tannerella forsythensis, Treponema denticola, and Haemophilus influenza than control participants, the differences were not significant. Some periodontal pathogens and respiratory pathogens were positively correlated with each other (r2 =0.29 to 0.47, all P < 0.05). The PCA graph showed that the distributions of pathogens were more dispersed but less discriminated in the COPD group than those in the control group. PLI (P = 0.045) and CAL >= 5mm site percentages (P = 0.01) were significantly associated with an increased risk of COPD, while pathogens were not associated with COPD. Conclusion: Our results from this study do not indicate periodontal pathogens as potential predictors of COPD risk, despite significantly poor periodontal status associated with COPD.\"}, {'qas': [{'id': '21163827:Cardiobacterium valvarum', 'question': 'Can Cardiobacterium valvarum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Cardiobacterium valvarum infective endocarditis and phenotypic/molecular characterization of 11 Cardiobacterium species strains.\\nABSTRACT:Cardiobacterium valvarum is a newly recognized human pathogen related to infective endocarditis. Cardiobacterium species are, however, only rarely the aetiology of infective endocarditis. An infective endocarditis case is presented and, additionally, phenotypic and phylogenetic comparison of a further 10 collection strains, representing the two species within the genus, was performed. C. valvarum was isolated from the blood and DNA was present in valvular tissue (partial 16S rRNA gene analysis) from a 64-year-old man with infective endocarditis of the mitral valve, rupture of chordae and prolapse of pulmonary valves in addition to a fluttering excrescence. A mechanical mitral valve and neochordae were inserted successfully. Phenotypically, the two species within the genus Cardiobacterium resemble each other greatly. When using the Vitek 2 Neisseria-Haemophilus identification card, the reaction for phenylphosphonate was positive for all Cardiobacterium hominis strains, but negative for all C. valvarum strains, thereby separating the two species. The two species made up two separate clusters by phylogenetic examination using 16S rRNA gene sequence analysis.'}, {'qas': [{'id': '25831585:Ochrobactrum intermedium', 'question': 'Can Ochrobactrum intermedium cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Valvulopathy consistent with endocarditis in an Argentine boa (Boa constrictor occidentalis).\\nABSTRACT:An Argentine boa (Boa constrictor occidentalis) of 5 yr 7 mo of age was presented for respiratory problems and regurgitation. Radiographs revealed evidence of cardiomegaly and pneumonia. Blood smear examination revealed the presence of intracytoplasmic inclusion bodies in peripheral lymphocytes, consistent with inclusion body disease. Cultures of a tracheal wash sample resulted in growth of Ochrobactrum intermedium and Pseudomonas putida. Echocardiographic examination revealed a large vegetative lesion on the right atrioventricular valve with valvular insufficiency, a mildly dilated right atrium, and pulmonary hypertension. Postmortem examination confirmed the presence of pneumonia and bacterial endocarditis with dystrophic mineralization of the right atrioventricular valve, associated with different bacteria than those cultured from the tracheal wash. The present case is the first report of endocarditis in a boa constrictor and contributes to the rare reports of cardiac disease in snakes.'}, {'qas': [{'id': '15545412:Nocardia shimofusensis', 'question': 'Can Nocardia shimofusensis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Nocardia shimofusensis sp. nov., isolated from soil, and Nocardia higoensis sp. nov., isolated from a patient with lung nocardiosis in Japan.\\nABSTRACT:Three actinomycete strains isolated from soils and one strain isolated from a patient with lung nocardiosis in 1999 and 2001 in Japan have been provisionally assigned to the genus Nocardia on the basis of morphological criteria. These isolates were further investigated to determine their specific taxonomic status. Detailed chemotaxonomic characterization and 16S rRNA gene sequence analysis of these isolates confirmed that they belong to the genus Nocardia. The 16S rRNA gene sequences of the four strains were most similar to that of Nocardia farcinica. However, the sequence similarity values between these four strains and N. farcinica were &lt;98.9 %. These four strains were susceptible to 5-fluorouracil, and they have the ability to decompose urea, which is a very characteristic trait. Furthermore, DNA-DNA relatedness data revealed that IFM 10311(T), IFM 10312 and IFM 10313 comprise a single novel species of Nocardia, that IFM 10084(T) represents another novel species of Nocardia and that these two novel species could be distinguished from N. farcinica. The names Nocardia shimofusensis sp. nov. and Nocardia higoensis sp. nov. are proposed, with IFM 10311(T) (=NBRC 100134(T)=JCM 12122(T)=DSM 44733(T)) and IFM 10084(T) (=NBRC 100133(T)=JCM 12121(T)=DSM 44732(T)) as the respective type strains.'}, {'qas': [{'id': '28076350:Mycobacterium arupense', 'question': 'Can Mycobacterium arupense cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Emergence of Rare Species of Nontuberculous Mycobacteria as Potential Pathogens in Saudi Arabian Clinical Setting.\\nABSTRACT:\\n'}, {'qas': [{'id': '33302682:Roseburia', 'question': 'Can Roseburia cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:The gut microbiome can be used to predict the gastrointestinal response and efficacy of lung cancer patients undergoing chemotherapy.\\nABSTRACT:\\n'}, {'qas': [{'id': '30083836:Klebsiella variicola', 'question': 'Can Klebsiella variicola cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Isolation and characterization of chromium(VI)-reducing bacteria from tannery effluents and solid wastes.\\nABSTRACT:In the present investigation, five novel Cr(VI) reducing bacteria were isolated from tannery effluents and solid wastes and identified as Kosakonia cowanii MKPF2, Klebsiella pneumonia MKPF5, Acinetobacter gerneri MKPF7, Klebsiella variicola MKPF8 and Serratia marcescens MKPF12 by 16S rDNA gene sequence analysis. The maximum tolerance concentration of Cr(VI) as K2Cr2O7 of the bacterial isolates was varying up to 2000 mg/L. Among the investigated bacterial isolates, A. gerneri MKPF7 was best in terms of reduction rate. The optimum temperatures for growth and Cr(VI) reduction by the bacterial isolates were 35 and 40  C, respectively except A. gerneri MKPF7 which grew and reduced Cr(VI) optimally at 40  C. The optimum pH for growth and Cr(VI) reduction by K. cowanii MKPF2, A. gerneri MKPF7 and S. marcescens MKPF12 was 7.0 whereas the optimum pH for growth and Cr(VI) reduction by K. pneumoniae MKPF5 and K. variicola MKPF8 were 7.0, 8.0 and 6.0, 7.0, respectively. All the bacterial isolates showed maximum tolerance against Ni2+ and Zn2+ whereas minimum tolerance was observed against Hg2+ and Cd2+. The bacteria isolated in the present study thus can be used as eco-friendly biological expedients for the remediation and detoxification of Cr(VI) from the contaminated environments.'}, {'qas': [{'id': '31843547:Escherichia fergusonii', 'question': 'Can Escherichia fergusonii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Gold nanoparticles synthesised by flavonoid tricetin as a potential antibacterial nanomedicine to treat respiratory infections causing opportunistic bacterial pathogens.\\nABSTRACT:In this study, flavonoid tricetin was used as a reducing and capping agent for the synthesis of gold nanoparticles (AuNPs). Further, the antibacterial efficacy of the synthesised AuNPs was evaluated against the opportunistic bacterial pathogens that cause respiratory infections. The optimum levels for the synthesis of AuNPs were found to be pH 8, temperature 30  C, tricetin 125 muM and chloroauric acid 250 muM. The tricetin synthesised AuNPs exhibited in spherical shape with an average size of 12 nm. FT-IR results confirmed that the hydroxyl (OH) and carbonyl (C=O) groups of tricetin were mainly participated in the synthesis of AuNPs. The opportunistic bacterial pathogens isolated from immunocompromised patients suffering with different respiratory infections were identified as Staphylococcus aureus, Enterobacter xiangfangensis, Bacillus licheniformis, Escherichia fergusonii, Acinetobacter pittii, Pseudomonas aeruginosa, Aeromonas enteropelogenes and Proteus mirabilis. The antibacterial studies confirmed the broad-spectrum antibacterial activity of AuNPs against the tested Gram-positive and Gram-negative bacteria. The synthesised AuNPs showed high biocompatibility on primary normal human dermal fibroblast (NHDF-c) cells up to 50 muM mL-1. Best of our knowledge, this is the first report on the synthesis of AuNPs using tricetin, which may be a potential antibacterial nanomedicine to treat bacterial infections.'}, {'qas': [{'id': '24096355:Actinomyces radicidentis', 'question': 'Can Actinomyces radicidentis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Actinomyces haliotis sp. nov., a bacterium isolated from the gut of an abalone, Haliotis discus hannai.\\nABSTRACT:A novel, Gram-staining-positive, facultatively anaerobic, non-motile and coccus-shaped bacterium, strain WL80(T), was isolated from the gut of an abalone, Haliotis discus hannai, collected from the northern coast of Jeju in Korea. Optimal growth occurred at 30  C, pH 7-8 and with 1% (w/v) NaCl. Phylogenetic analyses based on the 16S rRNA gene sequence revealed that strain WL80(T) fell within the cluster of the genus Actinomyces, with highest sequence similarity to the type strains of Actinomyces radicidentis (98.8% similarity) and Actinomyces urogenitalis (97.0% similarity). The major cellular fatty acids were C18 : 1omega9c and C16 : 0. Menaquinone-10 (H4) was the major respiratory quinone. The genomic DNA G+C content of the isolate was 70.4 mol%. DNA-DNA hybridization values with closely related strains indicated less than 7.6% genomic relatedness. The results of physiological, biochemical, chemotaxonomic and genotypic analyses indicated that strain WL80(T) represents a novel species of the genus Actinomyces, for which the name Actinomyces haliotis sp. nov. is proposed. The type strain is WL80(T) ( = KACC 17211(T) = JCM 18848(T)).'}, {'qas': [{'id': '21221948:Bacillus mycoides', 'question': 'Can Bacillus mycoides cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Bacillus gaemokensis sp. nov., isolated from foreshore tidal flat sediment from the Yellow Sea.\\nABSTRACT:A Gram-positive, rod-shaped, endospore-forming organism, strain BL3-6(T), was isolated from tidal flat sediments of the Yellow Sea in the region of Tae-An. A 16S rRNA gene sequence analysis demonstrated that this isolate belongs to the Bacillus cereus group, and is closely related to Bacillus mycoides (99.0% similarity), Bacillus thuringiensis (99.0%), Bacillus weihenstephanensis (99.0%), Bacillus cereus (98.9%), Bacillus anthracis (98.8%), and Bacillus pseudomycoides (98.1%). The phylogenetic distance from any validly described Bacillus species outside the Bacillus cereus group was less than 95.6%. The DNA G+C content of the strain was 39.4 mol% and the major respiratory quinone was menaquinone-7. The major cellular fatty acids were iso-C(14:0) (17.8%), iso-C(16:0) (15.8%), and iso-C(12:0) (11.3%). The diagnostic amino acid of the cell wall was meso-diaminopimelic acid and the major cell wall sugar was galactose. The results of DNA-DNA hybridization (&lt;55.6%) and physiological and biochemical tests allowed genotypic and phenotypic differentiation of strain BL3-6(T) from the published Bacillus species. BL3-6(T) therefore represents a new species, for which the name Bacillus gaemokensis sp. nov. is proposed, with the type strain BL3-6(T) (=KCTC 13318(T) =JCM 15801(T)).'}, {'qas': [{'id': '22860594:Aeromonas media', 'question': 'Can Aeromonas media cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:The influences of natural zeolite (cliptinolite) on ammonia and biogenic amine formation by foodborne pathogen.\\nABSTRACT:UNLABELLED: The influence of natural zeolite on biogenic amines (BAs) and ammonia (AMN) production by eight common gram negative and positive foodborne pathogens (FBP) were investigated in histidine decarboxylase broth (HDB). Presence of 1% zeolite in the HDB resulted in significantly higher AMN production. Histamine (HIS) production by gram positive bacteria was as low as 0.5 mg/L, whereas Escherichia coli produced 18.96 mg/L of HIS. The use of zeolite also significantly suppressed HIS accumulation by E. coli, Pseudomonas aeruginosa, S. paratyphi A (P &lt; 0.05), although zeolite addition stimulated HIS production by K. pneumonia and Aeromonas hydrophila. The range of tyramine (TYR) production by gram positive bacteria was 1.19 and 4.06 mg/L for Enteroccus faecalis and Listeria monocytogenes respectively. The results of study showed that the effect of zeolite on BAs and AMN production was dependent on bacterial strains, as well as zeolite concentrations used. PRACTICAL APPLICATION: Natural zeolites are the main absorptive, low-cost material used in agriculture and industry. Although the effect of zeolite on ammonia formation in some industrial systems is well known, there is limited information regarding the impact of zeolite on biogenic amine (BA) production by foodborne pathogens. The data presented in this article will help us to understand the impact of natural zeolite on BA and ammonia production by eight common foodborne pathogens.'}, {'qas': [{'id': '16082002:Mycobacterium sherrisii', 'question': 'Can Mycobacterium sherrisii cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Pulmonary Mycobacterium sherrisii infection in a human immunodeficiency virus type 1-infected patient.\\nABSTRACT:We report the first case of a pulmonary infection with Mycobacterium sherrisii in a patient with advanced human immunodeficiency virus infection. Mycobacterium sherrisii is a newly described nontuberculous mycobacterium related to Mycobacterium simiae. Sequencing of the 16S rRNA gene was used for species identification. Treatment and antibiotic susceptibilities are described.'}, {'qas': [{'id': '26297212:Chlamydia abortus', 'question': 'Can Chlamydia abortus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Improving the molecular diagnosis of Chlamydia psittaci and Chlamydia abortus infection with a species-specific duplex real-time PCR.\\nABSTRACT:Chlamydia psittaci and Chlamydia abortus are closely related intracellular bacteria exhibiting different tissue tropism that may cause severe but distinct infection in humans. C. psittaci causes psittacosis, a respiratory zoonotic infection transmitted by birds. C. abortus is an abortigenic agent in small ruminants, which can also colonize the human placenta and lead to foetal death and miscarriage. Infections caused by C. psittaci and C. abortus are underestimated mainly due to diagnosis difficulties resulting from their strict intracellular growth. We developed a duplex real-time PCR to detect and distinguish these two bacteria in clinical samples. The first PCR (PCR1) targeted a sequence of the 16S-23S rRNA operon allowing the detection of both C. psittaci and C. abortus. The second PCR (PCR2) targeted the coding DNA sequence CPSIT_0607 unique to C. psittaci. The two PCRs showed 100 % detection for &gt;= 10 DNA copies per reaction (1000 copies ml(- 1)). Using a set of 120 samples, including bacterial reference strains, clinical specimens and infected cell culture material, we monitored 100 % sensitivity and 100 % specificity for the detection of C. psittaci and C. abortus for PCR1. When PCR1 was positive, PCR2 could discriminate C. psittaci from C. abortus with a positive predictive value of 100 % and a negative predictive value of 88 %. In conclusion, this new duplex PCR represents a low-cost and time-saving method with high-throughput potential, expected to improve the routine diagnosis of psittacosis and pregnancy complication in large-scale screening programs and also during outbreaks.'}, {'qas': [{'id': '21601873:Pseudomonas luteola', 'question': 'Can Pseudomonas luteola cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Pyogranulomatous pleuropneumonia and mediastinitis in ferrets (Mustela putorius furo) associated with Pseudomonas luteola Infection.\\nABSTRACT:Between 2008 and 2009, three pet ferrets from different sources presented with acute episode of dyspnoea. Cytological examination of pleural exudates revealed severe purulent inflammation with abundant clusters of rod-shaped microorganisms with a clear surrounding halo. Treatment was ineffective and the ferrets died 2-5 days later. Two ferrets were subjected to necropsy examination, which revealed pyothorax, mediastinal lymphadenopathy and multiple white nodules (1-2mm) in the lungs. Microscopical examination showed multifocal necrotizing-pyogranulomatous pleuropneumonia and lymphadenitis with aggregates of encapsulated microorganisms, some of which were positively stained by periodic acid-Schiff and alcian blue. In-situ hybridization for Pneumocystis spp., Ziehl-Neelsen staining and immunohistochemistry for distemper, coronavirus and influenza antigen were negative in all cases. Electron microscopically, the bacteria were 2-3 mum long with a thick electron-lucent capsule. Microbiology from one ferret yielded a pure culture of gram-negative bacteria identified phenotypically as Pseudomonas luteola. This speciation was later confirmed by 16S RNA gene amplification.'}, {'qas': [{'id': '2966102:Propionibacterium granulosum', 'question': 'Can Propionibacterium granulosum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Suppression of the activity of natural killer-like cells by peritoneal macrophages obtained from Lewis lung tumor-bearing, Propionibacterium granulosum KP-45-treated, or normal, untreated mice.\\nABSTRACT:Peritoneal adherent cells (PAC) obtained from Propionibacterium granulosum KP-45-treated or Lewis lung carcinoma-bearing BDF1 mice suppressed in vitro the NK-like cytotoxic activity of murine splenocytes against YAC-1 tumor target cells. Maximum inhibition occurred when suppressor and effector cells were preincubated together for 18 h, but the effect was demonstrable also when the two groups of cells were mixed only at the onset of the 4-h cytotoxic assay (i.e. without previous contact). Inhibitory cells appeared to be mostly macrophages, as judged by adherence to plastic and morphologic features, and as little as 5 to 20% of PAC, relative to the total number of co-incubated cells, were required for the clear demonstration of the effect. In addition to activated also normal, resident PAC obtained from untreated animals inhibited the NK cell-mediated cytotoxicity, but the effect was significantly pronounced only when 20% of suppressor cells were incubated overnight with effector splenocytes. The results favor the hypothesis that both functionally activated as well as resting macrophages operate as important regulators of the activity of NK cells in vivo.'}, {'qas': [{'id': '12373493:Rickettsia japonica', 'question': 'Can Rickettsia japonica cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Fulminant Japanese spotted fever definitively diagnosed by the polymerase chain reaction method.\\nABSTRACT:A 72-year-old man was admitted to the emergency ward in our hospital on July 20, 2001, because of consciousness disturbance, fever, generalized skin eruption, and severe general weakness beginning 7 days previously. Physical examination on admission revealed marked systemic cyanosis, erythema, and purpura. Laboratory findings indicated disseminated intravascular coagulation (DIC) and multiorgan failure (platelet count, 0.9 x 10(4)/micro l; fibrin degradation product, 110 micro g/ml; C-reactive protein, 22.6 mg/dl). Soluble interleukin 2-receptor (sIL-2R) was markedly increased to 14 710 U/ml. Blood gas analysis demonstrated severe metabolic acidosis. He was diagnosed with multiorgan failure due to DIC. Administration of heparin and sodium bicarbonate was started immediately, but respiratory failure was exacerbated and systemic spasm caused by encephalitis was noted. Although he was supported by an artificial ventilator, deterioration of metabolic acidosis occurred, and the blood pressure decreased to less than 60 mm Hg. He died 5.5 h after admission. The serological test showed no positive antibody titers against Orientia tsutsugamushi, Rickettsia japonica, or Rickettsia typhi. However, a specific DNA band derived from R. japonica was detected by the polymerase chain reaction (PCR) method using a primer from a blood clot. Therefore, he was definitively diagnosed as having Japanese spotted fever. The PCR method may be markedly useful for establishing a definitive diagnosis of Japanese spotted fever during the critical stage.'}, {'qas': [{'id': '25612804:Brevibacterium', 'question': 'Can Brevibacterium cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Modification of chimeric (2S, 3S)-butanediol dehydrogenase based on structural information.\\nABSTRACT:A chimeric (2S, 3S)-butanediol dehydrogenase (cLBDH) was engineered to have the strict (S)-configuration specificity of the (2S, 3S)-BDH (BsLBDH) derived from Brevibacterium saccharolyticum as well as the enzymatic stability of the (2R, 3S)-BDH (KpMBDH) from Klebsiella pneumonia by swapping the domains of two native BDHs. However, while cLBDH possesses the stability, it lacks the specificity. In order to assist in the design a BDH having strict substrate specificity, an X-ray structural analysis of a cLBDH crystal was conducted at 1.58 A. The results obtained show some readily apparent differences around the active sites of cLBDH and BsLBDH. Based on this structural information, a novel (2S, 3S)-BDH having a preferred specificity was developed by introducing a V254L mutation into cLBDH. The influence of this mutation on the stability of cLBDH was not evaluated. Nevertheless, the technique described herein is an effective method for the production of a tailor-made BDH. '}, {'qas': [{'id': '32209379:Acinetobacter haemolyticus', 'question': 'Can Acinetobacter haemolyticus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Comparative genomics analysis of Acinetobacter haemolyticus isolates from sputum samples of respiratory patients.\\nABSTRACT:Acinetobacter haemolyticus (A. haemolyticus) is a significant Acinetobacter pathogen, and the resistance of A. haemolyticus continues to rise due to abuse of antibiotics and the frequent gene exchange between bacteria in hospital. In this study, we performed complete genome sequencing of two A. haemolyticus strains TJR01 and TJS01 to improve our understanding of pathogenic and resistance of A. haemolyticus. Both TJR01 and TJS01 contain one chromosome and two plasmids. Compared to TJS01, more virulence factors (VFs) associated pathogenicity and resistant genes were predicted in TJR01 due to T4SS and integron associated with combination and transport. Antimicrobial susceptibility results were consistent with sequencing. We suppose TJS01 was a susceptive strain and TJR01 was an acquired multidrug resistance strain due to plasmid-mediated horizontal gene transfer. We hope these findings may be helpful for clinical treatment of A. haemolyticus infection and reduce the risk of potential outbreak infection.'}, {'qas': [{'id': '28291631:Mycobacterium yongonense', 'question': 'Can Mycobacterium yongonense cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Distribution and clinical significance of Mycobacterium avium complex species isolated from respiratory specimens.\\nABSTRACT:Mycobacterium avium complex (MAC) was originally composed of 2 species, M. avium and M. intracellulare. However, several new species closely related to M. intracellulare have recently been identified. In addition, M. avium has been further subdivided into 4 subspecies. The aim of this study was to determine the proportion of different MAC species recovered from respiratory specimens and to elucidate the clinical relevance of these species. Clinical isolates, from 219 patients, that had been initially identified as M. avium or M. intracellulare by non-sequencing methods were reidentified using multilocus sequence typing, and the clinical significance of the identified species was then investigated. Of 91 isolates originally identified as M. intracellulare, 75 (82%) were confirmed to be M. intracellulare, 8 (9%) isolates were identified as M. chimaera, and 4 (4%) isolates each were identified as \"M. indicus pranii\" and M. yongonense. The 128 isolates originally designated as M. avium were determined to be M. avium subsp. hominissuis. Of the 219 patients, 146 (67%) met the diagnostic criteria for MAC lung disease, and for each MAC species, the proportion of patients meeting these criteria was as follows: M. intracellulare (54/75, 72%), M. chimaera (3/8, 38%), \"M. indicus pranii\" (3/4, 75%), M. yongonense (2/4, 50%), and M. avium subsp. hominissuis (84/128, 66%). In summary, multilocus sequence typing of respiratory isolates initially identified as MAC revealed that, although most isolates were M. avium subsp. hominissuis or M. intracellulare, approximately 7% were newer MAC members, with clinical evidence supporting their potential pathogenicity for humans.'}, {'qas': [{'id': '14576947:Borrelia garinii', 'question': 'Can Borrelia garinii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:Frequent detection of Mycoplasma pneumoniae in Bell's palsy.\\nABSTRACT:The cause of Bell's palsy (BP) remains unknown despite various hints to an infectious etiology. Mycoplasma pneumoniae is a common pathogen of the respiratory tract causing pharyngitis, tracheobronchitis or pneumonia. Neurological complications are the most frequent extrapulmonary manifestation. So far, only a few case reports suggested an association between cranial nerve palsy and M. pneumoniae infection. Patients with a BP who were admitted to the Department of Otorhinolaryngology or Neurology of the University of Wuerzburg between 2000 and 2002 were tested serologically for the presence of antibodies against Borrelia burgdorferi, herpes viruses (HSV-1/2, VZV) and M. pneumoniae. The diagnosis of mycoplasmal infection was made when at least one of the following criteria was met: a threefold rise or more in the titer of antibody of M. pneumoniae in paired sample or a microparticle agglutination assay (MAG) of &gt; or =1:40 and the detection of IgA and/or IgM antibodies in the acute phase serum. Ninety-one consecutive patients could be included. Fifteen patients showed a reactivation of a VZV ( n=12) or of a HSV-1 ( n=3) infection. In six cases the immunoblot revealed specific antibody bands for B. burgdorferi. In 24 patients (26.4%) a seroconversion of M. pneumoniae could be detected. Only two patients complained of mild respiratory symptoms. According to our results, M. pneumoniae is frequently associated with Bell's palsy. Thus, a routine screening for this pathogen, even in the absence of respiratory symptoms, is necessary.\"}, {'qas': [{'id': '23207125:Nocardia concava', 'question': 'Can Nocardia concava cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Disseminated Nocardiosis caused by Nocardia concava with acute respiratory failure and central nervous system involvement treated with linezolid.\\nABSTRACT:Nocardia concava was identified as a new species in 2005; however, the clinical manifestations of Nocardia concava infection have yet to be clarified. We herein present the case of an immunosuppressed patient who developed disseminated nocardiosis caused by N. concava with multiple abscesses in the lungs, cutis, subcutaneous tissue, skeletal muscles and kidneys accompanied by central nervous system involvement, including meningitis and ventriculitis. The patient was cured with appropriate treatment including linezolid after testing for susceptibility. Linezolid should be considered as an alternative agent for treating disseminated nocardiosis because of its effective distribution to multiple sites.'}, {'qas': [{'id': '27902326:Candidimonas', 'question': 'Can Candidimonas cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Caenimicrobium hargitense gen. nov., sp. nov., a new member of the family Alcaligenaceae (Betaproteobacteria) isolated from activated sludge.\\nABSTRACT:A new betaproteobacterium, CGII-59m2T, was isolated from an activated sludge bioreactor which treated landfill leachate. The 16S rRNA gene sequence analysis revealed that strain CGII-59m2T belonged to the family Alcaligenaceae and shared the highest pairwise similarity values with Parapusillimonas granuli LMG 24012T (97.7 %), various species of the genus Bordetella (97.3-97.0 %) and Candidimonas nitroreducens LMG 24812T (97.0 %). Cells of strain CGII-59m2T were rod-shaped, non-motile, and oxidase- and catalase-positive. The predominant fatty acids were C16 : 1omega7c, C16 : 0, cyclo C17 : 0 and C18 : 1omega7c, the major respiratory quinone was Q-8, and the main polar lipids were phosphatidylethanolamine, phosphatidylglycerol, diphosphatidylglycerol and an unknown phospholipid. The G+C content of the genomic DNA of strain CGII-59m2T was 62.3 mol%. The new bacterium can be distinguished from the closely related type strains based on its non-motile cells and its high C16 : 1omega7c fatty acid content. On the basis of the phenotypic, chemotaxonomic and molecular data, strain CGII-59m2T is considered to represent a novel species of a new genus, for which the name Caenimicrobium hargitense gen. nov., sp. nov. is proposed. The type strain is CGII-59m2T (=DSM 29806T=NCAIM B.02615T).'}, {'qas': [{'id': '18065662:Pandoraea sputorum', 'question': 'Can Pandoraea sputorum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Evaluation of in vitro virulence characteristics of the genus Pandoraea in lung epithelial cells.\\nABSTRACT:Pandoraea species are emerging opportunistic pathogens capable of causing chronic lung infections in cystic fibrosis patients. This study examined the interactions of 17 Pandoraea isolates from the five identified species (Pandoraea apista, Pandoraea norimbergensis, Pandoraea pulmonicula, Pandoraea sputorum and Pandoraea pnomenusa) plus two Pandoraea genomospecies isolates with lung epithelial cells and their ability to form biofilms in vitro. Only three isolates showed an ability to invade A549 lung epithelial cells, and only one isolate was able to form biofilms. In contrast, all isolates triggered a pronounced pro-inflammatory response, with elevation of both interleukin (IL)-6 (two- to 19-fold) and IL-8 (10- to 50-fold) above that observed for a control strain of Escherichia coli. This property is likely to be a major factor in the pathogenesis of the genus.'}, {'qas': [{'id': '24453231:Empedobacter brevis', 'question': 'Can Empedobacter brevis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Moheibacter sediminis gen. nov., sp. nov., a member of the family Flavobacteriaceae isolated from sediment, and emended descriptions of Empedobacter brevis, Wautersiella falsenii and Weeksella virosa.\\nABSTRACT:A Gram-reaction-negative, yellow-pigmented, strictly aerobic bacterium, designated M0116T, was isolated from the sediment of the Mohe Basin in north-east China. Flexirubin-type pigments were produced. Cells were catalase- and oxidase-positive and non-gliding rods. Phylogenetic analysis based on 16S rRNA gene sequences revealed that strain M0116T was a member of the family Flavobacteriaceae and was most closely related to members of the genera Empedobacter, Wautersiella and Weeksella with 90.5-91.0% sequence similarities. The major cellular fatty acids were iso-C15:0 and iso-C17:0 3-OH. The major respiratory quinone was MK-6 and the major polar lipid was phosphatidylethanolamine. The DNA G+C content was 38.2 mol%. Based on phenotypic, phylogenetic and genotypic data, strain M0116T is considered to represent a novel species of a new genus in the family Flavobacteriaceae, for which the name Moheibacter sediminis gen. nov., sp. nov. is proposed. The type strain is M0116T (=CGMCC 1.12708T=JCM 19634T). Emended descriptions of Empedobacter brevis, Wautersiella falsenii and Weeksella virosa are also proposed.'}, {'qas': [{'id': '27826114:Nocardia elegans', 'question': 'Can Nocardia elegans cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Nocardia elegans infection: a case report and literature review.\\nABSTRACT:A case of disseminated nocardiosis caused by Nocardia elegans in a 72-year-old man with rheumatoid arthritis, treated with tacrolimus and prednisolone, is reported herein. The patient had impaired vision and was diagnosed with endophthalmitis and an abdominal skin abscess. He was started on trimethoprim-sulfamethoxazole treatment, followed by cefepime. The patient was then switched to a combination of imipenem-cilastatin and minocycline. Although the patient survived as a result of surgery and prolonged antibiotic treatment, he eventually lost vision after the infection became resistant to antibiotic treatment. Molecular analysis of samples from the abscess and vitreous fluid confirmed the extremely rare pathogen N. elegans, which accounts for only 0.3-0.6% of infections caused by Nocardia species. This organism is almost always associated with pulmonary infection, and disseminated infections are rare. As with previously reported norcardial infections, the current case was treated successfully with trimethoprim-sulfamethoxazole, carbapenems, and aminoglycosides. However, the clinical characteristics of this organism remain unclear. Further studies are therefore required to develop more effective treatment protocols for disseminated nocardiosis caused by this problematic pathogen.'}, {'qas': [{'id': '32255852:Corynebacterium tuberculostearicum', 'question': 'Can Corynebacterium tuberculostearicum cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Respiratory Bacterial Microbiota and Individual Bacterial Variability in Lung Cancer and Bronchiectasis Patients.\\nABSTRACT:Respiratory bacterial microbiota plays a key role in human health. Lung cancer microbiome is a significant yet an understudied area while bronchiectasis microbiome is often studied. We assessed the bacterial microbiota in the upper and lower respiratory tract of the patients with lung cancer and bronchiectasis against a healthy group and their variations in individuality. 16S rRNA gene based metagenomic sequencing was used to detect entire bacterial community along with conventional aerobic bacterial culturing. In comparison to healthy, increased bacterial diversity was observed in diseased population. Abundance of more than 1% was considered and bacteria were identified in 97% similarity. Only lung cancer patients exhibited bacteria specific to the disease: Corynebacterium tuberculostearicum and Keratinibaculum paraultunense. However, Enterococcus faecalis and Delftia tsuruhatensis were also observed limited to lung cancer and bronchiectasis respectively, in less than 1% but supported with bacterial culturing. In conclusion the disease condition and intra-group variability should be considered in future with larger cohorts to understand individual patient variability highlighting the social habits and gender of the individual.'}, {'qas': [{'id': '29985760:Chlamydia caviae', 'question': 'Can Chlamydia caviae cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Chlamydia caviae Causing Community-Acquired Pneumonia: An Emerging Zoonosis.\\nABSTRACT:We describe a case of community-acquired pneumonia due to Chlamydia caviae in a patient with no direct animal exposure, raising questions about the zoonotic reservoirs and potential transmission routes. Genotyping of Chamydia isolates that cause pneumonia should be performed for a precise diagnosis and to initiate adequate infection control measures.'}, {'qas': [{'id': '24277860:Bacillus lentus', 'question': 'Can Bacillus lentus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Bacillus panacisoli sp. nov., isolated from ginseng soil.\\nABSTRACT:A Gram-staining-positive, motile, facultatively anaerobic, endospore-forming and rod-shaped bacterium, designated strain CJ32(T), was isolated from ginseng soil at Geumsan in Korea. The isolate grew optimally at 30  C, 2% (w/v) NaCl and pH 7.0. Colonies of strain CJ32(T) were beige and circular with an entire margin on LB agar plates. Phylogenetic analysis based on 16S rRNA gene sequences indicated that strain CJ32(T) was associated with the genus Bacillus and was most closely related to Bacillus graminis YC6957(T) (97.3% similarity) and Bacillus lentus IAM 12466(T) (97.1%). DNA-DNA hybridization with closely related strains was below 31.3%. The major respiratory isoprenoid quinone was MK-7. The diagnostic diamino acid in the cell-wall peptidoglycan was meso-diaminopimelic acid. The polar lipid profile of strain CJ32(T) consisted of phosphatidylethanolamine, phosphatidylglycerol, diphosphatidylglycerol and several unidentified lipids, including phospholipids, aminolipids and aminophospholipids. The predominant fatty acids of strain CJ32(T) were iso-C15:0 and anteiso-C15:0. The G+C content of the genomic DNA was 35.1 mol%. Based on phenotypic, genotypic and phylogenetic data, strain CJ32(T) should be classified within a novel species of the genus Bacillus, for which the name Bacillus panacisoli sp. nov. is proposed. The type strain is strain CJ32(T) ( = KACC 17503(T) = JCM 19226(T)).'}, {'qas': [{'id': '9883600:Mycobacterium intermedium', 'question': 'Can Mycobacterium intermedium cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:[Bacteriology of mycobacteria: taxonomic and morphological characteristics].\\nABSTRACT:Bacteriological characteristics of organisms belonging to Genus Mycobacterium which involves more than 60 species are described. Mycobacterial organisms can be divided into the following groups having differential characteristics, on the basis of the results of biological, biochemical, and genetic investigations, including lipid analysis, DNA probe test, and comparative 16S ribosomal RNA sequencing. First, Mycobacterium tuberculosis complex (M. tuberculosis, M. bovis, M. africanum, etc.). Second, cultivable but slowly growing nontuberculous mycobacteria, including photochromogens (Runyon Group I) such as M. kansasii, M. marinum, M. simiae, M. intermedium, and M. asiaticum, scotochromogens (Runyon Group II) such as M. scrofulaceum, M. szulgai, M. injectum, M. lentiflavum, and M. gordonae, nonphotochromogenens (Runyon Group III) such as M. avium, M. intracellulare, M. xenopi, M. malmoense, M. genavense, M. celatum, and M. gastri. Third, cultivable rapidly growing nontuberculous mycobacteria (Runyon Group IV) including M. fortuitum, M. chenolae, M. abscessus, M. phlei, and M. smegmatis. Fourth, noncultivable mycobacteria including M. leprae. About 30 species of Mycobacterium cause pulmonary, dermal, lymphatic, and disseminated infections in human beings. This paper mainly deals with the taxonomic, morphological, and other biological characteristics of these mycobacterial organisms.'}, {'qas': [{'id': '27591952:Shigella boydii', 'question': 'Can Shigella boydii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:Development of recombinant vaccine candidate molecule against Shigella infection.\\nABSTRACT:Shigellosis is an acute bacillary diarrheal disease caused by the gram negative bacillus Shigella. The existence of multiple Shigella serotypes and their growing resistance to antibiotics stress the urgent need for the development of vaccine that is protective across all serotypes. Shigella's IpaB antigen is involved in translocon pore formation, promotes bacterial invasion and induces apoptosis in macrophages. S. Typhi GroEL (Hsp 60) is the immunodominant antigen inducing both arms of immunity and has been explored as adjuvant in this study. The present study evaluates the immunogenicity and protective efficacy of recombinant IpaB domain-GroEL fusion protein in mice against lethal Shigella infection. The IpaB domain and GroEL genes were fused using overlap extension PCR and cloned in pRSETA expression vector. Fused gene was expressed in Escherichia coli BL-21 cells and the resulting 90 KDa fusion protein was purified by affinity chromatography. Intranasal (i.n.) immunization of mice with fusion protein increased the IgG and IgA antibody titers as compared to the group immunized with IpaB and GroEL and control PBS immunized group. Also IgG1 and IgG2a antibodies induced in fusion protein immunized mice were higher than co-immunized group. Significant increase in lymphocyte proliferation and cytokine levels (IFN-gamma, IL-4 and IL-10), indicates induction of both Th1 and Th2 immune responses in both immunized groups. Immunization with fusion protein protected 90-95% of mice whereas 80-85% survivability was observed in co-immunized group against lethal challenge with S. flexneri, S. boydii and S. sonnei. Passive immunization conferred 60-70% protection in mice against all these Shigella species. Organ burden and histopathology studies also revealed significant decrease in lung infection as compared to the co-immunized group. Since IpaB is the conserved dominant molecule in all Shigella species, this study will lead to an ideal platform for the development of safe, efficacious and cost-effective recombinant vaccine against Shigella serotypes.\"}, {'qas': [{'id': '16808975:Nocardia paucivorans', 'question': 'Can Nocardia paucivorans cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Nocardia paucivorans--a cause of disseminated nocardiosis.\\nABSTRACT:We describe a 53-year-old man without discernable immunocompromise who presented with cerebral and cutaneous dissemination of primary pulmonary Nocardia paucivorans infection. This report also identifies 32 other patients in our health area with clinical isolates of N. paucivorans over the last 20 years, extending the reported spectrum of disease caused by this organism. At least 30% (10/33) of these patients had disseminated disease indicating a propensity of this species to disseminate.'}, {'qas': [{'id': '12202591:Actinomyces turicensis', 'question': 'Can Actinomyces turicensis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Genotypic diversity of clinical Actinomyces species: phenotype, source, and disease correlation among genospecies.\\nABSTRACT:We determined the frequency distribution of Actinomyces spp. recovered in a routine clinical laboratory and investigated the clinical significance of accurate identification to the species level. We identified 92 clinical strains of Actinomyces, including 13 strains in the related Arcanobacterium-Actinobaculum taxon, by 16S rRNA gene sequence analysis and recorded their biotypes, sources, and disease associations. The clinical isolates clustered into 21 genogroups. Twelve genogroups (74 strains) correlated with a known species, and nine genogroups (17 strains) did not. The individual species had source and disease correlates. Actinomyces turicensis was the most frequently isolated species and was associated with genitourinary tract specimens, often with other organisms and rarely with inflammatory cells. Actinomyces radingae was most often associated with serious, chronic soft tissue abscesses of the breast, chest, and back. Actinomyces europaeus was associated with skin abscesses of the neck and genital areas. Actinomyces lingnae, Actinomyces gravenitzii, Actinomyces odontolyticus, and Actinomyces meyeri were isolated from respiratory specimens, while A. odontolyticus-like strains were isolated from diverse sources. Several of the species were commonly coisolated with a particular bacterium: Actinomyces israelii was the only Actinomyces spp. coisolated with Actinobacillus (Haemophilus) actinomycetemcomitans; Actinomyces meyeri was coisolated with Peptostreptococcus micros and was the only species other than A. israelii associated with sulfur granules in histological specimens. Most genogroups had consistent biotypes (as determined with the RapID ANA II system); however, strains were misidentified, and many codes were not in the database. One biotype was common to several genogroups, with all of these isolates being identified as A. meyeri. Despite the recent description of new Actinomyces spp., 19% of the isolates recovered in our routine laboratory belonged to novel genospecies. One novel group with three strains, Actinomyces houstonensis sp. nov., was phenotypically similar to A. meyeri and A. turicensis but was genotypically closest to Actinomyces neuii. A. houstonensis sp. nov. was associated with abscesses. Our data documented consistent site and disease associations for 21 genogroups of Actinomyces spp. that provide greater insights into appropriate treatments. However, we also demonstrated a complexity within the Actinomyces genus that compromises the biochemical identification of Actinomyces that can be performed in most clinical laboratories. It is our hope that this large group of well-defined strains will be used to find a simple and accurate biochemical test for differentiation of the species in routine laboratories.'}, {'qas': [{'id': '6561066:Propionibacterium granulosum', 'question': 'Can Propionibacterium granulosum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Inhibition of Lewis lung carcinoma in mice by local microwave hyperthermia combined with immunomodulating Propionibacterium granulosum KP-45.\\nABSTRACT:C-57 BL/6 mice with Lewis lung carcinoma were treated 2 weeks after tumor implantation with local microwave hyperthermia (2450 MHz, tumor temperature 43.5 degrees C, 30 min) and/or intratumoral or intraperitoneal injection of 1 mg cell walls of Propionibacterium granulosum KP-45. Tumor growth, survival time of the animals, and the delayed skin hypersensitivity to oxazolone were followed up, as well as the 3H-thymidine uptake of tumorous tissues and the number of lung metastases. The combined treatment of microwave hyperthermia with immunomodulating P. granulosum KP-45 resulted in significantly stronger inhibition of tumor growth than with each of these methods alone. The number of lung metastases could be significantly lowered, and the skin reactivity to oxazolone remained enhanced during the whole observation period (over 70 days after tumor implantation). The implications of the test observation are discussed.'}, {'qas': [{'id': '30227256:Photobacterium damselae', 'question': 'Can Photobacterium damselae cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Physiopathological responses of sole (Solea senegalensis) subjected to bacterial infection and handling stress after probiotic treatment with autochthonous bacteria.\\nABSTRACT:This study aimed to evaluate the protective effects of four autochthonous bacteria isolated from juvenile sole (Solea senegalensis) intestine as dietary probiotic supplement against bacterial pathogen infection and handling/transport stressors. Growth performance and immune responses were evaluated after 85 days of feeding trial. Sole (IBW = 16.07 +- 0.11 g) were fed six experimental diets, a control diet (CTRL, without the dietary probiotic supplementation), and five diets supplemented with probiotic bacteria: PB1 (Shewanella hafniensis), PB2 (Enterococcus raffinosus), PB3 (Shewanella hafniensis + Arthrobacter soli), PB4 (Pseudomonas protegens + Arthrobacter soli) and PB5 (Shewanella hafniensis + Arthrobacter soli + Enterococcus raffinosus). All bacteria were selected based on their in vitro antimicrobial activity. After the growth trial, fish were submitted to a stress factor (transport) and then each dietary group was divided in two additional groups: non-infected (placebo) and infected with Photobacterium damselae subsp. piscicida. Immune and antioxidant responses were evaluated at day 10 post-infection. In infection trial A, fish were infected on the same day of transport, whereas in trial B fish were infected after a 7-day recovery from the transport stress. At the end of the feeding trial, fish fed with PB2 and PB4 showed lower final body weight when compared with the other dietary groups. Respiratory burst activity and nitric oxide production were not affected by probiotic supplementation. Fish fed with PB5 presented lower peroxidase activity compared to CTRL. Lysozyme and alternative complement pathway activity (ACH50) showed no significant differences between treatments. The innate immune responses were significantly affected after handling stress and bacterial infection. In trial A, the ACH50 levels of infected fish were significantly lower than the placebo groups. On the other hand, in trial B fish infected with Pdp demonstrated higher ACH50 levels when compared to placebos. Peroxidase levels were strongly modulated by bacterial infection and handling stress. In trials A and B, infection had a clear downgrade effect in peroxidase levels. Lipid peroxidation, catalase, glutathione S-transferase and glutathione reductase were altered by both bacterial infection and transport. Overall, dietary probiotic supplementation did not influence growth performance of sole. The immune and oxidative defenses of sole responded differently to infection depending on the probiotic and the synergy between pathogen infection and transport.'}, {'qas': [{'id': '30755259:Erwinia uredovora', 'question': 'Can Erwinia uredovora cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Pantoea dispersa bacteremia in an immunocompetent patient: a case report and review of the literature.\\nABSTRACT:\\n'}, {'qas': [{'id': '22121154:Streptococcus uberis', 'question': 'Can Streptococcus uberis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Survey of marbofloxacin susceptibility of bacteria isolated from cattle with respiratory disease and mastitis in Europe.\\nABSTRACT:A monitoring programme conducted in Europe since 1994 to survey the marbofloxacin susceptibility of bacterial pathogens isolated from cattle has established the susceptibility of bacterial strains isolated before any antibiotic treatment from bovine mastitis and bovine respiratory disease (BRD) cases between 2002 and 2008. Minimum inhibitory concentration (MIC) was determined by a standardised microdilution technique. For respiratory pathogens, Pasteurella multocida and Mannheimia haemolytica isolates (751 and 514 strains, respectively) were highly susceptible to marbofloxacin (MIC&lt;=0.03 microg/ml for 77.39 per cent of the strains) and only 1.75 per cent of M haemolytica strains were resistant (MIC&gt;=4 microg/ml). Histophilus somni isolates (73 strains) were highly susceptible to marbofloxacin (0.008 to 0.06 microg/ml). Mycoplasma bovis MIC (171 strains) ranged from 0.5 to 4 microg/ml. For mastitis pathogens, the majority of Escherichia coli isolates were highly susceptible to marbofloxacin (95.8 per cent of 617 strains). Staphylococcus aureus and coagulase-negative staphylococci (568 and 280 strains) had a homogenous population with MIC centred on 0.25 microg/ml. Streptococcus uberis and Streptococcus dysgalactiae (660 and 217 strains) were moderately susceptible with MIC centred on 1 microg/ml. Marbofloxacin MIC for these various pathogens appeared stable over the seven years of the monitoring programme and was similar to previously published MIC results.'}, {'qas': [{'id': '22700551:a novel variant of Mycobacterium chimaera', 'question': 'Can a novel variant of Mycobacterium chimaera cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Characterization of a novel variant of Mycobacterium chimaera.\\nABSTRACT:In this study, nonchromogenic mycobacteria were isolated from pulmonary samples of three patients in the Netherlands. All isolates had identical, unique 16S rRNA gene and 16S-23S ITS sequences, which were closely related to those of Mycobacterium chimaera and Mycobacterium marseillense. The biochemical features of the isolates differed slightly from those of M. chimaera, suggesting that the isolates may represent a possible separate species within the Mycobacterium avium complex (MAC). However, the cell-wall mycolic acid pattern, analysed by HPLC, and the partial sequences of the hsp65 and rpoB genes were identical to those of M. chimaera. We concluded that the isolates represent a novel variant of M. chimaera. The results of this analysis have led us to question the currently used methods of species definition for members of the genus Mycobacterium, which are based largely on 16S rRNA or rpoB gene sequencing. Definitions based on a single genetic target are likely to be insufficient. Genetic divergence, especially in the MAC, yields strains that cannot be confidently assigned to a specific species based on the analysis of a single genetic target.'}, {'qas': [{'id': '24421815:Nocardia carnea ', 'question': 'Can Nocardia carnea  cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Molecular identification and susceptibility pattern of clinical Nocardia species: Emergence of Nocardia crassostreae as an agent of invasive nocardiosis.\\nABSTRACT:\\n'}, {'qas': [{'id': '8771727:Borrelia garinii', 'question': 'Can Borrelia garinii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:[Molecular genetic approaches to the study of the persistence of pathogenic Mycoplasma].\\nABSTRACT:The inhibition of the amplification of different regions of Mycoplasma pneumonia genome, depending on the conditions of cultivation, was observed with the use of polymerase chain reaction. A protein, stably associated with DNA, is responsible for this inhibitory effect. When selectively associated with different sites of DNA, the protein seems to be capable to inhibit the expression of genes, encoding pathogenicity factors of mycoplasmas and thus promoting their transformation into persistent forms.'}, {'qas': [{'id': '32342176:Shigella dysenteriae', 'question': 'Can Shigella dysenteriae cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:SERS-based immunocapture and detection of pathogenic bacteria using a boronic acid-functionalized polydopamine-coated Au@Ag nanoprobe.\\nABSTRACT:A surface-enhanced Raman scattering (SERS)-based immunocapture nanoprobe is described for the detection of pathogenic bacteria. The probe uses boronic acid-functionalized polydopamine-coated Au@Ag nanoparticles as an advanced SERS nanotag. Modified magnetic IgG@Fe3O4 nanoparticles are used for magnetic separation. Au@Ag@PDA nanoparticles, where PDA stands for polydopamine, were functionalized with boronic acid to bind to pathogenic bacteria and induce signal amplification. The Raman signal is amplified 108 times when the SERS tag binds the surface of bacteria. The SERS spectra exhibit fingerprint-like patterns that enable bacterial classification. The results of principal component analysis (PCA) and hierarchical cluster analysis (HCA) of the spectral regions were compared. The bacterial surface protein and glycan signals (1300-1450 cm-1) were the best regions for bacterial classification. Staphylococcus aureus, Escherichia coli, Shigella dysenteriae, Pseudomonas aeruginosa, and Klebsiella pneumonia were successfully classified by this method. The lowest detection limit was 10 colonies/mL (CFU mL). The assay can be completed within 30 min. Conceivably, this method may be extended to the quantitative detection or classification of bacteria under various other conditions. Graphical abstract Schematic representation of immunocapture and detection of pathogenic bacteria using boronic acid-functionalized polydopamine-coated Au@Ag nanoprobe through the bacterial surface protein and glycan signals. Green arrow: laser; black arrow: SERS; red ball: bacteria; grey ball: IgG@Fe3O4; golden ball: boronic acid-functionalized Au@Ag@PDA.'}, {'qas': [{'id': '2358715:Capnocytophaga sputigena', 'question': 'Can Capnocytophaga sputigena cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:[A case of septicemia due to Capnocytophaga ochracea beginning post radiation therapy for lung cancer].\\nABSTRACT:A 46-year-old man was admitted to our hospital on Mar. 16, 1988 with the chief complaint of productive cough. The chest roentgenogram and tomogram showed a tumorous shadow in the right upper lobe, accompanied with stenosis of the trachea and the right main bronchus. Bronchofiberscopic examination confirmed a nodular tumor protruding into the lower part of the trachea from the right lateral wall and nearly complete obstruction of the right main bronchus. Pathological specimen obtained by transbronchial biopsy revealed \"low differentiated adenocarcinoma\". It was highly suspected that the primary lung cancer had directly invaded the trachea and the right main bronchus. His symptoms and roentgenological findings remarkably improved after radiation therapy. He was discharged on May 12. On Sep. 14, he was admitted to our hospital again because of hoarseness, general fatigue and increasing dyspnea. The chest CT demonstrated severe stenosis of the trachea, which was treated with another radiation therapy. Although his symptoms diminished, he had a sudden onset of high fever on Oct. 15. Immediately a blood culture and transtracheal aspiration (TTA) were performed. Gram-negative bacilli were isolated from the blood culture four days later. The administration of fosfomycin and tobramycin was started. However he died because of massive hemoptysis on Oct. 23. Several days after the death, an isolated strain was identified as Capnocytophaga ochracea by the biochemical characteristics. Culture of sputum obtained by TTA was negative for Capnocytophaga ochracea. Septicemia due to Capnocytophaga spp. is very rare and only one case (due to Capnocytophaga sputigena) has been reported until now in Japan. Our patient is thought to be the first case of septicemia due to Capnocytophaga ochracea in Japan.'}, {'qas': [{'id': '11594605:Globicatella sanguinis', 'question': 'Can Globicatella sanguinis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Globicatella sulfidifaciens sp. nov., isolated from purulent infections in domestic animals.\\nABSTRACT:DNA-DNA hybridization experiments and comparative 165 rDNA sequence analysis revealed that six isolates from purulent joint and lung infections in calves, from a lung lesion in a sheep, and from a joint infection of a pig represented a novel species belonging to the genus Globicatella. Whole-cell protein electrophoresis and biochemical activity testing revealed that the isolates formed a homogeneous group differing from Globicatella sanguinis, the only species of this genus described to date. These animal isolates were classified as Globicatella sulfidifaciens sp. nov. with LMG 188441 (= CCUG 44365T), isolated from the lung of a calf, as the type strain. A detailed description of its phenotypic characteristics is presented. Hydrogen sulphide production was found to be a very useful diagnostic feature.'}, {'qas': [{'id': '26838716:Corynebacterium accolens', 'question': 'Can Corynebacterium accolens cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Unraveling the Molecular Mechanisms Underlying the Nasopharyngeal Bacterial Community Structure.\\nABSTRACT:The upper respiratory tract is colonized by a diverse array of commensal bacteria that harbor potential pathogens, such as Streptococcus pneumoniae. As long as the local microbial ecosystem-also called \"microbiome\"-is in balance, these potentially pathogenic bacterial residents cause no harm to the host. However, similar to macrobiological ecosystems, when the bacterial community structure gets perturbed, potential pathogens can overtake the niche and cause mild to severe infections. Recent studies using next-generation sequencing show that S. pneumoniae, as well as other potential pathogens, might be kept at bay by certain commensal bacteria, including Corynebacterium and Dolosigranulum spp. Bomar and colleagues are the first to explore a specific biological mechanism contributing to the antagonistic interaction between Corynebacterium accolens and S. pneumoniae in vitro [L. Bomar, S. D. Brugger, B. H. Yost, S. S. Davies, K. P. Lemon, mBio 7(1):e01725-15, 2016, doi:10.1128/mBio.01725-15]. The authors comprehensively show that C. accolens is capable of hydrolyzing host triacylglycerols into free fatty acids, which display antipneumococcal properties, suggesting that these bacteria might contribute to the containment of pneumococcus. This work exemplifies how molecular epidemiological findings can lay the foundation for mechanistic studies to elucidate the host-microbe and microbial interspecies interactions underlying the bacterial community structure. Next, translation of these results to an in vivo setting seems necessary to unveil the magnitude and importance of the observed effect in its natural, polymicrobial setting. '}, {'qas': [{'id': '10698751:coculture model', 'question': 'Can coculture model cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Influence of Acanthamoeba castellanii on intracellular growth of different Legionella species in human monocytes.\\nABSTRACT:Previous studies using a murine model of coinhalation of Legionella pneumophila and Hartmannella vermiformis have shown a significantly enhanced intrapulmonary growth of L. pneumophila in comparison to inhalation of legionellae alone (J. Brieland, M. McClain, L. Heath, C. Chrisp, G. Huffnagle, M. LeGendre, M. Hurley, J. Fantone, and C. Engleberg, Infect. Immun. 64:2449-2456, 1996). In this study, we introduce an in vitro coculture model of legionellae, Mono Mac 6 cells (MM6) and Acanthamoeba castellanii, using a cell culture chamber system which separates both cell types by a microporous polycarbonate membrane impervious to bacteria, amoebae, and human cells. Whereas L. pneumophila has shown a maximal 4-log-unit multiplication within MM6, which could not be further increased by coculture with Acanthamoeba castellanii, significantly enhanced replication of L. gormanii, L. micdadei, L. steigerwaltii, L. longbeachae, and L. dumoffii was seen after coculture with amoebae. This effect was seen only with uninfected amoebae, not with Legionella-infected amoebae. The supporting effect for intracellular multiplication in MM6 could be reproduced in part by addition of a cell-free coculture supernatant obtained from a coincubation experiment with uninfected A. castellanii and Legionella-infected MM6, suggesting that amoeba-derived effector molecules are involved in this phenomenon. This coculture model allows investigations of molecular and biochemical mechanisms which are responsible for the enhancement of intracellular multiplication of legionellae in monocytic cells after interaction with amoebae.'}, {'qas': [{'id': '29577680:Streptococcus dysgalactiae subsp.dysgalactiae', 'question': 'Can Streptococcus dysgalactiae subsp.dysgalactiae cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Streptococcus dysgalactiae subsp. dysgalactiae isolated from milk of the bovine udder as emerging pathogens: In vitro and in vivo infection of human cells and zebrafish as biological models.\\nABSTRACT:Streptococcus dysgalactiae subsp. dysgalactiae (SDSD) is a major cause of bovine mastitis and has been regarded as an animal-restricted pathogen, although rare infections have been described in humans. Previous studies revealed the presence of virulence genes encoded by phages of the human pathogen Group A Streptococcus pyogenes (GAS) in SDSD isolated from the milk of bovine udder with mastitis. The isolates SDSD VSD5 and VSD13 could adhere and internalize human primary keratinocyte cells, suggesting a possible human infection potential of bovine isolates. In this work, the in vitro and in vivo potential of SDSD to internalize/adhere human cells of the respiratory track and zebrafish as biological models was evaluated. Our results showed that, in vitro, bovine SDSD strains could interact and internalize human respiratory cell lines and that this internalization was dependent on an active transport mechanism and that, in vivo, SDSD are able to cause invasive infections producing zebrafish morbidity and mortality. The infectious potential of these isolates showed to be isolate-specific and appeared to be independent of the presence or absence of GAS phage-encoded virulence genes. Although the infection ability of the bovine SDSD strains was not as strong as the human pathogenic S. pyogenes in the zebrafish model, results suggested that these SDSD isolates are able to interact with human cells and infect zebrafish, a vertebrate infectious model, emerging as pathogens with zoonotic capability.'}, {'qas': [{'id': '17350201:Pseudomonas alcaligenes', 'question': 'Can Pseudomonas alcaligenes cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Salvage treatment of pneumonia and initial treatment of tracheobronchitis caused by multidrug-resistant Gram-negative bacilli with inhaled polymyxin B.\\nABSTRACT:Systemic colistin has shown efficacy against multidrug-resistant Pseudomonas aeruginosa and Acinetobacter spp., but it has presented poor results in pneumonia. Aerosolized polymyxin in cystic fibrosis patients has had good results. In this study, inhaled polymyxin B was used to treat respiratory infections by multidrug-resistant Gram-negative bacilli (MR-GNBs). Nineteen patients were treated with inhaled polymyxin B: 14 pneumonia, most of which had previously failed treatment with intravenous polymyxin B, and 5 tracheobronchitis. Inhaled polymyxin B was given at the dose of 500,000 IU twice a day after an aerosolized beta(2)-agonist. In pneumonia, inhaled and intravenous polymyxin B was given together. Median age was 69 years; 89% were in the intensive care unit. Sixteen infections (84%) were caused by P. aeruginosa. Klebsiella pneumoniae, Alcaligenes xylosoxidans, and Burkholderia sp. caused one infection each. In the 14 pneumonia cases, median of previous use of intravenous polymyxin B was 20 days (range, 0-32). Inhaled polymyxin B was used for a mean of 14 days (range, 4-25). Cure occurred in 10 (53%) patients, improvement in 8 (42%), and failure in 1. Nine patients died during hospitalization (all with pneumonia). Adverse events occurred in 4 patients without interruption of inhalation. This is the largest report using inhaled polymyxin B to treat nosocomial pneumonia by MR-GNB that had failed intravenous polymyxin B. It was also effective alone in P. aeruginosa tracheobronchitis.'}, {'qas': [{'id': '7425784:Campylobacter sputorum', 'question': 'Can Campylobacter sputorum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Oxygen consumption by Campylobacter sputorum subspecies Bubulus with formate as substrate.\\nABSTRACT:The kinetics of oxygen utilization by the microaerophile Campylobacter sputorium subspecies bubulus was studied. With formate as substrate two enzyme systems were found to be responsible for electron transfer between formate and oxygen. In the case of lactate oxidation one enzyme system could account for the activity measured. One of the formate-oxidizing systems possessed a high affinity for oxygen [Km(O2) = approx. 4 microM O2]. From inhibitor studies it was concluded that a respiratory chain was involved in its activity. Respiration by this system must be responsible for proton translocation and electron transport-linked phosphorylation at formate oxidation. The other enzyme system had an extremely low affinity for oxygen [Km (O2) = approx. 1 mM O2]. It was tentatively identified as the H2O2-producing formate oxidase previously found in C. sputorum. The H2O2 production by this enzyme is implicated in an explantation of the microaerophilic nature of C. sputorum. Sensitivity of formate dehydrogenase to H2O2 was demonstrated. The influence of the formate concentration on aerobic formate oxidation was determined. The pH- and temperature dependencies of oxygen uptake with formate as substrate were examined at air-saturation and at a low dissolved oxygen tension.'}, {'qas': [{'id': '28713344:Veillonella alcalescens', 'question': 'Can Veillonella alcalescens cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:Outer Membrane Proteome of Veillonella parvula: A Diderm Firmicute of the Human Microbiome.\\nABSTRACT:Veillonella parvula is a biofilm-forming commensal found in the lungs, vagina, mouth, and gastro-intestinal tract of humans, yet it may develop into an opportunistic pathogen. Furthermore, the presence of Veillonella has been associated with the development of a healthy immune system in infants. Veillonella belongs to the Negativicutes, a diverse clade of bacteria that represent an evolutionary enigma: they phylogenetically belong to Gram-positive (monoderm) Firmicutes yet maintain an outer membrane (OM) with lipopolysaccharide similar to classic Gram-negative (diderm) bacteria. The OMs of Negativicutes have unique characteristics including the replacement of Braun's lipoprotein by OmpM for tethering the OM to the peptidoglycan. Through phylogenomic analysis, we have recently provided bioinformatic annotation of the Negativicutes diderm cell envelope. We showed that it is a unique type of envelope that was present in the ancestor of present-day Firmicutes and lost multiple times independently in this phylum, giving rise to the monoderm architecture; however, little experimental data is presently available for any Negativicutes cell envelope. Here, we performed the first experimental proteomic characterization of the cell envelope of a diderm Firmicute, producing an OM proteome of V. parvula. We initially conducted a thorough bioinformatics analysis of all 1,844 predicted proteins from V. parvula DSM 2008's genome using 12 different localization prediction programs. These results were complemented by protein extraction with surface exposed (SE) protein tags and by subcellular fractionation, both of which were analyzed by liquid chromatography tandem mass spectrometry. The merging of proteomics and bioinformatics results allowed identification of 78 OM proteins. These include a number of receptors for TonB-dependent transport, the main component of the BAM system for OM protein biogenesis (BamA), the Lpt system component LptD, which is responsible for insertion of LPS into the OM, and several copies of the major OmpM protein. The annotation of V. parvula's OM proteome markedly extends previous inferences on the nature of the cell envelope of Negativicutes, including the experimental evidence of a BAM/TAM system for OM protein biogenesis and of a complete Lpt system for LPS transport to the OM. It also provides important information on the role of OM components in the lifestyle of Veillonella, such as a possible gene cluster for O-antigen synthesis and a large number of adhesins. Finally, many OM hypothetical proteins were identified, which are priority targets for further characterization.\"}, {'qas': [{'id': '16014483:Staphylococcus delphini', 'question': 'Can Staphylococcus delphini cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Staphylococcus pseudintermedius sp. nov., a coagulase-positive species from animals.\\nABSTRACT:Four staphylococcal isolates from clinical and necropsy specimens from a cat, a dog, a horse and a parrot (Psittacus erithacus timneh) were found to constitute a distinct taxon. 16S rRNA gene sequence analysis revealed that its closest phylogenetic relatives are Staphylococcus intermedius and Staphylococcus delphini. Growth characteristics, biochemical features and DNA-DNA hybridizations demonstrated that the strains differ from these and other known species and that they represent a single, novel Staphylococcus species for which the name Staphylococcus pseudintermedius sp. nov. is proposed. The novel species is commonly confused with S. intermedius in routine diagnostic veterinary bacteriology. Although the strains described were isolated from lesions and show several characteristics typical of pathogenic staphylococci, such as coagulase, DNase and beta-haemolysin production, the pathogenic significance of the novel species remains unclear. The type strain, LMG 22219(T) (=ON 86(T)=CCUG 49543(T)), was isolated from lung tissue of a cat.'}, {'qas': [{'id': '25093856:Staphylococcus delphini', 'question': 'Can Staphylococcus delphini cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Detection of airborne bacteria in a duck production facility with two different personal air sampling devices for an exposure assessment.\\nABSTRACT:Prevalent airborne microorganisms are not well characterized in industrial animal production buildings with respect to their quantity or quality. To investigate the work-related microbial exposure, personal bioaerosol sampling during the whole working day is recommended. Therefore, bioaerosol sampling in a duck hatchery and a duck house with two personal air sampling devices, a filter-based PGP and a NIOSH particle size separator, was performed. Subsequent, quantitative and qualitative analyses were carried out with\" culture independent methods. Total cell concentrations (TCC) determined via fluorescence microscopy showed no difference between the two devices. In average, 8 x 10(6) cells/m(3) were determined in the air of the duck hatchery and 5 x 10(7) cells/m(3) in the air of the duck house. A Generated Restriction Fragment Length Polymorphism (RFLP) pattern revealed deviant bacterial compositions comparing samples collected with both devices. Clone library analyses based on 16S rRNA gene sequence analysis from the hatchery\\'s air showed 65% similarity between the two sampling devices. Detailed 16S rRNA gene sequence analyses showed the occurrence of bacterial species like Acinetobacter baumannii, Enterococcus faecalis, Escherichia sp., and Shigella sp.; and a group of Staphylococcus delphini, S. intermedius, and S. pseudintermedius that provided the evidence of potential exposure to risk group 2 bacteria at the hatchery workplace. Size fractionated sampling with the developed by the Institute for Occupational Safety and Health of the German Social Accident Insurance (IFA) device revealed that pathogenic bacteria would deposit in the inhalable, the thorax, and possibly alveolar dust fraction according to EN481. TCC analysis showed the deposition of bacterial cells in the third stage (&lt; 1mum) at the NIOSH device which implies that bacteria can reach deep into the lungs and contaminate the alveolus after inhalation. Nevertheless, both personal sampling devices could be recommended for exposure assessment at agricultural workplaces. '}, {'qas': [{'id': '9097849:Tsukamurella paurometabola', 'question': 'Can Tsukamurella paurometabola cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:[Tsukamurella infections. Review of the literature apropos of a case].\\nABSTRACT:The genus Tsukamurella belongs to the family Nocardiaceae, and is an environmental saprophyte. The type species is Tsukamurella paurometabola. Its microbiological identification and differentiation from the other species containing mycolic acids can be difficult. There has been a few cases of human infections reported, usually in patients with special conditions, such as chronic lung pathology, immuno-suppression (leukemia, solid tumors, maybe HIV-infection) or the long-term use of indwelling catheters. The treatment of choice, despite the lack of adequate guidelines, is an antibiotherapy combining a beta-lactam and an aminoglycoside; catheter removal appears to be essential for cure.'}, {'qas': [{'id': '26095034:Chlamydia abortus', 'question': 'Can Chlamydia abortus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Intratracheal infection as an efficient route for testing vaccines against Chlamydia abortus in sheep.\\nABSTRACT:Pregnant ewes have been widely used to test vaccines against Chlamydia abortus. However, this model entails many disadvantages such as high economic costs and long periods of pregnancy. The murine model is very useful for specific studies but cannot replace the natural host for the later stages of vaccine evaluation. Therefore, a non-pregnant model of the natural host might be useful for a vaccine trial to select the best vaccine candidates prior to use of the pregnant model. With this aim, two routes of infection were assessed in young non-pregnant sheep, namely, intranasal (IN) and intratracheal (IT). In addition, groups of non-vaccinated sheep and sheep immunised with an inactivated vaccine were established to investigate the suitability of the model for testing vaccines. After the experimental infection, isolation of the microorganism in several organs, with pathological and immunohistochemical analyses, antibody production assessment and investigation by PCR of the presence of chlamydia in the vagina or rectum were carried out. Experimental IT inoculation of C. abortus induced pneumonia in sheep during the first few days post-infection, confirming the suitability of the IT route for testing vaccines in the natural host. The course of infection and the resulting pathological signs were less severe in vaccinated sheep compared with non-vaccinated animals, demonstrating the success of vaccination. IN infection did not produce evident lesions or demonstrate the presence of chlamydial antigen in the lungs and cannot be considered an appropriate model for testing vaccines. '}, {'qas': [{'id': '26848055:Clostridium baratii', 'question': 'Can Clostridium baratii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:A cluster of three cases of botulism due to Clostridium baratii type F, France, August 2015.\\nABSTRACT:A cluster of three cases of food-borne botulism due to Clostridium baratii type F occurred in France in August 2015. All cases required respiratory assistance. Consumption of a Bolognese sauce at the same restaurant was the likely source of contamination. Clostridium baratii was isolated both from stool specimens from the three patients and ground meat used to prepare the sauce. This is the second episode reported in France caused by this rare pathogen. '}, {'qas': [{'id': '9286063:Mycoplasma spermatophilum', 'question': 'Can Mycoplasma spermatophilum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Genital mycoplasma infections.\\nABSTRACT:Since 1937, 13 Mycoplasma species, two Acholeplasma species, and one Ureaplasma species have been isolated from humans. Six of these have the urogenital tract as the primary site of colonisation but others, which have the oropharynx and respiratory tract as the primary site, are found occasionally in the urogenital tract because of orogenital contact. Mycoplasma hominis was the first to be isolated and is most strongly associated with bacterial vaginosis (BV), together with a variety of other bacteria. Its involvement in pelvic inflammatory disease (PID) and other conditions may be as part of BV, although when isolated in pure culture from the blood of women who have postpartum or postabortal fever there is no reason to suspect its aetiological role. There is evidence for an aetiological role for Ureaplasma urealyticum organisms (ureaplasmas) in acute non-gonococcal urethritis (NGU) and particularly chronic NGU in men, but they rank third to Chlamydia trachomatis and M. genitalium. Whether the association of ureaplasmas with miscarriage and preterm labour is in the context of BV is not clear. Of no doubt, however, is the ability of ureaplasmas to cause septic arthritis in hypogammaglobulinaemic patients and there is evidence that they may cause some cases of sexually acquired reactive arthritis. The advent of polymerase chain reaction technology has seen an advance in the understanding of the role of M. genitalium; there is strong evidence that it is one of the causes of both acute and chronic NGU independent of C. trachomatis. There is some support for the role of M. genitalium in PID, but this needs to be substantiated. Other mycoplasmas, for example M. fermentans, M. pivum, M. primatum, M. penetrans, M. spermatophilum and even M. pneumoniae have the capacity to cause urogenital tract disease but there is no evidence to indicate that they do so.'}, {'qas': [{'id': '28884664:Porphyromonas melaninogenica', 'question': 'Can Porphyromonas melaninogenica cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Early nasopharyngeal microbial signature associated with severe influenza in children: a retrospective pilot study.\\nABSTRACT:A few studies have highlighted the importance of the respiratory microbiome in modulating the frequency and outcome of viral respiratory infections. However, there are insufficient data on the use of microbial signatures as prognostic biomarkers to predict respiratory disease outcomes. In this study, we aimed to evaluate whether specific bacterial community compositions in the nasopharynx of children at the time of hospitalization are associated with different influenza clinical outcomes. We utilized retrospective nasopharyngeal (NP) samples (n=36) collected at the time of hospital arrival from children who were infected with influenza virus and had been symptomatic for less than 2 days. Based on their clinical course, children were classified into two groups: patients with mild influenza, and patients with severe respiratory or neurological complications. We implemented custom 16S rRNA gene sequencing, metagenomic sequencing and computational analysis workflows to classify the bacteria present in NP specimens at the species level. We found that increased bacterial diversity in the nasopharynx of children was strongly associated with influenza severity. In addition, patients with severe influenza had decreased relative abundance of Staphylococcus aureus and increased abundance of Prevotella (including P. melaninogenica), Streptobacillus, Porphyromonas, Granulicatella (including G. elegans), Veillonella (including V. dispar), Fusobacterium and Haemophilus in their nasopharynx. This pilot study provides proof-of-concept data for the use of microbial signatures as prognostic biomarkers of influenza outcomes. Further large prospective cohort studies are needed to refine and validate the performance of such microbial signatures in clinical settings.'}, {'qas': [{'id': '28680649:Streptococcus defectivus', 'question': 'Can Streptococcus defectivus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Aorto-right ventricular fistula: a rare complication of Abiotrophia Endocarditis.\\nABSTRACT:A 41-year-old African male presented with worsening dyspnea and cachexia concerning for congestive heart failure. Transesophageal echocardiogram revealed a large mass attached to the aortic valve leaflet, mass attached to the flail anterior mitral valve leaflet, severe pulmonary hypertension and dilatation of the aortic root along with fistula between the right coronary aortic cusp and the right ventricular (RV) outflow tract. Blood cultures grew Abiotrophia Defectiva (AD) sensitive to vancomycin. Patient underwent emergent surgical closure of aorto RV fistula and aortic root replacement along with pulmonary and mitral valve replacement. Endocarditis caused by AD has been reported to result in heart failure, septic embolization and destruction of the valve despite use of appropriate antibiotics. To our knowledge, this is the only case of AD endocarditis without any identified entrance route; requiring replacement of pulmonary, mitral and aortic valve due to extensive valvular damage and large vegetations.'}, {'qas': [{'id': '28985698:Brucella pinnipedialis', 'question': 'Can Brucella pinnipedialis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Application of real-time quantitative PCR assays for detecting marine Brucella spp. in fish.\\nABSTRACT:Brucella ceti and Brucella pinnipedialis have been documented as occurring in marine mammals, and B. ceti has been identified in 3 naturally acquired human cases. Seroconversion and infection patterns in Pacific Northwest harbor seals ( Phoca vitulina richardii) and North Atlantic hooded seals ( Cystophora cristata) indicate post-weaning exposure through prey consumption or lungworm infection, suggesting fish and possibly invertebrates play an epizootiologic role in marine Brucella transmission and possible foodborne risk to humans. We determined if real-time quantitative PCR (qPCR) assays can detect marine Brucella DNA in fish DNA. Insertion sequence (IS) 711 gene and sequence type (ST)27 primer-probe sets were used to detect Brucella associated with marine mammals and human zoonotic infections, respectively. First, DNA extracts from paired-species fish (containing 2 species) samples were tested and determined to be Brucella DNA negative using both IS 711 and ST27 primer-probe sets. A representative paired-species fish DNA sample was spiked with decreasing concentrations of B. pinnipedialis DNA to verify Brucella detection by the IS 711 primer-probe within fish DNA. A standard curve, developed using isolated DNA from B. pinnipedialis, determined the limit of detection. Finally, the IS 711 primer-probe was used to test Atlantic cod ( Gadus morhua) DNA extracts experimentally infected with the B. pinnipedialis hooded seal strain. In culture-positive cod tissue, the IS 711 limit of detection was ~1 genome copy of Brucella. Agreement between culture and PCR results for the 9 positive and 9 negative cod tissues was 100%. Although a larger sample set is required for validation, our study shows that qPCR can detect marine Brucella in fish.'}, {'qas': [{'id': '31220185:Brucella canis', 'question': 'Can Brucella canis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Characterization of Brucella canis infection in mice.\\nABSTRACT:Canine brucellosis, caused by Brucella canis, is a disease of dogs and represents a public health concern as it can be transmitted to humans. Canine brucellosis is on the rise in the United States and there is currently no vaccine for use in dogs. Mice have been extensively utilized to investigate host-pathogen interactions and vaccine candidates for smooth Brucella species and could serve a similar role for studying B. canis. However, comparatively little is known about B. canis infection in mice. The objective of this study was to characterize the kinetics of colonization and pathogenicity of B. canis in mice in order to evaluate the mouse as a model for studying this pathogen. C57BL/6 mice were inoculated intraperitoneally with 105, 107, or 109 CFU of Brucella canis RM6/66 and euthanized 1-, 2-, 4-, 6-, 9-, and 12-weeks post-inoculation. B. canis induced splenomegaly in mice infected with 109 CFU at 1- and 2 weeks post-inoculation while no gross lesions were observed in other dose groups. Infection at the two higher doses resulted in dose-dependent granulomatous hepatitis and histiocytic infiltration of the spleen and mesenteric lymph nodes by 1-2 weeks. B. canis was cultured from the liver, spleen, uterus, bone marrow, lung, and kidney in all groups with colonization declining at a slow but steady rate throughout the experiment. Clearance was achieved by 9 weeks 105 CFU group and by 12 weeks in the 107 CFU group, while B. canis persisted in the spleen until 12 weeks in the highest dose group. Although B. canis does not demonstrate significant replication in C57BL/6 mice, it has the ability to establish an infection, induce splenomegaly, and persist for several weeks in multiple organs. Moreover, 1 x 107 CFU appears to be a suitable challenge dose for investigating vaccine safety.'}, {'qas': [{'id': '27894775:Bacillus circulans', 'question': 'Can Bacillus circulans cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Synthesis, characterization and biological studies of Schiff bases derived from heterocyclic moiety.\\nABSTRACT:Some new Schiff bases (H1-H7) have been synthesized by the condensation of 2-aminophenol, 2-amino-4-nitrophenol, 2-amino-4-methylphenol, 2-amino benzimidazole with thiophene-2-carboxaldehyde and pyrrole-2-carboxaldehyde. The structures of newly synthesized compounds were characterized by elemental analysis, FT-IR, 1H NMR, UV-VIS, and single crystal X-ray crystallography. The in vitro antibacterial activity of the synthesized compounds has been tested against Salmonella typhi, Bacillus coagulans, Bacillus pumills, Escherichia coli, Bacillus circulans, Pseudomonas, Clostridium and Klebsilla pneumonia by disk diffusion method. The quantitative antimicrobial activity of the test compounds was evaluated using Resazurin based Microtiter Dilution Assay. Ampicillin was used as standard antibiotics. Schiff bases individually exhibited varying degrees of inhibitory effects on the growth of the tested bacterial species. The antioxidant activity of the synthesized compounds was determined by the 1,1-diphenyl-2-picrylhydrazyl(DPPH) method. IC50 value of synthesized Schiff bases were calculated and compared with standard BHA.'}, {'qas': [{'id': '23041635:Brevundimonas diminuta', 'question': 'Can Brevundimonas diminuta cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Brevundimonas abyssalis sp. nov., a dimorphic prosthecate bacterium isolated from deep-subsea floor sediment.\\nABSTRACT:A novel Gram-negative, aerobic, psychrotolerant, alkali-tolerant, heterotrophic and dimorphic prosthecate bacterium, designated strain TAR-001(T), was isolated from deep-sea floor sediment in Japan. Cells of this strain had a dimorphic life cycle and developed an adhesive stalk at a site not coincident with the centre of the cell pole, and the other type of cell, a swarm cell, had a polar flagellum. Colonies were glossy, viscous and yellowish-white in colour. The temperature, pH and salt concentration range for growth were 2-41  C, pH 6.5-10.0 and 1-4% (w/v) NaCl. Phylogenetic analysis based on 16S rRNA gene sequences confirmed that strain TAR-001(T) belongs to the family Caulobacteraceae of the class Alphaproteobacteria, and lies between the genus Brevundimonas and the genus Caulobacter. Levels of similarity between the 16S rRNA gene sequence of strain TAR-001(T) and those of the type strains of Brevundimonas species were 93.3-95.7%; highest sequence similarity was with the type strain of Brevundimonas diminuta. Levels of sequence similarity between those of the type strains of Caulobacter species were 94.9-96.0%; highest sequence similarity was with the type strain of Caulobacter mirabilis. The G+C content of strain TAR-001(T) was 67.6 mol%. Q-10 was the major respiratory isoprenoid quinone. The major fatty acids were C18:1omega7c and C16:0, and the presence of 1,2-di-O-acyl-3-O-[D-glucopyranosyl-(1 4)-alpha-D-glucopyranuronosyl]glycerol suggests strain TAR-001(T) is more closely to the genus Brevundimonas than to the genus Caulobacter. The mean DNA-DNA hybridization levels between strain TAR-001(T) and the type strains of two species of the genus Brevundimonas were higher than that of the genus Caulobacter. On the basis of polyphasic biological features and the 16S rRNA gene sequence comparison presented here, strain TAR-001(T) is considered to represent a novel species of the genus Brevundimonas, for which the name Brevundimonas abyssalis sp. nov. is proposed; the type strain is TAR-001(T) (=JCM 18150(T)=CECT 8073(T)).'}, {'qas': [{'id': '33083918:Herbaspirillum', 'question': 'Can Herbaspirillum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Bloodstream infection due to Herbaspirillum sp.: case series and review of literature.\\nABSTRACT:Herbaspirillum species are Gram-negative bacteria belonging to the class Betaproteobacteria, order Burkholderiales. The phylogenetic and phenotypic similarities among these groups easily lead to species misidentification. Herbaspirillum bacteraemia is an uncommon clinical entity. The objective of this review is to collect information to contribute to the management of this infection. We describe our own case series and review the cases reported in the literature. Cancer appears as the major underlying disease. The main source of bacteraemia was respiratory. Phenotypic identification methods often misidentify this species. Matrix-assisted laser desorption/ionization time-of-flight mass spectrometry (MALDI-TOF MS) and molecular methods identify at genus level, but species assignment is not reliable. Herbaspirillum spp. showed a highly susceptible antimicrobial profile. beta-Lactams showed good activity with low MIC values, except ampicillin. All isolates were resistant to colistin, suggesting an intrinsic resistance mechanism. Herbaspirillum spp. is an uncommon pathogen. MALDI-TOF MS or molecular methods are necessary to achieve a reliable genus identification. These species are not multidrug resistant. Piperacillin/tazobactam or ceftazidime might be a good treatment for this microorganism.'}, {'qas': [{'id': '24379289:Filifactor alocis', 'question': 'Can Filifactor alocis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Filifactor alocis infection and inflammatory responses in the mouse subcutaneous chamber model.\\nABSTRACT:Recent microbiome studies have implicated a role for Filifactor alocis in periodontal disease. In this study, we investigated the colonization and survival properties of F. alocis in a mouse subcutaneous chamber model of infection and characterized host innate immune responses. An infection of 10(9) F. alocis successfully colonized all chambers; however, the infection was cleared after 72 h. F. alocis elicited a local inflammatory response with neutrophils recruited into the chambers at 2 h postinfection along with an increase in levels of the proinflammatory cytokines interleukin 1beta (IL-1beta), IL-6, and tumor necrosis factor (TNF). F. alocis also induced apoptosis in chamber epithelial cells and neutrophils. Consistent with resolution of infection, neutrophil numbers and cytokine levels returned to baseline by 72 h. Fluorescent in situ hybridization (FISH) and quantitative PCR demonstrated that F. alocis exited the chambers and spread to the spleen, liver, lung, and kidney. Massive neutrophil infiltration was observed in the spleen and lungs, and the recruited neutrophils were in close proximity to the infecting bacteria. Significant epithelial injury was observed in the kidneys. Infection of all tissues was resolved after 7 days. This first in vivo study of the pathogenicity of F. alocis shows that in the chamber model the organism can establish a proinflammatory, proapoptotic local infection which is rapidly resolved by the host concordant with neutrophil influx. Moreover, F. alocis can spread to, and transiently infect, remote tissues where neutrophils can also be recruited. '}, {'qas': [{'id': '22700551:Mycobacterium marseillense', 'question': 'Can Mycobacterium marseillense cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Characterization of a novel variant of Mycobacterium chimaera.\\nABSTRACT:In this study, nonchromogenic mycobacteria were isolated from pulmonary samples of three patients in the Netherlands. All isolates had identical, unique 16S rRNA gene and 16S-23S ITS sequences, which were closely related to those of Mycobacterium chimaera and Mycobacterium marseillense. The biochemical features of the isolates differed slightly from those of M. chimaera, suggesting that the isolates may represent a possible separate species within the Mycobacterium avium complex (MAC). However, the cell-wall mycolic acid pattern, analysed by HPLC, and the partial sequences of the hsp65 and rpoB genes were identical to those of M. chimaera. We concluded that the isolates represent a novel variant of M. chimaera. The results of this analysis have led us to question the currently used methods of species definition for members of the genus Mycobacterium, which are based largely on 16S rRNA or rpoB gene sequencing. Definitions based on a single genetic target are likely to be insufficient. Genetic divergence, especially in the MAC, yields strains that cannot be confidently assigned to a specific species based on the analysis of a single genetic target.'}, {'qas': [{'id': '29702266:Mycobacterium yongonense', 'question': 'Can Mycobacterium yongonense cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Genome sequence of Mycobacterium yongonense RT 955-2015 isolate from a patient misdiagnosed with multidrug-resistant tuberculosis: First clinical detection in Tanzania.\\nABSTRACT:\\n'}, {'qas': [{'id': '32030727:Brucella ceti', 'question': 'Can Brucella ceti cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Brucellosis associated with stillbirth in a bottlenose dolphin in Australia.\\nABSTRACT:A captive adult female bottlenose dolphin presented with stillbirth. The placenta appeared oedematous. No other gross lesions were evident in the placenta or the stillborn calf. Histopathology revealed mild multifocal placentitis and foetal encephalitis. Brucella sp. was isolated from lung, liver, spleen and kidney. Sequence and phylogenetic analysis demonstrated this organism to be most similar to Brucella ceti sequence type (ST) 27. Brucella sp. DNA was detected in formalin-fixed paraffin-embedded placenta and brain by real-time PCR using primers targeting the IS711 gene. Immunohistochemical staining revealed Brucella sp. antigen in placental inflammation. This is the first report of isolation of Brucella sp. from a marine mammal in the Southern Hemisphere and the first report of marine Brucella-associated disease in Australia.'}, {'qas': [{'id': '14535581:Enterococcus canis', 'question': 'Can Enterococcus canis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:First cases of animal diseases published since 2000. 1. Dogs.\\nABSTRACT:In this first article of a series of papers listing first case reports of animal diseases published since 2000, the following 19 cases of dog diseases are discussed: Blastomycotic granuloma involving the cranial vena cava. Congenital myocardial hamartoma. Discospondylitis: three cases caused respectively by Pseudomonas aeruginosa, Enterococcus faecalis and Staphylococcus epidermidis. Dystrophin deficient muscular dystrophy in a Labrador Retriever. Emphysematous prostatitis. Erythema multiforme major caused by a Parvovirus infection of keratinocytes. Hemochromatosis due to repeated blood transfusions. Intraspinal synovial cyst. Juvenile nephropathy in the Collie and the Irish Wolfhound. Primary cerebellar cortical degeneration (abiotrophy) in a Scottish terrier. Primary pulmonary artery chondrosarcoma. Renal dysplasia in a Bull Mastiff. Rhabdomyosarcoma (botryoid sarcoma) of the urinary bladder in a Maltese. Spinal mast cell tumor. Spongiform degeneration of the white matter in the central nervous system of Australian Cattle dog. Systemic pasteurellosis caused by Pasteurella canis. Thymic hemorrhage caused by dicumarol intoxication. Undimerized biclonal gammopathy with a single heavy chain class IgA in a dog with multiple myeloma. After a short introduction, the bibliographical data and the abstract of the author(s) and mostly some additional information derived from the article are given. The article will be regularly updated adding overlooked as well as new first reports.'}, {'qas': [{'id': '30521551:Eggerthella', 'question': 'Can Eggerthella cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Gut microbiota signatures in cystic fibrosis: Loss of host CFTR function drives the microbiota enterophenotype.\\nABSTRACT:\\n'}, {'qas': [{'id': '31310211:Leptotrichia amnionii', 'question': 'Can Leptotrichia amnionii cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Sneathia amnii and Maternal Chorioamnionitis and Stillbirth, Mozambique.\\nABSTRACT:We report a case of Sneathia amnii as the causative agent of maternal chorioamnionitis and congenital pneumonia resulting in a late fetal death in Mozambique, with strong supportive postmortem molecular and histopathologic confirmation. This rare, fastidious gram-negative coccobacillus has been reported to infrequently cause abortions, stillbirths, and neonatal infections.'}, {'qas': [{'id': '3195881:Vibrio vulnificus', 'question': 'Can Vibrio vulnificus cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Vibrio vulnificus and pulmonary infection.\\nABSTRACT:'}, {'qas': [{'id': '23710815:Arcanobacterium haemolyticum', 'question': 'Can Arcanobacterium haemolyticum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:[Lemierre syndrome caused by Arcanobacterium haemolyticum: a case report and review of the literature].\\nABSTRACT:\\n'}, {'qas': [{'id': '32672529:Staphylococcus cornubiensis', 'question': 'Can Staphylococcus cornubiensis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:Staphylococcus ursi sp. nov., a new member of the 'Staphylococcus intermedius group' isolated from healthy black bears.\\nABSTRACT:Six Staphylococcus strains were isolated from healthy black bears (Ursus americanus) in the Great Smoky Mountains National Park, Tennessee, USA. Phylogenetic analysis based on complete genome, 16S rRNA, dnaJ, hsp60, rpoB and sodA genes, and MALDI-TOF-MS main spectral profiles revealed that the strains belonged to one species and showed the closest relatedness to members of the 'Staphylococcus intermedius group' (SIG), which include Staphylococcus intermedius, Staphylococcus pseudintermedius, Staphylococcus delphini and Staphyloccoccus cornubiensis. The strains were positive in SIG-specific and negative in individual species-specific PCR assays for the nuc gene. The strains can be differentiated from the other SIG species by the absence of sucrose fermentation, from S. intermedius DSM 20373T, S. pseudintermedius CCUG 49543T and S. cornubiensis DSM 105366T by the absence of methyl beta-d-glucopyranoside fermentation and from S. delphini DSM 20771T by fermentation of trehalose. DNA relatedness of the type strain MI 10-1553T with the type strains of S. delphini, S. pseudintermedius, S. intermedius and S. cornubiensis was <=48.2 % by digital DNA-DNA hybridization and <=92.3 % by average nucleotide identity calculations. Iso-C15:0, anteiso-C15 : 0 and anteiso-C17 : 0 were the most common fatty acids. Polar lipids consisted of phosphadidylglycerols, phospholipids, glycolipid, diphosphatidylglycerol and aminophospholipid. Cell-wall peptidoglycan was of type A3alpha l-Lys-Gly3 (Ser; similar to A11.2 and A11.3). The respiratory quinone belonged to menaquinone 7 (MK-7). The G+C content of MI 10-1553T was 39.3 mol%. The isolated strains represent a novel species of the genus Staphylococcus, for which we propose the name Staphylococcus ursi sp. nov. The type strain is MI 10-1553T (=ATCC TSD-55T=CCOS 1900T).\"}, {'qas': [{'id': '24363295:Propioniferax', 'question': 'Can Propioniferax cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Mariniluteicoccus flavus gen. nov., sp. nov., a new member of the family Propionibacteriaceae, isolated from a deep-sea sediment.\\nABSTRACT:A Gram-staining-positive, aerobic, non-motile, irregular coccus, designated strain YIM M13146(T), was isolated from a sediment sample collected from the South China Sea at a depth of 2439 m, and its taxonomic position was determined by a polyphasic approach. Optimal growth of the strain was observed at 30  C (range 5-40  C), pH 7.0 (pH 6.0-9.0) and 0-1% NaCl (0-6%, w/v) on/in tryptic soy agar/broth. Strain YIM M13146(T) had the major cellular fatty acid anteiso-C15:0, the predominant respiratory menaquinone MK-9(H4), peptidoglycan type A3gamma (ll-DAP-Gly) containing alanine, glycine, glutamic acid and ll-diaminopimelic acid (ll-DAP) and the polar lipids phosphatidylcholine, diphosphatidylglycerol, one unknown phospholipid and several glycolipids. The G+C content of the DNA was 67.2 mol%. Phenotypic and chemotaxonomic characteristics together with 16S rRNA gene sequence analyses showed that strain YIM M13146(T) was distinct from its close phylogenetic relatives in the genera Propioniferax and Granulicoccus of the family Propionibacteriaceae. Hence, a new genus and species, Mariniluteicoccus flavus gen. nov., sp. nov., is proposed. The type strain of Mariniluteicoccus flavus is YIM M13146(T) ( = DSM 25892(T) = CCTCC AB 2012055(T)).'}, {'qas': [{'id': '1777388:Yersinia kristensenii', 'question': 'Can Yersinia kristensenii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Occurrence of Yersinia spp. in foods in Brazil.\\nABSTRACT:Over the past 9 years, 468 bacterial strains isolated from raw and pasteurized milk, beef and pork, bovine and chicken liver, chicken heart, gizzards and lung sausage, hamburger, cheese and lettuce in different regions of the State of Sao Paulo and in the city of Rio de Janeiro were received by the Reference Laboratory for Yersinia in Brazil. All were confirmed to be Yersinia spp. The 468 Yersinia isolates were grouped as 184 strains because some of the bacteria isolated from the same food sample belonged to the same species, and were considered to be a single strain. The Yersinia food strains were classified as Y. enterocolitica (46), Y. intermedia (67), Y. frederiksenii (20), Y. kristensenii (8) and 43 of them were biochemically atypical. Pathogenic types were not detected.'}, {'qas': [{'id': '19060075:Corynebacterium sputi', 'question': 'Can Corynebacterium sputi cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Corynebacterium sputi sp. nov., isolated from the sputum of a patient with pneumonia.\\nABSTRACT:A coryneform bacterium isolated from the sputum of a patient with pneumonia was characterized by phenotypic and molecular taxonomic methods. Chemotaxonomic investigations revealed the presence of cell-wall chemotype IV and short chain mycolic acids consistent with the genus Corynebacterium. Comparative 16S rRNA gene sequencing studies confirmed this assignment, with the organism forming a hitherto unknown subline within the genus associated with a subcluster containing Corynebacterium hansenii, Corynebacterium freneyi, Corynebacterium xerosis, Corynebacterium amycolatum and Corynebacterium sphenisci. Sequence divergence values of &gt;2.7 % from established corynebacterial species suggested that the new isolate represented a novel species. This was also supported by the results of the biochemical tests. On the basis of phenotypic and phylogenetic evidence, it is proposed that the unknown bacterium be classified as a novel species of the genus Corynebacterium, Corynebacterium sputi sp. nov. (type strain IMMIB L-999(T)=DSM 45148(T)=CCUG 55795(T)).'}, {'qas': [{'id': '32212561:Mycobacterium marinum', 'question': 'Can Mycobacterium marinum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:MYCOBACTERIAL DISEASE AND SUBSEQUENT DIAGNOSTIC INVESTIGATIONS IN A GROUP OF CAPTIVE PINNIPEDS IN NEW ZEALAND.\\nABSTRACT:This case series includes a single case of disseminated tuberculous disease due to Mycobacterium pinnipedii in a New Zealand fur seal (Arctocephalus forsteri), which was being cared for by a zoo in New Zealand. The remaining five pinnipeds in the colony underwent extensive mycobacterial disease surveillance over the following 4 yr, involving a total of 26 anesthetic procedures and numerous diagnostic tests that included comparative intradermal tuberculin skin tests, mycobacterial antibody serology, respiratory and gastric lavages, and computed tomography (CT) scans. An additional case of chronic sinusitis due to Mycobacterium marinum and Pseudomonas aeruginosa was identified in a California sea lion (Zalophus californianus). Results from CT and the respiratory lavages were the most helpful antemortem diagnostic tests for active mycobacterial disease in this case series. Of the remaining four animals, two were euthanatized and two remain alive, and none of them had evidence of active mycobacterial disease. Further mycobacterial disease surveillance in staff and animals was performed, and no other case was identified. There are no validated mycobacterial surveillance tests available for pinnipeds and so it remains unknown whether the two surviving pinnipeds are truly negative or whether they have latent mycobacterial infection that could develop into active mycobacterial disease in the future. For this reason, increased levels of biosecurity and quarantine remain permanently in place for the pinniped colony.'}, {'qas': [{'id': '28551873:Achromobacter marplatensis', 'question': 'Can Achromobacter marplatensis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Achromobacter panacis sp. nov., isolated from rhizosphere of Panax ginseng.\\nABSTRACT:A novel strain DCY105T was isolated from soil collected from the rhizosphere of ginseng (Panax ginseng), in Gochang, Republic of Korea. Strain DCY105T is Gram-reaction-negative, white, non-motile, non-flagellate, rod-shaped and aerobic. The bacteria grow optimally at 30 C, pH 6.5-7.0 and in the absence of NaCl. Phylogenetically, strain DCY105T is most closely related to Achromobacter marplatensis LMG 26219T (96.81%). The DNA G+C content of strain DCY105T was 64.4 mol%. Ubiquinone 8 was the major respiratory quinone, and phosphatidylethanolamine, phosphatidylglycerol, and diphosphatidylglycerol were amongst the major polar lipids. C16:00, C8:03OH and iso-C17:03OH were identified as the major fatty acids present in DCY105T. The results of physiological and biochemical tests allowed strain DCY105T to be differentiated phenotypically from other recognized species belonging to the genus Achromobacter. Therefore, it is suggested that the newly isolated organism represents a novel species, for which the name Achromobacter panacis sp. nov. is proposed with the type strain designated as DCY105T (=CCTCCAB 2015193T =KCTC 42751T).'}, {'qas': [{'id': '19567344:Leclercia adecarboxylata', 'question': 'Can Leclercia adecarboxylata cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Antibiotic susceptibility profiles of uncommon bacterial species causing severe infections in Italy.\\nABSTRACT:This study presents the results of the italian \"Severe infections project\" involving bacteria that can be considered rare causes of disease. we isolated 30 uncommon human pathogens from a total of 60 strains (1.2% of all the isolates). The most frequent sources of uncommon human pathogens were primary bloodstream infections (48.3%) and pneumonia (20%). Species such as Comamonas testosteroni, Enterococcus hirae, Kluyvera ascorbata, Kluyvera cryocrescens, Leclercia adecarboxylata and Ochrobactrum anthropi were recovered from bacteremia patients. Clinically useful antimicrobial agents were tested against each isolate. Resistance to 4 or more antibiotics tested was found in Achromobacter xylosoxidans, O. anthropi, Pseudomonas stutzeri, Citrobacter braakii, Enterobacter sakazakii, K. ascorbata, Proteus penneri and Serratia plymuthica. About 16% of the Gram-negative species were resistant to third-generation cephalosporins and 28.6% of the staphylococci were oxacillin-resistant. the results from this study offer indications for empirical therapy for severe infections from uncommon human pathogens.'}, {'qas': [{'id': '26904200:Achromobacter ruhlandii', 'question': 'Can Achromobacter ruhlandii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Distribution of innate efflux-mediated aminoglycoside resistance among different Achromobacter species.\\nABSTRACT:Achromobacter spp. are emerging respiratory pathogens in cystic fibrosis patients. Since 2013 the genus Achromobacter includes 15 species for which innate antibiotic resistance is unknown. Previously the AxyXY-OprZ efflux system has been described to confer aminoglycoside (AG) resistance in A. xylosoxidans. Nevertheless, some Achromobacter spp. strains are susceptible to AG. This study including 49 Achromobacter isolates reveals that AG resistance is correlated with different Achromobacter spp. It is noteworthy that the axyXY-oprZ operon is detected only in AG-resistant species, including the most frequently encountered in cystic fibrosis patients: A. xylosoxidans, A. ruhlandii, A. dolens and A. insuavis. '}, {'qas': [{'id': '9559181:Borrelia persica', 'question': 'Can Borrelia persica cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:[A great Franco-Mauritian epidemiologist: Joseph Desire Tholozan (1820-18970].\\nABSTRACT:Born in 1820 from French parents in Diego Garcia, an islet then linked to Mauritius where he started in Port-Louis his school years, Joseph Desire Tholozan was an original personality. He undertook medical studies in France (M. D. thesis, Paris, 1843) after having joined the military Health Service (1841) as a surgeon serving in various garrisons in the country and later at the Hospital of the Valde-Grace in Paris (1849). Successful at the \"agregation\" of Medicine in 1853, he later participated to the Crimean War (1854-1855) where he performed-interesting medical observations. In 1858, he was appointed personal physician to Nasreddin Shah and remained in Persia until his death in Teheran (1897) where he is buried. Tholozan published between 1847 and 1892 over fifty articles and books dealing chiefly with infectious pathology and epidemiology, written at a time when microbial etiology and specificity of such diseases were wholly unknown. He considered chiefly bubonic plague, studying as soon as 1871 the focus of the Iranian Kurdistan, a research which will be resumed by M. Baltazard and his collaborators between 1947 and 1971, i.e. a century later. He was also deeply interested by the \"oriental\" cholera of which he recalled masterly the history and geography in the Near and Middle East. He also performed, while in Crimea and Persia, personal observations on tuberculosis, diptheria, remittent fever, acrodynia and had studied in France in his early years various other diseases such as cutaneous staphylococcic infections, glanders, pulmonary haemorrhagies, etc. In Persia, he reorganized Public Health and medical teaching and educated many local physicians and surgeons. Being assured of the unlimited confidence of the Shah, he played an important cultural role, promoting French influence in Persia. Holder of many French and foreign decorations, Tholozan was Fellow of the French Academies of Sciences and Medicine. His name was given by Laboulbene to Ornithodoros tholozani, a tick vector of a recurrent fever (spirochetosis due to Borrelia persica), of which he had described both the symptoms and the vector in 1882.'}, {'qas': [{'id': '4436927:Acholeplasma laidlawii', 'question': 'Can Acholeplasma laidlawii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Acholeplasma laidlawii and Pasteurella multocida isolated from the pneumonic lung of a lynx.\\nABSTRACT:'}, {'qas': [{'id': '9142784:Pseudomonas oryzihabitans', 'question': 'Can Pseudomonas oryzihabitans cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Flavimonas oryzihabitans bacteremia: clinical features and microbiological characteristics of isolates.\\nABSTRACT:Flavimonas oryzihabitans is rarely reported as a pathogen in humans. Twelve cases of F. oryzihabitans bacteremia were diagnosed at National Taiwan University Hospital over a 3-year period. The clinical features of these patients were analyzed, and antimicrobial susceptibilities and random amplified polymorphic DNA (RAPD) patterns of the 12 isolates were studied. Among these 12 patients, eight (67%) had underlying neoplastic diseases and all acquired F. oryzihabitans bacteremia while hospitalized. The clinical syndromes included primary bacteremia in 5 patients (42%), biliary tract infection in 3 (25%), and peritonitis, subdural empyema, infusion-related bacteremia, and pneumonia in 1 each. Polymicrobial bacteremia or concomitant fungemia was seen in three patients (25%). All the patients survived after antibiotic treatment. All isolates were susceptible to piperacillin, third-generation cephalosporins, aminoglycosides, and quinolones but resistant to cephalothin, cefuroxime, and trimethoprim. Susceptibility to aztreonam was variable (25%). The RAPD patterns differed among the isolates, indicating the epidemiological unrelatedness of these infections. F. oryzihabitans should be included as an etiology of severe nosocomial infection in patients with underlying debilitating diseases.'}, {'qas': [{'id': '24434511:Haemophilus ducreyi', 'question': 'Can Haemophilus ducreyi cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Cytolethal distending toxin B as a cell-killing component of tumor-targeted anthrax toxin fusion proteins.\\nABSTRACT:Cytolethal distending toxin (Cdt) is produced by Gram-negative bacteria of several species. It is composed of three subunits, CdtA, CdtB, and CdtC, with CdtB being the catalytic subunit. We fused CdtB from Haemophilus ducreyi to the N-terminal 255 amino acids of Bacillus anthracis toxin lethal factor (LFn) to design a novel, potentially potent antitumor drug. As a result of this fusion, CdtB was transported into the cytosol of targeted cells via the efficient delivery mechanism of anthrax toxin. The fusion protein efficiently killed various human tumor cell lines by first inducing a complete cell cycle arrest in the G2/M phase, followed by induction of apoptosis. The fusion protein showed very low toxicity in mouse experiments and impressive antitumor effects in a Lewis Lung carcinoma model, with a 90% cure rate. This study demonstrates that efficient drug delivery by a modified anthrax toxin system combined with the enzymatic activity of CdtB has great potential as anticancer treatment and should be considered for the development of novel anticancer drugs. '}, {'qas': [{'id': '32626315:Pediococcus acidilactici', 'question': 'Can Pediococcus acidilactici cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Assessment of the application for renewal of authorisation of Bactocell  (Pediococcus acidilactici CNCM I-4622) as a feed additive for weaned piglets, pigs for fattening, minor porcine species (weaned and for fattening), chickens for fattening, laying hens and minor avian species for fattening and for laying and its extension of use to all growing pigs and all avian species.\\nABSTRACT:Following a request from the European Commission, the Panel on Additives and Products or Substances used in Animal Feed (FEEDAP) was asked to deliver a scientific opinion on Bactocell  (Pediococcus acidilactici CNCM I-4622) in the context of the renewal of the authorisation for weaned piglets, pigs for fattening, minor porcine species (weaned and for fattening), chickens for fattening, laying hens and minor avian species for fattening and for laying when used as a zootechnical feed additive (gut flora stabiliser) in feed or in water for drinking. In addition, the applicant requested the extension of use for suckling piglets, minor pig species (growing/for fattening), chickens reared for laying, chickens reared for breeding purposes, chickens for breeding purposes, turkeys and minor avian species (including non-food producing/ornamental birds) reared for laying/breeding purposes and for breeding purposes when used as in feed or in water for drinking. The applicant has provided evidence that the additive currently on the market complies with the conditions of authorisation. The additive is safe for the target species, consumers and the environment as well. The additive is non-irritant to skin and eyes and is not a dermal sensitiser but should be considered a respiratory sensitiser. Considering the high dusting potential of the formulations, exposure of users by inhalation is very likely. The additive, at the level of 1 x 109 CFU/kg feed (5 x 108 when delivered in water), has the potential to be efficacious in the new species proposed: chickens reared for laying, chickens reared for breeding purposes, chickens for breeding purposes, turkeys and minor avian species reared for laying/breeding purposes and for breeding purposes (including non-food producing/ornamental birds) and in suckling piglets and minor porcine species (growing/for fattening).'}, {'qas': [{'id': '23827995:Mycobacterium holsaticum', 'question': 'Can Mycobacterium holsaticum cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Nontuberculous mycobacteria in respiratory samples from patients with pulmonary tuberculosis in the state of Rondonia, Brazil.\\nABSTRACT:The main cause of pulmonary tuberculosis (TB) is infection with Mycobacterium tuberculosis (MTB). We aimed to evaluate the contribution of nontuberculous mycobacteria (NTM) to pulmonary disease in patients from the state of Rondonia using respiratory samples and epidemiological data from TB cases. Mycobacterium isolates were identified using a combination of conventional tests, polymerase chain reaction-based restriction enzyme analysis of hsp65 gene and hsp65 gene sequencing. Among the 1,812 cases suspected of having pulmonary TB, 444 yielded bacterial cultures, including 369 cases positive for MTB and 75 cases positive for NTM. Within the latter group, 14 species were identified as Mycobacterium abscessus, Mycobacterium avium, Mycobacterium fortuitum, Mycobacterium intracellulare, Mycobacterium gilvum, Mycobacterium gordonae, Mycobacterium asiaticum, Mycobacterium tusciae, Mycobacterium porcinum, Mycobacterium novocastrense, Mycobacterium simiae, Mycobacterium szulgai, Mycobacterium phlei and Mycobacterium holsaticum and 13 isolates could not be identified at the species level. The majority of NTM cases were observed in Porto Velho and the relative frequency of NTM compared with MTB was highest in Ji-Parana. In approximately half of the TB subjects with NTM, a second sample containing NTM was obtained, confirming this as the disease-causing agent. The most frequently observed NTM species were M. abscessus and M. avium and because the former species is resistant to many antibiotics and displays unsatisfactory cure rates, the implementation of rapid identification of mycobacterium species is of considerable importance. '}, {'qas': [{'id': '31388484:Gardnerella vaginalis', 'question': 'Can Gardnerella vaginalis cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Gardnerella vaginalis causing pulmonary infection in a young adult: A novel case.\\nABSTRACT:Gardnerella vaginalis is an anaerobic, gram-variable bacterium primarily found in vaginal microflora of women. Previous reports of G. vaginalis cultured in men are few and have primarily been limited to the gastrointestinal and genitourinary tract.2-4 Few reports of G. vaginalis causing severe infections have been reported in the literature, including septicemia7 and two cases of perinephric abscess.8,9 There has been one previously reported case of G. vaginalis causing pulmonary complications that occurred in a male alcohol abuser. In our case review, we aim to demonstrate an unusual source of a pulmonary infection and highlight the importance of proper microbial isolation to guide treatment. Our patient is a young male who presented following multiple gunshot wounds including one to his head causing an intracranial hemorrhage, hydrocephalus, and a dural sinus thrombosis. His hospital course was complicated by a decline in neurological status treated with a craniotomy and external drain placement and multiple pulmonary infections. During his fever work-ups, he found to have G. vaginalis on mini-bronchoalveolar lavage and was subsequently treated with metronidazole. After treating his G. vaginalis pneumonia and other infectious sources, namely Haemaphilus influenzae and coagulase-negative staphylococcus pneumonias, his fevers and leukocytosis resolved and he was successfully discharged to a rehabilitation facility for neurologic recovery. To our knowledge, this is the second reported case of G. vaginalis isolated from a pulmonary culture and the first in a previously healthy, immunocompetent young male outside of the urinary tract.'}, {'qas': [{'id': '28125697:Streptococcus hyovaginalis', 'question': 'Can Streptococcus hyovaginalis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Usefulness of MALDI-TOF MS as a Diagnostic Tool for the Identification of Streptococcus Species Recovered from Clinical Specimens of Pigs.\\nABSTRACT:The application of MALDI-TOF MS for identifying streptococcal isolates recovered from clinical specimens of diseased pigs was evaluated. For this proposal, the MALDI BDAL Database (Bruker Daltoniks, Germany) was supplemented with the main spectrum profiles (MSP) of the reference strains of S. porci, S. porcorum and S. plurextorum associated with pneumonia and septicemia. Although these three species showed similar MALDI profiles, several peaks were recognized that can be useful for their differentiation: S. porci (4113, 6133, 7975 and 8228 m/z Da), S. plurextorum (3979, 4078, 4665, 6164, 6491, 6812, 7959 and 9330 m/z Da) and S. porcorum (3385, 3954, 4190, 6772, 7908, and 8381 m/z Da). After adding these MSPs, an evaluation was conducted to determine the accuracy of MALDI-TOF MS for the identification of streptococci from diseased pigs using 74 field isolates. Isolates were identified as S. suis, S. porcinus, S. dysgalactiae, S. hyovaginalis, S. porcorum, S. alactolyticus, S. hyointestinalis and S. orisratti. This is the first time that the latter three species have been reported from clinical specimens of pigs. Overall, there was good concordance (95.9%) between the results obtained from MALDI-TOF MS identification (best hint) and those from genotyping. Our results demonstrate the good performance of MALDI-TOF MS (100% sensitivity and specificity) for identifying most of the species of streptococci that can frequently be isolated from diseased pigs. However, conflicting results were observed in the correct identification of some isolates of S. dysgalactiae and S. alactolyticus.'}, {'qas': [{'id': '27165798:Mycobacterium pinnipedii', 'question': 'Can Mycobacterium pinnipedii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Emerging Tuberculosis Pathogen Hijacks Social Communication Behavior in the Group-Living Banded Mongoose (Mungos mungo).\\nABSTRACT:UNLABELLED: An emerging Mycobacterium tuberculosis complex (MTC) pathogen, M. mungi, infects wild banded mongooses (Mungos mungo) in Northern Botswana, causing significant mortality. This MTC pathogen did not appear to be transmitted through a primary aerosol or oral route. We utilized histopathology, spoligotyping, mycobacterial interspersed repetitive units-variable number of tandem repeats (MIRU-VNTR), quantitative PCR (qPCR), and molecular markers (regions of difference [RDs] from various MTC members, including region of difference 1 [RD1] from M. bovis BCG [RD1(BCG)], M. microti [RD1(mic)], and M. pinnipedii [RD1(seal)], genes Rv1510 [RD4], Rv1970 [RD7], Rv3877/8 [RD1], and Rv3120 [RD12], insertion element IS1561, the 16S RNA gene, and gene Rv0577 [cfp32]), including the newly characterized mongoose-specific deletion in RD1 (RD1(mon)), in order to demonstrate the presence of M. mungi DNA in infected mongooses and investigate pathogen invasion and exposure mechanisms. M. mungi DNA was identified in 29% of nasal planum samples (n = 52), 56% of nasal rinses and swabs (n = 9), 53% of oral swabs (n = 19), 22% of urine samples (n = 23), 33% of anal gland tissue (n = 18), and 39% of anal gland secretions (n = 44). The occurrence of extremely low cycle threshold values obtained with qPCR in anal gland and nasal planum samples indicates that high levels of M. mungi can be found in these tissue types. Histological data were consistent with these results, suggesting that pathogen invasion occurs through breaks in the nasal planum and/or skin of the mongoose host, which are in frequent contact with anal gland secretions and urine during olfactory communication behavior. Lesions in the lung, when present, occurred only with disseminated disease. No environmental sources of M. mungi DNA could be found. We report primary environmental transmission of an MTC pathogen that occurs in association with social communication behavior. IMPORTANCE: Organisms causing infectious disease evolve modes of transmission that exploit environmental and host conditions favoring pathogen spread and persistence. We report a novel mode of environmental infectious disease transmission that occurs in association with olfactory secretions (e.g., urine and anal gland secretions), allowing pathogen exposure to occur within and between social groups through intricate social communication behaviors of the banded mongoose host. The presence of M. mungi in these environmentally deposited secretions would effectively circumvent natural social barriers (e.g., territoriality), facilitating between-group pathogen transmission in the absence of direct physical contact, a rare occurrence in this highly territorial species. This work identifies an important potential mechanism of pathogen transmission of epidemiological significance in social species. We also provide evidence of a novel mechanism of pathogen transmission for the MTC complex, where pathogen movement in the environment and host exposure dynamics are driven by social behavior.'}, {'qas': [{'id': '6500835:Dorea', 'question': 'Can Dorea cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Concentration of vitamin A in the liver of foetuses and infants dying of various causes in Brasilia, Brazil.\\nABSTRACT:The vitamin A concentration in liver samples taken at autopsy from the central portion of the right lobe of 57 infants 0-1 year old was determined by a dual spectrophotometric and colorimetric assay. Death was due to respiratory disease (30%), complications of premature birth (16%), infections (14%), hemorrhage (14%), pneumonia (10%), cerebral edema (7%), and miscellaneous causes (9%). Gross malnutrition was noted in only 2 of these children. The median vitamin A concentration was 61 micrograms of retinol/g liver, with a range of 6-293 micrograms/g. The percent distribution of liver vitamin A levels in micrograms/g was: less than 5 (0%); 5-10 (7%); 10-20 (5%); 20-40 (16%); 40-80 (42%); 80-120 (14%); greater than 120 (16%). The mean liver level in 9 stillborn full-term infants (60 micrograms/g) was markedly lower than in 7 stillborn premature infants (125 micrograms/g). The median value for 22 infants from indigent families (54 micrograms/g) was lower than that of 35 infants from non-indigent families (65 micrograms/g). By applying the criteria that liver reserves of vitamin A less than or equal to 5 micrograms retinol/g of liver indicate a high risk of vitamin A deficiency and those less than 20 micrograms retinol/g of liver denote an inadequate reserve, no infant was at high risk but 12% had insufficient reserves.'}, {'qas': [{'id': '28124138:Staphylococcus xylosus', 'question': 'Can Staphylococcus xylosus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Monitoring of bacterial pathogens at workplaces in power plant using biochemical and molecular methods.\\nABSTRACT:PURPOSE: The aim of this study was to characterize the ways of spreading of the most common bacterial species isolated from workers as well as from the air and raw materials at the workplaces in power plant utilizing biomass sources. To monitor microbial transmission and identify the source of contamination in the working environment, a combination of molecular and biochemical methods was applied. \\n'}, {'qas': [{'id': '17533989:Staphylococcus lentus', 'question': 'Can Staphylococcus lentus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:[Exposure assessment to bioaerosols among sewer workers].\\nABSTRACT:\\n'}, {'qas': [{'id': '32153546:Mycobacterium marinum', 'question': 'Can Mycobacterium marinum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Identification of Anti-Mycobacterium and Anti-Legionella Compounds With Potential Distinctive Structural Scaffolds From an HD-PBL Using Phenotypic Screens in Amoebae Host Models.\\nABSTRACT:Tubercular Mycobacteria and Legionella pneumophila are the causative agents of potentially fatal respiratory diseases due to their intrinsic pathogenesis but also due to the emergence of antibiotic resistance that limits treatment options. The aim of our study was to explore the antimicrobial activity of a small ligand-based chemical library of 1255 structurally diverse compounds. These compounds were screened in a combination of three assays, two monitoring the intracellular growth of the pathogenic bacteria, Mycobacterium marinum and L. pneumophila, and one assessing virulence of M. marinum. We set up these assays using two amoeba strains, the genetically tractable social amoeba Dictyostelium discoideum and the free-living amoeba Acanthamoeba castellanii. In summary, 64 (5.1%) compounds showed anti-infective/anti-virulence activity in at least one of the three assays. The intracellular assays hit rate varied between 1.7% (n = 22) for M. marinum and 2.8% (n = 35) for L. pneumophila with seven compounds in common for both pathogens. In parallel, 1.2% (n = 15) of the tested compounds were able to restore D. discoideum growth in the presence of M. marinum spiked in a lawn of food bacteria. We also validated the generality of the hits identified in the A. castellanii-M. marinum anti-infective screen using the D. discoideum-M. marinum host-pathogen model. The characterization of anti-infective and antibacterial hits in the latter infection model revealed compounds able to reduce intracellular growth more than 50% at 30 muM. Moreover, the chemical space and physico-chemical properties of the anti-M. marinum hits were compared to standard and candidate Mycobacterium tuberculosis (Mtb) drugs using ChemGPS-NP. A principle component analysis identified separate clusters for anti-M. marinum and anti-L. pneumophila hits unveiling the potentially new physico-chemical properties of these hits compared to standard and candidate M. tuberculosis drugs. Our studies underscore the relevance of using a combination of low-cost and low-complexity assays with full 3R compliance in concert with a rationalized focused library of compounds to identify new chemical scaffolds and to dissect some of their properties prior to taking further steps toward compound development.'}, {'qas': [{'id': '24129837:Providencia alcalifaciens', 'question': 'Can Providencia alcalifaciens cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Pathogenic Providencia alcalifaciens strain that causes fatal hemorrhagic pneumonia in piglets.\\nABSTRACT:Investigation of a serious pig disease with high mortality and typical lung lesions yielded a bacterial isolate identified as Providencia alcalifaciens based on the 16S ribosomal DNA sequence analysis. The pathogenicity of this bacterial isolate was confirmed in piglets and mice. The bacterial strain caused the typical illness in piglets, which suffered serious dyspnea and hemorrhagic pneumonia. The drug resistance spectrum of the bacterium was also determined. The results indicated that the isolate is resistant to 12 antibiotics and intermediately resistant to 10 antibiotics out of the 34 antibiotics tested. The current study is the first to report a serious lung disease in piglets caused by a multidrug resistant P. alcalifaciens isolate, which should be given more attention during surveillance and diagnostics.'}, {'qas': [{'id': '10458530:Capnocytophaga sputigena', 'question': 'Can Capnocytophaga sputigena cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Second-trimester abortion caused by Capnocytophaga sputigena: case report.\\nABSTRACT:Intra-amniotic infection is often the cause of a second-trimester abortion. The bacterial species involved include bacteria with low pathogenicity like Ureaplasma urealyticum and various Mycoplasma species. In this case we describe an intra-amniotic infection caused by Capnocytophaga sputigena, often found in the normal bacterial flora of the oral cavity, but not in the vagina. Oral sex during pregnancy was the most probable source of the infection. The aborted fetus showed signs of pneumonia upon histologic examination. The bacterial species was identified using broad-spectrum 16S rDNA polymerase chain reaction (PCR) directly from the amniotic fluid and after bacterial culture. Amniotic fluid glucose was below detection level, confirming the presence of an intra-amniotic infection.'}, {'qas': [{'id': '30056210:Lactococcus garvieae', 'question': 'Can Lactococcus garvieae cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Efficacy of a Eudragit L30D-55 encapsulated oral vaccine containing inactivated bacteria (Lactococcus garvieae/Streptococcus iniae) in rainbow trout (Oncorhynchus mykiss).\\nABSTRACT:The efficacy of a Eudragit L30D-55 encapsulated vaccine against Lactococcus garvieae and Streptococcus iniae was investigated in rainbow trout. Fish were divided into four groups and fed the different experimental feeds. Groups were: A) fish immunized by Eudragit-coated pellets containing vaccine, B) fish immunized by vaccine-coated pellets without Eudragit, C) fish fed Eudragit-coated pellets without vaccine and D) fish fed pellets without vaccine orEudragit (control group). In groups A and B, the vaccination was conducted for 14 days. Similar to groups A and B, fish of group C were fed 14 days with pellets coated with Eudragit and afterwards they were fed control diet. Serum samples were taken on day 0, 20, 40 and 60 of the experiment. After 60 days, fish were challenged with L. garvieae and S. iniae. In almost all groups, innate immunity components including alternative complement activity, lysozyme activity, bactericidal activity, IgM and total protein showed no significant changes during the 60 days that the experiment lasted. However, the blood respiratory burst activity and lysozyme activity showed a significant increase on day 20 of experiment in groups B and D respectively (P &lt; 0.05). The relative expression of immune-related genes including IL-6 and IgM genes was higher in vaccinated fish, with the highest expression in those immunized by Eudragit-coated pellets (Group A). In addition, the relative expression of IL-6 and IgM peaked on day 20 but decreased on day 60 in vaccinated groups. The ELISA antibody titer against L. garvieae increased from day 20 and peaked on day 60 of experiment (P &lt; 0.05). Also, the antibody titer against L. garvieae was higher in fish immunized by Eudragit-coated pellets (Group A) compared to fish of group C and control. After bacterial challenge, a survival percentages of % 85 +- 7.07% (challenged with S. iniae) and % 72.21 +- 7.8% (challenged with L. garvieae) were observed respectively in groups immunized with pellets coated with Eudragit L30D-55 (Group A), which were higher than survival percentages obtained in other experimental groups (P &lt; 0.05). The results of the present study demonstrate that the oral administration of Eudragit L30D-55-encapsulated vaccine appropriately protects rainbow trout against Lactococcus garvieae and Streptococcus iniae.'}, {'qas': [{'id': '19734285:Pusillimonas', 'question': 'Can Pusillimonas cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Eiseniicola composti gen. nov., sp. nov., with antifungal activity against plant pathogenic fungi.\\nABSTRACT:A Gram-negative, short rod-shaped bacterial strain, YC06271T, was isolated from the vermicompost (VC) collected at Masan, Korea and its taxonomic position was investigated by a polyphasic taxonomic approach. Strain YC06271T grew optimally at 28-30 degrees C and at pH 7.0-9.0. The 16S rRNA gene sequence of strain YC06271T was most closely related to members of the genera Bordetella (96.4-95.8 %), Achromobacter (96.0-95.7 %), Alcaligenes (96.0-94.2 %), Pusillimonas noertemannii (95.9 %), Pigmentiphaga (95.8-95.5 %) and less than 95.5 % similarity with the members of the other genera of the family Alcaligenaceae. Strain YC06271T contained ubiquinone-8 (Q-8) as the major respiratory quinone system and putrescine as the major polyamine. The major fatty acids of strain YC06271T were C16:1omega7c and/or C15:0 iso 2-OH and C16:0. The major polar lipids were phosphatidylglycerol and phosphatidylethanolamine. The G+C content of the genomic DNA was 55.4 mol%. Phylogenetic analysis, biochemical, chemotaxonomic and phenotypic characteristics strongly supported the differentiation of strain YC06271T from the validly published genera of the family Alcaligenaceae. Therefore, it is proposed that strain YC06271T represents a novel species within a novel genus, with the name Eiseniicola composti gen. nov., sp. nov. The type strain is YC06271T (= KCTC 22250T = DSM 21045T).'}, {'qas': [{'id': '29984751:Campylobacter rectus', 'question': 'Can Campylobacter rectus cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Clival Osteomyelitis with Cavernous Sinus Thrombosis Due to Fusobacterium nucleatum and Campylobacter rectus Induced by Tooth Extraction.\\nABSTRACT:A 70-year-old woman who had suffered from aseptic meningitis complained of chronic headache after dental treatment including tooth extraction. She developed a fever and respiratory failure. Based on chest computed tomography and head magnetic resonance imaging (MRI), she was diagnosed with osteomyelitis in the clivus accompanying moderate pituitary involvement, cavernous sinus thrombosis and septic pulmonary embolism. Both of the causal bacteria, Fusobacterium nucleatum and Campylobacter rectus, were isolated from her blood. Dual infection leading to clival osteomyelitis and cavernous sinus thrombosis has not been reported. It is important to perform enhanced MRI and blood culture for patients with chronic headache related to dental treatment.'}, {'qas': [{'id': '32943860:Porphyromonas denticola', 'question': 'Can Porphyromonas denticola cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:Periodontal Status and Microbiologic Pathogens in Patients with Chronic Obstructive Pulmonary Disease and Periodontitis: A Case-Control Study.\\nABSTRACT:Purpose: To evaluate clinical periodontal status and microbiologic pathogens in patients with chronic obstructive pulmonary disease (COPD) and periodontitis. Patients and Methods: We conducted a case-control study of 60 periodontitis patients with COPD (case group) and 60 periodontitis patients with normal pulmonary function (control group). Their periodontal status and respiratory function were clinically examined. Real-time polymerase chain reaction assays were used to measure five dental pathogens and four respiratory pathogens in subgingival dental plaque. Spearman's rank correlation coefficients (r2) were calculated to assess correlations of pathogens. Principal component analysis (PCA) was employed to assess the similarity of bacterial diversity between the two groups. Logistic regression was performed to examine the associations of periodontal variables and pathogens with COPD risk. Results: COPD patients had fewer remaining teeth, higher plaque index (PLI), and more severe site percentages of clinical attachment level (CAL) than the controls. Although COPD patients tended to have relatively higher ranked means of Porphyromonas gingivalis, Tannerella forsythensis, Treponema denticola, and Haemophilus influenza than control participants, the differences were not significant. Some periodontal pathogens and respiratory pathogens were positively correlated with each other (r2 =0.29 to 0.47, all P < 0.05). The PCA graph showed that the distributions of pathogens were more dispersed but less discriminated in the COPD group than those in the control group. PLI (P = 0.045) and CAL >= 5mm site percentages (P = 0.01) were significantly associated with an increased risk of COPD, while pathogens were not associated with COPD. Conclusion: Our results from this study do not indicate periodontal pathogens as potential predictors of COPD risk, despite significantly poor periodontal status associated with COPD.\"}, {'qas': [{'id': '3862729:Treponema pertenue', 'question': 'Can Treponema pertenue cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:The participation of activated peritoneal macrophages in Treponema pallidum subspecies pertenue infection in Syrian hamsters.\\nABSTRACT:The role of cell-mediated immunity in hamsters during treponemal infection appears to involve the activated macrophage. To date, studies have been hindered by the inability to confirm that macrophages exhibit enhanced treponemicidal activity at the infection site. We show that lipopolysaccharide and thioglycollate-treated animals, when inoculated with Treponema pallidum subsp. pertenue, exhibit enhanced clearance of these organisms compared with controls. Macrophages from these infected groups display an enhanced respiratory burst, as detected by NBT reduction, as well as a marked increase in C3b receptor-mediated ingestion activity. Significant changes in these parameters indicate that alterations in macrophage activation are occurring in the infected compartment. Thus the stimulatory agents apparently modify the host's immune responses to promote subsequent reduction of treponemal infection. In addition, hamster peritoneal macrophages demonstrate enhanced activation behavior as a result of exposure to at least two signals, which may be prerequisite for processing this organism efficiently.\"}, {'qas': [{'id': '20042465:Corynebacterium falsenii', 'question': 'Can Corynebacterium falsenii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:Quantification and identification of culturable airborne bacteria from duck houses.\\nABSTRACT:Employees at agricultural working places are often exposed to complex bioaerosols. Investigations of bioaerosols in duck houses revealed concentrations of cultivable bacteria between 0.4 and 3 x 10(5) colony forming units (CFU) m(-3) on tryptone soy agar, 0.3 and 2 x 10(5) CFU m(-3) on actinomycetes isolation agar, and 0.8 and 5 x 10(3) CFU m(-3) on Middlebrook agar, respectively, when incubated at 25 degrees C. At an incubation temperature of 37 degrees C, 0.6-3 x 10(2) CFU m(-3) were counted on MacConkey agar and 0.3-2 x 10(3) CFU m(-3) on Middlebrook agar, and the concentrations of bacteria on glycerol-arginine agar and oatmeal agar incubated at 50 degrees C varied between 0.1 and 2 x 10(3) and 1 and 7 x 10(3) CFU m(-3), respectively. In addition, high concentrations of cells were observed by fluorescence microscope quantification of cell counts after 4',6-diamidino-2-phenylindol staining with 3-8 x 10(7) cells m(-3). A total of 213 colonies with different morphological appearance were selected and the isolated pure cultures were identified at the genus level using the 16S rRNA gene sequence analyses. In summary, 19 different genera of Actinobacteria, four genera of the Firmicutes, one genus of the Bacteroidetes, and five genera of the Proteobacteria were identified. Several isolates represent new phylogenetic lineages. Based on 16S rRNA gene analyses, some isolates were most closely related to Cellulosimicrobium funkei, Corynebacterium falsenii, Corynebacterium xerosis, Mycobacterium arupense, and Staphylococcus epidermidis, which have been grouped into Risk group 2 of biological agents and may cause negative pulmonary health effects. These bacterial species were present in high concentrations up to 10(4) CFU m(-3). For this reason, we recommend an adequate personal breathing protection at these working places.\"}, {'qas': [{'id': '11294736:Campylobacter coli', 'question': 'Can Campylobacter coli cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:Emerging mechanisms of fluoroquinolone resistance.\\nABSTRACT:Broad use of fluoroquinolones has been followed by emergence of resistance, which has been due mainly to chromosomal mutations in genes encoding the subunits of the drugs' target enzymes, DNA gyrase and topoisomerase IV, and in genes that affect the expression of diffusion channels in the outer membrane and multidrug-resistance efflux systems. Resistance emerged first in species in which single mutations were sufficient to cause clinically important levels of resistance (e.g., Staphylococcus aureus and Pseudomonas aeruginosa). Subsequently, however, resistance has emerged in bacteria such as Campylobacter jejuni, Escherichia coli, and Neisseria gonorrhoeae, in which multiple mutations are required to generate clinically important resistance. In these circumstances, the additional epidemiologic factors of drug use in animals and human-to-human spread appear to have contributed. Resistance in Streptococcus pneumoniae, which is currently low, will require close monitoring as fluoroquinolones are used more extensively for treating respiratory tract infections.\"}, {'qas': [{'id': '33006801:Shewanella algae', 'question': 'Can Shewanella algae cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Macrocyclic polyketides with siderophore mode of action from marine heterotrophic Shewanella algae: Prospective anti-infective leads attenuate drug-resistant pathogens.\\nABSTRACT:AIMS: Biotechnological and chemical characterization of previously undescribed homologous siderophore-type macrocyclic polyketides from heterotrophic Shewanella algae MTCC 12715 affiliated with Rhodophycean macroalga Hypnea valentiae of marine origin, with significant anti-infective potential against drug-resistant pathogens. METHODS AND \\n'}, {'qas': [{'id': '3705446:Staphylococcus gallinarum', 'question': 'Can Staphylococcus gallinarum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Comparative characteristics of Staphylococcus sciuri, Staphylococcus lentus and Staphylococcus gallinarum isolated from healthy and sick hosts.\\nABSTRACT:Of 136 strains of coagulase-negative staphylococci isolated from healthy and sick human beings, goats, sheep, antelope and other animals, 88 (64.7%) were Staphylococcus sciuri and 35 (25.7%) were S. lentus and the remainder Staphylococcus gallinarum. The strains of S. sciuri were isolated from humans with boils and wounds, goats with pestes des petits ruminants (PPR) and dogs with nasal discharge. One isolate of S. gallinarum came from a fowl with chronic respiratory disease and 11 others were isolated from goats. The characteristics of S. sciuri, S. lentus and S. gallinarum isolated from different sources were similar.'}, {'qas': [{'id': '2410491:Mycoplasma primatum', 'question': 'Can Mycoplasma primatum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Demonstration of cross-reactive antigens in F38 and related mycoplasmas by enzyme-linked immunosorbent assay (ELISA) and immunoblotting.\\nABSTRACT:The ELISA and an immunoblotting technique were used to study F38-type mycoplasmas - an important cause of contagious caprine pleuropneumonia - and a number of related mycoplasma species, subspecies, types or serogroups. Two-way ELISA cross-reactivity was demonstrated between five mycoplasmas, namely strain F38, Mycoplasma mycoides subsp. mycoides (LC strain), M. equigenitalium, M. primatum and bovine serogroup 7. In addition one-way cross-reactivity was demonstrated between F38 and each of the following mycoplasmas: M. mycoides subsp. mycoides (two SC strains), M. mycoides subsp. capri, and bovine serogroup L. F38 and M. capricolum did not cross-react. Immunoblot analysis, unlike ELISA, revealed that F38 and M. capricolum were closely related. At least four major protein antigens were shared between F38, M. mycoides subsp. mycoides (SC and LC strains), M. mycoides subsp. capri and bovine serogroup 7. The ELISA cross-reactions (above) shown by M. equigenitalium and M. primatum with each other, with F38 and with other mycoplasmas were not apparent by immunoblotting.'}, {'qas': [{'id': '31036687:Mycobacterium ulcerans', 'question': 'Can Mycobacterium ulcerans cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Shortening Buruli Ulcer Treatment with Combination Therapy Targeting the Respiratory Chain and Exploiting Mycobacterium ulcerans Gene Decay.\\nABSTRACT:Buruli ulcer is treatable with antibiotics. An 8-week course of rifampin (RIF) and either streptomycin (STR) or clarithromycin (CLR) cures over 90% of patients. However, STR requires injections and may be toxic, and CLR shares an adverse drug-drug interaction with RIF and may be poorly tolerated. Studies in a mouse footpad infection model showed that increasing the dose of RIF or using the long-acting rifamycin rifapentine (RPT), in combination with clofazimine (CFZ), a relatively well-tolerated antibiotic, can shorten treatment to 4 weeks. CFZ is reduced by a component of the electron transport chain (ETC) to produce reactive oxygen species toxic to bacteria. Synergistic activity of CFZ with other ETC-targeting drugs, the ATP synthase inhibitor bedaquiline (BDQ) and the bc 1:aa 3 oxidase inhibitor Q203 (now named telacebec), was recently described against Mycobacterium tuberculosis Recognizing that M. tuberculosis mutants lacking the alternative bd oxidase are hypersusceptible to Q203 and that Mycobacterium ulcerans is a natural bd oxidase-deficient mutant, we tested the in vitro susceptibility of M. ulcerans to Q203 and evaluated the treatment-shortening potential of novel 3- and 4-drug regimens combining RPT, CFZ, Q203, and/or BDQ in a mouse footpad model. The MIC of Q203 was extremely low (0.000075 to 0.00015 mug/ml). Footpad swelling decreased more rapidly in mice treated with Q203-containing regimens than in mice treated with RIF and STR (RIF+STR) and RPT and CFZ (RPT+CFZ). Nearly all footpads were culture negative after only 2 weeks of treatment with regimens containing RPT, CFZ, and Q203. No relapse was detected after only 2 weeks of treatment in mice treated with any of the Q203-containing regimens. In contrast, 15% of mice receiving RIF+STR for 4 weeks relapsed. We conclude that it may be possible to cure patients with Buruli ulcer in 14 days or less using Q203-containing regimens rather than currently recommended 56-day regimens.'}, {'qas': [{'id': '1402084:Pasteurella stomatis', 'question': 'Can Pasteurella stomatis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:[Clinicobacteriological study of Pasteurella multocida as a zoonosis (1). Condition of dog and cat carriers of Pasteurella, and the influence for human carrier rate by kiss with the pets].\\nABSTRACT:Pasteurella multocida is a gram-negative short rod-shaped bacteria, which is a part of the indigenous flora of the oral cavity of many animals other than man. The number of reports on cases of infections with this bacterium due to animal bites and/or scratches, bacterial infections of the respiratory tract, sepsis due to this organism and death caused by the bacteria have been increasing in recent years. We investigated P. multocida in the hair and oral cavity of 3 dogs and 29 cats according to the classification of Mutters et al.. We also studied the relationship between the carrier rate for Pasteurella in the oral cavity and kissing of pets in 24 pet owners (3 dogs and 11 cats). No P. multocida was isolated from the hair of neither dogs nor cats. One strain of P. multocida subsp. multocida and two strains of P. stomatis, were isolated from the oral cavity of dogs, and 35 strains of Pasteurella were isolated from the oral cavity of cats. Two strains of P. multocida subsp. multocida, whose biochemical properties were different, were detected in the oral cavity of one cat. In three cats, Pasteurella other than P. multocida subsp. multocida was isolated from the same oral cavity. No Pasteurella was detected in the oral cavity of 19 pet owners who had not kissed their cats, whereas P. stomatis was isolated from the oral cavity of one of 2 pet owners who had kissed their cats and in 2 of 3 pet owners who had kissed their dogs (the same bacteria was isolated from a dog that was being kept by some of these positive pet owners).(ABSTRACT TRUNCATED AT 250 WORDS)'}, {'qas': [{'id': '17205698:Ruminococcus productus', 'question': 'Can Ruminococcus productus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:[Liver abscess and infective endocarditis cases caused by Ruminococcus productus].\\nABSTRACT:The genus Ruminococcus which are anaerobe Gram positive cocci, previously classified as Peptostreptococcus, may colonize the upper respiratory tract, gastrointestinal tract, vagina and skin of humans and animals. In this report a case of liver abscess and a case of infective endocarditis caused by Ruminoccocus productus, which is very rarely encountered in the clinical practice were presented. The first case was a 32 years old male who was admitted to the hospital in 2002, with the complaints of fever lasting for 20 days and pain while breathing. The abdominal ultrasonography revealed the presence of a liver abscess, and the drainage material from the abscess yielded Ruminococcus productus, identified in BACTEC 9200 (Becton Dickinson, Sparks, Md) anaerobe system. As the isolate was found to be sensitive to penicilin, the empirical gentamicin and ampicillin/sulbactam therapy was continued. The second case was a 25 years old male who was admitted to the hospital in 2005, with the signs of fever lasting for 3-4 months, chills, bone and joint pains. As multiple vegetations were detected in echocardiography, blood cultures were collected and empirical therapy with ceftriaxone and gentamicin was initiated with the preliminary diagnosis of infective endocarditis. Bacteria which were isolated from blood cultures by BACTEC 9200 system have been identified as R. productus. As this strain was also sensitive to penicillin, the empirical therapy was changed to penicilin and gentamicin. These two cases indicated that R. productus should be considered in complicated infections even if it is a rarely isolated species from the clinical samples.'}, {'qas': [{'id': '19860823:Nocardia puris ', 'question': 'Can Nocardia puris  cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Clinical and microbiological characteristics of Nocardiosis including those caused by emerging Nocardia species in Taiwan, 1998-2008.\\nABSTRACT:The genus of Nocardia is rapidly expanding and the species distribution varies with different geographical locations. We retrospectively reviewed the laboratory records of the bacteriology laboratory at National Taiwan University Hospital from January 1998 to June 2008 to identify patients with nocardiosis. During the study period, 164 isolates of Nocardia spp. were identified from 134 patients but only 113 patients had Nocardia infection. Nocardia brasiliensis (n = 54) was the most common pathogen, followed by N. asteroides (n = 36), N. farcinica (n = 7), N. flavorosea (n = 4), N. otitidiscaviarum (n = 3), N. nova (n = 3), N. beijingensis (n = 2) and one each of N. puris, N. jinanensis and N. takedensis. The major types of infection were cutaneous infection (56.6%), pulmonary infection (33.6%) and disseminated infection (7.1%). Eighty-eight patients received sulfonamide-containing antibiotic and eight of 100 patients with available data on outcomes died during the episode of nocardiosis. In conclusion, the clinical and microbiological manifestations of Nocardiosis vary with the different Nocardia species. Accurate identification of the species is crucial to make the diagnosis.'}, {'qas': [{'id': '19149091:Cedecea lapagei', 'question': 'Can Cedecea lapagei cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:[A pneumonia case caused by Cedecea lapagei].\\nABSTRACT:Cedecea spp. which are the members of Enterobacteriaceae family, are mostly isolated from sputum and their clinical importance is not yet demonstrated. This report presents a pneumonia case caused by Cedecea lapagei. A 38-years-old male patient admitted to Inonu University Faculty of Medicine Emergency department with prediagnosis of subarachnoid haemorrhage was operated and transferred to Intensive Care Unit of Reanimation where he underwent artificial ventilation. On the third day of hospitalization his temperature was 39 degrees C, white blood cell count was 27.000/ml and he was still unconscious. He had a history of chronic obstructive pulmonary disease. Chest X-ray revealed opacities in the right lower lobe and mucoid tracheal secretion ensued following tracheal entubation performed after operation. Direct microscopic examination of bronchoalveolar lavage (BAL) fluid yielded abundant number of leukocytes and gram-negative bacilli. Bacteria isolated from BAL specimen were identified as C. lapagei by Phoenix 100 (Becton Dickinson, USA) automated system and also by API 20E kit (Biomerieux, France). Upon the initiation of intravenous amikacin (1 x 1 g) and meropenem (3 x 1 g), the signs of infection decreased in intensity, however, the patient was lost due to subarachnoid hemorrhage on the 12th day of hospitalization. In this case it was estimated that C. lapagei pneumonia originated from the aspiration of upper airway secretion owing to unconsciousness of the patient. Although there were reports of Cedecea infections in the literature, this was the first documented case of C. lapagei pneumonia when the accessible related literature was concerned.'}, {'qas': [{'id': '3818940:Pseudomonas stutzeri', 'question': 'Can Pseudomonas stutzeri cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Pseudomonas stutzeri pneumonia and septicemia in a patient with multiple myeloma.\\nABSTRACT:A case of septicemia caused by Pseudomonas stutzeri belonging to the unusual biotype Vb-3 in a patient with multiple myeloma is described. The origin of the septicemia was attributed to a community-acquired pneumonia. The bacteriology and pathogenicity of P. stutzeri are reviewed.'}, {'qas': [{'id': '32274497:Leptospira biflexa', 'question': 'Can Leptospira biflexa cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:M16-type metallopeptidases are involved in virulence for invasiveness and diffusion of Leptospira interrogans and transmission of leptospirosis.\\nABSTRACT:BACKGROUD: Leptospirosis is a global zoonotic infectious disease caused by Leptospira interrogans. The pathogen rapidly invades into hosts and diffuses from bloodstream into internal organs and excretes from urine to cause transmission of leptospirosis. However, the mechanism of leptospiral invasiveness remains poorly understood. \\n'}, {'qas': [{'id': '21920817:Staphylococcus capitis', 'question': 'Can Staphylococcus capitis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Acute endocarditis of the patch caused by Staphylococcus capitis in treated tetralogy of Fallot. An unusual location by an unusual bacterium.\\nABSTRACT:We present the case of a 46 year-old male, HCV infected, treated with corrective surgery for tetralogy of Fallot (TOF) immediately after percutaneous closure of the Blalock Taussig shunt. Four months later, the patient had infective endocarditis by Staphylococcus capitis localised on the right side of the patch, treated by oxacillin and gentamycin. The particularity of our report is the unusual location of the acute endocarditis and the bacterium involved: the pulmonary valve is much more likely to be involved in endocarditis in TOF patients and the patch endocarditis has rarely been reported. Moreover, Staphylococcus capitis has never been reported as a cause of acute endocarditis in corrected TOF patients. We believe that antibiotic therapy should be instituted as soon as possible even though an aggressive surgical treatment is mandatory to achieve complete recovery, mainly when clinical condition and inflammation markers do not improve.'}, {'qas': [{'id': '28945536:Bacteroides xylanisolvens', 'question': 'Can Bacteroides xylanisolvens cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Bacteroides koreensis sp. nov. and Bacteroides kribbi sp. nov., two new members of the genus Bacteroides.\\nABSTRACT:Three bacterial isolates from human faeces, YS-aM39T, R2F3-3-3T and R2F3-5-1, were characterized as Gram-negative, strictly anaerobic, non-spore-forming, non-motile, and rod-shaped. Isolate YS-aM39T formed a distinct line of descent, showing greatest 16S rRNA gene sequence relatedness with R2F3-3-3T (97.5 %), R2F3-5-1 (97.5 %), Bacteroides ovatus (98.8 %) and Bacteroides xylanisolvens (97.2 %). Isolates R2F3-3-3T and R2F3-5-1 also formed a distinct line of descent, sharing greatest 16S rRNA gene sequence relatedness with B. ovatus (98.2 %) and B. xylanisolvens (97.2 %). The DNA G+C content of YS-aM39T was 44.8 mol%, that of R2F3-3-3T was 42.4 mol% and that of R2F3-5-1 was 42.6 mol%. The respiratory quinone of all three isolates was menaquinone MK-10. Polar lipid analysis identified phosphatidylethanolamine as the major lipid. The predominant fatty acids in all three isolates were anteiso-C15 : 0, iso-C15 : 0, C16 : 0 3-OH and iso-C17 : 0 3-OH. The major end products of glucose fermentation were acetic acid, lactic acid and formic acid. DNA-DNA hybridization data indicated that two isolates, YS-aM39T and R2F3-3-3T, represent a species distinct from B. ovatus and B. xylanisolvens. Finally, in this study, the two isolates represented two new species in the genus Bacteroides, for which we propose the names Bacteroides koreensis sp. nov. (type strain, YS-aM39T=KCTC 15520T=JCM 31393T) and Bacteroides kribbi sp. nov. (type strain, R2F3-3-3T=KCTC 15460T=JCM 31391T).'}, {'qas': [{'id': '33036349:Enterococcus casseliflavus', 'question': 'Can Enterococcus casseliflavus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Culturomics Discloses Anti-Tubercular Enterococci Exclusive of Pulmonary Tuberculosis: A Preliminary Report.\\nABSTRACT:Mycobacterium tuberculosis causes pulmonary tuberculosis, a deadly infection of which the clinical expression and prognosis are not fully understood at the individual level, apart from genetic susceptibility traits. We investigated whether individual gut microbiota may correlate with pulmonary tuberculosis status. Culturomics investigations of gut microbiota in two pulmonary tuberculosis patients and two controls in Burkina Faso found 60 different bacterial species in patients and 97 in controls, including 45 in common. Further analysis of the results at the individual level indicated seven bacteria, including Enterococcus mundtii and Enterococcus casseliflavus, which were exclusively cultured in controls. Blind quantitative PCR-based exploration of faeces samples in two cohorts in Burkina Faso and in France confirmed a nonsignificant association of E. mundtii and E. casseliflavus with controls. Further in vitro explorations found four E. mundtii and E. casseliflavus strains inhibiting the growth of M. tuberculosis strains representative of four different lineages as well as Mycobacterium africanum, Mycobacterium canettii, and Mycobacterium bovis, in an inoculum-dependent manner. Heat-killed E. mundtii or E. casseliflavus were ineffective. These unprecedented observations of direct interactions between gut E. mundtii and E. casseliflavus with M. tuberculosis complex mycobacteria suggest that gut microbiota may modulate the expression of pulmonary tuberculosis.'}, {'qas': [{'id': '1997461:Vibrio vulnificus', 'question': 'Can Vibrio vulnificus cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Pulmonary infiltrates associated with Vibrio vulnificus septicemia.\\nABSTRACT:In the case described, fatal Vibrio vulnificus primary septicemia was complicated by development of diffuse pulmonary infiltrates. Clinical features of this recently discovered infectious disease are discussed. Various aspects of diagnoses of pulmonary infiltrates in patients with V vulnificus septicemia are also presented. Vibriosis should be suspected in patients with chronic, underlying, predisposing diseases, recent ingestion of raw seafood, and characteristic skin lesions.'}, {'qas': [{'id': '28768141:Campylobacter hyointestinalis', 'question': 'Can Campylobacter hyointestinalis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:A Randomized Trial to Assess the Effect of Fluoroquinolone Metaphylaxis on the Fecal Prevalence and Quinolone Susceptibilities of Salmonella and Campylobacter in Feedlot Cattle.\\nABSTRACT:The study objective was to determine effects of fluoroquinolone metaphylaxis on fecal prevalence of Salmonella and Campylobacter and fecal prevalence of quinolone-resistant Salmonella and Campylobacter in feedlot cattle. On Day 0, cattle (n = 288) at risk for bovine respiratory disease (BRD) were randomly assigned to either a nontreated control pen (12 pens) or a fluoroquinolone-treated (enrofloxacin; Baytril  100) pen (12 pens). Rectal fecal samples were collected from cattle on days 0, 7, 14, 21, and 28. Feces were cultured for Salmonella enterica and Campylobacter spp. using enrichment and selective isolation methods, and confirmed by serology and PCR. Susceptibilities to nalidixic acid and ciprofloxacin were determined using microbroth dilution methods. Data analyses were performed using linear mixed models. Overall, Salmonella sp. and Campylobacter spp. were recovered from 10.2% (139/1,364) and 12.4% (170/1,364) of the fecal samples, respectively. Campylobacter species included hyointestinalis, jejuni, and coli. Neither Salmonella sp. nor Campylobacter spp. prevalence was significantly impacted by fluoroquinolone treatment (p = 0.80, p = 0.61, respectively). However, Salmonella prevalence differed between study weeks (p &lt; 0.01) with prevalence decreasing over time. Before treatment, 98.9% (91/92) of Salmonella isolates were susceptible to nalidixic acid and ciprofloxacin. All Salmonella recovered posttreatment (n = 43) were susceptible to both antimicrobials. The majority of Campylobacter spp. recovered before treatment were resistant to nalidixic acid (23/35; 65.7%) and ciprofloxacin (21/35; 60.0%). There was no significant treatment by week interaction (p = 0.85) or treatment effects (p = 0.61) on the posttreatment prevalence of Campylobacter resistance. There was, however, a significant week effect (p = 0.05), with Campylobacter resistance prevalence decreasing over time. In this 28-day study, we found no evidence that a fluoroquinolone used for metaphylaxis significantly impacts fecal prevalence of Salmonella sp. or Campylobacter spp. or the fecal prevalence of nalidixic acid or ciprofloxacin resistance.'}, {'qas': [{'id': '19567344:Kluyvera ascorbata', 'question': 'Can Kluyvera ascorbata cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Antibiotic susceptibility profiles of uncommon bacterial species causing severe infections in Italy.\\nABSTRACT:This study presents the results of the italian \"Severe infections project\" involving bacteria that can be considered rare causes of disease. we isolated 30 uncommon human pathogens from a total of 60 strains (1.2% of all the isolates). The most frequent sources of uncommon human pathogens were primary bloodstream infections (48.3%) and pneumonia (20%). Species such as Comamonas testosteroni, Enterococcus hirae, Kluyvera ascorbata, Kluyvera cryocrescens, Leclercia adecarboxylata and Ochrobactrum anthropi were recovered from bacteremia patients. Clinically useful antimicrobial agents were tested against each isolate. Resistance to 4 or more antibiotics tested was found in Achromobacter xylosoxidans, O. anthropi, Pseudomonas stutzeri, Citrobacter braakii, Enterobacter sakazakii, K. ascorbata, Proteus penneri and Serratia plymuthica. About 16% of the Gram-negative species were resistant to third-generation cephalosporins and 28.6% of the staphylococci were oxacillin-resistant. the results from this study offer indications for empirical therapy for severe infections from uncommon human pathogens.'}, {'qas': [{'id': '11888910:Rhodococcus fascians', 'question': 'Can Rhodococcus fascians cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Peripheral benzodiazepine receptor ligands reverse apoptosis resistance of cancer cells in vitro and in vivo.\\nABSTRACT:The mitochondrial peripheral benzodiazepine receptor (mPBR) is involved in a functional structure designated as the permeability transition pore, which controls apoptosis. Binding of Fas/APO-1/CD95 triggers a prototypic apoptosis-inducing pathway. Using four different human tumor cell lines (T-cell Jurkat, neuroblastoma SHEP, osteosarcoma 143N2, and glioblastoma SNB79 cell lines), all of which express CD95 and mPBR, we investigated the potential role of mPBR ligands in CD95-induced apoptosis. We show that, in vitro, the three mPBR ligands tested (RO5-4864, PK11195, and diazepam) enhanced apoptosis induced by anti-CD95 antibody in Jurkat cells, as demonstrated by mitochondrial transmembrane potential drop and DNA fragmentation. In contrast, RO5-4864, but not PK11195 or diazepam, enhanced anti-CD95 apoptosis in all other cell lines. These effects were obtained in Bcl-2-overexpressing SHEP cell lines, but not in Bcl-X(L) SHEP cell lines. Enhancement of anti-CD95 antibody-induced apoptosis by RO5-4864 was characterized by an increased mitochondrial release of cytochrome c and Smac/DIABLO proteins and an enhanced activation of caspases 9 and 3, suggesting a mitochondrion-dependent mechanism. Preincubation of cells with the different mPBR ligands or anti-CD95 did not affect the levels of expression of either mPBR or CD95. In vivo, we found that the RO5-4864 mPBR ligand significantly increased the growth inhibition induced by two chemotherapeutic agents, etoposide and ifosfamide, using two human small cell lung cancers xenografted into nude mice. Peripheral benzodiazepine receptor ligands may therefore act as chemosensitizing agents for the treatment of human neoplasms.'}, {'qas': [{'id': '33036892:Tsukamurella paurometabola', 'question': 'Can Tsukamurella paurometabola cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Septic pulmonary emboli caused by Tsukamurella inchonensis: A case report.\\nABSTRACT:The genus Tsukamurella is a fastidious, environmental organism that potentially causes various infections in humans. Due to the morphological and biochemical similarities to others pathogens, such as Gordona, Rhodococcus, Corynebacterium, Nocardia, and Mycobacterium, a molecular-based approach is indispensable to correctly identify them. Herein, we describe a case of Tsukamurella inchonensis bacteremia complicated with septic pulmonary emboli (SPE), which is the first in the literature. A 44-year-old Japanese woman diagnosed with tongue cancer had undergone partial tongue resection. While receiving chemotherapy and radiotherapy, she developed high fever. Chest computed tomography suggested multiple emboli at the bilateral, peripheral lungs, indicating SPE. Blood culture detected Gram-positive rods, to which matrix-assisted laser desorption/ionization-time of flight mass spectrometry failed to identify. Then, we attempted to characterize it by 16S rRNA sequence, which suggested the organism to be Tsukamurella species but resulted in low resonance of the species-level identification. Additionally, we employed a confidence gene targeting groEL, leading to 100% matching (753/753 bps) with T. inchonensis NCTC 10741 (GenBank accession no. LR131273.1), which has been incorrectly registered as wrong species name Tsukamurella paurometabola in the database. Under the diagnosis of T. inchonensis-associated SPE, we successfully treated the patient with imipenem/cilastatin administration for 4 weeks. Sequencing analysis of groEL was of great use in identifying the organism in this case. More clinical cases based on molecular diagnosis of the fastidious pathogens need to be accumulated to further understand the characteristics and appropriate treatment regimen.'}, {'qas': [{'id': '28247494:Helicobacter cinaedi', 'question': 'Can Helicobacter cinaedi cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Extraintestinal infection of Helicobacter cinaedi induced by oral administration to Balb/c mice.\\nABSTRACT:Although Helicobacter cinaedi was initially considered an opportunistic pathogen in immunocompromised patients, it was later shown to also infect immunocompetent and healthy individuals. Sporadic bacteremia due to H. cinaedi has frequently been reported; however, whether the bacterium can be translocated after passage through the intestinal mucosa remains unclear. In the present study, a preclinical small animal model that faithfully reproduces H. cinaedi infection in humans was developed. Balb/c male mice were orally inoculated with a single dose of 6.8 x 107 CFU of a human clinical H. cinaedi strain. The organism persistently colonized the intestinal tract of the mice, particularly the cecum and colon, for at least 56 days, and the bacteria were excreted in the feces. Although inoculated bacteria were recovered from the spleen, liver, kidney, lung, bladder and mesenteric lymph nodes during the first 2 weeks of bacteremia, the organism was not isolated from these organs after 4 weeks, suggesting that complement- and antibody-mediated serum sensitivity account for the relatively low frequency of systemic infection. However, H. cinaedi was isolated from the biceps femoris, triceps branchii, latissimus dorsi, and trapezius muscles beyond 2 weeks after infection and after production of specific anti-H. cinaedi IgM and IgG antibodies. The present findings suggest that experimental infection of Balb/c mice with H. cinaedi may be a useful model for further studies of H. cinaedi pathogenesis, prophylaxis or therapeutic interventions in vivo.'}, {'qas': [{'id': '24908812:Mycobacterium kyorinense', 'question': 'Can Mycobacterium kyorinense cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:[Fatal nontuberculous mycobacterial lung disease caused by Mycobacterium kyorinense: a case report with five years of follow-up].\\nABSTRACT:An 85-year-old man with dementia first visited our hospital 5 years ago, complaining of hemoptysis. He was hospitalized 2 years later owing to fever, cough, and dyspnea. A chest computed tomography scan showed infiltration with a cavity in the left upper lobe. He was diagnosed with nontuberculous mycobacterial lung infection on the basis of the presence of acid-fast bacilli in the sputum and repeated bronchoalveolar lavage specimens; however, we were unable to identify the isolate by DNA-DNA hybridization. Although his general condition had slightly improved after treatment initiation, intermittent chemotherapy owing to the adverse effects of the drugs and dementia led to rapid disease progression and death. After his death, the isolated mycobacterium was identified as Mycobacterium kyorinense by sequence analysis of the hsp 65 and rpoB genes.'}, {'qas': [{'id': '18626541:Ewingella americana', 'question': 'Can Ewingella americana cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Multi-drug resistant Ewingella americana.\\nABSTRACT:We report a case of pneumonia due to multi-drug resistant Ewingella americana in a young patient admitted in the Intensive Care Unit of Hera General Hospital, Makkah, Saudi Arabia with severe head injury in a road traffic accident. He was an Indonesian pilgrim who had traveled to the Kingdom of Saudi Arabia to perform Hajj in December 2007. Ewingella americana was identified to be the pathogen of pneumonia with clinical signs and symptoms along with positive radiological findings.'}, {'qas': [{'id': '30510316:Streptococcus uberis', 'question': 'Can Streptococcus uberis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Antimicrobial resistance in mastitis, respiratory and enteric bacteria isolated from ruminant animals from the Atlantic Provinces of Canada from 1994-2013.\\nABSTRACT:Diagnostic laboratory antimicrobial susceptibility data for bacteria isolated from clinical samples of cattle, sheep, and goats from 1994 to 2013 were evaluated retrospectively. Among bacteria from bovine mastitis, Staphylococcus aureus and Streptococcus uberis were the most commonly isolated organisms. Pasteurella multocida, Mannheimia haemolytica, and Histophilus somni were commonly isolated from the respiratory tract, while Escherichia coli isolates were frequently recovered from the intestinal tract. Isolates from mastitis were generally highly susceptible to the antimicrobials tested, except neomycin and oxytetracycline. Respiratory tract isolates were highly susceptible to trimethoprim-sulfamethoxazole, penicillin, florfenicol, and ceftiofur, while enteric bacteria were frequently susceptible to ceftiofur. Antimicrobial resistance trends over the study period were generally stable for small ruminant and cattle isolates. Multi-drug resistance was more common among respiratory isolates from small ruminants compared to those from cattle but more common in enteric bacteria from cattle compared to those from small ruminants. This information may guide clinicians when they are choosing empirical therapies for the treatment of ruminant animals in Atlantic Canada.'}, {'qas': [{'id': '11526154:Mycobacterium bohemicum', 'question': 'Can Mycobacterium bohemicum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Direct identification of mycobacteria from MB/BacT alert 3D bottles: comparative evaluation of two commercial probe assays.\\nABSTRACT:The new INNO-LiPA Mycobacteria (Innogenetics, Ghent, Belgium), a reverse-hybridization-based line probe assay, and the AccuProbe assay (Gen-Probe Inc., San Diego, Calif.) were applied to MB/BacT Alert 3D (MB/BacT) system (Organon Teknika, Boxtel, The Netherlands) culture bottles and evaluated for mycobacterial identification. From 2,532 respiratory and extrapulmonary specimens submitted for culture, 168 were flagged positive by the MB/BacT system and promptly evaluated for identification (within 24 h). Each of 163 vials grew one mycobacterial isolate, including Mycobacterium tuberculosis complex (n = 73), M. avium complex (n = 3), M. avium (n = 8), M. intracellulare (n = 5), M. kansasii (n = 15), M. gordonae (n = 8), M. malmoense (n = 3), M. chelonae (n = 13), M. abscessus (n = 2), M. xenopi (n = 11), M. scrofulaceum (n = 2), M. fortuitum (n = 7), M. terrae (n = 3), M. simiae (n = 2), M. celatum (n = 3), M. flavescens (n = 1), M. interjectum (n = 1), M. bohemicum (n = 1), and M. pulveris (n = 2). Five cultures yielded mixed growth of two mycobacterial species: M. tuberculosis complex plus M. gordonae (n = 2), M. tuberculosis complex plus M. chelonae (n = 1), M. tuberculosis complex plus M. xenopi (n = 1), and M. avium plus M. chelonae (n = 1). In testing of one-isolate vials, both systems showed excellent sensitivity and specificity for all species and complexes for which they are licensed (nine for INNO-LiPA Mycobacteria versus six for AccuProbe). There were minor discrepancies in results for two isolates identified by INNO-LiPA Mycobacteria as M. avium - M. intracellulare - M. scrofulaceum (MAIS) complex and by AccuProbe as M. intracellulare. In testing of two-isolate vials, INNO-LiPA Mycobacteria correctly identified all isolates, while the AccuProbe assay failed to identify three M. tuberculosis complex isolates and one M. avium isolate. The AccuProbe assay was completed within 2 h, while INNO-LiPA Mycobacteria required a 6-h period. In our opinion, INNO-LiPA Mycobacteria offers the following advantages: (i) it contains a genus-specific probe that, in addition to being used in genus identification, may be used as an internal control for both the amplification and hybridization steps; (ii) it simultaneously identifies M. tuberculosis complex, MAIS complex, and seven other mycobacterial species, even from mixed cultures; (iii) its mycobacterial DNA amplification ensures reliable results independent from the concentration of viable microorganisms; and (iv) it genotypically identifies M. kansasii and M. chelonae. In conclusion, even though INNO-LiPA Mycobacteria is considerably less easy to use than AccuProbe, requiring personnel skilled in molecular biology techniques, it represents an excellent approach for routine identification of frequently encountered mycobacteria.'}, {'qas': [{'id': '28403835:Bergeyella zoohelcum', 'question': 'Can Bergeyella zoohelcum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Bacteremia caused by Bergeyella zoohelcum in an infective endocarditis patient: case report and review of literature.\\nABSTRACT:\\n'}, {'qas': [{'id': '25522092:Streptobacillus moniliformis', 'question': 'Can Streptobacillus moniliformis cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Notes from the field: fatal rat-bite fever in a child - San Diego County, California, 2013.\\nABSTRACT:In August 2013, the County of San Diego Health and Human Services Agency was notified of a fatal case of rat-bite fever (RBF) in a previously healthy male, aged 10 years, who owned pet rats. Two days before his death, the patient experienced rigors, fevers, vomiting, headaches, and leg pains. His physician noted a fever of 102.6 F (39.2oC), documented a normal examination, diagnosed viral gastroenteritis, and prescribed anti-nausea medication. During the next 24 hours, the patient experienced vomiting and persistent fever. He was confused and weak before collapsing at home. Paramedics reported the patient was unresponsive and had dilated pupils; resuscitation was initiated in the field and was continued for &gt;1 hour after arrival at the emergency department but was unsuccessful. A complete blood count performed during resuscitation revealed anemia (hemoglobin 10.0 g/dL [normal = 13.5-18.0 g/dL], thrombocytopenia (platelets 40,000/microL [normal = 140,000-440,000/microL]), leukocytosis (white blood cells 17,900 cells/microL [normal = 4,000-10,500/microL]) with 16% band neutrophils; the patient also had evidence of disseminated intravascular coagulation. No rash or skin breakdown was noted. Lung, liver, and epiglottis tissue collected postmortem was positive for Streptobacillus moniliformis DNA by polymerase chain reaction. '}, {'qas': [{'id': '29201017:Aeromonas media', 'question': 'Can Aeromonas media cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Characterization of Metagenomes in Urban Aquatic Compartments Reveals High Prevalence of Clinically Relevant Antibiotic Resistance Genes in Wastewaters.\\nABSTRACT:The dissemination of antimicrobial resistance (AMR) is an escalating problem and a threat to public health. Comparative metagenomics was used to investigate the occurrence of antibiotic resistant genes (ARGs) in wastewater and urban surface water environments in Singapore. Hospital and municipal wastewater (n = 6) were found to have higher diversity and average abundance of ARGs (303 ARG subtypes, 197,816 x/Gb) compared to treated wastewater effluent (n = 2, 58 ARG subtypes, 2,692 x/Gb) and surface water (n = 5, 35 subtypes, 7,985 x/Gb). A cluster analysis showed that the taxonomic composition of wastewaters was highly similar and had a bacterial community composition enriched in gut bacteria (Bacteroides, Faecalibacterium, Bifidobacterium, Blautia, Roseburia, Ruminococcus), the Enterobacteriaceae group (Klebsiella, Aeromonas, Enterobacter) and opportunistic pathogens (Prevotella, Comamonas, Neisseria). Wastewater, treated effluents and surface waters had a shared resistome of 21 ARGs encoding multidrug resistant efflux pumps or resistance to aminoglycoside, macrolide-lincosamide-streptogramins (MLS), quinolones, sulfonamide, and tetracycline resistance which suggests that these genes are wide spread across different environments. Wastewater had a distinctively higher average abundance of clinically relevant, class A beta-lactamase resistant genes (i.e., blaKPC, blaCTX-M, blaSHV, blaTEM). The wastewaters from clinical isolation wards, in particular, had a exceedingly high levels of blaKPC-2 genes (142,200 x/Gb), encoding for carbapenem resistance. Assembled scaffolds (16 and 30 kbp) from isolation ward wastewater samples indicated this gene was located on a Tn3-based transposon (Tn4401), a mobilization element found in Klebsiella pneumonia plasmids. In the longer scaffold, transposable elements were flanked by a toxin-antitoxin (TA) system and other metal resistant genes that likely increase the persistence, fitness and propagation of the plasmid in the bacterial host under conditions of stress. A few bacterial species (Enterobacter cloacae, Klebsiella pneumoniae, Citrobacter freundii, Pseudomonas aeruginosa) that were cultured from the isolation ward wastewaters on CHROMagar media harbored the blaKPC-2 gene. This suggests that hospital wastewaters derived from clinical specialty wards are hotspots for the spread of AMR. Assembled scaffolds of other mobile genetic elements such as IncQ and IncF plasmids bearing quinolone resistance genes (qnrS1, qnrS2) and the class A beta-lactamase gene (blaTEM-1) were recovered in wastewater samples which may aid the transfer of AMR.'}, {'qas': [{'id': '32212561:Mycobacterium pinnipedii', 'question': 'Can Mycobacterium pinnipedii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:MYCOBACTERIAL DISEASE AND SUBSEQUENT DIAGNOSTIC INVESTIGATIONS IN A GROUP OF CAPTIVE PINNIPEDS IN NEW ZEALAND.\\nABSTRACT:This case series includes a single case of disseminated tuberculous disease due to Mycobacterium pinnipedii in a New Zealand fur seal (Arctocephalus forsteri), which was being cared for by a zoo in New Zealand. The remaining five pinnipeds in the colony underwent extensive mycobacterial disease surveillance over the following 4 yr, involving a total of 26 anesthetic procedures and numerous diagnostic tests that included comparative intradermal tuberculin skin tests, mycobacterial antibody serology, respiratory and gastric lavages, and computed tomography (CT) scans. An additional case of chronic sinusitis due to Mycobacterium marinum and Pseudomonas aeruginosa was identified in a California sea lion (Zalophus californianus). Results from CT and the respiratory lavages were the most helpful antemortem diagnostic tests for active mycobacterial disease in this case series. Of the remaining four animals, two were euthanatized and two remain alive, and none of them had evidence of active mycobacterial disease. Further mycobacterial disease surveillance in staff and animals was performed, and no other case was identified. There are no validated mycobacterial surveillance tests available for pinnipeds and so it remains unknown whether the two surviving pinnipeds are truly negative or whether they have latent mycobacterial infection that could develop into active mycobacterial disease in the future. For this reason, increased levels of biosecurity and quarantine remain permanently in place for the pinniped colony.'}, {'qas': [{'id': '25406238:Actinobaculum schaalii', 'question': 'Can Actinobaculum schaalii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Dissection of the genus Actinobaculum: Reclassification of Actinobaculum schaalii Lawson et al. 1997 and Actinobaculum urinale Hall et al. 2003 as Actinotignum schaalii gen. nov., comb. nov. and Actinotignum urinale comb. nov., description of Actinotignum sanguinis sp. nov. and emended descriptions of the genus Actinobaculum and Actinobaculum suis; and re-examination of the culture deposited as Actinobaculum massiliense CCUG 47753T ( = DSM 19118T), revealing that it does not represent a strain of this species.\\nABSTRACT:The remarkable host specificity of the species of the genus Actinobaculum led us to recharacterize these species by a polyphasic approach. A comparative chemotaxonomic study including analysis of whole-cell sugars, amino acid composition of the peptidoglycan, fatty acid methyl esters, respiratory quinones and polar lipids revealed significant differences that, in combination with molecular data, support a dissection of the genus Actinobaculum. The proposals of this study include the reclassification of Actinobaculum schaalii and Actinobaculum urinale as Actinotignum schaalii gen. nov., comb. nov. (type strain DSM 15541(T) = CCUG 27420(T)) and Actinotignum urinale comb. nov. (type strain DSM 15805(T) = CCUG 46093(T)), respectively. Emended descriptions of the genus Actinobaculum and Actinomyces suis are also provided. The results of 16S rRNA gene sequence analysis and DNA-DNA hybridization also indicated that the type strain of Actinobaculum massiliense deposited as CCUG 47753(T) ( = DSM 19118(T)) should in fact be considered a member of the species Actinobaculum schaalii. In addition, comparative 16S rRNA gene sequencing and DNA-DNA relatedness studies of four strains recovered from clinical materials demonstrated that three of the isolates belonged to Actinotignum schaalii; the remaining strain represents a novel species, for which the name Actinotignum sanguinis sp. nov. is proposed. The type strain is IMMIB L-2199(T) ( = DSM 26039(T) = CCUG 64068(T)).'}, {'qas': [{'id': '25406238:Actinobaculum urinale', 'question': 'Can Actinobaculum urinale cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Dissection of the genus Actinobaculum: Reclassification of Actinobaculum schaalii Lawson et al. 1997 and Actinobaculum urinale Hall et al. 2003 as Actinotignum schaalii gen. nov., comb. nov. and Actinotignum urinale comb. nov., description of Actinotignum sanguinis sp. nov. and emended descriptions of the genus Actinobaculum and Actinobaculum suis; and re-examination of the culture deposited as Actinobaculum massiliense CCUG 47753T ( = DSM 19118T), revealing that it does not represent a strain of this species.\\nABSTRACT:The remarkable host specificity of the species of the genus Actinobaculum led us to recharacterize these species by a polyphasic approach. A comparative chemotaxonomic study including analysis of whole-cell sugars, amino acid composition of the peptidoglycan, fatty acid methyl esters, respiratory quinones and polar lipids revealed significant differences that, in combination with molecular data, support a dissection of the genus Actinobaculum. The proposals of this study include the reclassification of Actinobaculum schaalii and Actinobaculum urinale as Actinotignum schaalii gen. nov., comb. nov. (type strain DSM 15541(T) = CCUG 27420(T)) and Actinotignum urinale comb. nov. (type strain DSM 15805(T) = CCUG 46093(T)), respectively. Emended descriptions of the genus Actinobaculum and Actinomyces suis are also provided. The results of 16S rRNA gene sequence analysis and DNA-DNA hybridization also indicated that the type strain of Actinobaculum massiliense deposited as CCUG 47753(T) ( = DSM 19118(T)) should in fact be considered a member of the species Actinobaculum schaalii. In addition, comparative 16S rRNA gene sequencing and DNA-DNA relatedness studies of four strains recovered from clinical materials demonstrated that three of the isolates belonged to Actinotignum schaalii; the remaining strain represents a novel species, for which the name Actinotignum sanguinis sp. nov. is proposed. The type strain is IMMIB L-2199(T) ( = DSM 26039(T) = CCUG 64068(T)).'}, {'qas': [{'id': '30533234:Porphyromonas asaccharolytica', 'question': 'Can Porphyromonas asaccharolytica cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:Porphyromonas asaccharolytica as a Rare Causative Agent for Lemierre's Syndrome.\\nABSTRACT:Lemierre's syndrome is a rare disease associated with significant morbidity and mortality. It begins with an oropharyngeal infection, which spreads locally to involve the internal jugular vein causing thrombophlebitis, followed by distant spread and metastatic infections. Affected individuals are commonly young adults. Causative organisms are usually oropharyngeal flora, most commonly being the anaerobe Fusobacterium necrophorum. Porphyromonas asaccharolytica is a rare etiological agent with only three cases being reported in the literature. This case report describes a previously healthy 22-year-old man who initially presented with acute tonsillitis and was later found to have left internal jugular vein thrombophlebitis along with bilateral septic emboli to the lungs. The patient was treated with a five-week course of ampicillin-sulbactam and metronidazole. Subsequent imaging also showed progression of internal jugular vein thrombus, for which warfarin was given for three months for anticoagulation.\"}, {'qas': [{'id': '9226890:Tsukamurella tyrosinosolvens', 'question': 'Can Tsukamurella tyrosinosolvens cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Tsukamurella tyrosinosolvens sp. nov.\\nABSTRACT:Chemotaxonomic and 16S ribosomal DNA sequence analyses of four bacterial isolates from blood cultures from patients with cardiac pacemaker implants and sputa of patients with chronic lung infections clearly demonstrated that these bacteria belong to the genus Tsukamurella. DNA-DNA hybridization data, as well as the physiological characteristics of the isolates, indicate that they are closely related and belong to a single species that differs from previously described members of the genus Tsukamurella. The name Tsukamurella tyrosinosolvens sp. nov. is proposed for these isolates, and the new species is represented by strain IMMIB D-1397T (= DSM 44234T). Strain IMMIB D-1397T exhibits 53.4, 53.5, and 54.7% DNA-DNA relatedness to Tsukamurella paurometabola DSM 20162T, Tsukamurella inchonensis DSM 44067T, and Tsukamurella pulmonis DSM 44142T, respectively.'}, {'qas': [{'id': '20833885:Mycobacterium senuense', 'question': 'Can Mycobacterium senuense cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Mycobacterium algericum sp. nov., a novel rapidly growing species related to the Mycobacterium terrae complex and associated with goat lung lesions.\\nABSTRACT:A previously undescribed, rapid-growing, non-chromogenic Mycobacterium isolate from a goat lung lesion in Algeria is reported. Biochemical and molecular tools were used for its complete description and showed its affiliation to the Mycobacterium terrae complex. 16S rRNA, rpoB and hsp65 gene sequences were unique. Phylogenetic analyses showed a close relationship with M. terrae sensu stricto and Mycobacterium senuense. Culture and biochemical characteristics were generally similar to those of M. terrae and M. senuense. However, in contrast to M. terrae and M. senuense, the isolate was positive for urease production and had faster growth. The mycolic acid profile was distinct from those of M. terrae and M. senuense, thus further supporting the new taxonomic position of the isolate. We propose the name Mycobacterium algericum sp. nov. for this novel species. The type strain is TBE 500028/10(T) ( = Bejaia(T) = CIP 110121(T) = DSM 45454(T)).'}, {'qas': [{'id': '17605768:Bacteroides ureolyticus', 'question': 'Can Bacteroides ureolyticus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Femoral vein thrombophlebitis and septic pulmonary embolism due to a mixed anaerobic infection including Solobacterium moorei: a case report.\\nABSTRACT:\\n'}, {'qas': [{'id': '21886861:Tsukamurella paurometabola', 'question': 'Can Tsukamurella paurometabola cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Complete genome sequence of Tsukamurella paurometabola type strain (no. 33).\\nABSTRACT:Tsukamurella paurometabola corrig. (Steinhaus 1941) Collins et al. 1988 is the type species of the genus Tsukamurella, which is the type genus to the family Tsukamurellaceae. The species is not only of interest because of its isolated phylogenetic location, but also because it is a human opportunistic pathogen with some strains of the species reported to cause lung infection, lethal meningitis, and necrotizing tenosynovitis. This is the first completed genome sequence of a member of the genus Tsukamurella and the first genome sequence of a member of the family Tsukamurellaceae. The 4,479,724 bp long genome contains a 99,806 bp long plasmid and a total of 4,335 protein-coding and 56 RNA genes, and is a part of the Genomic Encyclopedia of Bacteria and Archaea project.'}, {'qas': [{'id': '24790091:Pandoraea pulmonicola', 'question': 'Can Pandoraea pulmonicola cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Inhibition of co-colonizing cystic fibrosis-associated pathogens by Pseudomonas aeruginosa and Burkholderia multivorans.\\nABSTRACT:Cystic fibrosis (CF) is a recessive genetic disease characterized by chronic respiratory infections and inflammation causing permanent lung damage. Recurrent infections are caused by Gram-negative antibiotic-resistant bacterial pathogens such as Pseudomonas aeruginosa, Burkholderia cepacia complex (Bcc) and the emerging pathogen genus Pandoraea. In this study, the interactions between co-colonizing CF pathogens were investigated. Both Pandoraea and Bcc elicited potent pro-inflammatory responses that were significantly greater than Ps. aeruginosa. The original aim was to examine whether combinations of pro-inflammatory pathogens would further exacerbate inflammation. In contrast, when these pathogens were colonized in the presence of Ps. aeruginosa the pro-inflammatory response was significantly decreased. Real-time PCR quantification of bacterial DNA from mixed cultures indicated that Ps. aeruginosa significantly inhibited the growth of Burkholderia multivorans, Burkholderia cenocepacia, Pandoraea pulmonicola and Pandoraea apista, which may be a factor in its dominance as a colonizer of CF patients. Ps. aeruginosa cell-free supernatant also suppressed growth of these pathogens, indicating that inhibition was innate rather than a response to the presence of a competitor. Screening of a Ps. aeruginosa mutant library highlighted a role for quorum sensing and pyoverdine biosynthesis genes in the inhibition of B. cenocepacia. Pyoverdine was confirmed to contribute to the inhibition of B. cenocepacia strain J2315. B. multivorans was the only species that could significantly inhibit Ps. aeruginosa growth. B. multivorans also inhibited B. cenocepacia and Pa. apista. In conclusion, both Ps. aeruginosa and B. multivorans are capable of suppressing growth and virulence of co-colonizing CF pathogens. '}, {'qas': [{'id': '8369583:Pasteurella stomatis', 'question': 'Can Pasteurella stomatis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:Epidemiology of human infections by Pasteurella and related groups in France.\\nABSTRACT:A retrospective study of infections due to Pasteurella (P.) and related groups was performed between the Pasteurella National Center and Nancy's hospital from 1985 to 1991. Among the 958 cases recorded, wound infections (bites, scratches and punctures) were the common forms of pasteurellosis (66%) caused by P. multocida (48%), P. canis (11%), P. dagmatis (5%), P. stomatis (4%), and in few cases by groups EF-4 and M-5 (14 and 13%, respectively). In human infections unrelated to animal wounds, respiratory tract diseases and bacteremia-septicemia were the predominant infections with respectively 19 and 11%, and caused by P. multocida. Next in importance were urogenital (2.5%), abdominal (1%) and central nervous system (&lt; 1%) infections. The majority of animal bite wound infections was treated with penicillins or tetracyclines; with other forms, penicillins and cephalosporins were more likely.\"}, {'qas': [{'id': '32673645:Lactococcus garvieae', 'question': 'Can Lactococcus garvieae cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Resistant and susceptible rainbow trout (Oncorhynchus mykiss) lines show distinctive immune response to Lactococcus garvieae.\\nABSTRACT:Lactococcosis is one of the main bacterial diseases affecting rainbow trout (Oncorhynchus mykiss), with significant economic and sanitary repercussion. Vaccination and antibiotic treatments are commonly used to prevent and control the infection outbreaks; however, these strategies have some drawbacks including limited coverage, handling costs, induction of antibiotic resistance and chemical residues in the environment. Selective breeding programs represent a promising complementary approach for increasing fish disease resistance in commercial farms and some immunological parameters may be tentatively used as indirect indicators for this purpose. The present study investigated for the first time some innate and adaptive immune responses in two groups of rainbow trout derived from selected lines (susceptible and resistant) showing a different \"in field\" phenotypical resistance to Yersinia ruckeri, Flavobacterium branchiophilum, F. psychrophilum, and Ichthyophthirius multifiliis, after an immersion-dilution based exposure to Lactococcus garvieae carried out in controlled experimental conditions. Twenty-six resistant and twenty-six susceptible female rainbow trout (mean body weight 80 g, 9 months aged, F5 generation) were obtained from an intensive farm considered L. garvieae free and were exposed to the pathogen. Moreover, 10 resistant and 10 susceptible fish were used as uninfected controls. After 5 days, blood and tissue samples were collected for immunological analyses. A significantly higher serum and mucus lysozyme activity was recorded in resistant rainbow trout compared to susceptible fish (P <= 0.05), both before and after exposure to L. garvieae. Similarly, respiratory burst activity of head kidney leukocytes resulted more intense in resistant fish (P <= 0.05), suggesting that phagocytes could more quickly activate their microbicidal mechanisms to counteract the bacterial spread. Resistant group displayed also an up-regulation of immunoglobulins M (IgM), major histocompatibility complex II (MHC-II) and interleukin 8 (IL-8) gene expression (P <= 0.05) and a significantly higher blood lymphocytes count (P <= 0.05), highlighting their potential better ability to trigger the recruitment of defensive cells and the initiation of specific immune processes such as antigen presentation to CD4+ T lymphocytes and IgM synthesis. The results herein presented might be useful for the identification of immunological markers to be used as indirect indicators in rainbow trout selective breeding programs.'}, {'qas': [{'id': '19648340:Actinomadura sputi', 'question': 'Can Actinomadura sputi cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Actinomadura sputi sp. nov., isolated from the sputum of a patient with pulmonary infection.\\nABSTRACT:The taxonomic position of an actinomycete, strain IMMIB L-889(T), isolated from the sputum of a 64-year-old man, was determined using a polyphasic taxonomic approach. The strain had chemical and morphological properties that were consistent with its classification in the genus Actinomadura. It formed a distinct phyletic line in the 16S rRNA gene tree of Actinomadura and was most closely related to the type strain of Actinomadura hallensis (98.4 % sequence similarity), but could be readily distinguished from the latter species using DNA-DNA relatedness and phenotypic data. The combined genotypic and phenotypic data indicate that strain IMMIB L-889(T) represents a novel species of the genus Actinomadura, for which the name Actinomadura sputi sp. nov. is proposed. The type strain is IMMIB L-889(T) =DSM 45233(T)=CCUG 56589(T). [corrected]'}, {'qas': [{'id': '11417826:Staphylococcus capitis', 'question': 'Can Staphylococcus capitis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Bacterial flora of the conjunctiva and nasal cavity in normal and diseased captive bustards.\\nABSTRACT:A survey was carried out to describe the normal aerobic bacterial flora of the conjunctiva and nasal cavity of captive houbara bustards (Chlamydotis undulata), kori bustards (Ardeotis kori), and white-bellied bustards (Eupodotis senegalensis) maintained at the National Avian Research Center, Abu Dhabi, United Arab Emirates. A total of 58 samples were examined from the nasal cavity and 55 samples from the conjunctiva of healthy bustards. There was no bacterial growth in 45% of conjunctival samples. Bacteria isolated from the conjunctiva of healthy birds included Micrococcus spp., Staphylococcus auricularis, Staphylococcus xylosus, Staphylococcus capitis, Staphylococcus warneri, Bacillus spp., and Enterobacter amigenus. Bacteria isolated from the nasal cavity of healthy birds included Bacillus spp., Micrococcus spp., S. auricularis, S. xylosus, Staphylococcus simulans, Staphylococcus saprophyticus, Staphylococcus hyicus, Staphylococcus cohnii, Staphylococcus sciuri, Aerococcus spp., and Providencia rettgeri. These findings were compared with bacterial isolates from bustards with clinical signs of ocular or upper respiratory tract diseases. Mycoplasma spp., Pseudomonas aeruginosa, Pseudomonas stutzeri, Proteus mirabilis, Escherichia coli, Klebsiella spp., Aeromonas hydrophila, and Staphylococcus aureus were the pathogenic bacteria isolated from the conjunctiva of 34.3% bustards with ocular discharges. Mycoplasma spp., P. aeruginosa, Pseudomonas spp., P. mirabilis, E. coli, Klebsiella pneumoniae, and S. aureus were the pathogenic bacteria isolated from the nasal cavity of 74% bustards with upper respiratory tract diseases.'}, {'qas': [{'id': '22760049:Actinomyces graevenitzii', 'question': 'Can Actinomyces graevenitzii cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Multiple lung abscesses caused by Actinomyces graevenitzii mimicking acute pulmonary coccidioidomycosis.\\nABSTRACT:Actinomyces graevenitzii is a newly recognized Actinomyces species that is seldom isolated from clinical specimens. A case of multiple pulmonary abscesses mimicking acute pulmonary coccidioidomycosis is described in this study, and the findings indicate that this organism is an opportunistic human pathogen.'}, {'qas': [{'id': '23132688:Bartonella clarridgeiae', 'question': 'Can Bartonella clarridgeiae cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Cat-scratch disease in Northern Italy: atypical clinical manifestations in humans and prevalence of Bartonella infection in cats.\\nABSTRACT:In this paper, we report an investigation on cat-scratch disease (CSD) in Northern Italy. Seventy-four cases of CSD were diagnosed at the San Matteo hospital, Pavia, during the period 2005-2010. Of these 74 patients, 18 (24.3 %) reported atypical clinical manifestations such as ocular papillitis, maculopapular eruptions, vertebral infection, pulmonary infiltrates, and granulomatous hepatitis. Contact with cats was documented for 61 patients (82.4 %), while cat-related trauma was reported for 49 patients (66.2 %). We subsequently investigated the presence of Bartonella infection in cats belonging to the above patients and in other domestic and stray cats from three provinces of Northern Italy. Among the 27 domestic cats tested, nine of the 11 belonging to the CSD patients and two of the remaining 16 were infected by B. henselae (81.8 % vs. 12.5 %). Out of over 1,300 stray cats examined, 23.1 % were seropositive for B. henselae; after culturing and genotyping, 17 % were found to be infected by B. henselae (15.5 %) or B. clarridgeiae (1.5 %).'}, {'qas': [{'id': '12784178:Actinomadura madurae', 'question': 'Can Actinomadura madurae cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:[Unusual gram positive rods, causing pneumonia].\\nABSTRACT:In the last decade, a growing number of patients with pneumonia, caused by unusual gram positive rods have been observed. Mostly, the patients had been infected as a consequence of impaired immunity. In some cases, bioterrorist activities may also induce pneumonia by gram positive rods (B. anthracis). In order to bring these organisms to the attention of the medical community, we present three clinical cases and describe six species of gram positive rods, known to provoke this kind of pneumonias. Case 1 was a 84 years old patient with impaired lung function. He was suspicious of tuberculosis (Tbc). Nocardia spec. was isolated. Case 2 was an alcoholic of 46 years with pneumonia. Reactivation of Tbc was suspected. Actinomadura madurae has been isolated. Case 3 was a patient of 58 years with myelodysplastic syndrome (MDS) and pneumonia. N. asteroides was isolated. All patients shared impaired immunity (age, alcoholism, MDS) with impaired lung functions; Tbc had been suspected (Case 1 + 2). Infection by A. madurae was contained by Clindamycin. Therapy of Nocardia with Moxifloxacin (Case 1) or Bactrim (Case 3) was only partly effective. In the appendix, six species of gram positive rods which are known to cause pneumonia, are summarized (Nocardia, Actinomyceta, Actinomadura, Rhodococcus, Corynebacterium and Bacillus).'}, {'qas': [{'id': '29740101:Brucella canis', 'question': 'Can Brucella canis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Tissue distribution and cell tropism of Brucella canis in naturally infected canine foetuses and neonates.\\nABSTRACT:Brucella canis infection is an underdiagnosed zoonotic disease. Knowledge about perinatal brucellosis in dogs is extremely limited, although foetuses and neonates are under risk of infection due to vertical transmission. In this study, immunohistochemistry was used to determine tissue distribution and cell tropism of B. canis in canine foetuses and neonates. Diagnosis of B. canis in tissues of naturally infected pups was based on PCR and sequencing of amplicons, bacterial isolation, and immunohistochemistry, whose specificity was confirmed by laser capture microdissection. PCR positivity among 200 puppies was 21%, and nine isolates of B. canis were obtained. Tissues from 13 PCR-positive puppies (4 stillborn and 9 neonates) presented widespread immunolabeling. Stomach, intestines, kidney, nervous system, and umbilicus were positive in all animals tested. Other frequently infected organs included the liver (92%), lungs (85%), lymph nodes (69%), and spleen (62%). Immunolabeled coccobacilli occurred mostly in macrophages, but they were also observed in erythrocytes, epithelial cells of gastrointestinal mucosa, renal tubules, epidermis, adipocytes, choroid plexus, ependyma, neuroblasts, blood vessels endothelium, muscle cells, and in the intestinal lumen. These results largely expand our knowledge about perinatal brucellosis in the dog, clearly demonstrating a pantropic distribution of B. canis in naturally infected foetuses and neonates.'}, {'qas': [{'id': '31832844:Paenibacillus macerans', 'question': 'Can Paenibacillus macerans cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Paenibacillus oralis sp. nov., Isolated from Human Subgingival Dental Plaque of Gingivitis Lesion.\\nABSTRACT:A Gram-stain-negative, facultative anaerobic, spore-forming, motile, and rod-shaped bacterium, strain ChDC PVNT-B20T, was isolated from the human subgingival dental plaque of a gingivitis lesion. Phylogenetic analysis based on the 16S ribosomal RNA gene (16S rDNA) showed that the strain belonged to the genus Paenibacillus. BLAST analysis of 16S rDNA sequence of the strain displayed high identity to those of Paenibacillus faecis DSM 23593T (97.7% similarity) and Paenibacillus macerans ATCC 8244T (97.6% similarity). Draft genome of strain ChDC PVNT-B20T was composed of 8,112,407 bp. The DNA G+C content of the strain was 51.3 mol%. Average nucleotide identity values between strain ChDC PVNT-B20T and P. faecis DSM 23593T or P. macerans ATCC 8244T were 75.71% and 91.5%, respectively. Genome-to-genome distance values between strain ChDC PVNT-B20T and P. faecis DSM 23593T or P. macerans ATCC 8244T were 21.6% (19.3-24.0%) and 44.9% (42.3-47.4%), respectively. Major cellular fatty acids of strain ChDC PVNT-B20T were anteiso-C15:0 (43.4%), C16:0 (16.6%), iso-C16:0 (14.4%), and anteiso-C17:0 (12.4%). The sole respiratory quinone of the strain was menaqinone-7. Major polar lipids of the strain were phosphatidylglycerol (PG), diphosphatidylglycerol (DPG), and one unidentified glycolipid (GL). Minor polar lipids were one unidentified aminolipid (AL), one unidentified phospholipid (PL), and three unidentified lipids (L1-L3). Based on these results, strain ChDC PVNT-B20T is considered to represent a novel species of the genus Paenibacillus, for which the name Paenibacillus oralis sp. nov. is proposed. Type strain is ChDC PVNT-B20T (= KCOM 3021T = JCM 33462 T).'}, {'qas': [{'id': '12624081:Acinetobacter ursingii', 'question': 'Can Acinetobacter ursingii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Bacteremia caused by Acinetobacter ursingii.\\nABSTRACT:Acinetobacter ursingii has not been reported in infectious processes apart from its recent description as a new species. A bacteremia caused by A. ursingii in a patient with a pulmonary adenocarcinoma confirms that this microorganism is an opportunistic human pathogen. The isolate was susceptible to imipenem, aminoglycosides, rifampin, and fluoroquinolones.'}, {'qas': [{'id': '18251785:Vibrio vulnificus', 'question': 'Can Vibrio vulnificus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Autopsy cases of fulminant-type bacterial infection with necrotizing fasciitis: group A (beta) hemolytic Streptococcus pyogenes versus Vibrio vulnificus infection.\\nABSTRACT:Two autopsy cases of fulminant-type infection associated with necrotizing fasciitis were analyzed clinicopathologically. Both cases involved 57-year-old alcohol abusers. The former was a woman with group A (beta) hemolytic Streptococcus pyogenes infection, and the latter was a man with Vibrio vulnificus infection. The sudden onset of shock with high fever resulted in sepsis, decreased clotting, and hepatorenal symptoms, followed by death within a few days. Post-mortem examination showed widespread congestion and bleeding, and alcoholic liver cirrhosis was observed. Necrotizing fasciitis was identified in both cases. Bacteria from the pharynx or intestinal tract invaded the blood, and marked bacterial proliferation produced sepsis, resulting in necrotizing fasciitis. Despite the presence of sepsis, bilateral pulmonary congestion and bleeding were observed without pneumonia. Due to the rapid progression of sepsis, there was no time for granulocyte migration from the bone marrow. It seems that almost all mature granulocytes which had already existed in the bone marrow accumulated at the focus of necrotizing fasciitis because the bone marrow had few mature granulocytes and lacked hypercellularity. The cause of death in each case was circulatory collapse due to septic shock. It was difficult to distinguish the type of infection on histopathology. Cultures were necessary to determine the bacterial agents involved.'}, {'qas': [{'id': '25135044:Aerococcus urinae', 'question': 'Can Aerococcus urinae cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Infectious Endocarditis in 49-Year-Old Man and Discussion of Phenotypic Characteristics of Aerococcus urinae and Viridans Streptococci.\\nABSTRACT:PATIENT: 49 year-old man. CHIEF COMPLAINT: Dyspnea at rest and dyspnea on exertion. HISTORY OF PRESENT ILLNESS: Diagnosed with upper respiratory tract infection 10 days previously. PAST MEDICAL HISTORY: Obese (BMI not available), but no significant past medical history. SOCIAL HISTORY: Noncontributory. FAMILY HISTORY: Noncontributory.'}, {'qas': [{'id': '11417826:Staphylococcus xylosus', 'question': 'Can Staphylococcus xylosus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Bacterial flora of the conjunctiva and nasal cavity in normal and diseased captive bustards.\\nABSTRACT:A survey was carried out to describe the normal aerobic bacterial flora of the conjunctiva and nasal cavity of captive houbara bustards (Chlamydotis undulata), kori bustards (Ardeotis kori), and white-bellied bustards (Eupodotis senegalensis) maintained at the National Avian Research Center, Abu Dhabi, United Arab Emirates. A total of 58 samples were examined from the nasal cavity and 55 samples from the conjunctiva of healthy bustards. There was no bacterial growth in 45% of conjunctival samples. Bacteria isolated from the conjunctiva of healthy birds included Micrococcus spp., Staphylococcus auricularis, Staphylococcus xylosus, Staphylococcus capitis, Staphylococcus warneri, Bacillus spp., and Enterobacter amigenus. Bacteria isolated from the nasal cavity of healthy birds included Bacillus spp., Micrococcus spp., S. auricularis, S. xylosus, Staphylococcus simulans, Staphylococcus saprophyticus, Staphylococcus hyicus, Staphylococcus cohnii, Staphylococcus sciuri, Aerococcus spp., and Providencia rettgeri. These findings were compared with bacterial isolates from bustards with clinical signs of ocular or upper respiratory tract diseases. Mycoplasma spp., Pseudomonas aeruginosa, Pseudomonas stutzeri, Proteus mirabilis, Escherichia coli, Klebsiella spp., Aeromonas hydrophila, and Staphylococcus aureus were the pathogenic bacteria isolated from the conjunctiva of 34.3% bustards with ocular discharges. Mycoplasma spp., P. aeruginosa, Pseudomonas spp., P. mirabilis, E. coli, Klebsiella pneumoniae, and S. aureus were the pathogenic bacteria isolated from the nasal cavity of 74% bustards with upper respiratory tract diseases.'}, {'qas': [{'id': '23555735:Segniliparus rugosus', 'question': 'Can Segniliparus rugosus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Differential immune responses to Segniliparus rotundus and Segniliparus rugosus infection and analysis of their comparative virulence profiles.\\nABSTRACT:Two closely related bacterial species, Segniliparus rotundus and Segniliparus rugosus, have emerged as important human pathogens, but little is known about the immune responses they elicit or their comparative pathophysiologies. To determine the virulence and immune responses of the two species, we compared their abilities to grow in phagocytic and non-phagocytic cells. Both species maintained non-replicating states within A549 epithelial cells. S. rugosus persisted longer and multiplied more rapidly inside murine bone marrow-derived macrophages (BMDMs), induced more pro-inflammatory cytokines, and induced higher levels of macrophage necrosis. Activation of BMDMs by both species was mediated by toll-like receptor 2 (TLR2), followed by mitogen-activated protein kinases (MAPK) and nuclear factor kappaB (NF-kappaB) signaling pathways, indicating a critical role for TLR2 in Segniliparus-induced macrophage activation. S. rugosus triggered faster and stronger activation of MAPK signaling and IkappaB degradation, indicating that S. rugosus induces more pro-inflammatory cytokines than S. rotundus. Multifocal granulomatous inflammations in the liver and lung were observed in mice infected with S. rugosus, but S. rotundus was rapidly cleared from all organs tested within 15 days post-infection. Furthermore, S. rugosus induced faster infiltration of innate immune cells such as neutrophils and macrophages to the lung than S. rotundus. Our results suggest that S. rugosus is more virulent and induces a stronger immune response than S. rotundus.'}, {'qas': [{'id': '18447205:Micromonas micros', 'question': 'Can Micromonas micros cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:[Prediction on the efficacy of fluoroquinolones for anaerobic infections using Monte Carlo simulation].\\nABSTRACT:Various analysis based on PK-PD (pharmacokinetics-pharmacodynamics) theory were performed for prediction on the efficacy of antimicrobial agents for infectious diseases. However, there have been few reports when causative organisms would be anaerobic bacteria. Although the antimicrobial activity of fluoroquinolones against anaerobes has been weak, gatifloxacin (GFLX), moxifloxacin (MFLX), and garenoxacin (GRNX) are considered as anti-anaerobic quinolones in recent days. Therefore, we investigated prediction on the efficacy of fluoroquinolones for anaerobic infections using Monte Carlo Simulation. Although the target attainment of fluoroquinolones (levofloxacin, GFLX, and MFLX) on AUC/MIC value has not established yet, we have assumed 40 and 125 as the target attainment of AUC/MIC value. GFLX and MFLX would be effective against anaerobic infections caused by Prevotella intermedia, Fusobacterium nucleatum, Micromonas micros, which are the causative organisms for respiratory tract infections. From these results, GFLX and MFLX would be effective against anaerobic infections in respiratory tract as anti-anaerobic fluoroquinolones.'}, {'qas': [{'id': '24581117:Taylorella', 'question': 'Can Taylorella cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:A genomic perspective on a new bacterial genus and species from the Alcaligenaceae family, Basilea psittacipulmonis.\\nABSTRACT:\\n'}, {'qas': [{'id': '22798514:Fusobacterium varium', 'question': 'Can Fusobacterium varium cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': \"TITLE:Lemierre syndrome: from pharyngitis to fulminant sepsis.\\nABSTRACT:We report a case of a previously healthy 33-year-old male who presented to his primary care physician with nausea, vomiting, diarrhoea and fever. One week prior to presentation the patient reported a history of sore throat which he presumed to be a viral infection and sought no medical attention. Upon hospital presentation, the patient was admitted and rapidly progressed to sepsis and respiratory failure. Goal directed therapy was initiated and the patient was intubated. Further clinical work up included blood cultures revealing Fusobacterium varium bacteraemia, and CT and ultrasound imaging demonstrated thrombosis of the internal jugular vein and septic pulmonary emboli. A diagnosis of Lemierre syndrome was made, and antibiotics as well as anticoagulation therapy were initiated. The patient's clinical condition improved with treatment, and he was discharged home on hospital day 12 with completion of an uneventful 4-week course of outpatient antibiotic and anticoagulation therapy.\"}, {'qas': [{'id': '31271000:Rahnella aquatilis', 'question': 'Can Rahnella aquatilis cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Bacteremia due to Rahnella aquatilis in a Patient with a Chemoport.\\nABSTRACT:Rahnella aquatilis, a saprophytic organism, is a member of the Enterobacteriaceae family. The natural habitat of this organism is fresh water, and it is rarely found in clinical specimens. Clinical conditions ascribed to this organism include bacteremia, respiratory infection, urinary tract infection, wound infection in an immunocompromised host, and infective endocarditis in patients with congenital heart diseases. Here, we report a case of bacteremia due to R. aquatilis in a woman with breast cancer who had received chemotherapy through a chemoport. To our knowledge, this is the second case of bacteremia caused by this organism in a patient with cancer in Korea.'}, {'qas': [{'id': '23903543:Rickettsia prowazekii', 'question': 'Can Rickettsia prowazekii cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Reverse transcription-PCR-electrospray ionization mass spectrometry for rapid detection of biothreat and common respiratory pathogens.\\nABSTRACT:Electrospray ionization mass spectrometry (ESI-MS) analysis of reverse transcription (RT)-PCR amplicons from human respiratory samples allows for broad pathogen identification approximately 8 h after collection. We investigated the performance characteristics of a high-throughput RT-PCR-coupled ESI-MS assay for distinguishing biothreat (BT) agents from common bacterial, fungal, and viral respiratory pathogens in bronchoalveolar lavage (BAL) fluid specimens from subjects with suspected respiratory infections. In a retrospective case series, 202 BAL fluid specimens were collected at the Johns Hopkins Hospital between August 2010 and February 2011 from patients with suspected acute respiratory infections. Samples were processed using standard bacterial, viral, and fungal testing in the clinical microbiology laboratory as part of routine care and then were blindly spiked with either water or nucleic acids from BT organisms (Bacillus anthracis, Yersinia pestis, Francisella tularensis, Brucella spp., Burkholderia spp., and Rickettsia prowazekii) and tested by RT-PCR-ESI-MS. The sensitivities and specificities of RT-PCR-ESI-MS versus standard clinical methods were as follows: for mock BT DNA, 98.5% sensitivity (95% confidence interval [CI], 94.2 to 99.7%) and 100% specificity (95% CI, 93.1 to 100.0%); for bacterial pathogens, 81.8% sensitivity (95% CI, 74.3 to 87.6%) and 73.6% specificity (95% CI, 64.2 to 81.4%); for viral pathogens, 93.3% sensitivity (95% CI, 66.0 to 99.7%) and 97.3% specificity (95% CI, 89.7 to 99.5%); for fungal pathogens, 42.6% sensitivity (95% CI, 29.5 to 56.7%) and 97.8% specificity (95% CI, 91.8 to 99.6%). Our data suggest that RT-PCR-ESI-MS is a useful adjunct to standard culture protocols for rapid detection of both BT and common respiratory pathogens; further study is required for assay validation, especially for fungal detection, and potential implementation. '}, {'qas': [{'id': '8086575:Anaerobiospirillum', 'question': 'Can Anaerobiospirillum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Selected nonsurgical anaerobic infections: therapeutic choices and the effective armamentarium.\\nABSTRACT:Anaerobic bacteria play an important role in diverse clinical infections. In this review the historical role and prevalence of anaerobes and their potential impact on the management of patients are discussed in terms of pleuropulmonary infections, animal and human bite-wound infections, diabetic foot infections, bacteremia, tetanus, diarrhea due to Anaerobiospirillum succiniciproducens, and diarrhea due to toxigenic Bacteroides fragilis.'}, {'qas': [{'id': '28330077:Staphylococcus simulans', 'question': 'Can Staphylococcus simulans cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Diversity and antagonistic potential of marine microbes collected from south-west coast of India.\\nABSTRACT:The diversity of some of the culturable microorganisms associated with marine flora and fauna collected off Vizhinjam and Mulloor coast of South India was evaluated and their bioactive production potential determined. From a total of 24 bacteria, 4 actinomycetes and 8 fungi isolated from diverse marine sources, five bacterial species-BLM3, BSP2, BCS1, BCS4 and BMA6 showed inhibitory activity against at least one of the tested pathogens viz., Klebsiella pneumonia KU1, Pseudomonas aeruginosa VL3, Salmonella enterica typhimurium MTCC 98, Escherichia coli MTCC 40, Micrococcus luteus MTCC 105, Staphylococcus simulans MTCC 3610, Proteus vulgaris MTCC 426, Vibrio fluvialis, Vibrio sp. P3a and Vibrio sp. P3b. The isolated actinomycetes and fungi did not produce significant inhibition zones against the tested pathogens; however, the macroalgal isolated actinomycetes strain AMA1 produced reddish pigment in Starch Casein medium which remained stable till the stationary phase of growth. The marine sediment isolate BCS4, identified as Bacillus sp. showed wide spectrum of activity against the tested Gram positive bacteria, S. simulans MTCC 3610 and Gram negative bacteria, Proteus vulgaris with zone of inhibitions of 25 and 11 mm respectively. Better extraction of the bioactive compound was obtained with ethyl acetate when compared with methanol, benzene and hexane and TLC analysis revealed the presence of an active compound. The 16SrRNA sequencing confirmed the potent strain belong to Bacillus sp. and hence designated Bacillus sp. BCS4.'}, {'qas': [{'id': '27848164:Legionella cincinnatiensis', 'question': 'Can Legionella cincinnatiensis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Septic arthritis due to Legionella cincinnatiensis: case report and review of the literature.\\nABSTRACT:Legionella spp. are an important cause of pulmonary and rarely extrapulmonary infections. L. cincinnatiensis has only been implicated in five cases to date. We herein report the first case of L. cincinnatiensis septic arthritis in a 90-year old lady with a past medical history of chronic kidney disease. She developed septic arthritis of her left wrist after having received intraarticular corticosteroid injections and oral corticosteroids administered for presumed chondrocalcinosis. Appropriate antimicrobial treatment of L. cincinnatiensis septic arthritis was delayed until identification of this organism in joint biopsies by broad-range bacterial PCR targeting the 16S rRNA gene with subsequent rDNA sequence analysis and by culture on special media. Reviewing all reported cases of septic arthritis caused by Legionella spp. other than L. cincinnatiensis it is notable that diagnosis was established by PCR in the majority of cases and only subsequently confirmed by special culture. Although most patients were immunosuppressed, outcome was favourable. Treatment consisted of a fluoroquinolone alone or in combination with rifampicin or a macrolide. Our case highlights the need for a high index of suspicion for infections with unusual/fastidious organisms when symptoms are suggestive of septic arthritis but conventional methods fail to identify a causative organism.'}, {'qas': [{'id': '24844262:Wautersiella falsenii', 'question': 'Can Wautersiella falsenii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Description of Chishuiella changwenlii gen. nov., sp. nov., isolated from freshwater, and transfer of Wautersiella falsenii to the genus Empedobacter as Empedobacter falsenii comb. nov.\\nABSTRACT:A Gram-reaction-negative, strictly aerobic, non-pigmented, non-gliding, rod-shaped bacterium, designated strain BY4(T), was isolated from freshwater. Cells were catalase- and oxidase-positive and indole was produced. Phylogenetic analyses based on 16S rRNA gene sequences indicated that strain BY4(T) belonged to the family Flavobacteriaceae and showed 91.6-95.9% sequence similarities to the most closely related strains. The major respiratory quinone was MK-6 and the major polar lipid was phosphatidylethanolamine. The major polyamine was homospermidine and the major fatty acids were iso-C(15 : 0), iso-C(17 : 0) 3-OH and summed feature 3 (C(16 : 1)omega7c and/or C(16 : 1)omega6c). The DNA G+C content was 30.0 mol%. On the basis of phenotypic, phylogenetic and genotypic features, strain BY4(T) is suggested to represent a novel species in a new genus within the family Flavobacteriaceae, for which the name Chishuiella changwenlii gen. nov., sp. nov. is proposed. The type strain of this type species is BY4(T) ( = CGMCC 1.12707(T) = JCM 19633(T)). On the basis of data collected from previous and present studies, it is proposed to reclassify Wautersiella falsenii to the genus Empedobacter as the new combination Empedobacter falsenii comb. nov. (type strain NF 993(T) = CCUG 51536(T) = CIP 108861(T)).'}, {'qas': [{'id': '2358715:Capnocytophaga ochracea', 'question': 'Can Capnocytophaga ochracea cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:[A case of septicemia due to Capnocytophaga ochracea beginning post radiation therapy for lung cancer].\\nABSTRACT:A 46-year-old man was admitted to our hospital on Mar. 16, 1988 with the chief complaint of productive cough. The chest roentgenogram and tomogram showed a tumorous shadow in the right upper lobe, accompanied with stenosis of the trachea and the right main bronchus. Bronchofiberscopic examination confirmed a nodular tumor protruding into the lower part of the trachea from the right lateral wall and nearly complete obstruction of the right main bronchus. Pathological specimen obtained by transbronchial biopsy revealed \"low differentiated adenocarcinoma\". It was highly suspected that the primary lung cancer had directly invaded the trachea and the right main bronchus. His symptoms and roentgenological findings remarkably improved after radiation therapy. He was discharged on May 12. On Sep. 14, he was admitted to our hospital again because of hoarseness, general fatigue and increasing dyspnea. The chest CT demonstrated severe stenosis of the trachea, which was treated with another radiation therapy. Although his symptoms diminished, he had a sudden onset of high fever on Oct. 15. Immediately a blood culture and transtracheal aspiration (TTA) were performed. Gram-negative bacilli were isolated from the blood culture four days later. The administration of fosfomycin and tobramycin was started. However he died because of massive hemoptysis on Oct. 23. Several days after the death, an isolated strain was identified as Capnocytophaga ochracea by the biochemical characteristics. Culture of sputum obtained by TTA was negative for Capnocytophaga ochracea. Septicemia due to Capnocytophaga spp. is very rare and only one case (due to Capnocytophaga sputigena) has been reported until now in Japan. Our patient is thought to be the first case of septicemia due to Capnocytophaga ochracea in Japan.'}, {'qas': [{'id': '19915112:Nocardia mikamii', 'question': 'Can Nocardia mikamii cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Nocardia mikamii sp. nov., isolated from human pulmonary infections in the USA.\\nABSTRACT:Four nocardioform bacterial strains isolated from clinical respiratory sources were characterized using a polyphasic taxonomic approach. On the basis of 16S rRNA gene sequence analyses, these strains were found to be 100 % similar to each other and were shown to belong to the genus Nocardia. Chemotaxonomic data [major menaquinone: omega-cyclic isoprene side chain MK-8(H4(cycl)); major polar lipids: diphosphatidylglycerol, phosphatidylethanolamine, phosphatidylinositol and phosphatidylinositol mannosides; major fatty acids: monounsaturated fatty acids with a considerable amount of tuberculostearic acid; and mycolic acids (52-62 carbon atoms)] were consistent with the assignment of the novel strains to the genus Nocardia. Comparative phylogenetic analysis of the 16S rRNA gene sequences showed that the novel strains were related to Nocardia cerradoensis DSM 44546(T) (99.8 %) and Nocardia aobensis JCM 12352(T) (99.6 %). Analysis of gyrB gene sequences showed these strains were related to N. aobensis (96.6 %) and to N. cerradoensis (96.3 %). The results suggest that gyrB gene sequencing is a more powerful tool than 16S rRNA gene sequencing for taxonomic identification within the genus Nocardia. DNA-DNA hybridization and physiological and biochemical tests supported the genotypic and phenotypic differentiation of the novel strains from related species. These data indicated that the new strains represent a novel species within the genus Nocardia, for which the name Nocardia mikamii sp. nov. is proposed, with strain W8061(T) (=DSM 45174(T)=JCM 15508(T)) as the type strain.'}, {'qas': [{'id': '30555142:Kerstersia gyiorum', 'question': 'Can Kerstersia gyiorum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Three cases of Chronic Suppurative Otitis Media (CSOM) caused by Kerstersia gyiorum and a review of the literature.\\nABSTRACT:New identification techniques such as gene sequencing and mass spectrometry have increased the incidence of novel agents such as Kerstersia gyiorum. As a new member of the Alcaligenaceae family, K. gyiorum was isolated from wounds, respiratory tract, urine specimens and most frequently from chronic suppurative otitis media (CSOM). We isolated three K. gyiorum strains from three CSOM cases over a one-year period. The strains were analyzed by mass spectrometry and identified by Bruker Biotyper 3.1 (Bruker Daltonics, USA). The cases were young patients without chronic diseases and immunodeficiencies. Two strains were resistant to ciprofloxacin.'}, {'qas': [{'id': '27902326:Parapusillimonas', 'question': 'Can Parapusillimonas cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Caenimicrobium hargitense gen. nov., sp. nov., a new member of the family Alcaligenaceae (Betaproteobacteria) isolated from activated sludge.\\nABSTRACT:A new betaproteobacterium, CGII-59m2T, was isolated from an activated sludge bioreactor which treated landfill leachate. The 16S rRNA gene sequence analysis revealed that strain CGII-59m2T belonged to the family Alcaligenaceae and shared the highest pairwise similarity values with Parapusillimonas granuli LMG 24012T (97.7 %), various species of the genus Bordetella (97.3-97.0 %) and Candidimonas nitroreducens LMG 24812T (97.0 %). Cells of strain CGII-59m2T were rod-shaped, non-motile, and oxidase- and catalase-positive. The predominant fatty acids were C16 : 1omega7c, C16 : 0, cyclo C17 : 0 and C18 : 1omega7c, the major respiratory quinone was Q-8, and the main polar lipids were phosphatidylethanolamine, phosphatidylglycerol, diphosphatidylglycerol and an unknown phospholipid. The G+C content of the genomic DNA of strain CGII-59m2T was 62.3 mol%. The new bacterium can be distinguished from the closely related type strains based on its non-motile cells and its high C16 : 1omega7c fatty acid content. On the basis of the phenotypic, chemotaxonomic and molecular data, strain CGII-59m2T is considered to represent a novel species of a new genus, for which the name Caenimicrobium hargitense gen. nov., sp. nov. is proposed. The type strain is CGII-59m2T (=DSM 29806T=NCAIM B.02615T).'}, {'qas': [{'id': '33089823:Brucella ceti', 'question': 'Can Brucella ceti cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Occurrence of Brucella ceti in stranded bottlenose dolphins Tursiops truncatus coincides with calving season.\\nABSTRACT:Brucellosis is a disease caused by the Gram-negative facultative intracellular bacterium Brucella spp. In terrestrial species, this zoonotic bacterium is a global public health risk, but there is also concern over the zoonotic potential of marine forms, such as B. ceti, which affects cetaceans. Due to the detection of B. ceti in samples from bottlenose dolphins Tursiops truncatus during the 2010-2014 Gulf of Mexico Unusual Mortality Event, a long-term study of the prevalence of Brucella in stranded bottlenose dolphins from South Carolina, USA, was conducted. From 2012 through 2017, 282 stranded bottlenose dolphins were tested for B. ceti via real-time PCR. Nearly 32% of the dolphins tested positive in at least one sample (brain, lung, blowhole swab). Very little information exists in the literature on the occurrence of Brucella spp. in marine mammals, though in terrestrial species, such as cattle and elk, higher prevalence is often reported in spring. Similar results were found in this study with the peak occurrence being between March and June, a known period of calving in South Carolina. Results from this study provide important insights into the occurrence of the marine bacterium B. ceti.'}, {'qas': [{'id': '12058317:Legionella wadsworthii', 'question': 'Can Legionella wadsworthii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:In vitro antimycobacterial and antilegionella activity of licochalcone A from Chinese licorice roots.\\nABSTRACT:Licochalcone A, extracted and purified from Chinese licorice roots, showed in vitro inhibitory effect on human pathogenic Mycobacteria species and Legionella species. M. tuberculosis, M. bovis and BCG were inhibited by &lt; 20 mg/l licochalcone A, whereas all non- M. tuberculosis complex species were resistant to &gt; 20 mg/l Legionella pneumophila (serogroups 1 - 7) and L. bozemanii, L. dumoffii, L. feelei, L. longbeacheae and L. wadsworthii were inhibited by licochalcone A 1 - 4 mg/l, whereas L. gormanii and L. micdadei were inhibited by licochalcone A 500 - 1000 mg/l. These data indicate that licochalcone A might be of interest as a new class of antibacterial drug in the treatment of severe lung-infections.'}, {'qas': [{'id': '8571983:Ehrlichia species', 'question': 'Can Ehrlichia species cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:Addressing the challenges of emerging infectious disease.\\nABSTRACT:Through the recent examples of diphtheria in the former Soviet Union, plague in India, and trends in pneumonia mortality in the United States, the author, in this article, illustrates issues in emerging infectious diseases. The Centers for Disease Control's plan, Addressing Emerging Infectious Disease Threats: A Prevention Strategy for the United States, is summarized. Initial efforts to implement this plan are described, with particular focus on the development of Emerging Infections Programs, which are conducting epidemiologic and laboratory projects on several infectious diseases, including invasive bacterial diseases, unexplained deaths, foodborne diseases, and ehrlichiosis in four population-based sites in the United States.\"}, {'qas': [{'id': '18804973:Capnocytophaga sputigena', 'question': 'Can Capnocytophaga sputigena cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:[Capnocytophaga sputigena pleuropneumonitis: a case report].\\nABSTRACT:Community-acquired pneumonia is complicated by purulent pleural effusion in approximately 28% of affected children. Its incidence has increased over the last decade. Streptococcus pneumoniae is the most frequent causal microorganism, encountered in roughly 80% of cases. We report the case of an immunocompetent 13-year-old teenager wearing orthodontic braces who suffered from pleuropneumonia due to Capnocytophaga sputigena. To date, this Gram-negative bacillus, a commensal of the buccal cavity, had never been reported to be responsible for pleural or lung infection. The clinical presentation is similar to that observed with usual bacteria, while bacterial sensitivity to betalactam antibiotics is excellent.'}, {'qas': [{'id': '25008832:Myroides odoratus', 'question': 'Can Myroides odoratus cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Myroides odoratus and Chryseobacterium indologenes: two rare isolates in the immunocompromised.\\nABSTRACT:Myroides spp and Chryseobacterium spp are uncommon clinical isolates, though more frequently reported to cause infections than other pigmented non-fermentors. Two cases of Myroides odoratus and Chryseobacterium indologenes infection in a diabetic with pulmonary tuberculosis and a patient with de-compensated alcoholic liver disease, respectively, are reported here. Anti-microbial susceptibility testing of the isolates was performed by determining the minimum inhibitory concentration. The clinical picture, characteristic features of the isolates and the antibiotic susceptibility pattern are discussed briefly. '}, {'qas': [{'id': '30311486:Corynebacterium xerosis', 'question': 'Can Corynebacterium xerosis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Microbiota of Honeybees, &lt;I&gt;Apis mellifera Adansonii &lt;/I&gt;(Hymenoptera: Apidae) from Selected Ecozones, South West Nigeria.\\nABSTRACT:BACKGROUND AND OBJECTIVES: Honeybees explore their environment in search of nectar and pollens including water whose sources are contaminated by variety of microbes. This has both negative and positive economic implications on health status of honeybees, the consumers of the products including the quality of the products. Hence, the study was designed to investigate the effect of geographical locations on the occurrence and diversity of microbiota on the external integument and the digestive gut of adult worker honeybees, Apis mellifera adansonii. MATERIALS AND \\n'}, {'qas': [{'id': '19841075:Legionella tucsonensis', 'question': 'Can Legionella tucsonensis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Mouse macrophages are permissive to motile Legionella species that fail to trigger pyroptosis.\\nABSTRACT:Legionella pneumophila, a motile opportunistic pathogen of humans, is restricted from replicating in the lungs of C57BL/6 mice. Resistance of mouse macrophages to L. pneumophila depends on recognition of cytosolic flagellin. Once detected by the NOD-like receptors Naip5 and Ipaf (Nlrc4), flagellin triggers pyroptosis, a proinflammatory cell death. In contrast, motile strains of L. parisiensis and L. tucsonensis replicate profusely within C57BL/6 macrophages, similar to flagellin-deficient L. pneumophila. To gain insight into how motile species escape innate defense mechanisms of mice, we compared their impacts on macrophages. L. parisiensis and L. tucsonensis do not induce proinflammatory cell death, as measured by lactate dehydrogenase (LDH) release and interleukin-1beta (IL-1beta) secretion. However, flagellin isolated from L. parisiensis and L. tucsonensis triggers cell death and IL-1beta secretion when transfected into the cytosol of macrophages. Neither strain displays three characteristics of the canonical L. pneumophila Dot/Icm type IV secretion system: sodium sensitivity, LAMP-1 evasion, and pore formation. Therefore, we postulate that when L. parisiensis and L. tucsonensis invade a mouse macrophage, flagellin is confined to the phagosome, protecting the bacteria from recognition by the cytosolic surveillance system and allowing Legionella to replicate. Despite their superior capacity to multiply in mouse macrophages, L. parisiensis and L. tucsonensis have been associated with only two cases of disease, both in renal transplant patients. These results point to the complexity of disease, a product of the pathogenic potential of the microbe, as defined in the laboratory, and the capacity of the host to mount a measured defense.'}, {'qas': [{'id': '23686599:Leclercia adecarboxylata', 'question': 'Can Leclercia adecarboxylata cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Pneumonia due to multidrug-resistant Leclercia adecarboxylata.\\nABSTRACT:'}, {'qas': [{'id': '19567344:Ochrobactrum anthropi', 'question': 'Can Ochrobactrum anthropi cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Antibiotic susceptibility profiles of uncommon bacterial species causing severe infections in Italy.\\nABSTRACT:This study presents the results of the italian \"Severe infections project\" involving bacteria that can be considered rare causes of disease. we isolated 30 uncommon human pathogens from a total of 60 strains (1.2% of all the isolates). The most frequent sources of uncommon human pathogens were primary bloodstream infections (48.3%) and pneumonia (20%). Species such as Comamonas testosteroni, Enterococcus hirae, Kluyvera ascorbata, Kluyvera cryocrescens, Leclercia adecarboxylata and Ochrobactrum anthropi were recovered from bacteremia patients. Clinically useful antimicrobial agents were tested against each isolate. Resistance to 4 or more antibiotics tested was found in Achromobacter xylosoxidans, O. anthropi, Pseudomonas stutzeri, Citrobacter braakii, Enterobacter sakazakii, K. ascorbata, Proteus penneri and Serratia plymuthica. About 16% of the Gram-negative species were resistant to third-generation cephalosporins and 28.6% of the staphylococci were oxacillin-resistant. the results from this study offer indications for empirical therapy for severe infections from uncommon human pathogens.'}, {'qas': [{'id': '27118979:Listeria grayi', 'question': 'Can Listeria grayi cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Cytotoxic Effect and TLC Bioautography-Guided Approach to Detect Health Properties of Amazonian Hedyosmum sprucei Essential Oil.\\nABSTRACT:Bioautography has been used as rapid and easy strategy to detect and identify bioactive fractions/molecules in the never before investigated Hedyosmum sprucei Solms (Chloranthaceae) essential oil (EO). The antioxidant activity, performed through DPPH bioautographic assay and spectrophotometric evaluations (IC50 = 230 +- 10 microg/mL), seemed to be mainly due to alpha-cadinol and alpha-muurolol. (HP)TLC bioautography, focused on antimicrobial capacities, pointed out alpha-cadinol, alpha-muurolol, tau-muurolol, caryophyllene oxide, and methyleugenol as the most effective compounds against Staphylococcus aureus, considered as testing strain. Moreover, the microdilution method, assessed among a wide panel of microorganisms, revealed Listeria grayi and Staphylococcus aureus as the most sensitive among human tested strains and Clavibacter michiganensis among phytopathogens. GC-MS chemical profile showed that bioactive molecules represented only a small quantity of the whole EO: germacrene D (23.16%), beta-caryophyllene (15.53%), delta-cadinene (5.50%), alpha-copaene (5.08%), and alpha-phellandrene (3.48%) were the main compounds, highlighting an uncommon composition among the genus Hedyosmum. Finally, H. sprucei EO was checked for cytotoxic potential against A549 (lung cancer) and MCF-7 (breast cancer) cell lines showing promising cytotoxic effects against both cell lines after 48 h (IC50 A549 = 44.05 +- 2.35 microg/mL; IC50 MCF-7 = 32.76 +- 4.92 microg/mL) and 72 h (IC50 A549 = 43.55 +- 2.80 microg/mL; IC50 MCF-7 = 33.64 +- 0.43 microg/mL). '}, {'qas': [{'id': '11803363:Capnocytophaga gingivalis', 'question': 'Can Capnocytophaga gingivalis cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Pneumonia and sepsis due to fluoroquinolone-resistant Capnocytophaga gingivalis after autologous stem cell transplantation.\\nABSTRACT:Human oral Capnocytophaga species have been only rarely described as a cause of sepsis in patients following stem cell or marrow transplantation, and pneumonia has not been reported in this setting. In addition, fluoroquinolone resistance is rarely seen in these species, and has never been reported in C. gingivalis. We report a case of pneumonia (confirmed by culture of bronchoalveolar lavage fluid) and sepsis due to fluoroquinolone- resistant Capnocytophaga gingivalis in a patient following autologous stem cell transplantation, who responded to treatment with linezolid and metronidazole. Capnocytophaga infections should be considered in patients with fever following stem cell or marrow transplantation, especially those with neutropenia and mucositis. Susceptibility testing is needed given the existence of multidrug-resistant isolates.'}, {'qas': [{'id': '11427591:Mycobacterium triplex', 'question': 'Can Mycobacterium triplex cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Pulmonary infection caused by an unusual, slowly growing nontuberculous Mycobacterium.\\nABSTRACT:Mycobacterium triplex, a recently described slowly growing nontuberculous mycobacterium, was isolated from a Finnish patient with pulmonary mycobacteriosis. The disease was successfully treated with antimycobacterial drugs. The strain isolated, which was similar to the type strain but differed slightly from the species description, was regarded as a variant of M. triplex sensu stricto. According to present knowledge this variant of the species has never been isolated before.'}, {'qas': [{'id': '17660147:Bergeyella zoohelcum', 'question': 'Can Bergeyella zoohelcum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Cellulitis and bacteremia caused by Bergeyella zoohelcum.\\nABSTRACT:Bergeyella zoohelcum is a rod-shaped, aerobic, Gram-negative, non-motile and non-saccharolytic bacterium. It is frequently isolated from the upper respiratory tract of dogs, cats and other mammals. Clinically, B. zoohelcum has been known to cause cellulitis, leg abscess, tenosynovitis, septicemia, pneumonia and meningitis, and is associated with animal bites. In addition, food-borne transmission was considered in a recent case report. We report a 73-year-old man with liver cirrhosis who had no history of dog bite but had dog exposure, who developed cellulitis of the left lower leg and B. zoohelcum was isolated from blood culture. This patient, without evidence of polymicrobial infection, was treated with cefazolin and gentamicin with a good outcome. B. zoohelcum is a zoonotic pathogen that may cause bacteremia in patients with underlying disease such as liver cirrhosis; it can be treated with a beta-lactam or quinolone.'}, {'qas': [{'id': '26378409:Aeromonas dhakensis', 'question': 'Can Aeromonas dhakensis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:Aeromonas dhakensis pneumonia and sepsis in a neonate Risso's dolphin Grampus griseus from the Mediterranean Sea.\\nABSTRACT:A neonate Risso's dolphin Grampus griseus was found stranded alive on a beach in Catalonia, Spain. Rehabilitation attempts were unsuccessful and it died 2 d later, showing pneumonia and sepsis. A pure bacterial culture was obtained from all tissues and blood and identified as Aeromonas hydrophila using the API 20NE. However, sequencing the rpoD gene showed that the strain in fact belongs to A. dhakensis, making this the first report of fatal haemorrhagic-necrotizing pneumonia and sepsis due to this species in a marine mammal. The A. dhakensis strain GMV-704 produced beta-haemolysis, possessed several virulence genes and showed sensitivity to several antimicrobials. This study provides a new potential host for A. dhakensis, and its potential virulence in dolphins and its presence in the marine environment may warrant considering this species a potential threat to marine mammals. \"}, {'qas': [{'id': '3384931:Acholeplasma laidlawii', 'question': 'Can Acholeplasma laidlawii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Distinction of species and strains of mycoplasmas (mollicutes) by genomic DNA fingerprints with an rRNA gene probe.\\nABSTRACT:Genomic fingerprints of Acholeplasma laidlawii, Mycoplasma hominis, and Mycoplasma pneumoniae strains were obtained by Southern blot hybridization of the digested mycoplasmal DNAs with an rRNA gene probe. The hybridization patterns revealed genotypic heterogeneity among A. laidlawii and M. hominis strains and a remarkable degree of homogeneity among M. pneumoniae strains isolated from pneumonia patients during a 10-year period. Genomic fingerprints with the rRNA gene probe can thus serve as indicators of intraspecies genetic homogeneity or heterogeneity and can provide a new, sensitive tool for strain identification with a potential for application in epidemiology.'}, {'qas': [{'id': '32198173:Paenibacillus polymyxa', 'question': 'Can Paenibacillus polymyxa cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:The role of Fnr paralogs for controlling anaerobic metabolism in diazotroph Paenibacillus polymyxa WLY78.\\nABSTRACT:Fnr is a transcriptional regulator that controls the expression of a variety of genes in response to oxygen limitation in bacteria. Genome sequencing revealed four genes (fnr1, fnr3, fnr5 and fnr7) coding for Fnr proteins in Paenibacillus polymyxa WLY78. Fnr1 and Fnr3 showed more similarity to each other than to Fnr5 and Fnr7. Also, Fnr1 and Fnr3 exhibited high similarity with Bacillus cereus Fnr and Bacillus subtilis Fnr in sequence and structures. Both of the aerobically purified His-tagged Fnr1 and His-tagged Fnr3 in Escherichia coli could bind to the specific DNA promoter. Deletion analysis showed that the four fnr genes, especially fnr1 and fnr3, have significant impacts on growth and nitrogenase activity. Single deletion of fnr1 or fnr3 led to a 50% reduction in nitrogenase activity and double deletion of fnr1 and fnr3 resulted to a 90% reduction in activity. Genome-wide transcription analysis showed that Fnr1 and Fnr3 indirectly activated expression of nif (nitrogen fixation) genes and Fe transport genes under anaerobic conditions. Fnr1 and Fnr3 inhibited expression of the genes involved in the aerobic respiratory chain and activated expression of genes responsible for anaerobic electron acceptor genes.IMPORTANCEThe members of the nitrogen-fixing Paenibacillus spp. have great potential to be used as a bacterial fertilizer in agriculture. However, the functions of fnr gene(s) in nitrogen fixation and other metabolisms in Paenibacillus spp. are not known. Here, we found that in P. polymyxa WLY78, Fnr1 and Fnr3 were responsible for regulation of numerous genes in response to changes in oxygen levels, but Fnr5 and Fnr7 exhibited little effect. Fnr1 and Fnr3 indirectly or directly regulated many types of important metabolism, such as nitrogen fixation, Fe uptake, respiration and electron transport. This study not only reveals the function of the fnr genes of P. polymyxa WLY78 in nitrogen fixation and other metabolisms, but also will provide insight into the evolution and regulatory mechanisms of fnr in Paenibacillus.'}, {'qas': [{'id': '28110789:Staphylococcus sciuri', 'question': 'Can Staphylococcus sciuri cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Exudative epidermitis of piglets caused by non-toxigenic Staphylococcus sciuri.\\nABSTRACT:Infections with strains of Staphylococcus sciuri are a potential threat to animal and public health, and a cause for considerable concern. We isolated and identified S. sciuri as a pathogen from an acute outbreak of exudative epidermitis in piglets for further genetic identification using experimental infections. The results of this study showed that S. sciuri strain NJ1306 reproduced exudative epidermitis in experimentally infected 5-day-old piglets. The isolated bacteria also caused sudden death in BALB/c mice following intraperitoneal injection with 5x108 CFU of the isolate. The data indicated that strain NJ1306 of S. sciuri was pathogenic to piglets and mice, and the study provided the first known report of clinical lung lesions and endocarditis in piglets due to S. sciuri.'}, {'qas': [{'id': '24257815:Leptospira borgpetersenii', 'question': 'Can Leptospira borgpetersenii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:High virulence in hamsters of four dominant Leptospira serovars isolated from rats in the Philippines.\\nABSTRACT:Leptospirosis is caused by pathogenic species of Leptospira. The aim of this study was to determine and characterize the pathogenicity of four dominant Leptospira isolates prevailing among rats in the Philippines. The isolates were Leptospira interrogans serovar Manilae strain K64, L. interrogans serovar Losbanos strain K37, L. interrogans serovar Ratnapura strain K5 and Leptospira borgpetersenii serovar Javanica strain K6. Pathogenicities were studied using hamsters, which reproduce severe human leptospirosis. The minimum lethal doses were 10(0) ( = 1) leptospires for K64, K37 and K5, and 10(1) leptospires for K6. Weight loss amongst the Leptospira-infected hamsters was observed from 1 day before death (K64-, K37- and K5-infected hamsters) to as much as 1 week before death for K6-infected hamsters. Similar and varied gross and microscopic lesions were observed amongst infected hamsters, even for strains belonging to the same species (i.e. L. interrogans). The most significant and common histopathological findings were congestion of the glomerulus, disarrangement of hepatic cords and erythrophagocytosis. Other findings were foamy splenic macrophages for K6, severe petechial pulmonary haemorrhage for K64, and hematuria and severe pulmonary congestion for K37. Immunostaining and culture revealed the presence of leptospires in different organs of the infected hamsters. Based on these results, Leptospira isolates from rats in the Philippines were shown to be highly virulent, causing pulmonary haemorrhage, severe hepato-renal damage and death in hamsters even at lower doses. The present findings on experimental leptospirosis support clinical data showing that patients with severe manifestations of leptospirosis, such as pulmonary haemorrhage, are increasing in the Philippines. These findings may serve as a basis to strengthen the early diagnosis and treatment of human leptospirosis. '}, {'qas': [{'id': '22469443:Ewingella americana', 'question': 'Can Ewingella americana cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Sensitization to Per a 2 of the American cockroach correlates with more clinical severity among airway allergic patients in Taiwan.\\nABSTRACT:\\n'}, {'qas': [{'id': '32754384:Rothia dentocariosa', 'question': 'Can Rothia dentocariosa cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Melody Valve Endocarditis Due to Rothia dentocariosa: A Diagnostic Challenge.\\nABSTRACT:Recently, there have been several advances in the field of adult congenital heart disease, such as the percutaneous pulmonic valve implantation (PPVI) to treat right ventricular outflow obstruction. Complications from this technique are seldom but essential to understand. We present a case of a 37-year-old Caucasian male with complicated congenital heart disease, including prior Melody valve implantation, who presented to our hospital with recurrent episodes of pneumonia of two months duration. He was diagnosed with prosthetic valve endocarditis (PVE) from an unusual organism, Rothia dentocariosa. He eventually underwent surgical replacement of the infected valve. Our report is the first case of Melody valve endocarditis due to Rothia dentocariosa reported from the United States.'}, {'qas': [{'id': '21882663:Corynebacterium xerosis', 'question': 'Can Corynebacterium xerosis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Studies on antimicrobial activities of solvent extracts of different spices.\\nABSTRACT:The antimicrobial activities of the ethyl acetate, acetone and methanol extract of 12 plant species were studied. The extract of Capsicum annuum (red pepper) (fruit) Zingiber officinale (ginger) (root), Cuminum cyminum (cumin), Alpinia ficinarum (galingale), Coriandrum sativum (coriander), Cinnamomun zeylanicum Nees (cinnamomun), Origanum onites L. (thyme), Folium sennae (senna), Eugenia caryophyllata (cloves), Flos tiliae (lime), Folium menthae crispae (peppermint) and Piper nigrum (blackpepper) were tested in vitro against 2 fungi and 8 bacterial species by the disc diffusion method. Klebsiella pneumonia 13883, Bacillus megaterium NRS, Pseudomonas aeroginosa ATCC 27859, Staphylococcus aureus 6538 P, Escherichia coil ATCC 8739, Enterobacter cloaca ATCC 13047, Corynebacterium xerosis UC 9165, Streptococcus faecalis DC 74, Kluyveromyces marxianus, Rhodotorula rubra were used in this investigation. The results indicated that extracts of different spices has shown antibacterial activity in the range of 7-24 mm 30 microl(-1) inhibition zone Eugenia caryophyllata (clove), 7-20 mm 30microl(-1) inhibition zone Capsicum annum (red pepper) and Cinnamomun zeylanicum (cinnamon) bark, 7-18 mm 30microl(-1) inhibition zone Folium sennae (senna) leaves, 7-16 mm 30 microl(-1) inhibition zone Zingiber officinale (ginger) root, 7-15 mm 30microl(-1) inhibition zone Cuminum cyminum (cumin) seed, 7-14 mm 30 microl(-1) inhibition zone Folium menthae crispae (peppermint), Origanum onites (thyme) leaves and Alpinia ficinarum (galingale) root, 7-12 mm 30 microl(-1) inhibiton zone Piper nigrum (blackpepper), 7-11 mm 30microl(-1) inhibition zone Flos tiliae (lime) leaves, 7-8 mm 30microl(-1) inhibition zone Coriandrum sativum (coriander) to the microorganisms tested.'}, {'qas': [{'id': '27641915:Porphyromonas copri', 'question': 'Can Porphyromonas copri cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:The Microbiome: a Revolution in Treatment for Rheumatic Diseases?\\nABSTRACT:PURPOSE OF REVIEW: The microbiome is the term that describes the microbial ecosystem that cohabits an organism such as humans. The microbiome has been implicated in a long list of immune-mediated diseases which include rheumatoid arthritis, ankylosing spondylitis, and even gout. The mechanisms to account for this effect are multiple. The clinical implications from observations on the microbiome and disease are broad. RECENT \\n'}, {'qas': [{'id': '28061799:Ralstonia insidiosa', 'question': 'Can Ralstonia insidiosa cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Ralstonia picketti neonatal sepsis: a case report.\\nABSTRACT:\\n'}, {'qas': [{'id': '24895247:Streptococcus infantis', 'question': 'Can Streptococcus infantis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:The potential role of lung microbiota in lung cancer attributed to household coal burning exposures.\\nABSTRACT:Bacteria influence site-specific disease etiology and the host's ability to metabolize xenobiotics, such as polycyclic aromatic hydrocarbons (PAHs). Lung cancer in Xuanwei, China has been attributed to PAH-rich household air pollution from burning coal. This study seeks to explore the role of lung microbiota in lung cancer among never smoking Xuanwei women and how coal burning may influence these associations. DNA from sputum and buccal samples of never smoking lung cancer cases (n = 8, in duplicate) and controls (n = 8, in duplicate) in two Xuanwei villages was extracted using a multi-step enzymatic and physical lysis, followed by a standardized clean-up. V1-V2 regions of 16S rRNA genes were PCR-amplified. Purified amplicons were sequenced by 454 FLX Titanium pyrosequencing and high-quality sequences were evaluated for diversity and taxonomic membership. Bacterial diversity among cases and controls was similar in buccal samples (P = 0.46), but significantly different in sputum samples (P = 0.038). In sputum, Granulicatella (6.1 vs. 2.0%; P = 0.0016), Abiotrophia (1.5 vs. 0.085%; P = 0.0036), and Streptococcus (40.1 vs. 19.8%; P = 0.0142) were enriched in cases compared with controls. Sputum samples had on average 488.25 species-level OTUs in the flora of cases who used smoky coal (PAH-rich) compared with 352.5 OTUs among cases who used smokeless coal (PAH-poor; P = 0.047). These differences were explained by the Bacilli species (Streptococcus infantis and Streptococcus anginosus). Our small study suggests that never smoking lung cancer cases have differing sputum microbiota than controls. Further, bacteria found in sputum may be influenced by environmental exposures associated with the type of coal burned in the home.\"}, {'qas': [{'id': '16091900:Mycobacterium tilburgii', 'question': 'Can Mycobacterium tilburgii cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Localized pulmonary infection associated with Mycobacterium tilburgii in an HIV-infected patient.\\nABSTRACT:The case of an HIV-infected patient with low CD4+ cell count and localized pulmonary infection associated with Mycobacterium tilburgii, a recently recognized atypical mycobacterial species, is reported. Diagnosis was confirmed by repeated detection of mycobacterial DNA in lung biopsy specimens. After surgical removal of the pulmonary nodules, treatment with antimycobacterial combination therapy led to a complete and sustained recovery of the respiratory symptoms.M. tilburgii so far has only been described to cause disseminated infection in three patients, two of them suffering from acquired immunodeficiency syndrome. To our knowledge, this is the first report of localized pulmonary disease attributed to this mycobacterial species. As it responds to antimycobacterial combination therapy, efforts to establish the diagnosis using PCR methods in patients with suspected pulmonary disease due to mycobacteria other than tuberculosis should be undertaken.'}, {'qas': [{'id': '28348783:Granulicatella elegans', 'question': 'Can Granulicatella elegans cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:An unusual case of thoracic empyema caused by Granulicatella elegans (nutritionally variant streptococci) in a patient with pulmonary tuberculosis and human immunodeficiency virus infection.\\nABSTRACT:INTRODUCTION: Nutritionally variant streptococci (NVS) are an infrequent cause of human infection with Granulicatella elegans being the least encountered species in clinical specimens. The most common infection caused by NVS is infective endocarditis. CASE PRESENTATION: We report an unusual case of thoracic empyema due to G. elegans in a patient with pulmonary tuberculosis (TB) and human immunodeficiency virus infection. The patient responded favourably to drainage and penicillin. \\n'}, {'qas': [{'id': '22992836:Delftia acidovorans', 'question': 'Can Delftia acidovorans cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Delftia acidovorans bacteremia caused by bacterial translocation after organophosphorus poisoning in an immunocompetent adult patient.\\nABSTRACT:A 46-year-old woman was transferred to our emergency unit because of impaired consciousness and respiratory failure with the history of excessive pesticide intake. The patient was hypersalivative and had bilateral pupillary miosis. Laboratory results showed markedly decreased cholinesterase. She was intubated and treated in the intensive care unit with the diagnosis of organophosphorus poisoning. The patient had persisted diarrhea, with a high fever and stomach tenderness on day 10. Whole-body contrast enhanced computed tomography revealed a swollen, enhanced small intestinal wall, and blood culture identified Delftia acidovorans. She was diagnosed as D. acidovorans bacteremia, probably caused by bacterial translocation based on the clinical presentation and the exclusion of other sources, and treated well with a total of 8 days of antibiotic therapy. So far as we know, this is the first case of D. acidovorans bacteremia that was presumably caused by bacterial translocation after organophosphorus poisoning in an immunocompetent adult patient.'}, {'qas': [{'id': '16213751:Vibrio damselae', 'question': 'Can Vibrio damselae cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Effect of salinity on the immune response of tiger shrimp Penaeus monodon and its susceptibility to Photobacterium damselae subsp. damselae.\\nABSTRACT:Addition of NaCl at 2.5% to tryptic soy broth (TSB) significantly increased the growth of Photobacterium damselae subsp. damselae. Tiger shrimp Penaeus monodon held in 25 per thousand seawater were injected with P. damsela subsp. damselae grown in TSB containing NaCl at 0.5%, 1.5%, 2.5% and 3.5% at a dose of 8.48 x 10(4)colony-forming units (cfu)shrimp(-1). Over 24-96 h, the cumulative mortality was significantly higher for the shrimp challenged with P. damselae subsp. damselae grown in 2.5% NaCl than those grown in 0.5%, 1.5% and 3.5% NaCl. In another experiment, P. monodon held in 25 per thousand were injected with TSB-grown P. damselae subsp. damselae (8.48 x 10(4)cfushrimp(-1)), and then transferred to 5 per thousand, 15 per thousand, 25 per thousand (control) and 35 per thousand. After 96 h, the mortality was highest for the P. damselae subsp. damselae-injected shrimp held in 5 per thousand, and the lowest for the P. damselae subsp. damselae-injected shrimp held in 25 per thousand. In a separate experiment, P. monodon held in 25 per thousand and then transferred to 5 per thousand, 15 per thousand, 25 per thousand (control) and 35 per thousand were examined for immune parameters, and phagocytic activity and clearance efficiency of P. damselae subsp. damselae after 12-96 h. The THC, hyaline cell, phenoloxidase activity, respiratory burst, superoxide dismutase (SOD) activity, phagocytic activity and clearance efficiency decreased significantly for the shrimp held in 5 per thousand, 15 per thousand and 35 per thousand after 12h. It is concluded that tiger shrimp P. monodon transferred from 25 per thousand to low salinity levels (5 per thousand and 15 per thousand) and high salinity (35 per thousand) had reduced immune ability and decreased resistance against P. damselae subsp. damselae infection.'}, {'qas': [{'id': '33412232:Vibrio parahaemolyticus', 'question': 'Can Vibrio parahaemolyticus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:A teleost bactericidal permeability-increasing protein-derived peptide that possesses a broad antibacterial spectrum and inhibits bacterial infection as well as human colon cancer cells growth.\\nABSTRACT:The bactericidal permeability-increasing protein (BPI) is a multifunctional cationic protein produced by neutrophils with antibacterial, antitumor, and LPS-neutralizing properties. In teleost, a number of BPIs have been reported, but their functions are very limited. In this study, an N-terminal peptide, BO18 (with 18 amino acids), derived from rock bream (Oplegnathus fasciatus) BPI, was synthesized and investigated for its antibacterial spectrum, action mechanism, immunoregulatory property as well as the inhibition effects on bacterial invasion and human colon cancer cells growth. The results showed that BO18 was active against Gram-positive bacteria Bscillus subiilis, Micrococcus luteus, and Staphylococcus aureus, as well as Gram-negative bacteria Vibrio alginolyticus, Vibrio litoralis, Vibrio parahaemolyticus and Vibrio vulnificus. BO18 treatment facilitated the bactericidal process of erythromycin and rifampicin by enhancing the permeability of the outer membrane. During its interaction with V. alginolyticus, BO18 exerted its antibacterial activity by destroying cell membrane integrity, penetrating into the cytoplasm and binding to genomic DNA and total RNA. In vitro analysis indicated BO18 could enhance the respiratory burst ability and regulate the expression of immune related genes of macrophages. In vivo detection showed the administration of fish with BO18 before bacterial infection significantly reduced pathogen dissemination and replication in tissues. In addition, BO18 exerted a cytotoxic effect on the growth of human colon cancer cells HT-29. Together, these results add new insights into the function of teleost BPIs, and support that BO18 is a novel and broad-spectrum antibacterial peptide with potential to apply in fighting pathogenic infection in aquaculture.'}, {'qas': [{'id': '21686384:Actinomyces graevenitzii', 'question': 'Can Actinomyces graevenitzii cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Pulmonary actinomycosis complicating infliximab therapy for Crohn disease.\\nABSTRACT:The use of anti-tumour necrosis factor (TNF) agents has expanded significantly over the past few years, particularly for rheumatological diseases and Crohn disease. A number of associated opportunistic infections have been observed as a result of suppression of T-cell-mediated immunity, the most frequent being tuberculosis. This report describes a case of pulmonary actinomycosis in a 52-year-old patient receiving regular infusions of infliximab, an anti-TNF agent, for Crohn disease. He presented with a 12-day history of fever, night sweats and a non-productive cough on a background of a 9-year history of Crohn terminal ileitis. There was radiological evidence of a left upper lobe non-cavitatory pneumonia and bronchoscopic lavage fluid eventually grew Actinomyces graevenitzii. The patient was hospitalised and improved with antibiotic therapy. Within 4 weeks there was almost complete radiological resolution and infliximab was restarted after 4 months without further complication.'}, {'qas': [{'id': '19667375:Alistipes indistinctus', 'question': 'Can Alistipes indistinctus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:Alistipes indistinctus sp. nov. and Odoribacter laneus sp. nov., common members of the human intestinal microbiota isolated from faeces.\\nABSTRACT:Two anaerobic, non-spore-forming, non-motile, Gram-negative-staining bacteria, strains YIT 12060(T) and YIT 12061(T), were isolated from human faeces. Cells of strain YIT 12060(T) were coccoid to rod-shaped with round ends, positive for catalase, negative for indole and oxidase production, produced succinic and acetic acids as end products of glucose metabolism in peptone/yeast extract/glucose medium and had a DNA G+C content of 55.2 mol%. The main respiratory quinones were MK-10 (40%) and MK-11 (57%). Fatty acid analysis demonstrated the presence of a high concentration of iso-C(15 : 0) (56%). Following 16S rRNA gene sequence analysis, this strain was found to be most closely related to species of the genus Alistipes, with 90.9-92.6% gene sequence similarities to type strains of this species. Phylogenetic analysis and biochemical data supported the affiliation of strain YIT 12060(T) to the genus Alistipes of the family 'Rikenellaceae'. Strain YIT 12060(T) therefore represents a novel species of the genus Alistipes for which the name Alistipes indistinctus sp. nov. is proposed; the type strain is YIT 12060(T) (=DSM 22520(T)=JCM 16068(T)). Cells of the other isolate, strain YIT 12061(T), were pleomorphic rods that were asaccharolytic, catalase- and oxidase-negative, positive for gelatin hydrolysis and indole production, produced small amounts of succinic, acetic and iso-valeric acids as end products of metabolism in peptone/yeast extract medium and had a DNA G+C content of approximately 42.4 mol%. On the basis of 16S rRNA gene sequence similarity values, this strain was shown to belong to the family 'Porphyromonadaceae' and related to the type strains of Odoribacter splanchnicus (89.6%) and Odoribacter denticanis (86.2%); similarity values with strains of recognized species within the family 'Porphyromonadaceae' were less than 84 %. Biochemical data supported the affiliation of strain YIT 12061(T) to the genus Odoribacter. Strain YIT 12061(T) therefore represents a novel species for which the name Odoribacter laneus sp. nov. is proposed; the type strain is YIT 12061(T) (=DSM 22474(T)=JCM 16069(T)).\"}, {'qas': [{'id': '30214111:Nocardia thailandica', 'question': 'Can Nocardia thailandica cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:First Report of the Isolation of Nocardia thailandica from the Bronchoalveolar Lavage of a Patient in Iran.\\nABSTRACT:Nocardiae are Gram-positive, filamentous, aerobic, relatively slow-growing, and weakly acid-fast bacteria which cause nocardiosis in humans. We describe a 53-year-old patient with chronic bronchitis referred to Al-Zahra Hospital, Isfahan. A bronchial washing sample was taken from the patient. A Nocardia-like microorganism was detected in microscopic evaluation. Based on the phenotypic and 16S rRNA gene sequencing, the isolate was identified as Nocardia thailandica. The patient was treated with trimethoprim-sulfamethoxazole and linezolid. This is the first report of the isolation of Nocardia thailandica from Iran.'}, {'qas': [{'id': '19666814:Staphylococcus condimenti', 'question': 'Can Staphylococcus condimenti cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Staphylococcus massiliensis sp. nov., isolated from a human brain abscess.\\nABSTRACT:Gram-positive, catalase-positive, coagulase-negative, non-motile, non-fermentative and novobiocin-susceptible cocci were isolated from a human brain abscess sample (strain 5402776(T)). This novel strain was analysed by a polyphasic taxonomic approach. The respiratory quinones detected were MK-7 (93 %) and MK-6 (7 %) and the major fatty acids were C(15 : 0) iso (60.5 %), C(17 : 0) iso (8.96 %) C(15 : 0) anteiso (7.93 %) and C(19 : 0) iso (6.78 %). The peptidoglycan type was A3alpha l-Lys-Gly(2-3)-l-Ser-Gly. Based on cellular morphology and biochemical criteria, the new isolate was assigned to the genus Staphylococcus, although it did not correspond to any recognized species. The G+C content of the DNA was 36.6 mol%. Phylogenetic analysis based on 16S rRNA gene sequence comparisons showed that the new isolate was most closely related to Staphylococcus piscifermentans, Staphylococcus condimenti, Staphylococcus carnosus subsp. carnosus, S. carnosus subsp. utilis and Staphylococcus simulans (97.7 %, 97.6 %, 97.6 %, 97.6 % and 96.5 % sequence similarity, respectively). Comparison of tuf, hsp60, rpoB, dnaJ and sodA gene sequences was also performed. In phylogenetic analysis inferred from tuf, dnaJ and rpoB gene sequence comparisons, strain 5402776(T) clustered with Staphylococcus pettenkoferi (93.7 %, 82.5 % and 89 % sequence similarity, respectively) and on phylogenetic analysis inferred from sodA gene sequence comparisons, it clustered with Staphylococcus chromogenes (82.8 %). On the basis of phenotypic and genotypic data, this isolate represents a novel species for which the name Staphylococcus massiliensis sp. nov. is proposed (type strain 5402776(T)=CCUG 55927(T)=CSUR P23(T)).'}, {'qas': [{'id': '7989533:Corynebacterium pseudogenitalium', 'question': 'Can Corynebacterium pseudogenitalium cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Genomic diversity among Corynebacterium jeikeium strains and comparison with biochemical characteristics and antimicrobial susceptibilities.\\nABSTRACT:Levels of DNA relatedness were determined by performing DNA-DNA hybridization experiments (S1 nuclease procedure) with 13 human isolates exhibiting various antimicrobial susceptibility patterns which had been identified as Corynebacterium jeikeium by classical tests and the API Coryne system and with reference strains of C. jeikeium and related taxa. Twelve of 13 isolates which formed three genomic groups showed between 22 and 75% relatedness with the type strain of C. jeikeium. One of these genomic groups included all the strains resistant to penicillin and gentamicin and is genomically related to the C. jeikeium type strain at the species level. In addition, the reference strain of \"Corynebacterium genitalium\" biotype II was found to belong to this genospecies and therefore can be considered as a synonym of C. jeikeium. In contrast, one isolate and the reference strains of \"Corynebacterium pseudogenitalium\" biotypes C-3 and C-4 which were assigned to C. jeikeium by the API Coryne system were less than 10% related to the C. jeikeium type strain. These nongenomically related strains can be differentiated from the jeikeium-related strains on the basis of positive acidification from fructose and growth under anaerobic conditions. Furthermore, these strains exhibited full susceptibility to penicillin whereas the strains related to the C. jeikeium type strain are resistant to or only moderately susceptible to penicillin. No genomic relationship was found between C. jeikeium-related strains and other lipophilic coryneforms, identified as Corynebacterium accolens or Corynebacterium group G or F. Our study demonstrates the necessity to perform the fructose fermentation test or respiratory-type test for the correct identification of lipophilic coryneforms as C. jeikeium. Although these strains show genomic diversity at the species level, in a practical aspect, biochemical properties as well as antimicrobial susceptibility may allow the classification of such isolates in this single taxon.'}, {'qas': [{'id': '30364596:Providencia stuartii', 'question': 'Can Providencia stuartii cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': \"TITLE:First report of carbapenem-resistant Providencia stuartii in Saudi Arabia.\\nABSTRACT:We present the case of 31-year-old man who developed hospital-acquired pneumonia in the intensive care unit. Pathogens were identified to be carbapenem-resistant isolates of Providencia stuartii and Klebsiella pneumoniae. The patient was treated with an extended infusion of double-dose meropenem (targeting the carbapenem-resistant P. stuartii) and colistin (targeting the carbapenem-resistant K. pneumoniae) for 2 weeks. The patient's disease responded well to the prescribed regimen; his chest X-ray became normal, and all other signs of infection subsided. To our knowledge, this is the first description of the emergence of carbapenem-resistant P. stuartii due to AmpC hyperproduction in Saudi Arabia.\"}, {'qas': [{'id': '3376617:Clostridium glycolicum', 'question': 'Can Clostridium glycolicum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:[Morphological changes in human embryonic lung fibroblasts caused by cytotoxins of various Clostridium species].\\nABSTRACT:A total of 243 strains of 35 Clostridium species were tested for cytotoxin production in cooked meat medium or liver broth within 48-72 h at 37 degrees C, using human embryonal lung fibroblasts in tissue-culture as indicator cells. Cytotoxin could be detected in the culture-filtrates of all toxigenic strains of C. chauvoei, C. difficile, C. histolyticum, C. novyi types A and B, C. septicum and C. tetani, but not in the atoxigenic ones. The cytotoxin of C. novyi correlated with alpha-toxin in the culture filtrate. All strains of C. perfringens and C. novyi D tested were not cytotoxic for lung fibroblasts despite their pathogenicity for guinea-pigs. Further cytotoxigenic strains were found among C. hastiforme, C. limosum, C. oceanicum, C. putrificum, C. ramosum, C. sordellii, C. sporogenes, and C. subterminale. The morphological changes in lung fibroblasts caused by the culture filtrates were characteristic and species-specific and corresponded with pathogenicity for guinea-pigs and/or mice. No cytotoxin was produced by C. absonum, C. barati, C. bifermentans, C. botulinum (atoxic), C. butyricum, C. cadaveris, C. carnis, C. clostridioforme, C. cochlearium, C. glycolicum, C. innocuum, C. malenominatum, C. mangenotii, C. paraputrificum, C. putrefaciens, C. rectum, C. tertium, and C. tyrobutyricum.'}, {'qas': [{'id': '602960:Clostridium sordellii', 'question': 'Can Clostridium sordellii cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Pneumonia and empyema caused by Clostridium sordellii.\\nABSTRACT:A case of pleuropulmonary infection caused by Clostridium sordellii is reported for the first time. The clinical presentation with acute onset resembling pulmonary infarction, the absence of toxicity, hemolysis and shock, and response to penicillin and drainage was similar to that of patients with pleuropulmonary infection caused by C. perfringens.'}, {'qas': [{'id': '22615839:Mycobacterium palustre', 'question': 'Can Mycobacterium palustre cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Failure to recognize nontuberculous mycobacteria leads to misdiagnosis of chronic pulmonary tuberculosis.\\nABSTRACT:\\n'}, {'qas': [{'id': '22605704:Corynebacterium jeikeium', 'question': 'Can Corynebacterium jeikeium cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Corynebacterium jeikeium bacteraemia and pulmonary infiltrates in a patient with acute myelogenous leukaemia.\\nABSTRACT:The authors used vancomycin (VCM) to treat Corynebacterium jeikeium bacteraemia and pulmonary infiltrates in a patient with acute myelogenous leukaemia. Pulmonary infiltrates are found in about one-third of patients with haematological pathologies accompanying C jeikeium bacteraemia. Although linezolid and daptomycin are antimicrobial alternatives for treating Corynebacterium infections, the authors believe that VCM is the best choice for immediate treatment success and to prevent pulmonary infiltrates from developing in the presence of bloodstream infection with Corynebacteria in patients with haematological malignancies.'}, {'qas': [{'id': '19643869:Pandoraea sputorum', 'question': 'Can Pandoraea sputorum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Pandoraea thiooxydans sp. nov., a facultatively chemolithotrophic, thiosulfate-oxidizing bacterium isolated from rhizosphere soils of sesame (Sesamum indicum L.).\\nABSTRACT:A facultatively chemolithoautotrophic, thiosulfate-oxidizing, Gram-negative, aerobic, motile, rod-shaped bacterial strain, designated ATSB16(T), was isolated from rhizosphere soils of sesame (Sesamum indicum L.). 16S rRNA gene sequence analysis demonstrated that this strain was closely related to Pandoraea pnomenusa LMG 18087(T) (96.7 % similarity), P. pulmonicola LMG 18016(T) (96.5 %), P. apista LMG 16407(T) (96.2 %), P. norimbergensis LMG 18379(T) (96.1 %) and P. sputorum LMG 18819(T) (96.0 %). Strain ATSB16(T) shared 96.0-96.4 % sequence similarity with four unnamed genomospecies of Pandoraea. The major cellular fatty acids of the strain ATSB16(T) were C(17 : 0) cyclo (33.0 %) and C(16 : 0) (30.6 %). Q-8 was the predominant respiratory quinone. The major polar lipids were phosphatidylmethylethanolamine, diphosphatidylglycerol, phosphatidylethanolamine and two unidentified aminophospholipids. Hydroxyputrescine and putrescine were the predominant polyamines. The genomic DNA G+C content of the strain was 64.0 mol%. On the basis of the results obtained from this study, strain ATSB16(T) represents a novel species of the genus Pandoraea, for which the name Pandoraea thiooxydans sp. nov. is proposed. The type strain is ATSB16(T) (=KACC 12757(T) =LMG 24779(T)).'}, {'qas': [{'id': '7869351:Clostridium paraputrificum', 'question': 'Can Clostridium paraputrificum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Clostridial infection in children.\\nABSTRACT:A survey of the isolation of Clostridium spp. from 1543 specimens sent to anaerobic microbiology laboratories revealed 113 isolates from 107 specimens (7.0% of all specimens) from 96 children. The isolates comprised 43 (38%) unidentified Clostridium spp., 37 (33%) C. perfringens, 13 (12%) C. ramosum, five (4%) C. innocuum, six (5%) C. botulinum, three (3%) C. difficile, two (2%) C. butyricum, and one isolate each of C. bifermentans, C. clostridiiforme, C. limosum and C. paraputrificum. Most clostridial isolates were from abscesses (38), peritonitis (26), bacteraemia (10), and chronic otitis media (7). Predisposing or underlying conditions were present in 31 (32%) cases. These were immunodeficiency (12), malignancy (9), diabetes (7), trauma (7), presence of a foreign body (6) and previous surgery (6). The clostridia were the only bacterial isolates in 14 (15%) cases; 82 (85%) cases had mixed infection. The species most commonly isolated with clostridia were anaerobic cocci (57); Bacteroides spp. (B. fragilis group) (50), Escherichia coli (22), pigmented Prevotella or Porphyromonas spp. (18) and Fusobacterium spp. (10). Most Bacteroides and Escherichia coli isolates with clostridia were from abdominal infections and skin and soft tissue infections adjacent to the rectal area; most pigmented Prevotella and Porphyromonas isolates were from oropharyngeal, pulmonary, and head and neck sites. Antimicrobial therapy was given to all patients, in conjunction with surgical drainage in 34 (35%). Only two patients died. These data illustrate the importance of Clostridium spp. in paediatric infections.'}, {'qas': [{'id': '27773758:Chlamydia abortus', 'question': 'Can Chlamydia abortus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Chlamydia related bacteria (Chlamydiales) in early pregnancy: community-based cohort study.\\nABSTRACT:OBJECTIVES: Serological case-control studies suggest that certain chlamydia-related bacteria (Chlamydiales) which cause cows to abort may do the same in humans. Chlamydiales include Waddlia chondrophila, Chlamydia abortus and Chlamydia trachomatis. Data on prevalence of Chlamydiales in pregnancy are sparse. Using stored urine samples from a carefully characterised cohort of 847 newly pregnant women recruited from 37 general practices in London, UK, we aimed to investigate the prevalence and types of Chlamydiales infections. We also explored possible associations with miscarriage or spontaneous preterm birth. \\n'}, {'qas': [{'id': '18473102:Leptospira santarosai', 'question': 'Can Leptospira santarosai cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Demographic and clinical features of leptospirosis: three-year experience in central Taiwan.\\nABSTRACT:BACKGROUND AND PURPOSE: Leptospirosis is a major cause of fever in subtropical and tropical areas. The clinical manifestations are protean, ranging from very mild and nonspecific symptoms to severe septic shock and death. This retrospective study investigated the demographic and clinical features of leptospirosis in central Taiwan over 3 years, with emphasis on pulmonary manifestations. \\n'}, {'qas': [{'id': '20331587:Rothia dentocariosa', 'question': 'Can Rothia dentocariosa cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Prevalence and diversity of microbes in the amniotic fluid, the fetal inflammatory response, and pregnancy outcome in women with preterm pre-labor rupture of membranes.\\nABSTRACT:PROBLEM: The role played by microbial invasion of the amniotic cavity (MIAC) in preterm pre-labor rupture of membranes (pPROM) is inadequately characterized, in part because of reliance on cultivation-based methods. METHOD OF STUDY: Amniotic fluid from 204 subjects with pPROM was analyzed with both cultivation and molecular methods in a retrospective cohort study. Broad-range and group-specific polymerase chain reaction (PCR) assays targeted small subunit ribosomal DNA (rDNA), or other gene sequences, from bacteria, fungi, and archaea. Results were correlated with measurements of host inflammation, as well as pregnancy and perinatal outcomes. \\n'}, {'qas': [{'id': '30556384:Herbaspirillum', 'question': 'Can Herbaspirillum cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Herbaspirillum huttiense pneumonia in a patient with essential thrombocythaemia.\\nABSTRACT:'}, {'qas': [{'id': '17137534:Arcanobacterium haemolyticum', 'question': 'Can Arcanobacterium haemolyticum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:Management of group A beta-hemolytic streptococcal pharyngotonsillitis in children.\\nABSTRACT:Acute pharyngotonsillitis is one of the most common infections encountered by pediatricians and family physicians. According to the US Vital Health Statistics report, acute pharyngotonsillitis is responsible for more than 6 million office visits each year by children younger than 15 years of age and an additional 1.8 million visits by adolescents and young adults aged 15 to 24 years. Most children with acute pharyngotonsillitis have symptoms that can be attributed to infection with a respiratory virus, such as adenovirus, influenza virus, parainfluenza virus, rhinovirus, and respiratory syncytial virus. However, in approximately 30% to 40% of cases, acute pharyngotonsillitis is of bacterial etiology. Group A beta-hemolytic streptococci (GABHS) are responsible for most bacterial cases of acute pharyngotonsillitis, although other pathogens, such as Neisseria gonorrhoeae, Arcanobacterium haemolyticum, Mycoplasma pneumoniae, and Chlamydia pneumoniae, may be the causative agents in sporadic cases. Pharyngotonsillitis caused by these latter pathogens can sometimes be distinguished from that caused by GABHS by considering the patient's medical history in concert with the clinical presentation. In some cases, acute pharyngotonsillitis may have an idiopathic etiology. An accurate diagnosis of GABHS infection is important because it is the only common form of acute pharyngotonsillitis for which antibiotic therapy is definitely indicated. Antibiotic therapy can shorten the clinical course of GABHS pharyngotonsillitis, reduce the rate of transmission, and prevent suppurative and nonsuppurative complications, such as peritonsillar abscess and acute rheumatic fever. Although the threat of rheumatic fever is much lower for children in the United States than in developing nations, preventing rheumatic fever and the spread of disease is the primary goal of antibiotic therapy in GABHS pharyngotonsillitis treatment and a cornerstone of practice guidelines.\"}, {'qas': [{'id': '10777042:Brachyspira pilosicoli', 'question': 'Can Brachyspira pilosicoli cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Detectability and prevalence of Brachyspira species in herds rearing health class feeder pigs in Finland.\\nABSTRACT:Faeces samples were taken three times at two-week intervals, from the farrowing units of four herds of known Brachyspira (formerly Serpulina) status and one of unknown Brachyspira status. Brachyspira hyodysenteriae, Brachyspira pilosicoli, Brachyspira intermedia and Brachyspira group III were isolated from the faecal samples from the weaners in the herds using either a maximum of 50 ppm of olaquindox or no feed additives. The detection rates were relatively consistent. However, B hyodysenteriae was not detected at one sampling in a known positive herd. The prevalence of Brachyspira species was also studied in feeder pigs originating from LSO 2000 health class farrowing units, comparable with specific pathogen-free herds. These farms were free from swine dysentery, sarcoptic mange, swine enzootic pneumonia and progressive atrophic rhinitis. Fifty of 428 herds were sampled once. B hyodysenteriae was not isolated from any of them, but B intermedia, B pilosicoli and Brachyspira group III were isolated from five, 14 and 37 of the herds, respectively. The detection of Brachyspira species did not relate to the prevalence of diarrhoea in the herds, as judged by the farmers. The herds using carbadox (40 to 50 ppm) had a lower prevalence of Brachyspira species than those using olaquindox (40 to 50 ppm).'}, {'qas': [{'id': '7276145:Eubacterium brachy', 'question': 'Can Eubacterium brachy cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Pleuropulmonary infection associated with Eubacterium brachy, a new species of Eubacterium.\\nABSTRACT:A new species of Eubacterium was isolated from a case of pleural effusion. A case history and description of the organism are given.'}, {'qas': [{'id': '33339327:Paracoccus yeei', 'question': 'Can Paracoccus yeei cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:Longitudinal Study of Viral and Bacterial Contamination of Hospital Pediatricians' Mobile Phones.\\nABSTRACT:Mobile phones (MPs) of healthcare workers (HCWs) may represent an important source of transmission of infectious agents. This longitudinal study documents the contamination of these tools. Ten MPs handled by senior pediatricians were sampled once a week during 23 weeks in three pediatric wards of the University Hospital of Saint-Etienne, France. Cultures were performed for bacteria and multiplex PCR assays for a panel of respiratory and enteric viruses. A questionnaire on hygiene habits regarding phoning and care was filled-in by pediatricians before and after the study. From a total of 230 samples, 145 (63%) were contaminated by at least one pathogen. The MPs from emergency departments were the most impacted. Viruses were detected in 179 samples; bacteria were isolated in 59 samples. Contamination increased during the winter epidemic peak. A cross-contamination by Paracoccus yeei between hands and MPs of different HCWs was demonstrated. The communication of the study results influenced the hygiene behaviors. This study highlights the contamination of MPs by pathogens that are resistant in the environment, and its sustainability along the winter season. The role of MPs as vectors of nosocomial infection needs to be better investigated.\"}, {'qas': [{'id': '26810057:Tannerella', 'question': 'Can Tannerella cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Oral and endotracheal tubes colonization by periodontal bacteria: a case-control ICU study.\\nABSTRACT:Periodontal infection is a possible risk factor for respiratory disorders; however, no studies have assessed the colonization of periodontal pathogens in endotracheal tubes (ET). This case-control study analyzed whether periodontal pathogens are able to colonize ET of dentate and edentulous patients in intensive care units (ICU) and whether oral and ET periodontal pathogen profiles have any correlation between these patients. We selected 18 dentate and 18 edentulous patients from 78 eligible ICU patients. Oral clinical examination including probing depth, clinical attachment level, gingival index , and plaque index was performed by a single examiner, followed by oral and ET sampling and processing by quantitative polymerase chain reaction (total bacterial load, Aggregatibacter actinomycetemcomitans, Porphyromonas gingivalis, and Tannerella forsythia). Data were statistically analyzed by Mann-Whitney U, two-way analysis of variance (p &lt; 0.05). Among dentate, there was no correlation between clinical parameters and ET bacterial levels. Both dentate and edentulous patients showed similar ET bacterial levels. Dentate patients showed no correlation between oral and ET bacterial levels, while edentulous patients showed positive correlations between oral and ET levels of A. actinomycetemcomitans, P. gingivalis, and T. forsythia. Periodontal pathogens can colonize ET and the oral cavity of ICU patients. Periodontal pathogen profiles tend to be similar between dentate and edentulous ICU patients. In ICU patients, oral cavity represents a source of ET contamination. Although accompanied by higher oral bacterial levels, teeth do not seem to influence ET bacterial profiles. '}, {'qas': [{'id': '27543089:Francisella guangzhouensis', 'question': 'Can Francisella guangzhouensis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Allofrancisella inopinata gen. nov., sp. nov. and Allofrancisella frigidaquae sp. nov., isolated from water-cooling systems, and transfer of Francisella guangzhouensis Qu et al. 2013 to the new genus as Allofrancisella guangzhouensis comb. nov.\\nABSTRACT:Five bacterial strains (SYSU YG23T, SYSU 10HL1970T, 10HP82-10, 10HL1938, 10HP457) isolated from water reservoirs of cooling systems were characterized using a polyphasic taxonomic approach. The isolates were Gram-stain-negative, strictly aerobic and non-motile. Growth was enhanced in the presence of l-cysteine. The major fatty acids (&gt;5 %) for the five strains were C10 : 0, C16 : 0, C16 : 0 3-OH, C18 : 0 3-OH and C18 : 1omega9c. Ubiquinone-8 was detected as the respiratory quinone while the polar lipid profile consisted of phosphatidylethanolamine, diphosphatidylglycerol, phosphatidylglycerol, phosphatidylcholine, three unidentified phospholipids, two unidentified aminophospholipids and three unidentified glycolipids. The strains shared 16S rRNA gene sequence similarities of 99.0-99.2 % with Francisella guangzhouensis 08HL01032T but less than 95.2 % with other members of the family Francisellaceae. The phylogenetic dendrogram based on 16S rRNA gene sequences showed that these strains form a separate cluster along with Francisella guangzhouensis. This cluster was also confirmed from multilocus-sequence typing based on sequences of the mdhA, rpoB and sdhA genes. Matrix-assisted laser desorption ionization time-of-flight MS analyses of the strains along with closely and distantly related Francisella strains also showed a distinct cluster for these strains. Based on the findings from the polyphasic taxonomy studies, the strains were considered to represent two novel species of a new genus for which the names Allofrancisella inopinata gen. nov., sp. nov. (type strain SYSU YG23T=KCTC 42968T=DSM 101834T) and Allofrancisella frigidaquae sp. nov. (type strain SYSU 10HL1970T=KCTC 42969T=DSM 101835T) are proposed. In addition, Francisella guangzhouensisQu et al. 2013 is proposed to be transferred to this new genus as Allofrancisella guangzhouensis comb. nov.'}, {'qas': [{'id': '27279269:Leminorella richardii', 'question': 'Can Leminorella richardii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Leminorella sepsis in very low birth weight neonate as cause of neonatal mortality.\\nABSTRACT:Leminorella is a member of Enterobacteriaceae family and was known previously as Enteric Group 57. Based upon genetic differences using DNA hybridization, it has three taxa: Leminorella grimontii, Leminorella richardii, and Leminorella sp. strain 3. The third one is similar biochemically to the L. grimontii strains. The generic name has been derived on the name of a French microbiologist, Leon Le Minor. The biochemical properties includes being facultative anaerobes, growth on sheep blood, TSI, and MacConkey agar; hydrogen sulfide producer, l-arabinose fermenter, and tyrosine hydrolyzer; and are negative for d-mannose fermentation, urea, and lipase. They usually infect in adulthood and result in urinary tract infection, surgical site infection, bacteremia, peritonitis, respiratory tract infection, and soft tissue infection. We report the first case of L. grimontii sepsis in a very low birth weight neonate that died because of neonatal sepsis.'}, {'qas': [{'id': '25955156:Providencia stuartii', 'question': 'Can Providencia stuartii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Understanding Voltage Gating of Providencia stuartii Porins at Atomic Level.\\nABSTRACT:Bacterial porins are water-filled beta-barrel channels that allow translocation of solutes across the outer membrane. They feature a constriction zone, contributed by the plunging of extracellular loop 3 (L3) into the channel lumen. Porins are generally in the open state, but undergo gating in response to external voltages. To date the underlying mechanism is unclear. Here we report results from molecular dynamics simulations on the two porins of Providenica stuartii, Omp-Pst1 and Omp-Pst2, which display distinct voltage sensitivities. Voltage gating was observed in Omp-Pst2, where the binding of cations in-between L3 and the barrel wall results in exposing a conserved aromatic residue in the channel lumen, thereby halting ion permeation. Comparison of Omp-Pst1 and Omp-Pst2 structures and trajectories suggests that their sensitivity to voltage is encoded in the hydrogen-bonding network anchoring L3 onto the barrel wall, as we observed that it is the strength of this network that governs the probability of cations binding behind L3. That Omp-Pst2 gating is observed only when ions flow against the electrostatic potential gradient of the channel furthermore suggests a possible role for this porin in the regulation of charge distribution across the outer membrane and bacterial homeostasis. '}, {'qas': [{'id': '24585374:Candidimonas', 'question': 'Can Candidimonas cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Eoetvoesia caeni gen. nov., sp. nov., isolated from an activated sludge system treating coke plant effluent.\\nABSTRACT:A novel bacterium, PB3-7B(T), was isolated on phenol-supplemented inorganic growth medium from a laboratory-scale wastewater purification system that treated coke plant effluent. 16S rRNA gene sequence analysis revealed that strain PB3-7B(T) belonged to the family Alcaligenaceae and showed the highest pairwise sequence similarity to Parapusillimonas granuli Ch07(T) (97.5%), Candidimonas bauzanensis BZ59(T) (97.3%) and Pusillimonas noertemannii BN9(T) (97.2%). Strain PB3-7B(T) was rod-shaped, motile and oxidase- and catalase-positive. The predominant fatty acids were C(16 : 0), C(17 : 0) cyclo, C(19 : 0) cyclo omega8c and C(14 : 0) 3-OH, and the major respiratory quinone was Q-8. The G+C content of the genomic DNA of strain PB3-7B(T) was 59.7 mol%. The novel bacterium can be distinguished from closely related type strains based on its urease activity and the capacity for assimilation of glycerol and amygdalin. On the basis of the phenotypic, chemotaxonomic and molecular data, strain PB3-7B(T) is considered to represent a new genus and species, for which the name Eoetvoesia caeni gen. nov., sp. nov. is proposed. The type strain of Eoetvoesia caeni is PB3-7B(T) ( = DSM 25520(T) = NCAIM B 02512(T)).'}, {'qas': [{'id': '22875690:Aggregatibacter aphrophilus', 'question': 'Can Aggregatibacter aphrophilus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:[Polymicrobial brain abscess in hereditary haemorrhagic telangiectasia  (Osler's disease)].\\nABSTRACT:HISTORY AND ADMISSION \\n\"}, {'qas': [{'id': '20339022:Aeromonas caviae', 'question': 'Can Aeromonas caviae cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Emergence of CTX-M-3, TEM-1 and a new plasmid-mediated MOX-4 AmpC in a multiresistant Aeromonas caviae isolate from a patient with pneumonia.\\nABSTRACT:Aeromonas species rarely cause pulmonary infection. We report, for what is believed to be the first time, a case of severe pneumonia in a cancer patient caused by Aeromonas caviae. Detailed microbiological investigation revealed that this isolate carried three beta-lactamase-encoding genes (encoding MOX-4, CTX-M-3 and TEM-1) conferring resistance to all beta-lactams but imipenem. The beta-lactamase with a pI of 9.0 was transferred by conjugation and associated with a 7.3 kb plasmid, as demonstrated by Southern blot hybridization. Analysis of the nucleotide and amino acid sequences showed a new ampC gene that was closely related to those encoding the MOX-1, MOX-2 and MOX-3 beta-lactamases. This new plasmid-mediated AmpC beta-lactamase from China was named MOX-4. This is believed to be the first report of MOX-4, CTX-M-3 and TEM-1 beta-lactamases in a multiresistant A. caviae.'}, {'qas': [{'id': '6528782:Pseudomonas stutzeri', 'question': 'Can Pseudomonas stutzeri cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:[Pleuropneumonia caused by Pseudomonas stutzeri].\\nABSTRACT:Biological characteristics and antibiotic sensitivity of P. stutzeri strain, isolated from a child with pleuropneumonia, are presented. Formation of rugous colonies, growth at 41 degrees C and in the presence of 6.5% of NaCl, the positive results of the oxidase and nitrate reductase tests, the negative signs of arginine hydrolase and lysine decarboxylase activity permit the identification of this Pseudomonas species. The isolated culture has proved to be sensitive to amino glycoside antibiotics, carbonicillin and polymyxin.'}, {'qas': [{'id': '31255535:Ehrlichia chaffeensis', 'question': 'Can Ehrlichia chaffeensis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Novel Ehrlichia sp. detected in Magellanic penguins (Sphenicus magellanicus) and in the seabird tick Ixodes uriae from Magdalena Island, southern Chile.\\nABSTRACT:Ehrlichia spp. are obligatory intracellular microorganisms that infect hematopoietic, endothelial or blood cells of mammals. Ticks are the only vectors of these agents in nature. To date, the role of birds and their associated ticks as reservoirs of ehrlichiae remains almost unexplored. In this study, we performed a molecular screening for bacteria of Anaplasmataceae family in samples of spleen (n = 72) and lung (n = 17), recovered from 72 carcasses of Magellanic penguins (Spheniscus magellanicus) in Brazil and Chile. One apparently unengorged tick (Ixodes uriae) was also collected while wandering upon one of the carcasses and submitted to molecular analyses as well. Through conventional and nested PCR protocols three genes (16S rRNA, dsb and groEL) of a new Ehrlichia sp. were partially characterized upon organs of three penguins and in the tick coming from Magdalena Island (Chile). First matches after BLASTn comparisons showed that our sequences share 99.4% (16S rRNA), 94.6% (groEL) and 79.3% (dsb) of identity with \"Candidatus Ehrlichia ornithorhynchi\", Ehrlichia sp. NS101 and Ehrlichia canis CCZ, respectively. Matrixes of genetic distance including other representatives of the Ehrlichia genus point a 99.4%, 94.0%, and 80.0% of identity with 16S rRNA, groEL and dsb genes from Ehrlichia sp. It25, Ehrlichia sp. NS101, and Ehrlichia chaffeensis San Louis, respectively. A Bayesian phylogenetic analysis of Anaplasmataceae 16S rRNA gene places the detected Ehrlichia sp. into a group with Ehrlichia sp. BAT and Ehrlichia sp. Natal. Although depicting different topologies, Bayesian unrooted phylogenetic trees constructed for groEL and dsb genes position this Ehrlichia sp. into well-supported branches, which reinforces the finding of a new taxon. For the moment, any pathogenic effect of this new Ehrlichia sp. on penguins is still unknown. However, this fact becomes important to assess from a conservation point of view since populations of Magellanic penguins are currently threatened and in an ongoing decrease.'}, {'qas': [{'id': '26654795:Acinetobacter haemolyticus', 'question': 'Can Acinetobacter haemolyticus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:New insights on the palate, lung, and nasal epithelium clone (PLUNC) proteins: Based on molecular and functional analysis of its homolog of YH1/SPLUNC1.\\nABSTRACT:The palate, lung, and nasal epithelium clone (PLUNC) proteins are intricate immune molecules and arisen questions from them are still unresolved. In order to identify the role of PLUNC family proteins, we had analyzed its homolog protein YH1/SPLUNC1, which highly expresses in nontumor nasopharyngeal epithelium while expresses weakly in nasopharyngeal carcinoma (NPC) tissues. It is found that YH1/SPLUNC1 protein expression level was higher in chronic normal nasopharynx inflammatory cells compared to NPC tissue cells. An approach to produce active YH1/SPLUNC1 protein had been established and recombinant YH1/SPLUNC1 protein could bind to all four Gram-positive and four Gram-negative bacteria we tested, and triggered the aggregation of those bacteria. Interestingly, YH1/SPLUNC1 protein has antimicrobial activity, and it can directly kill Escherichia coli and Acinetobacter haemolyticus. The microorganism cell showed morphological changes in cell wall such as cell damage and cytoplasmic leakage after exposure to YH1/SPLUNC1 protein, indicating that YH1/SPLUNC1 directly killed the microorganisms by cell wall permeabilization. All these results indicated that YH1/SPLUNC1 might be an important antimicrobial protein involved in innate immunity defense.'}, {'qas': [{'id': '7857794:Campylobacter gracilis', 'question': 'Can Campylobacter gracilis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Chemotaxonomic analyses of Bacteroides gracilis and Bacteroides ureolyticus and reclassification of B. gracilis as Campylobacter gracilis comb. nov.\\nABSTRACT:The cellular fatty acids, respiratory quinones, and proteins of the generically misnamed taxa Bacteroides gracilis and Bacteroides ureolyticus were analyzed and compared with the corresponding chemotaxonomic features of their closest relatives, the campylobacters. Our results and previously published data for genotypic and phenotypic characteristics were used in a polyphasic approach to reconsider the classification of these organisms. We transfer B. gracilis to the genus Campylobacter as Campylobacter gracilis comb. nov. B. ureolyticus can be considered a campylobacter on genotypic grounds; in contrast, the proteolytic metabolism and fatty acid components of this taxon exclude it from the genus Campylobacter. We prefer to consider this taxon a species incertae sedis pending the isolation and characterization of additional B. ureolyticus-like bacteria.'}, {'qas': [{'id': '33100541:Kocuria kristinae', 'question': 'Can Kocuria kristinae cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Kocuria kristinae pneumonia and bacteremia.\\nABSTRACT:Kocuria kristinae is an aerobic gram-positive bacterium that is part of the normal skin flora and not a common cause of infection. Here, we present the first reported case of community-acquired pneumonia and bacteremia caused by K. kristinae. The pneumonia was complicated by acute on chronic diastolic heart failure, resulting in acute hypoxic respiratory failure requiring intubation and mechanical ventilation.'}, {'qas': [{'id': '19860823:Nocardia', 'question': 'Can Nocardia cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Clinical and microbiological characteristics of Nocardiosis including those caused by emerging Nocardia species in Taiwan, 1998-2008.\\nABSTRACT:The genus of Nocardia is rapidly expanding and the species distribution varies with different geographical locations. We retrospectively reviewed the laboratory records of the bacteriology laboratory at National Taiwan University Hospital from January 1998 to June 2008 to identify patients with nocardiosis. During the study period, 164 isolates of Nocardia spp. were identified from 134 patients but only 113 patients had Nocardia infection. Nocardia brasiliensis (n = 54) was the most common pathogen, followed by N. asteroides (n = 36), N. farcinica (n = 7), N. flavorosea (n = 4), N. otitidiscaviarum (n = 3), N. nova (n = 3), N. beijingensis (n = 2) and one each of N. puris, N. jinanensis and N. takedensis. The major types of infection were cutaneous infection (56.6%), pulmonary infection (33.6%) and disseminated infection (7.1%). Eighty-eight patients received sulfonamide-containing antibiotic and eight of 100 patients with available data on outcomes died during the episode of nocardiosis. In conclusion, the clinical and microbiological manifestations of Nocardiosis vary with the different Nocardia species. Accurate identification of the species is crucial to make the diagnosis.'}, {'qas': [{'id': '26310805:Weeksella virosa', 'question': 'Can Weeksella virosa cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Algoriella xinjiangensis gen. nov., sp. nov., a new psychrotolerant bacterium of the family Flavobacteriaceae.\\nABSTRACT:An aerobic, Gram-stain negative, non-spore-forming and psychrotolerant bacterium, designated strain XJ109(T), was isolated from a sewage water sample collected from Xinjiang Uigur Autonomous Region, China. Phylogenetic analysis based on 16S rRNA gene sequence indicated that strain XJ109(T) represents a novel member of the family Flavobacteriaceae. The strain showed 95.5 % similarity with the 16S rRNA gene sequence of Empedobacter brevis LMG 4011(T), 95.4% with Chishuiella changwenlii BY4(T), 95.3% with Empedobacter falsenii NF 993(T) and 92.3% with Weeksella virosa DSM 16922(T). Strain XJ109(T) showed the common phenotypic and chemotaxonomic characteristics of the family Flavobacteriaceae, containing menaquinone-6 (MK-6) as the predominant respiratory quinone and iso-C17:0 3OH and iso-C15:0 as the major fatty acids. The polar lipid profile consisted of phosphatidylethanolamine, one unidentified phospholipid and two unidentified lipids. The genomic DNA G+C content was 38.0 mol%. Strain XJ109(T) was positive for catalase and oxidase activities, and it was observed to grow at 4-30  C (optimal 16-20  C), pH 6.5-10.0 (optimal 7.0-7.5) and in media containing 0-2.0% (w/v) NaCl (optimal 0.5 %). On the basis of the polyphasic evidence presented, strain XJ109(T) is considered to represent a novel genus and species of the family Flavobacteriaceae, for which the name Algoriella xinjiangensis gen. nov., sp. nov. is proposed. The type strain is XJ109(T) (=CGMCC 1.10229(T)=JCM 16590(T)).'}, {'qas': [{'id': '2402972:Acholeplasma laidlawii', 'question': 'Can Acholeplasma laidlawii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:[Examination of cattle with respiratory diseases for Mycoplasma and bacterial bronchopneumonia agents].\\nABSTRACT:A total of 247 mycoplasma strains was isolated from 435 lungs, tracheobronchial secretions and nasal swabs originating from cattle with symptoms of bronchopneumonia. Mycoplasma (M.) bovis was found 89 times (36%) and was the most common mycoplasma species in the lungs. M. bovirhinis, M. bovigenitalium, M. spec. and Acholeplasma (A.) laidlawii were isolated 158 times (64%). Among these mycoplasmas M. bovirhinis was the most widespread species (114 isolations). In 55 cases (62%) M. bovis was associated with Pasteurella or Actinomyces (A.) pyogenes. The other mycoplasma species were found in 67 cases (42%) together with these bacteria. Without mycoplasmas Pasteurella and A. pyogenes occurred in 33 of the probes investigated (21%). Beside mycoplasmas Haemophilus (H.) somnus was isolated from 16 of 162 tracheobronchial secretions investigated. The results confirm earlier suppositions that in most of the cases bronchopneumonia of cattle is a multifactorial event, frequently associated with mycoplasmas--especially M. bovis.'}, {'qas': [{'id': '24951305:Clostridium innocuum', 'question': 'Can Clostridium innocuum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:Gut microbiome in sudden infant death syndrome (SIDS) differs from that in healthy comparison babies and offers an explanation for the risk factor of prone position.\\nABSTRACT:The role of bacteria in the causation of sudden infant death syndrome (SIDS) is gaining acceptance. Mainstream research favouring respiratory compromise has failed to provide a plausible pathogenetic mechanism despite many years of investigation and thousands of research papers. Bacterial colonisation of the colon of the human infant is influenced by many factors including age, mode of delivery, diet, environment, and antibiotic exposure. The gut microbiome influences development of the immune system. The gut microflora could be important in protection against the bacteria and/or their toxins purportedly involved in SIDS pathogenesis. The aim was to perform a preliminary investigation of the gut microflora in sudden infant death syndrome (SIDS) compared with live comparison babies. The intestinal contents from 52 SIDS, and 102 faecal samples from age-matched live comparison infants were screened by PCR to target 16s RNA genes of Clostridium innocuum, Cl. Perfringens, Cl. difficile, Bacteroides thetaiotaomicron and Staphylococcus aureus. Gut colonisation of the babies with these bacteria was analysed in relation to age, gender and type of feeding; and for SIDS babies sleeping position. Cl. difficile, Cl. innocuum and B. thetaiotaomicron were significantly associated with SIDS with 25%, 46% and 30% of cases PCR positive for these respective bacteria compared with only 6%, 23% and 8.8% respectively in the comparison group. SIDS babies had dual colonisation by both Cl. perfringens and Cl. difficile significantly more often than comparison babies and also with triple colonisation by Cl. perfringens, Cl. difficile and Cl. innocuum. SIDS babies were more often colonised by S. aureus than comparison babies. In addition, SIDS babies found prone were significantly more likely to be colonised by S. aureus than for other positions recorded (OR =  ; CI = 2 04 -  ). No significant differences between breast and bottle-fed SIDS babies was observed in regard to each clostridial bacterium, or S. aureus, however Cl. innocuum was found to be significantly associated with formula feeding in the comparison cohort. Comparison of breast and formula feeding of SIDS babies with live comparison babies revealed significant differences with regards to some of the clostridial bacteria. Age-specific differences in gut bacterial microbiome were observed in both SIDS and comparison healthy babies. This study gives an insight into differences in the gut bacterial microbiome of SIDS babies compared with healthy babies. These differences could be important in contributing to a baby's susceptibility to infection and therefore to SIDS. The association of S. aureus colonisation with prone sleep position supports the hypothesis that prone sleep position could increase the risk of ingestion/inhalation of bacteria contaminating the sleeping surface and could account for the increased risk of SIDS in babies who are put to sleep prone. The study provides impetus for broader studies into the gut microbiome of babies and could lead to effective approaches to SIDS prevention.\"}, {'qas': [{'id': '28268195:Rickettsia peacockii', 'question': 'Can Rickettsia peacockii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:'Candidatus Rickettsia nicoyana': A novel Rickettsia species isolated from Ornithodoros knoxjonesi in Costa Rica.\\nABSTRACT:Rickettsiae are intracellular bacteria commonly associated with hematophagous arthropods. Most of them have been described in hard ticks, but some have been found in soft ticks. Here we report the detection and isolation of a new Rickettsia from Ornithodoros knoxjonesi larvae collected from Balantiopteryx plicata (Emballonuridae) in Nicoya, Costa Rica. Two ticks were processed to detect Rickettsia spp. genes gltA, ompA, ompB, and htrA by PCR. Part of the macerate was also inoculated into Vero E6 and C6/36 cell lines, and cells were evaluated by Gimenez stain, indirect immunofluorescence assay (IFA), and PCR. Both ticks were positive by PCR and rickettsial growth was successful in Vero E6 cells. Amplification and sequencing of near full length rrs, gltA, sca4 genes, and fragments of ompA and ompB showed that the Rickettsia sp. was different from described species. The highest homologies were with 'Candidatus Rickettsia wissemanii' and Rickettsia peacockii: 99.70% (1321/1325) with both sequences for rrs, 99.58% (1172/1177) and 99.76% (1246/1249) for gltA, 99.26% with both sequences (2948/2970 and 2957/2979) for sca4, 98.78% (485/491) and 98.39% (2069/2115) for ompA, and 98.58 (1453/1474) and 98.92% (1459/1475) for ompB; respectively. Bat blood, spleen, liver, and lung samples analyzed for Rickettsia detection were negative. Results demonstrate that the Rickettsia isolated from O. knoxjonesi is probably an undescribed species that belongs to the spotted fever group, for which 'Candidatus Rickettsia nicoyana' is proposed. Considering that B. plicata inhabits areas where contact with humans may occur and that human parasitism by Ornithodoros has been reported in the country, it will be important to continue with the characterization of this species and its pathogenic potential.\"}, {'qas': [{'id': '23231490:Eubacterium limosum', 'question': 'Can Eubacterium limosum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Nosocomial infections in leukemic and solid-tumor cancer patients: distribution, outcome and microbial spectrum of anaerobes.\\nABSTRACT:AIMS: Nosocomial infections cause significant morbidity and mortality in cancer patients. As a result of their debilitated immune system, cancer patients are likely candidates for colonization with anaerobes. We sought to compare the distribution of nosocomial infections in neutropenic and non-neutropenic cancer patients and to calculate the associated mortality rates. MATERIAL &amp; \\n'}, {'qas': [{'id': '29559113:Actinomyces graevenitzii', 'question': 'Can Actinomyces graevenitzii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Meta-analysis of the lung microbiota in pulmonary tuberculosis.\\nABSTRACT:The lung microbiota has received less attention compared to other body sites, in part because its study carries special technological difficulties related to obtaining reliable samples as compared to other body niches. The limited number of studies on the sputum microbiota on TB patients and controls available so far have reported inconsistent, and sometimes, contradictory results. Aiming to clarify if changes in the lung microbiota composition are associated with pulmonary TB, we performed a meta-analysis of available data on microbiota of the lower respiratory tract in TB patients and healthy controls. Re-processing next generation sequencing data under uniform parameters and utilizing state-of-the-art bioinformatics analysis, we obtained distinct clusterings of microbiota between TB cases vs. controls across multiple studies. We identified Tumebacillus ginsengisoli, Propionibacterium acnes, Haemophilus parahaemolyticus as differentially abundant species signature in healthy controls while Caulobacter henricii, Actinomyces graevenitzii, Rothia mucilaginosa, in addition to Mycobacterium tuberculosis as differentially abundant species signature in TB cases, and described R. mucilaginosa as the anchoring species in a network of bacteria co-occurring with Mycobacterium tuberculosis (Mtb) infection.'}, {'qas': [{'id': '33048281:Pandoraea pulmonicola', 'question': 'Can Pandoraea pulmonicola cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:A chronic strain of the cystic fibrosis pathogen Pandoraea pulmonicola expresses a heterogenous hypo-acylated lipid A.\\nABSTRACT:Pandoraea sp. is an emerging Gram-negative pathogen in cystic fibrosis causing severe and persistent inflammation and damage of the lungs. The molecular mechanisms underlying the high pathogenicity of Pandoraea species are still largely unknown. As Gram-negatives, Pandoraea sp. express lipopolysaccharides (LPS) whose recognition by the host immune system triggers an inflammatory response aimed at the bacterial eradication from the infected tissues. The degree of the inflammatory response strongly relies on the fine structure of the LPS and, in particular, of its glycolipid moiety, i.e. the lipid A. Here we report the structure of the lipid A isolated from the LPS of a chronic strain of P. pulmonicola (RL 8228), one of the most virulent identified so far among the Pandoraea species. Our data demonstrated that the examined chronic strain produces a smooth-type LPS with a complex mixture of hypoacylated lipid A species displaying, among other uncommon characteristics, the 2-hydroxylation of some of the acyl chains and the substitution by an additional glucosamine on one or both the phosphate groups.'}, {'qas': [{'id': '31304018:Nocardia exalbida', 'question': 'Can Nocardia exalbida cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Pulmonary nocardiosis caused by Nocardia exalbida mimicking lung cancer.\\nABSTRACT:Nocardiosis is an uncommon infection caused by Nocardia species, but it can often occur in immunocompromised patients. As computed tomography (CT) findings of pulmonary nocardiosis, consolidation, masses, and nodules are often found, but lymph node enlargement is infrequent. Nocardia exalbida was first isolated in 2006, and there are still few reports on this microorganism. We report a case of pulmonary nocardiosis caused by N. exalbida presenting as a mass and lymph node enlargement that mimicked lung cancer. A 76-year-old man was referred and admitted to our hospital because of persistent cough, sputum production, and chest discomfort. Chest CT showed a mass in the superior segment of the left upper lobe and mediastinal lymph node enlargement. After we performed bronchoscopies, Nocardia species was isolated in a cultured specimen and identified as N. exalbida. This case required differentiation from lung cancer and was difficult to diagnose.'}, {'qas': [{'id': '32874333:Bacillus coagulans', 'question': 'Can Bacillus coagulans cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Safety and efficacy of TechnoSpore  (Bacillus coagulans DSM 32016) for piglets, other growing Suidae, chickens for fattening, other poultry for fattening and ornamental birds.\\nABSTRACT:Following a request from the European Commission, the Panel on Additives and Products or Substances used in Animal Feed (FEEDAP) was asked to deliver a scientific opinion on the safety and efficacy of TechnoSpore  (Bacillus coagulans DSM 32016), when used as a zootechnical additive for piglets (suckling and weaned), other growing Suidae, chickens for fattening, other poultry for fattening and ornamental birds. The bacterial species present in the additive is considered suitable for the qualified presumption of safety approach to safety assessment. The identity of the active agent was established and the lack of toxigenic potential confirmed. B. coagulans DSM 32016 did not show resistance to antibiotics of human and veterinary importance, and therefore, was presumed safe for the target species, consumers of products derived from animals Fed the additive and the environment. Since the other components of the additive did not give rise to concerns, TechnoSpore  was also considered safe for the target species, consumer and the environment. The additive is not a skin/eye irritant or a skin sensitiser but should be considered a respiratory sensitiser. TechnoSpore  showed the potential to be efficacious as a zootechnical additive in weaned piglets and chickens for fattening at 1 x 109 CFU/kg complete feed. This conclusion was extended to suckling piglets and extrapolated to other growing Suidae at the same physiological stage and to other birds for fattening and ornamental birds at the same use level. B. coagulans DSM 32016 included in Technospore  is compatible with halofuginone and diclazuril. The Panel could not conclude on the compatibility of the additive with monensin sodium, decoquinate, robenidine hydrochloride, lasalocid sodium, narasin, salinomycin sodium, maduramicin ammonium, nicarbazin and narasin/nicarbazin.'}, {'qas': [{'id': '12448677:Burkholderia ubonensis', 'question': 'Can Burkholderia ubonensis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Evaluation of species-specific recA-based PCR tests for genomovar level identification within the Burkholderia cepacia complex.\\nABSTRACT:The Burkholderia cepacia complex presently comprises nine genomovars: B. cepacia (genomovar I), B. multivorans (genomovar II), B. cepacia genomovar III, B. stabilis (genomovar IV), B. vietnamiensis (genomovar V), B. cepacia genomovar VI, B. ambifaria (genomovar VII), B. anthina (genomovar VIII) and B. pyrrocinia (genomovar IX). Strains of each genomovar can colonise the respiratory tract of cystic fibrosis (CF) patients. However, the majority of infections in CF patients are caused by B. multivorans and B. cepacia genomovar III isolates. Accurate genomovar-level identification is best achieved through a polyphasic approach combining phenotypic and genotypic analyses. In the present study, the sensitivity and specificity of recA-based genomovar specific primer pairs were evaluated with a collection of 508 B. cepacia complex isolates representing all nine genomovars. The assays for the identification of B. multivorans (sensitivity and specificity, 100%), B. cepacia genomovar III (sensitivity, 92%; specificity, 100%), and B. ambifaria (sensitivity and specificity, 100%) were the most efficient. However, the B. cepacia genomovar I assay lacked sensitivity (72%) and cross-reacted with all B. pyrrocinia isolates examined. Several new recA RFLP types were also revealed within the B. cepacia complex. One of these profiles was shared by a clinical and an environmental B. cepacia-like isolate and by the B. ubonensis type strain. The latter organism is a recently described soil bacterium. Its relationship to the various B. cepacia complex genomovars needs further study.'}, {'qas': [{'id': '32988729:Mycobacterium europaeum', 'question': 'Can Mycobacterium europaeum cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Mycobacterium europaeum lung disease in an immunocompetent patient without underlying lung disease.\\nABSTRACT:Mycobacterium europaeum (M. europaeum) was recently identified as a nontuberculous mycobacterium belonging to the Mycobacterium simiae complex. There have been only a few reported cases of M. europaeum lung disease, all of which occurred in patients with immunodeficiency or prior lung disease. We herein report a case of M. europaeum lung disease in an otherwise healthy Japanese individual. A 70-year-old woman who had no apparent immunodeficiency or medical history was diagnosed with M. europaeum lung disease by multiple positive sputum cultures. The patient was successfully treated with clarithromycin, rifampin, ethambutol, and amikacin. This report is the first case of M. europaeum lung disease occurring in an individual without predisposing risk factors.'}, {'qas': [{'id': '21481624:Nocardia veterana', 'question': 'Can Nocardia veterana cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Bacteremic pneumonia caused by Nocardia veterana in an HIV-infected patient.\\nABSTRACT:Disseminated Nocardia veterana infection has rarely been reported. We describe the first reported case of N. veterana bacteremic pneumonia in an HIV-infected patient. The isolate was confirmed by 16S rRNA sequencing analysis. The patient initially responded well to trimethoprim-sulfamethoxazole treatment (minimum inhibitory concentration 0.25 mug/ml), but died of ventilator-associated pneumonia.'}, {'qas': [{'id': '29805463:Proteus hauser', 'question': 'Can Proteus hauser cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Comparison of virulence genes in Proteus species isolated from human and pet turtle.\\nABSTRACT:The current study was aimed to investigate the prevalence of ureC, rsbA, zapA and mrpA virulence genes using polymerase chain reaction (PCR) in Proteus spp. isolated from 5 commercially popular species of pet turtles and comparison of the mrpA gene sequences of Proteus mirabilis isolates with human clinical isolates. A total of 24 isolates in pet turtles were identified, comprised of P. mirabilis (15), Proteus vulgaris (7) and Proteus hauseri (2). The prevalence of ureC, rsbA, zapA and mrpA genes among all identified Proteus spp. isolates were 91.7%, 50%, 45.8% and 45.8%, respectively. The average percentage similarities of mrpA gene sequence of pet turtle P. mirabilis isolates to human urinary and respiratory isolates were 96.35% and 94.85%, respectively. The prevalence of virulence genes and high similarity of mrpA gene sequences between pet turtles and human P. mirabilis isolates revealed that though pet turtles are healthy, these animals may pose a potential risk of urinary and respiratory infections to humans.'}, {'qas': [{'id': '27617005:Pseudomonas mosselii', 'question': 'Can Pseudomonas mosselii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Pseudopyronine B: A Potent Antimicrobial and Anticancer Molecule Isolated from a Pseudomonas mosselii.\\nABSTRACT:In continuation of our search for new bioactive compounds from soil microbes, a fluorescent Pseudomonas strain isolated from paddy field soil of Kuttanad, Kerala, India was screened for the production of bioactive secondary metabolites. This strain was identified as Pseudomonas mosselii through 16S rDNA gene sequencing followed by BLAST analysis and the bioactive metabolites produced were purified by column chromatography (silica gel) and a pure bioactive secondary metabolite was isolated. This bioactive compound was identified as Pseudopyronine B by NMR and HR-ESI-MS. Pseudopyronine B recorded significant antimicrobial activity especially against Gram-positive bacteria and agriculturally important fungi. MTT assay was used for finding cell proliferation inhibition, and Pseudopyronine B recorded significant antitumor activity against non-small cell lung cancer cell (A549), and mouse melanoma cell (B16F10). The preliminary MTT assay results revealed that Pseudopyronine B recorded both dose- and time-dependent inhibition of the growth of test cancer cell lines. Pseudopyronine B induced apoptotic cell death in cancer cells as evidenced by Acridine orange/ethidium bromide and Hoechst staining, and this was further confirmed by flow cytometry analysis using Annexin V. Cell cycle analysis also supports apoptosis by inducing G2/M accumulation in both A549 and B16F10 cells. Pseudopyronine B treated cells recorded significant up-regulation of caspase 3 activity. Moreover, this compound recorded immunomodulatory activity by enhancing the proliferation of lymphocytes. The production of Pseudopyronine B by P. mosselii and its anticancer activity in A549 and B16F10 cell lines is reported here for the first time. The present study has a substantial influence on the information of Pseudopyronine B from P. mosselii as potential sources of novel drug molecule for the pharmaceutical companies, especially as potent antimicrobial and anticancer agent. '}, {'qas': [{'id': '26023207:Sphingobacterium multivorum', 'question': 'Can Sphingobacterium multivorum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Sphingobacterium mucilaginosum sp. nov., isolated from rhizosphere soil of a rose.\\nABSTRACT:A Gram-stain-negative, strictly aerobic, motile, short-rod-shaped bacterium, designated strain THG-SQA8(T), was isolated from rhizosphere soil of rose in PR China. Strain THG-SQA8(T) was closely related to members of the genus Sphingobacterium, showed the highest sequence similarities with Sphingobacterium multivorum KACC 14105(T) (98.0%) and Sphingobacterium ginsenosidimutans KACC 14526(T) (97.4%). DNA-DNA hybridization showed values of 35.2 +- 0.9% and 8.8 +- 0.3% DNA reassociation with S. multivorum KACC 14105(T) and S. ginsenosidimutans KACC 14526(T), respectively. Chemotaxonomic data revealed that strain THG-SQA8(T) possesses menaquinone-7 as the only respiratory quinone, and summed feature 3 (C16 : 1omega7c and/or C16 : 1omega6c), iso-C1 : 0 and C16 : 0 as the major fatty acids. The major polar lipid was phosphatidylethanolamine. The DNA G+C content was 40.7 mol%. These data corroborated the affiliation of strain THG-SQA8(T) to the genus Sphingobacterium. Thus, the isolate represents a novel species, for which the name Sphingobacterium mucilaginosum sp. nov. is proposed, with THG-SQA8(T) as the type strain ( = CCTCC AB 2014317(T) = KCTC 42503(T)).'}, {'qas': [{'id': '27440847:Aerococcus urinae', 'question': 'Can Aerococcus urinae cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Aerococcus urinae, a rare cause of infective endocarditis.\\nABSTRACT:We present the case of an elderly male who was initially seen in our hospital for a urinary tract infection that was treated with oral ciprofloxacin. He was admitted 2 weeks later with altered mental status and fever, and was found to have bacteraemia with Aerococcus urinae Owing to altered mental status a brain MRI was performed which showed evidence of embolic stroke. Following this, a transesophageal echocardiogram showed severe mitral regurgitation and a vegetation &gt;1 cm involving the mitral valve with associated destruction of posterior valve leaflets. The patient was started on antibiotics intravenous penicillin G and intravenous gentamicin for a total duration of 6 weeks. He underwent mitral valve replacement on day 4 of hospitalisation. The postoperative course was complicated by ventilator-dependent respiratory failure, requiring tracheostomy and eventual transfer to a skilled nursing facility. Unfortunately, he died after 2 weeks of stay at the facility. '}, {'qas': [{'id': '23837368:Brucella ceti', 'question': 'Can Brucella ceti cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Wildlife reservoirs of brucellosis: Brucella in aquatic environments.\\nABSTRACT:Neurobrucellosis and osteomyelitis are common pathologies of humans and cetaceans infected with Brucella ceti or B. pinnipedialis. Currently, 53 species of marine mammal are known to show seropositivity for brucellae, and B. ceti or B. pinnipedialis have been isolated or identified in polymerase chain reaction assays in 18 of these species. Brucellae have also been isolated from fish and identified in lungworm parasites of pinnipeds and cetaceans. Despite these circumstances, there are no local or global requirements for monitoring brucellosis in marine mammals handled for multiple purposes such as capture, therapy, rehabilitation, investigation, slaughter or consumption. Since brucellosis is a zoonosis and may be a source of infection to other animals, international standards for Brucella in potentially infected marine mammals are necessary.'}, {'qas': [{'id': '15634981:Pasteurella dagmatis', 'question': 'Can Pasteurella dagmatis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Characterization of sucrose-negative Pasteurella multocida variants, including isolates from large-cat bite wounds.\\nABSTRACT:To validate the identification of Pasteurella multocida-like bacteria negative for acid formation from sucrose, including isolates from bite wounds caused by large cats, 17 strains were phenotypically and genotypically characterized. Phylogenetic analysis of partially sequenced rpoB and infB genes showed the monophyly of the strains characterized and the reference strains of P. multocida. The sucrose-negative strains formed two groups, one related to reference strains of P. multocida and the other related to a separate species-like group (taxon 45 of Bisgaard). DNA-DNA hybridization further documented the species-like nature of this group. Ribotyping showed the heterogeneity of all strains except four strains that shared the same ribotype and that were isolated from bovine lungs. Phylogenetic analysis by 16S rRNA sequence comparison showed the monophyly of the strains characterized and the reference strains of P. multocida. Two strains isolated from leopard bite wounds were related to the type strain of P. dagmatis; however, they represented a new taxon (taxon 46 of Bisgaard), in accordance with their distinct phenotypic and genotypic identifications. The present study documents that sucrose-negative strains of P. multocida-like bacteria belong to two genotypically distinct groups. The study further confirms the phenotypic heterogeneity of P. multocida strains and documents two new species-like taxa of Pasteurella related to P. multocida. Until diagnostic tools have been further elaborated, special care should be taken in the identification of Pasteurella-like bacteria isolated from bite wounds caused by large cats. The evidence of phenotypic and genotypic divergence calls for the further development of PCR tests and DNA sequencing to document doubtful isolates.'}, {'qas': [{'id': '15329532:Legionella pneumophila', 'question': 'Can Legionella pneumophila cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': \"TITLE:Microbiological investigations on a nosocomial case of Legionella pneumophila pneumonia associated with water birth and review of neonatal cases.\\nABSTRACT:A case of Legionella pneumophila 1 pneumonia, confirmed by positive serology and urinary antigen, occurred in a 7-day old neonate after water birth in hospital. As respiratory samples were not available for culture, further microbiological investigations were performed in neonate and environment, in order to recognize the source of infection. The hospital water supply was contaminated by L. pneumophila 1 strains (300-2000 cfu/L) of two monoclonal subtypes of Pontiac subgroup. L. spiritensis (10-225 cfu/L) was isolated from cold tap water of the patient's home. PCR from tap and humidifiers water of the patient's home was positive for Legionella spp, but not for L. pneumophila. Because L. pneumophila 1, responsible of child infection, was only isolated from the hospital pool water for waterbirthing, we conclude that the infant acquired the nosocomial legionellosis by prolonged delivery in contaminated water, perhaps by aspiration. Infection control measures for waterbirthing are highly recommended. A review of neonatal case of legionellosis is also presented. As this rare infection may have a high fatality rate if unrecognized, pediatricians should be aware of the possibility of the legionellosis in newborns.\"}, {'qas': [{'id': '32561458:Vibrio alginolyticus', 'question': 'Can Vibrio alginolyticus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Injection of an extract of fresh cacao pod husks into Litopenaeus vannamei upregulates immune responses via innate immune signaling pathways.\\nABSTRACT:Immunostimulation is a novel method and a promising development in aquaculture. Products derived from plants exhibit various biological activities. In this study, the hot-water extract isolated from fresh cacao (Theobroma cacao) pod husks (CPHs) was administered by injection to evaluate cellular signaling pathways of innate immunity, the immunostimulating potential, disease resistance, and hypothermal tolerance of white shrimp, Litopenaeus vannamei. Results showed significant increases in the total hemocyte count, semigranular cells, granular cells, phenoloxidase activity, and respiratory bursts (RBs) of hemocytes per unit of hemolymph at 1 day, and in phagocytic activity toward and the clearance efficiency of Vibrio alginolyticus at 1-3 days after shrimp were injected with fresh CPH extract at 40 mug shrimp-1. However, only RBs per hemocyte had significantly decreased at 1 day after the injection. All immune parameters had returned to control levels by 3 days after receiving fresh CPH extract except for RBs, phagocytic activity, and the clearance efficiency, which had returned to control values by 7 days. Furthermore, at 1 day after the injection, the peroxinectin, prophenoloxidase (proPO) II, toll-like receptors (TLR) 3, signal transducer and activator of transcription (STAT), and crustin in shrimp receiving fresh CPH extract at 20 mug shrimp-1, and the lipopolysaccharide and beta-1,3-glucan-binding protein, proPO II, TLR1, and STAT in shrimp receiving fresh CPH extract at 40 mug shrimp-1 were significantly higher than those of shrimp receiving saline. After injecting fresh CPH extract at 10-40 mug shrimp-1 and 40 mug shrimp-1, the mortality of shrimp challenged with V. alginolyticus and exposed to 14  C for 96 h significantly decreased, respectively. These results suggest that fresh CPH extract can be used as an immunostimulant and a physiological regulator for shrimp through injection administration to enhance immunological and physiological responses, which can elevate the resistance against V. alginolyticus and tolerance against hypothermal stress in L. vannamei.'}, {'qas': [{'id': '19801389:Bacteroides ureolyticus', 'question': 'Can Bacteroides ureolyticus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Reclassification of Bacteroides ureolyticus as Campylobacter ureolyticus comb. nov., and emended description of the genus Campylobacter.\\nABSTRACT:The protein profiles, genomic amplified fragment length polymorphism patterns and 16S rRNA and cpn60 gene sequences of a diverse collection of 26 Bacteroides ureolyticus strains, along with published data on their DNA base, respiratory quinone and cellular fatty acid compositions, were used to reassess the taxonomy of this bacterial species. The results demonstrate that this organism is most appropriately allocated in the genus Campylobacter. The presence of much higher amounts of 18 : 1omega7c in its cellular fatty acid profile and its ability to digest gelatin and casein are the characteristics that differentiate it from present species of the genus Campylobacter. Therefore we propose to reclassify this species incertae sedis into the genus Campylobacter as Campylobacter ureolyticus with strain LMG 6451(T) (=CCUG 7319(T) =NCTC 10941(T)) as the type strain.'}, {'qas': [{'id': '30083836:Klebsiella singaporensis', 'question': 'Can Klebsiella singaporensis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Isolation and characterization of chromium(VI)-reducing bacteria from tannery effluents and solid wastes.\\nABSTRACT:In the present investigation, five novel Cr(VI) reducing bacteria were isolated from tannery effluents and solid wastes and identified as Kosakonia cowanii MKPF2, Klebsiella pneumonia MKPF5, Acinetobacter gerneri MKPF7, Klebsiella variicola MKPF8 and Serratia marcescens MKPF12 by 16S rDNA gene sequence analysis. The maximum tolerance concentration of Cr(VI) as K2Cr2O7 of the bacterial isolates was varying up to 2000 mg/L. Among the investigated bacterial isolates, A. gerneri MKPF7 was best in terms of reduction rate. The optimum temperatures for growth and Cr(VI) reduction by the bacterial isolates were 35 and 40  C, respectively except A. gerneri MKPF7 which grew and reduced Cr(VI) optimally at 40  C. The optimum pH for growth and Cr(VI) reduction by K. cowanii MKPF2, A. gerneri MKPF7 and S. marcescens MKPF12 was 7.0 whereas the optimum pH for growth and Cr(VI) reduction by K. pneumoniae MKPF5 and K. variicola MKPF8 were 7.0, 8.0 and 6.0, 7.0, respectively. All the bacterial isolates showed maximum tolerance against Ni2+ and Zn2+ whereas minimum tolerance was observed against Hg2+ and Cd2+. The bacteria isolated in the present study thus can be used as eco-friendly biological expedients for the remediation and detoxification of Cr(VI) from the contaminated environments.'}, {'qas': [{'id': '31576208:Actinomyces odontolyticus', 'question': 'Can Actinomyces odontolyticus cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:A pulmonary infection by Actinomyces odontolyticus and Veillonella atypica in an immunocompetent patient with dental caries.\\nABSTRACT:Actinomycosis is a rare, chronic granulomatous infection, frequently associated with immunocompromised states, but it can also affect healthy people. Here, we report a case of a pulmonary infection by Actinomyces odontolitycus and Veillonella atypica due to a dental caries in an immunocompetent 65-year-old man patient.'}, {'qas': [{'id': '29884324:Mycobacterium riyadhense', 'question': 'Can Mycobacterium riyadhense cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Mycobacterium riyadhense as the opportunistic infection that lead to HIV diagnosis: A report of 2 cases and literature review.\\nABSTRACT:Human immunodeficiency virus (HIV) infection usually presents with a wide range of manifestations. Although HIV patients are prone to pulmonary infections by opportunistic pathogens in the late stage of AIDS, manifesting the disease with pulmonary infections caused by Mycobacterium riyadhense (newly identified non-tuberculous mycobacteria) is extremely rare with only one case reported in the literature. We are describing two previously healthy patients who presented with M. riyadhense lung infection and were subsequently found to have HIV, illustrating the need for considering the possibility M. riyadhense lung infection as a presenting illness of HIV.'}, {'qas': [{'id': '26489004:Mycobacterium sherrisii', 'question': 'Can Mycobacterium sherrisii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Mycobacterium sherrisii Pulmonary Disease, Burkina Faso.\\nABSTRACT:'}, {'qas': [{'id': '28268125:Mycobacterium branderi', 'question': 'Can Mycobacterium branderi cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': \"TITLE:Mycobacterium branderi infection: Case report and literature review of an unusual and difficult-to-treat non-tuberculous mycobacterium.\\nABSTRACT:A 67-year-old man with significant smoking history presented with fever, unintentional weight loss, night sweats, productive cough, and progressive dyspnea. Multiple respiratory specimens grew Mycobacterium branderi. Computed tomography scanning of the chest revealed a cavitary right upper lung lesion. Bronchoscopy and thoracoscopic biopsy were negative for malignancy but showed necrotizing granulomatous inflammation, which was culture negative. Due to clinical and radiologic progression despite therapy with clarithromycin, ethambutol and moxifloxacin, the lesion was surgically resected and the patient's symptoms resolved. Mycobacteria were seen in histopathology but did not grow from resected tissue. The patient received an additional 6 months of medical therapy and remains asymptomatic 1 month after completing antimicrobials. Cases of M. branderi causing human infection are very rarely reported. This is a novel case of multi-drug resistant M. branderi pulmonary infection in an apparently immunocompetent patient, progressive despite medical therapy and requiring surgical resection for definitive management.\"}, {'qas': [{'id': '31176766:Photobacterium damselae', 'question': 'Can Photobacterium damselae cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:The intracellular signaling pathway of octopamine upregulating immune resistance functions in Penaeus monodon.\\nABSTRACT:Octopamine (OA), a biogenic monoamine, is known to mediate several immune responses. This study analyzed the effects of OA on immunological regulation in the tiger shrimp Penaeus monodon. The immune parameters including total haemocyte count, differential haemocyte count, phenoloxidase activity, respiratory bursts, superoxide dismutase activity, and phagocytic activity and clearance efficiency in response to the pathogen, Photobacterium damselae, were determined when shrimp were individually injected with saline or OA at 100 or 1000 pmol shrimp-1. In addition, the intracellular second messengers in haemocyte such as Ca2+ and adenosine 3',5'-cyclic monophosphate (cAMP) were examined in shrimp receiving saline or OA at 1 or 10 nmol shrimp-1. Results showed that all of the immune parameters significantly increased at 2-4 h in OA-injected shrimp except hyaline cells in 100 pmol shrimp-1-injected shrimp at 4 h, but phenoloxidase activity per granulocyte significantly decreased at 2-4 h. However, these had returned to saline control levels after receiving OA for 8 h except differential haemocyte count and phenoloxidase activity per granulocyte for 16 h. An injection of OA also significantly increased the survival rate of shrimp challenged with Pho. damselae. Shrimp receiving OA at 1 and 10 nmol shrimp-1 significantly increased the intracellular Ca2+ concentration ([Ca2+]i) at 30-60 min and 30 min, and cAMP concentration [cAMP]i) at 5-15 min and 15 min, respectively. However, [Ca2+]i at 50-60 min, and [cAMP]i at 30-60 min returned to saline control when the shrimp received OA at 10 nmol shrimp-1, and at 1 and 10 nmol shrimp-1, respectively. These results suggest that OA administration by injection at &lt;=1000 pmol shrimp-1 mediates transient upregulation of immunity together with the increased resistance of P. monodon to Pho. damselae, which are modulated through intracellular Ca2+ and cAMP second messenger pathways.\"}, {'qas': [{'id': '11325997:Pandoraea pnomenusa', 'question': 'Can Pandoraea pnomenusa cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Assignment of CDC weak oxidizer group 2 (WO-2) to the genus Pandoraea and characterization of three new Pandoraea genomospecies.\\nABSTRACT:CDC weak oxidizer group 2 (WO-2) consists of nine phenotypically similar human clinical isolates received by the Centers for Disease Control and Prevention between 1989 and 1998. Four of the isolates were from blood, three were from sputum, and one each was from bronchial fluid and maxillary sinus. All are aerobic nonfermentative, motile gram-negative rods with one to eight polar flagella per cell. All grew at 25 and 35 degrees C and were positive for catalase, urease (usually delayed 3 to 7 days), citrate, alkalinization of litmus milk, oxidization of glycerol (weakly), and growth on MacConkey agar and in nutrient broth without NaCl. All except one strain were oxidase positive with the Kovacs method, and all except one isolate weakly oxidized D-glucose. All were negative for oxidation of D-xylose, D-mannitol, lactose, sucrose, maltose, and 20 other carbohydrates, esculin hydrolysis, indole production, arginine dihydrolase, and lysine and ornithine decarboxylase. Only two of nine isolates reduced nitrate. Broth microdilution susceptibilities were determined for all strains against 13 antimicrobial agents. Most of the strains were resistant to ampicillin, extended-spectrum cephalosporins, and aminoglycosides, including gentamicin, tobramycin, and amikacin, but they varied in their susceptibility to fluoroquinolones. High-performance liquid chromatographic and mass spectrometric analyses of the WO-2 group identified ubiquinone-8 as the major quinone component. The percent G+C of the WO-2 strains ranged from 65.2 to 70.7% (thermal denaturation method). All shared a common cellular fatty acid (CFA) profile, which was characterized by relatively large amounts (7 to 22%) of 16:1omega7c, 16:0, 17:0cyc, 18:1omega7c, and 19:0cyc(11-12); small amounts (1 to 3%) of 12:0 and 14:0; and eight hydroxy acids, 2-OH-12:0 (4%), 2-OH-14:0 (trace), 3-OH-14:0 (12%), 2-OH-16:1 (1%), 2-OH-16:0 (3%), 3-OH-16:0 (4%), 2-OH-18:1 (2%), and 2-OH-19:0cyc (3%). This profile is similar to the CFA profile of Pandoraea, a recently described genus associated with respiratory infections in cystic fibrosis patients (T. Coenye et al., Int. J. Syst. Evol. Microbiol., 50:887-899, 2000). Sequencing of the 16S rRNA gene (1,300 bp) for all nine strains indicated a high level (&gt; or =98.8%) of homogeneity with Pandoraea spp. type strains. DNA-DNA hybridization analysis (hydroxyapatite method; 70 degrees C) confirmed the identity of WO-2 with the genus Pandoraea and assigned three strains to Pandoraea apista and three to Pandoraea pnomenusa, and identified three additional new genomospecies containing one strain each (ATCC BAA-108, ATCC BAA-109, ATCC BAA-110). This study also shows that Pandoraea isolates may be encountered in blood cultures from patients without cystic fibrosis.'}, {'qas': [{'id': '26283066:Acidovorax temperans', 'question': 'Can Acidovorax temperans cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:Community of thermoacidophilic and arsenic resistant microorganisms isolated from a deep profile of mine heaps.\\nABSTRACT:Soluble arsenic (As) in acidic feed solution may inhibit the copper (Cu) bioleaching process within mine heaps. To clarify the effect of soluble arsenic on the live biomass and bioxidative activity in heaps, toxicological assays were performed using a synthetic feed solution given by a mine company. The microorganisms had previously been isolated from two heap samples at up to 66 m depth, and cultured using specific media for chemolithotrophic acidophiles (pH 1-2) and moderate thermophiles (48 C), for arsenic tolerance assay. The four media with the highest biomass were selected to assay As-resistance; one culture (Q63h) was chosen to assay biooxidative activity, using a heap sample that contained chalcopyrite and covellite. We found that 0.5 g/L of As does not affect living biomass or biooxidative activity on Cu sulfides, but it dissolves Cu, while As precipitates as arsenic acid (H3AsO4 1/2H2O). The arsenic tolerant community, as identified by 16S rDNA gene sequence analysis, was composed of three main metabolic groups: chemolithotrophs (Leptospirillum, Sulfobacillus); chemolithoheterotrophs and organoheterotrophs as Acidovorax temperans, Pseudomonas alcaligenes, P. mendocina and Sphingomonas spp. Leptospirillum spp. and S. thermosulfidooxidans were the dominant taxa in the Q63-66 cultures from the deepest sample of the oldest, highest-temperature heap. The results indicated arsenic resistance in the microbial community, therefore specific primers were used to amplify ars (arsenic resistance system), aio (arsenite oxidase), or arr (arsenate respiratory reduction) genes from total sample DNA. Presence of arsB genes in S. thermosulfidooxidans in the Q63-66 cultures permits H3AsO4-As(V) detoxification and strengthens the community's response to As. \"}, {'qas': [{'id': '11696101:Staphylococcus lentus', 'question': 'Can Staphylococcus lentus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Antimicrobial susceptibility of staphylococci isolated from the faeces of wild turkeys (Meleagris gallopavo).\\nABSTRACT:AIMS: The purpose of this study was to investigate the staphylococcal flora associated with wild turkey populations. METHODS AND \\n'}, {'qas': [{'id': '8060118:Streptomyces thermovulgaris', 'question': 'Can Streptomyces thermovulgaris cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Occurrence of thermophilic actinomycetes in natural substrates in Nigeria.\\nABSTRACT:Thermophilic actinomycetes were isolated from 163 (48.95%) of 333 samples of vegetable substrates and soil from different sites in Anambra and Enugu States in Nigeria. Thermoactinomyces (Tha.) vulgaris was the most common, occurring in 32.4% of samples while Tha. thalpophilus was isolated from 20.1%. Tha. sacchari, Saccharomonospora (Sam.) viridis and Saccharopolyspora (Sap.) rectivirgula were isolated from 3-10.5% of the samples examined. Streptomyces (Stm.) thermovulgaris occurred in 2.7% and Thermomonospora spp. in 0.6% of samples. Except for Stm. thermovulgaris and, perhaps, Thermomonospora spp., all the species isolated are known to be allergenic in man. The five known allergenic species were abundant in hay and maize silage. T. vulgaris and T. thalpophilus were also frequently isolated from rice straw. Tha. vulgaris, Tha. thalpophilus and Sap. rectivirgula were all most numerous in hay while Tha. sacchari was most numerous in maize silage. The widespread occurrence of thermophilic actinomycetes in Nigeria emphasizes the need for investigation of their role in allergic respiratory disorders.'}, {'qas': [{'id': '31243332:Selenomonas', 'question': 'Can Selenomonas cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Diverse hydrogen production and consumption pathways influence methane production in ruminants.\\nABSTRACT:Farmed ruminants are the largest source of anthropogenic methane emissions globally. The methanogenic archaea responsible for these emissions use molecular hydrogen (H2), produced during bacterial and eukaryotic carbohydrate fermentation, as their primary energy source. In this work, we used comparative genomic, metatranscriptomic and co-culture-based approaches to gain a system-wide understanding of the organisms and pathways responsible for ruminal H2 metabolism. Two-thirds of sequenced rumen bacterial and archaeal genomes encode enzymes that catalyse H2 production or consumption, including 26 distinct hydrogenase subgroups. Metatranscriptomic analysis confirmed that these hydrogenases are differentially expressed in sheep rumen. Electron-bifurcating [FeFe]-hydrogenases from carbohydrate-fermenting Clostridia (e.g., Ruminococcus) accounted for half of all hydrogenase transcripts. Various H2 uptake pathways were also expressed, including methanogenesis (Methanobrevibacter), fumarate and nitrite reduction (Selenomonas), and acetogenesis (Blautia). Whereas methanogenesis-related transcripts predominated in high methane yield sheep, alternative uptake pathways were significantly upregulated in low methane yield sheep. Complementing these findings, we observed significant differential expression and activity of the hydrogenases of the hydrogenogenic cellulose fermenter Ruminococcus albus and the hydrogenotrophic fumarate reducer Wolinella succinogenes in co-culture compared with pure culture. We conclude that H2 metabolism is a more complex and widespread trait among rumen microorganisms than previously recognised. There is evidence that alternative hydrogenotrophs, including acetogenic and respiratory bacteria, can prosper in the rumen and effectively compete with methanogens for H2. These findings may help to inform ongoing strategies to mitigate methane emissions by increasing flux through alternative H2 uptake pathways, including through animal selection, dietary supplementation and methanogenesis inhibitors.'}, {'qas': [{'id': '26297330:Roseomonas cervicalis', 'question': 'Can Roseomonas cervicalis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Roseomonas oryzae sp. nov., isolated from paddy rhizosphere soil.\\nABSTRACT:A non-motile, coccus-shaped, pale-pink-pigmented bacterium, designated strain JC288T, was isolated from a paddy rhizosphere soil collected from Western Ghats, Kankumbi, Karnataka, India. Cells were found to be Gram-stain-negative, and catalase- and oxidase-positive; the major fatty acids were C16 : 0, C16 : 1omega7c/C16 : 1omega6c, C18 : 1omega7c/C18 : 1omega6c and C18 : 1 2-OH. The predominant respiratory quinone was Q-10 and the genomic DNA G+C content was 67.5 mol%. Strain JC288T contained diphosphatidylglycerol, phosphatidylglycerol, phosphatidylethanolamine, phosphatidylcholine, four unidentified aminolipids, three unidentified phospholipids, two unidentified lipids, an aminophospholipid and a glycolipid. Hydroxyspirilloxanthin was the major carotenoid of strain JC288T. 16S rRNA gene sequence comparisons indicated that strain JC288T represents a member of the genus Roseomonas within the family Acetobacteraceae of the phylum Proteobacteria. Strain JC288T shared the highest 16S rRNA gene sequence similarity with Roseomonas rhizosphaerae YW11T (97.3 %), Roseomonas aestuarii JC17T (97.1 %), Roseomonas cervicalis CIP 104027T (95.9 %) and other members of the genus Roseomonas ( &lt; 95.5 %). The distinct genomic difference and morphological, physiological and chemotaxonomic differences from the previously described taxa support the classification of strain JC288T as a representative of a novel species of the genus Roseomonas, for which the name Roseomonas oryzae sp. nov. is proposed. The type strain is JC288T ( = KCTC 42542T = LMG 28711T).'}, {'qas': [{'id': '26806258:Staphylococcus argenteus', 'question': 'Can Staphylococcus argenteus cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Comparison of community-onset Staphylococcus argenteus and Staphylococcus aureus sepsis in Thailand: a prospective multicentre observational study.\\nABSTRACT:Staphylococcus argenteus is a globally distributed cause of human infection, but diagnostic laboratories misidentify this as Staphylococcus aureus. We determined whether there is clinical utility in distinguishing between the two. A prospective cohort study of community-onset invasive staphylococcal sepsis was conducted in adults at four hospitals in northeast Thailand between 2010 and 2013. Of 311 patients analysed, 58 (19%) were infected with S. argenteus and 253 (81%) with S. aureus. Most S. argenteus (54/58) were multilocus sequence type 2250. Infection with S. argenteus was more common in males, but rates of bacteraemia and drainage procedures were similar in the two groups. S. argenteus precipitated significantly less respiratory failure than S. aureus (5.2% versus 20.2%, adjusted OR 0.21, 95% CI 0.06-0.74, p 0.015), with a similar but non-significant trend for shock (6.9% versus 12.3%, adjusted OR 0.46, 95% CI 0.15-1.44, p 0.18). This did not translate into a difference in death at 28 days (6.9% versus 8.7%, adjusted OR 0.80, 95% CI 0.24-2.65, p 0.72). S. argenteus was more susceptible to antimicrobial drugs compared with S. aureus, and contained fewer toxin genes although pvl was detected in 16% (9/58). We conclude that clinical differences exist in association with sepsis due to S. argenteus versus S. aureus. '}, {'qas': [{'id': '17900356:Mycobacterium engbaekii', 'question': 'Can Mycobacterium engbaekii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Bovine tuberculosis in African buffaloes: observations regarding Mycobacterium bovis shedding into water and exposure to environmental mycobacteria.\\nABSTRACT:\\n'}, {'qas': [{'id': '31366344:Streptococcus gallolyticus subsp.gallolyticus', 'question': 'Can Streptococcus gallolyticus subsp.gallolyticus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Neonatal meningitis, endocarditis, and pneumonitis due to Streptococcus gallolyticus subsp. pasteurianus: a case report.\\nABSTRACT:\\n'}, {'qas': [{'id': '26358298:Asaia', 'question': 'Can Asaia cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Complete genome and gene expression analyses of Asaia bogorensis reveal unique responses to culture with mammalian cells as a potential opportunistic human pathogen.\\nABSTRACT:Asaia bogorensis, a member of acetic acid bacteria (AAB), is an aerobic bacterium isolated from flowers and fruits, as well as an opportunistic pathogen that causes human peritonitis and bacteraemia. Here, we determined the complete genomic sequence of the As. bogorensis type strain NBRC 16594, and conducted comparative analyses of gene expression under different conditions of co-culture with mammalian cells and standard AAB culture. The genome of As. bogorensis contained 2,758 protein-coding genes within a circular chromosome of 3,198,265 bp. There were two complete operons encoding cytochrome bo3-type ubiquinol terminal oxidases: cyoABCD-1 and cyoABCD-2. The cyoABCD-1 operon was phylogenetically common to AAB genomes, whereas the cyoABCD-2 operon belonged to a lineage distinctive from the cyoABCD-1 operon. Interestingly, cyoABCD-1 was less expressed under co-culture conditions than under the AAB culture conditions, whereas the converse was true for cyoABCD-2. Asaia bogorensis shared pathogenesis-related genes with another pathogenic AAB, Granulibacter bethesdensis, including a gene coding pathogen-specific large bacterial adhesin and additional genes for the inhibition of oxidation and antibiotic resistance. Expression alteration of the respiratory chain and unique hypothetical genes may be key traits that enable the bacterium to survive under the co-culture conditions. '}, {'qas': [{'id': '22164420:Ralstonia mannitolilytica', 'question': 'Can Ralstonia mannitolilytica cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:[The first experience of application of PCR techniques in real-time mode to diagnose bacteriemia during postoperational period in cardiosurgery patients].\\nABSTRACT:The results comparison is made between PCR polymerase chain reaction techniques in real-time mode and cultural techniques of analysis of blood samples in patients with suspicion of bacteriemia during early postoperational period. From September to December 2009 43 blood samples from 37 adult patients aged 59+9.2 years were analyzed on suspicion of bacteriemia. The blood samples for PCR analysis (SeptiFast technique Roche diagnostics, Switzerland) were taken parallel to the inoculation. Among 43 samples 20 (46.5%) were positive. The species structure of identified germs was as follows: Enteroccocus Faecalis - 2, Klebsiella pneumonia - 2, Enteroccocusfaecium - 1, Pseudomonas aeruginosa - 2, coagulase negative Staphilococcus - 11. All cases of seeding of coagulase negative Staphilococcus were interpreted using PCR techniques as negative or contamination. In four cases a full matching of results of compared techniques was obtained. In cases when hemoculture was negative (n = 23) the PCR techniques permitted to detect two positive results (Staphilococcus spp. and Stenotrophomonas maltophilia). In three cases of positive hemoculture the PCR techniques brought negative result and at that in two cases the causative agents (Pseudomonas fluorescens and Ralstonia mannitolilytica) were out of specter of pathogens detected by this method. The PCR techniques in real-time mode permit to reduce significantly time of pathogen investigation and to detect the false positive results conditioned by contamination. This techniques has to be applied in parallel with cultural technique because only this approach provide a possibility to get the clinically needed data related to the sensitivity of causative agent to antibacterial preparations.'}, {'qas': [{'id': '3624445:Eubacterium brachy', 'question': 'Can Eubacterium brachy cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Characteristics and sites of infection of Eubacterium nodatum, Eubacterium timidum, Eubacterium brachy, and other asaccharolytic eubacteria.\\nABSTRACT:Three new species, Eubacterium nodatum, Eubacterium timidum, and Eubacterium brachy, were described, primarily from subgingival samples taken from patients with moderate and severe adult periodontitis. Except for the isolation of E. brachy from a pleuropulmonary infection, these species have not been reported from other infected body sites. We report on the isolation of these species and an undescribed group (D-6) of asaccharolytic eubacteria also found in periodontal disease from numerous different sites of infection, mostly the head and neck. A similarity in cellular morphological properties of E. nodatum and Actinomyces sp. was noted previously. Additional similarities, particularly to Actinomyces israelii, that we found are the formation of molar tooth colonies and the isolation from cases of lumpy jaw and from the genital tract of women in association with the use of an intrauterine contraceptive device. E. timidum and E. brachy did not occur more often from any particular site outside of the head, neck, and respiratory tract. The group D-6 strains came from a variety of sites in the trunk and pelvis. These species are all obligately anaerobic, asaccharolytic, and generally nonreactive, and they grow poorly and slowly on media commonly used to isolate anaerobic bacteria. L-Lysine (0.5%) markedly stimulated the growth of E. nodatum and, to a lesser extent, another acetate- and butyrate-producing group, Eubacterium sp. group D-6, but we did not find comparable stimulants for the other species. We found the production of phenyl acetate to be a helpful marker in the identification of E. timidum and Eubacterium sp. group D-6. Although the isolation and identification of most of these species remain somewhat difficult, the evidence from dental infections and the present report suggests that these species are potential pathogens that are likely to be overlooked in infected clinical material without special attention to more prolonged incubation and use of enriched isolation media.'}, {'qas': [{'id': '31420951:Burkholderia tropica', 'question': 'Can Burkholderia tropica cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Evaluating antibacterial and anticancer activity of crude extracts of bacterial endophytes from Crinum macowanii Baker bulbs.\\nABSTRACT:The results from this study revealed that crude extracts isolated from bacterial endophytes obtained from Crinum macowanii bulbs showed activity against both Gram-positive and Gram-negative pathogenic bacteria, while Acinetobacter guillouiae crude extracts displayed anticancer activity. This study aimed to isolate and characterize bacterial endophytes and their crude extracts from C. macowanii bulbs. Endophytes were isolated using validated surface sterilization techniques, followed by phenotypic and genotypic profiles of the isolates. Crude extracts were extracted from the endophytes using ethyl acetate, while methanol:dichloromethane (1:1) was used to obtain crude extracts from the bulbs. Antibacterial activity of crude extract from each endophyte was investigated against selected pathogenic strains using the broth microdilution method, and anticancer activity against U87MG glioblastoma and A549 lung carcinoma cells was determined by the MTS (3-(4,5-dimethylthiazol-2-yl)-5-(3-carboxymethoxy-phenyl)-2-(4-sulfophenyl)-2H-tetrazolium) assay. Acinetobacter guillouiae, Pseudomonas moraviensis, Pseudomonas sp., Rahnella aquatilis, Bacillus cereus, Novosphingobium sp., Raoultella ornithinolytica, and Burkholderia tropica were successfully isolated. The crude extracts from the majority of endophytes showed antibacterial activity, ranging from 0.125 to &gt;16.00 mg/ml against Gram-negative and Gram-positive pathogenic bacteria. Acinetobacter guillouiae extracts showed a high bioactive potential against U87MG glioblastoma cell lines by reducing their growth by 50% at concentrations of 12.5, 6.25, and 3.13 microg/ml. Crude extracts isolated from C. macowanii bulbs showed potential for possible drug lead against common pathogenic bacteria.'}, {'qas': [{'id': '24135316:Haematobacter missouriensis', 'question': 'Can Haematobacter missouriensis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Paenirhodobacter enshiensis gen. nov., sp. nov., a non-photosynthetic bacterium isolated from soil, and emended descriptions of the genera Rhodobacter and Haematobacter.\\nABSTRACT:A Gram-reaction-negative, facultatively anaerobic, non-motile, rod-shaped, non-photosynthetic bacterial strain, DW2-9(T), was isolated from soil. The highest 16S rRNA gene sequence similarities were found to Rhodobacter capsulatus ATCC 11166(T) (97.1%), Rhodobacter viridis JA737(T) (96.4%), Rhodobacter maris JA276(T) (96.2%), Rhodobacter veldkampii ATCC 35703(T) (96.0%), Haematobacter massiliensis CCUG 47968(T) (96.0%), Haematobacter missouriensis CCUG 52307(T) (95.9%) and Rhodobacter aestuarii JA296(T) (95.7%). The genomic DNA G+C content was 67.2 mol% and the major respiratory quinone was ubiquinone 10 (Q-10). The major cellular fatty acids (&gt;5%) were C(18 : 1)omega7c, C(16 : 0), C(19 : 0) cyclo omega8c and summed feature 3 (one or more of iso-C(15 : 0) 2-OH, C(16 : 1)omega6c and C(16 : 1)omega7c). However, unlike species of the genus Rhodobacter, strain DW2-9(T) neither formed internal photosynthetic membranes nor produced photosynthetic pigments. DNA-DNA hybridization between strain DW2-9(T) and R. capsulatus JCM 21090(T) showed a relatedness of 33%. Strain DW2-9(T) contained phosphatidylethanolamine, phosphatidylglycerol and an unknown aminophospholipid as major polar lipids, which differed from those of species of the genera Rhodobacter and Haematobacter. In addition to the differences in phylogenetic position and polar lipid types, strain DW2-9(T) could be distinguished from species of the genus Haematobacter by the cultivation conditions. On the basis of our polyphasic taxonomic analysis, strain DW2-9(T) is considered to represent a novel genus and species, for which the name Paenirhodobacter enshiensis gen. nov., sp. nov. is proposed. The type strain of Paenirhodobacter enshiensis is DW2-9(T) ( = CCTCC AB 2011145(T) = KCTC 15169(T)). Emended descriptions of the genera Rhodobacter and Haematobacter are also proposed.'}, {'qas': [{'id': '24122117:Azohydromonas', 'question': 'Can Azohydromonas cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Caldimonas meghalayensis sp. nov., a novel thermophilic betaproteobacterium isolated from a hot spring of Meghalaya in northeast India.\\nABSTRACT:While studying the microbial diversity of hot springs of North-east India we isolated a strain AK31T from the Jakrem hot spring of Meghalaya. The strain formed light yellow colonies on nutrient agar and was Gram negative, non spore-forming rods, motile with single polar flagellum. The strain was positive for oxidase and catalase and hydrolysed starch and weakly urea. The predominant cellular fatty acids were C16:0 (34.8 %), C17:0 cyclo (27.1 %), C16:1 omega7c and/or iso-C15:0 2OH (summed feature 3) (9.6 %), C10:0 3OH (8.0 %), C12:0 (5.8 %), C14:0 (5.3 %) and C18:1 omega7c (5.3 %). Strain AK31T contained ubiquinone-8 as the major respiratory quinone and diphosphatidylglycerol, phosphatidylethanolamine, three unidentified phospholipids and one unidentified glycolipid as the polar lipids. The G + C content of the DNA of the strain AK31T was 66.7 mol%. The 16S rRNA gene sequence analysis indicated that strain AK31T was member of the genus Caldimonas and closely related to Caldimonas manganoxidans JCM 10698T and Caldimonas taiwanensis On1T with 96.9 % similarity and with Aquincola tertiaricarbonis L10T and Azohydromonas australica IAM 12664T with 96.5 and 96.4 % similarity respectively. Phylogenetic analyses indicated that the strain AK31T clustered with C. manganoxidans JCM 10698T and C. taiwanensis On1T with a phylogenetic distance of 3.25 %. Based on data from the current polyphasic study, strain AK31T is proposed as a novel species of the genus Caldimonas, for which the name Caldimonas meghalayensis sp. nov. is proposed. The type strain of C. meghalayensis is AK31T (= MTCC 11703T = JCM 18786T).'}, {'qas': [{'id': '32670977:Streptococcus pseudoporcinus', 'question': 'Can Streptococcus pseudoporcinus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Bacteremia because of Streptococcus pseudoporcinus in a Syphilis-HIV co-infected patient: A case report.\\nABSTRACT:A 43 year old male patient came to the emergency department with complaints of severe breathlessness and pedal edema with on & off fever since 15 days. The patient also gave history of sexual exposures with multiple partners. ECHO revealed moderate LV dysfunction, severe aortic regurgitation (AR), trivial mitral regurgitation (MR) with mild pulmonary edema. The patient was diagnosed as a case of severe Aortic regurgitation with atrial fibrillation. The patient was found sero-positive for HIV and Syphilis. His blood cultures obtained prior to initiation of antibiotics showed growth of small 0.5-1mm in diameter, beta- hemolytic colonies on blood agar The isolate was identified to be Streptococcus pseudoporcinous by VITEK2 Compact system and was sensitive to vancomycin, linezolid, penicillin, cotrimoxazole and ciprofloxacin. Streptococcus pseudoporcinus is usually found as colonizer of female genital tract has been rarely associated with bacteremia. In the present report the patient possibly has acquired the infection from female genital tract because of his abnormal sexual behaviour. Association of Streptococcus pseudoporcinus with increased numbers of sexual partners and sexually transmitted infections suggests that further studies of this organism are warranted.'}, {'qas': [{'id': '9663026:Actinomyces viscosus', 'question': 'Can Actinomyces viscosus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:The efficacy of antimicrobial mouth rinses in oral health care.\\nABSTRACT:There is growing public recognition of the importance of oral health, as symbolized by the theme. \"Oral Health for a Healthy Life\" proposed for the 1994 World Health Day. In this report, the efficacy of antimicrobial mouth rinses, mainly Listerine, was reviewed by three investigators who are working as a microbiologist, a microbiologist, a dentist, and a dental hygienist participating in oral health care. Listerine, an antimicrobial mouth rinse, completely killed microorganisms in 10 to 30 seconds; the microbes includes methicillin-resistant Staphylococcus aureus, Streptococcus pyogenes, Helicobacter pylori, Candida albicans, Streptococcus mutans, Actinomyces viscosus, Porphyromonas gingivalis, Prevotella intermedia, and Actinobacillus actinomycetemcomitans. Listerine was also weakly effective in inactivating human immunodeficiency viruses. Bacteria in samples collected from human dental plaque and saliva were completely killed within 30 seconds when exposed to Listerine. When saliva samples were collected from subjects who had rinsed their mouths with 20 ml of Listerine or 1:50 diluted povidone-iodine, levels of viable anaerobic bacteria in the samples were reduced to 1%. When Listerine was used for oral surgery such as tooth extraction and periodontal surgery, the agent was effective in relieving toothache. This was probably due to a decrease in oral bacteria by the antimicrobial action of Listerine, leading to lowering the inflammatory response of the host. The use of antimicrobial mouth rinse during dental treatments such as endodontic treatment proved effective for more reliable infection control. In Japan, there are an increasing number of elderly and medically compromised hosts who are potentially at risk for developing pneumonia due to silent aspiration of microbes in the oral cavity and throat. For the aged with such potential risk, using of antimicrobial mouth rinse may be effective in preventing dental plaque accumulation when used in addition to the mechanical control of plaque, since they tend to have difficulty in brushing teeth by themselves. Indeed, the use of antimicrobial mouth rinse in these elderly people proved useful not only in preventing bacterial pneumonia, but also in improving their quality of life by preserving their oral health.'}, {'qas': [{'id': '27882322:Mycobacterium abscess', 'question': 'Can Mycobacterium abscess cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Identification of Species of Nontuberculous Mycobacteria Clinical Isolates from 8 Provinces of China.\\nABSTRACT:Pulmonary diseases caused by nontuberculous mycobacteria (NTM) are increasing in incidence and prevalence worldwide. In this study, we identified NTM species of the clinical isolates from 8 provinces in China, in order to preliminarily provide some basic scientific data in the different species and distribution of NTM related to pulmonary disease in China. A total of 523 clinical isolates from patients with tuberculosis (TB) diagnosed clinically from 2005 to 2012 were identified to the species using conventional and molecular methods, including multilocus PCR, rpoB and hsp65 PCR-PRA, hsp65, rpoB, and 16S-23S internal transcribed spacer region sequencing. The isolates were identified into 3 bacterium genera, including NTM, Gordonia bronchialis, and Nocardia farcinica, and, for the 488 NTM isolates, 27 species were identified. For all the 27 species of NTM which were found to cause pulmonary infections in humans, the most prevalent species was M. intracellulare, followed by M. avium and M. abscessus. And seven other species were for the first time identified in patients with TB in China. NTM species identification is very important for distinguishing between tuberculosis and NTM pulmonary diseases, and the species diversity drives the creation of diverse and integrated identification methods with higher accuracy and efficacy.'}, {'qas': [{'id': '31516793:Ralstonia pickettii', 'question': 'Can Ralstonia pickettii cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Ralstonia pickettii Bacteremia: An Emerging Infection in a Tertiary Care Hospital Setting.\\nABSTRACT:Ralstonia species are Gram-negative bacilli that have increasingly been recognized as emerging nosocomial pathogens, particularly in immunocompromised hosts. Ralstonia pickettii is the most clinically important pathogen from the Ralstonia genus. Nosocomial outbreaks of Ralstonia pickettii infections brought about by the use of contaminated medical solutions, including saline, sterile water, as well as disinfectants, have been reported. There have been case reports of invasive infections with variable presentations. Here, we describe three cases of Ralstonia pickettii bacteremia during a period of one year in a tertiary care hospital in Karachi, Pakistan. The first case was a 76-year-old male, known case of type 2 diabetes mellitus (DM), hypertension, and amyotrophic lateral sclerosis, who presented with complaints of burning micturition, hematuria, and fever. The patient had a history of multiple hospital admissions in the recent past. His blood culture was found to be positive for Ralstonia pickettii. A computed tomography scan of the kidneys, ureter, and bladder (CT KUB) was suggestive of pyelonephritis. The patient improved on intravenous meropenem. The second case was a 47-year-old man, who was admitted with a gunshot injury to the neck, resulting in complete cervical cord resection and mild hydrocephalus with intraventricular hemorrhage. The patient had a prolonged intensive care unit (ICU) stay, which was complicated by ventilator-associated pneumonia with Acinetobacter and central line-associated bloodstream infection (CLABSI) with Ralstonia pickettii. He was treated with meropenem and colistin but continued to deteriorate and expired. The third case was a 46-year-old lady, known case of end-stage renal disease (ESRD), who was admitted with prosthetic valve endocarditis. She had a prolonged hospital stay complicated by CLABSI with Ralstonia pickettii, improved on meropenem, but later died due to fungemia. Ralstonia pickettii is an emerging cause of nosocomial infection in patients, particularly those with a prolonged hospital stay, and can cause invasive and severe infections.'}, {'qas': [{'id': '19061488:Ochrobactrum anthropi', 'question': 'Can Ochrobactrum anthropi cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:First isolation of two colistin-resistant emerging pathogens, Brevundimonas diminuta and Ochrobactrum anthropi, in a woman with cystic fibrosis: a case report.\\nABSTRACT:INTRODUCTION: Cystic fibrosis afflicted lungs support the growth of many bacteria rarely implicated in other cases of human infections. CASE PRESENTATION: We report the isolation and identification, by 16S rRNA amplification and sequencing, of two emerging pathogens resistant to colistin, Brevundimonas diminuta and Ochrobactrum anthropi, in a 17-year-old woman with cystic fibrosis and pneumonia. The patient eventually responded well to a 2-week regime of imipenem and tobramycin. \\n'}, {'qas': [{'id': '19933592:Pusillimonas', 'question': 'Can Pusillimonas cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Pusillimonas soli sp. nov., isolated from farm soil.\\nABSTRACT:A Gram-negative, motile, non-spore-forming bacterial strain, designated MJ07(T), was isolated from a farm soil and was characterized to determine its taxonomic position by using a polyphasic approach. Comparative 16S rRNA gene sequence analysis showed that strain MJ07(T) belongs to the family Alcaligenaceae, class Betaproteobacteria, and is related most closely to Pusillimonas ginsengisoli KCTC 22046(T) (98.6 % sequence similarity) and Pusillimonas noertemannii BN9(T) (96.9 %). The levels of 16S rRNA gene sequence similarity between strain MJ07(T) and members of all other recognized species of the family Alcaligenaceae were below 95.2 %. The G+C content of the genomic DNA of strain MJ07(T) was 59.4 mol%. The detection of a quinone system with ubiquinone Q-8 as the major respiratory lipoquinone, putrescine as the predominant polyamine, phosphatidylethanolamine, phosphatidylglycerol, diphosphatidylglycerol and two unknown aminolipids as major polar lipids and a fatty acid profile with C16:0 (32.0 %), C17:0cyclo (24.7 %) and C19:0cyclo omega8c (11.5 %) as the major components supported the affiliation of strain MJ07(T) to the genus Pusillimonas. Strain MJ07(T) exhibited relatively low levels of DNA-DNA relatedness with respect to P. ginsengisoli KCTC 22046(T) (50 +- 8 %) and P. noertemannii KACC 13183(T) (18 +- 7 %). On the basis of its phenotypic and genotypic properties together with its phylogenetic distinctiveness, strain MJ07(T) (=KCTC 22455(T) =JCM 16386(T)) should be classified in the genus Pusillimonas as the type strain of a novel species, for which the name Pusillimonas soli sp. nov. is proposed.'}, {'qas': [{'id': '32032761:Photobacterium damselae', 'question': 'Can Photobacterium damselae cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:An in vitro study of the effect of carob (Ceratonia siliqua L.) leaf extracts on gilthead seabream (Sparus aurata L.) leucocyte activities. Antioxidant, cytotoxic and bactericidal properties.\\nABSTRACT:Carob leaves, the main residues of the carob tree, were investigated as a renewable and abundant source of bioactive compounds for fish aquaculture. Aqueous and ethanolic extracts obtained from carob leaves were characterized in terms of biochemical composition, antiradical and cytotoxic effects and immunostimulant and antibacterial activities. The ethanolic extract showed higher levels of total phenolics, flavonoids and condensed tannins and higher antioxidant activity than the aqueous extract. No significant immunostimulant effects were observed on gilthead seabream (Sparus aurata) head kidney leucocytes (viability, phagocytosis and respiratory burst activities and peroxidase content) after incubation for 24 h with different extracts. Furthermore, the ethanolic extracts used at 0.5, 0.75 and 1 mg mL-1 and aqueous extracts at 1 gmL-1 had a cytotoxic effect on PLHC-1 cells. When the bactericidal activity was tested against three fish pathogenic bacteria (Vibrio harveyi, Vibrio anguillarum and Photobacterium damselae) notable activity of the different extracts was detected against P. damselae at all three concentrations. A similar effect was demonstrated against V. haryeri when ethanolic extracts were used in the same range of concentrations. This work demonstrates interesting in vitro effects of carob leaf extracts and suggests it could be used as an alternative to chemical compounds with farmed fish. The concentration and nature of the extracts were very important in terms of any positive results.'}, {'qas': [{'id': '11852504:Sphingobacterium multivorum', 'question': 'Can Sphingobacterium multivorum cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:[Respiratory infection caused by Sphingobacterium multivorum].\\nABSTRACT:'}, {'qas': [{'id': '27524544:Aggregatibacter aphrophilus', 'question': 'Can Aggregatibacter aphrophilus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Aggregatibacter aphrophilus misidentified as Brucella spp. in an immunocompetent patient with brain abscess.\\nABSTRACT:Aggregatibacter aphrophilus rarely causes brain abscesses. Here we report a case of the brain abscess caused by Aggregatibacter aphrophilus. Cultivated gram negative coccobacilli from cerebral abscess were initially misidentified as Brucella spp. because it gave false positive agglutination with anti-Brucella sera. Definite identification was made with MALDI-TOF assay. Right to left shunt through the pulmonary arteriovenous malformation was speculated to be the underlying cause for the brain abscess. The patient was treated successfully with ampicillin-sulbactam after failing ceftriaxone treatment. '}, {'qas': [{'id': '8327781:Balneatrix alpica', 'question': 'Can Balneatrix alpica cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Balneatrix alpica gen. nov., sp. nov., a bacterium associated with pneumonia and meningitis in a spa therapy center.\\nABSTRACT:In 1987, an outbreak of pneumonia and meningitis caused by an unknown bacterium occurred in a spa therapy centre. Nine isolates of this pathogen constituted a tight DNA hybridization group. rRNA-DNA hybridization and 16S rRNA sequencing showed that the studied bacteria represented a new branch in superfamily II (= gamma subclass) of the Proteobacteria, close to the genus Oceanospirillum. The new bacterium was highly polymorphic and, in young cultures, had curved Gram-negative cells, motile by polar single flagella. The new bacterium differed from the genus Oceanospirillum by its lacking the NaCl requirement and by reducing nitrate into nitrite, producing indole from tryptophan and producing acid from carbohydrates. The name Balneatrix alpica gen. nov., sp. nov. is proposed for the studied organism. The type strain is strain 4-87 (= CIP 103589).'}, {'qas': [{'id': '31870880:Gardnerella vaginalis', 'question': 'Can Gardnerella vaginalis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Infectious disease symptoms and microbial carriage among French medical students travelling abroad: A prospective study.\\nABSTRACT:\\n'}, {'qas': [{'id': '33140140:Bacillus amyloliquefaciens', 'question': 'Can Bacillus amyloliquefaciens cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:Seaweed-associated heterotrophic bacteria: new paradigm of prospective anti-infective and anticancer agents.\\nABSTRACT:Ever since the development of the first antibiotic compound with anticancer potential, researchers focused on isolation and characterization of prospective microbial natural products with potential anti-infective and anticancer activities. The present work describes the production of bioactive metabolites by heterotrophic bacteria associated with intertidal seaweeds with potential anti-infective and anticancer activities. The bacteria were isolated in a culture-dependent method and were identified as Shewanella algae MTCC 12715 (KX272635) and Bacillus amyloliquefaciens MTCC 12716 (KX272634) based on combined phenotypic and genotypic methods. Further, the bacteria were screened for their ability to inhibit drug-resistant infectious pathogens and prevent cell proliferation of human liver carcinoma (HepG2) and breast cancer (MCF7) cell lines, without affecting the normal cells. Significant anti-infective activity was observed with bacterial cells and their organic extracts against broad-spectrum multidrug-resistant pathogens, such as vancomycin-resistant Enterococcus faecalis, methicillin-resistant Staphylococcus aureus, Klebsiella pneumonia and Pseudomonas aeruginosa with minimum inhibitory concentration <= 3.0 microg mL-1 as compared to the antibiotic agents' chloramphenicol and ampicillin, which were active at >= 6.25 mg mL-1. The extracts also exhibited anticancer activity in a dose-responsive pattern against HepG2 (with IC50, half maximal inhibitory concentration ~ 78-83 microg mL-1) and MCF7 (IC50 ~ 45-48 microg mL-1) on tetrazolium bromide screening assay with lesser cytotoxic effects on normal fibroblast (L929) cell lines (IC50 > 100 microg mL-1). The results revealed that seaweed-associated heterotrophic bacteria could occupy a predominant role for a paradigm shift towards the development of prospective anti-infective and anticancer agents.\"}, {'qas': [{'id': '28699577:Sphingobacterium mizutaii', 'question': 'Can Sphingobacterium mizutaii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Sphingobacteriumsoli sp. nov., isolated from soil.\\nABSTRACT:A Gram-stain-negative, strictly aerobic, rod-shaped, non-motile, non-spore-forming bacterial strain, designated YIM X0211T, was isolated from a soil sample of Shiling County, Yunnan Province, south-west China. The new isolate was characterized taxonomically by using a polyphasic approach. The strain grew optimally at 30  C, at pH 7.0 and with 0-3 % (w/v) NaCl. It was positive for catalase and oxidase but negative for H2S production. Comparative 16S rRNA gene sequence analysis showed that strain YIM X0211T fell within the cluster comprising Sphingobacterium species and clustered with Sphingobacterium mizutaii DSM 11724T (97.93 % similarity). The G+C content of the genomic DNA was 41.2 mol%. The predominant respiratory quinone was menaquinone MK-7. The major fatty acids were iso-C15 : 0 2-OH, iso-C17 : 0 3-OH and summed feature 3 (C16 : 1omega7c/C16 : 1omega6c). The polar lipids consisted of phosphatidylethanolamine, sphingolipid, and several unknown phospholipids or lipids. The DNA-DNA hybridization value between strain YIM X0211T and S. mizutaii DSM 11724T was 42.3+-0.4 %, which is below the 70 % limit for species delineation. These chemotaxonomic data supported the affiliation of strain YIM X0211T to the genus Sphingobacterium. Based on the recorded phenotypic and genotypic characteristics, it is determined that the isolate represents a novel species of the genus Sphingobacterium, for which the name Sphingobacterium soli sp. nov. is proposed. The type strain is YIM X0211T (=KCTC 42696T=CGMCC 1.15966T).'}, {'qas': [{'id': '16132425:Pannonibacter phragmitetus', 'question': 'Can Pannonibacter phragmitetus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Diversity of alkaliphilic and alkalitolerant bacteria cultivated from decomposing reed rhizomes in a Hungarian soda lake.\\nABSTRACT:Bacterial communities associated with decomposing rhizomes of Phragmites australis were investigated in Lake Ferto (Neusiedlersee, Hungary). Alkaliphilic and alkalitolerant strains were isolated on cellulose-containing alkaline medium spread with dilutions of scrapings taken from the surface of the decaying plant material. Fifty-one strains were grouped by numerical analysis based on physiological tests and BIOLOG sole carbon source utilization data. The strains identified by 16S rDNA sequence comparisons included members of low G+C Gram positives (Marinibacillus marinus, Bacillus cereus, and Exiguobacterium aurantiacum), high G+C Gram positives (Nesterenkonia halobia and Dietzia natronolimnea), alpha-proteobacteria (Pannonibacter phragmitetus), and gamma-proteobacteria (Pseudomonas pseudoalcaligenes and Halomonas venusta). Most of the strains were characterized by aerobic chemoorganotrophic respiratory metabolism and utilized several different carbon sources, although no direct cellulolytic activity was observed. Results of the pH and salt tolerance tests revealed optimuma in most cases at pH 11 and at the presence of 2.5-5% NaCl. These bacteria probably occupy niches in the aerobic, alkaline, water-influenced environments on the decomposing reed surfaces.'}, {'qas': [{'id': '32890724:Gardnerella vaginalis', 'question': 'Can Gardnerella vaginalis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Risk factors for symptoms of infection and microbial carriage among French medical students abroad.\\nABSTRACT:OBJECTIVES: To investigate symptoms of infections and their risk factors among French medical students undertaking an internship abroad. \\n'}, {'qas': [{'id': '24040782:Capnocytophaga cynodegmi', 'question': 'Can Capnocytophaga cynodegmi cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Mycoplasma edwardii peritonitis in a patient on maintenance peritoneal dialysis.\\nABSTRACT:Mycoplasma edwardii (M. edwardii) is an anthropozoonotic microorganism found in the upper respiratory and urogenital tracts of dogs. M. edwardii was one of the microbes isolated from peritoneal fluid of a 10-year-old child diagnosed with polymicrobial peritonitis following a puncture of dialysis tubing by a pet dog. Other unique pathogens representative of canine oral microflora isolated from this patient on peritoneal dialysis were Kingella denitrificans, Actinomycetes species and Capnocytophaga cynodegmi. '}, {'qas': [{'id': '1752209:Propionibacterium avidum', 'question': 'Can Propionibacterium avidum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Prophylactic and therapeutic application of Propionibacterium avidum KP-40 in swine and calves with acute enzootic bronchopneumonia.\\nABSTRACT:The usefulness of the prophylactic or therapeutic application of Propionibacterium avidum KP-40, a potent stimulator of the monocyte-macrophage-system, was demonstrated in piglets and calves. After a 3-month-period of observation PA-treated piglets showed a significantly improved development (decreased number of infections, gain of body weight). In piglets and calves the therapeutic use of PA together with oxytetracycline proved to be superior in the treatment of acute endemic enzootic bronchopneumonia (AEB) as compared to groups of animals receiving PA or oxytetracycline alone.'}, {'qas': [{'id': '32672529:Staphylococcus delphini', 'question': 'Can Staphylococcus delphini cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:Staphylococcus ursi sp. nov., a new member of the 'Staphylococcus intermedius group' isolated from healthy black bears.\\nABSTRACT:Six Staphylococcus strains were isolated from healthy black bears (Ursus americanus) in the Great Smoky Mountains National Park, Tennessee, USA. Phylogenetic analysis based on complete genome, 16S rRNA, dnaJ, hsp60, rpoB and sodA genes, and MALDI-TOF-MS main spectral profiles revealed that the strains belonged to one species and showed the closest relatedness to members of the 'Staphylococcus intermedius group' (SIG), which include Staphylococcus intermedius, Staphylococcus pseudintermedius, Staphylococcus delphini and Staphyloccoccus cornubiensis. The strains were positive in SIG-specific and negative in individual species-specific PCR assays for the nuc gene. The strains can be differentiated from the other SIG species by the absence of sucrose fermentation, from S. intermedius DSM 20373T, S. pseudintermedius CCUG 49543T and S. cornubiensis DSM 105366T by the absence of methyl beta-d-glucopyranoside fermentation and from S. delphini DSM 20771T by fermentation of trehalose. DNA relatedness of the type strain MI 10-1553T with the type strains of S. delphini, S. pseudintermedius, S. intermedius and S. cornubiensis was <=48.2 % by digital DNA-DNA hybridization and <=92.3 % by average nucleotide identity calculations. Iso-C15:0, anteiso-C15 : 0 and anteiso-C17 : 0 were the most common fatty acids. Polar lipids consisted of phosphadidylglycerols, phospholipids, glycolipid, diphosphatidylglycerol and aminophospholipid. Cell-wall peptidoglycan was of type A3alpha l-Lys-Gly3 (Ser; similar to A11.2 and A11.3). The respiratory quinone belonged to menaquinone 7 (MK-7). The G+C content of MI 10-1553T was 39.3 mol%. The isolated strains represent a novel species of the genus Staphylococcus, for which we propose the name Staphylococcus ursi sp. nov. The type strain is MI 10-1553T (=ATCC TSD-55T=CCOS 1900T).\"}, {'qas': [{'id': '23725776:Ralstonia pickettii', 'question': 'Can Ralstonia pickettii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Deja vu: Ralstonia mannitolilytica infection associated with a humidifying respiratory therapy device, Israel, June to July 2011.\\nABSTRACT:Following a bloodstream infection in June 2011 with Ralstonia mannitolilytica in a premature infant treated with a humidifying respiratory therapy device, an investigation was initiated at the Hadassah Medical Centres in Jerusalem. The device delivers a warmed and humidified mixture of air and oxygen to patients by nasal cannula. The investigation revealed colonisation with R. mannitolilytica of two of 15 patients and contamination of components of five of six devices deployed in the premature units of the Hadassah hospitals. Ten isolates from the investigation were highly related and indistinguishable from isolates described in an outbreak in 2005 in the United States (US). Measures successful in containing the US outbreak were not included in user instructions provided to our hospitals by the distributor of the device.'}, {'qas': [{'id': '25494219:Leuconostoc lactis', 'question': 'Can Leuconostoc lactis cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Bacteremia due to vancomycin-resistant Leuconostoc lactis in a patient with pneumonia and abdominal infection.\\nABSTRACT:'}, {'qas': [{'id': '8432835:Serratia proteamaculans', 'question': 'Can Serratia proteamaculans cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Fatal pneumonia due to Serratia proteamaculans subsp. quinovora.\\nABSTRACT:Serratia proteamaculans subsp. quinovora was isolated from several samples (blood cultures, tracheal aspirates, pleural effusion) from a patient with pneumonia. This is the first clinical isolate and the first documented human infection caused by this organism.'}, {'qas': [{'id': '25406238:Actinobaculum massiliense', 'question': 'Can Actinobaculum massiliense cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Dissection of the genus Actinobaculum: Reclassification of Actinobaculum schaalii Lawson et al. 1997 and Actinobaculum urinale Hall et al. 2003 as Actinotignum schaalii gen. nov., comb. nov. and Actinotignum urinale comb. nov., description of Actinotignum sanguinis sp. nov. and emended descriptions of the genus Actinobaculum and Actinobaculum suis; and re-examination of the culture deposited as Actinobaculum massiliense CCUG 47753T ( = DSM 19118T), revealing that it does not represent a strain of this species.\\nABSTRACT:The remarkable host specificity of the species of the genus Actinobaculum led us to recharacterize these species by a polyphasic approach. A comparative chemotaxonomic study including analysis of whole-cell sugars, amino acid composition of the peptidoglycan, fatty acid methyl esters, respiratory quinones and polar lipids revealed significant differences that, in combination with molecular data, support a dissection of the genus Actinobaculum. The proposals of this study include the reclassification of Actinobaculum schaalii and Actinobaculum urinale as Actinotignum schaalii gen. nov., comb. nov. (type strain DSM 15541(T) = CCUG 27420(T)) and Actinotignum urinale comb. nov. (type strain DSM 15805(T) = CCUG 46093(T)), respectively. Emended descriptions of the genus Actinobaculum and Actinomyces suis are also provided. The results of 16S rRNA gene sequence analysis and DNA-DNA hybridization also indicated that the type strain of Actinobaculum massiliense deposited as CCUG 47753(T) ( = DSM 19118(T)) should in fact be considered a member of the species Actinobaculum schaalii. In addition, comparative 16S rRNA gene sequencing and DNA-DNA relatedness studies of four strains recovered from clinical materials demonstrated that three of the isolates belonged to Actinotignum schaalii; the remaining strain represents a novel species, for which the name Actinotignum sanguinis sp. nov. is proposed. The type strain is IMMIB L-2199(T) ( = DSM 26039(T) = CCUG 64068(T)).'}, {'qas': [{'id': '33384344:Hafnia alvei', 'question': 'Can Hafnia alvei cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': \"TITLE:Hafnia alvei pneumonia: a rare cause of infection in the multimorbid or immunocompromised.\\nABSTRACT:Hafnia alvei is a rare, poorly understood commensal bacterium which has, on occasion, been shown to infect humans. We present two cases. The first patient presented with a 1-week history of dyspnoea, pleurisy and a productive cough, and the second with a prodrome of fatigue and night sweats. The former had a history of severe chronic obstructive pulmonary disease and the latter had a history of Crohn's disease. Both patients had underlying comorbidities and immunosuppression, but differed in presentation, radiological findings and recovery. This case series aims to remind readers of the broad differential of pathogens that can lead to disease in the immunocompromised and that one should not dismiss atypical cultured bacteria as commensal too hastily.\"}, {'qas': [{'id': '7831266:Campylobacter avium', 'question': 'Can Campylobacter avium cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:[Macrolides. New therapeutic prospects].\\nABSTRACT:The aim of the development of semisynthetic derivatives was to overcome the problem of chemical stability of erythromycin A in acid medium, with less variability in gastro-intestinal absorption and leading to renewed interest in macrolides. The new macrolides have the same antibacterial spectrum as erythromycin A including Gram-positive and Gram-negative cocci, intracellular bacteria, mycoplasma, Campylobacter sp., Helicobacter pylori, mycobacteria spp., Gram-negative bacilli including Haemophilus influenzae, Bordetella pertussis, Pasteurella multocida, Gram-positive bacilli including Corynebacterium diphtheriae and anaerobic species. In vitro activity against Haemophilus influenzae is still a controversial subject. Macrolides are among the best tolerated antibacterial agents. Theoretically, macrolides could be given to a large range of patients even those suffering from underlying diseases. The new macrolides, roxithromycin, azithromycin, clarithromycin, dirithromycin, rokitamycin and miokamycin, are indicated for the treatment of upper respiratory tract infections and lower respiratory tract infections due to intracellular bacteria or Mycoplasma pneumoniae. Macrolides could be used as first line therapy for non-gonococcal urethritis, especially those due to Chlamydia trachomatis or Ureaplasma urealyticum. In pelvic inflammatory infections in which Chlamydia trachomatis is involved macrolides could also be used. Other non-conventional indications under discussion are H. pylori and Lyme's disease. Macrolides in combination with other antibacterials could be an alternative for Mycobacterium avium-intracellulare infections. The antiparasite effect of erythromycin has been known since the 1950s. Extensive experimental work is currently underway to determine the potential use of these drugs in this setting. Research during the 80s in the macrolide field, led to enhanced pharmacokinetic properties. Current research is focused on expanding the antibacterial spectrum and to overcome cross-resistance among 14-membered-ring macrolides.\"}, {'qas': [{'id': '26887928:Mycobacterium kumamotonense', 'question': 'Can Mycobacterium kumamotonense cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Increased isolation of nontuberculous mycobacteria among TB suspects in Northeastern, Tanzania: public health and diagnostic implications for control programmes.\\nABSTRACT:\\n'}, {'qas': [{'id': '16055008:Capnocytophaga cynodegmi', 'question': 'Can Capnocytophaga cynodegmi cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:Lower respiratory tract infection due to Capnocytophaga cynodegmi in a cat with pulmonary carcinoma.\\nABSTRACT:A 10-year-old male castrated domestic shorthair cat was evaluated for coughing and lethargy. Thoracic radiographs revealed a soft tissue lung mass and diffuse peribronchial infiltrates. Bronchoscopy was performed and Capnocytophaga cynodegmi was cultured from bilateral bronchoalveolar lavage samples. Clinical signs and bacterial colonization resolved following treatment with enrofloxacin. A lung lobectomy was performed to remove the lung mass, which was diagnosed as pulmonary carcinoma. C cynodegmi is most frequently isolated from localized wound and corneal infections in humans. Specialized growth characteristics of C cynodegmi may result in low sensitivity for bacterial culture. To the authors' knowledge, this case represents the first report of C cynodegmi infection in a veterinary patient and only the second case in human or veterinary medicine where the organism has been isolated from a bronchoalveolar lavage sample. Based on this report, Capnocytophaga species should be considered as potential opportunistic pathogens.\"}, {'qas': [{'id': '28680649:Abiotrophia defectiva', 'question': 'Can Abiotrophia defectiva cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Aorto-right ventricular fistula: a rare complication of Abiotrophia Endocarditis.\\nABSTRACT:A 41-year-old African male presented with worsening dyspnea and cachexia concerning for congestive heart failure. Transesophageal echocardiogram revealed a large mass attached to the aortic valve leaflet, mass attached to the flail anterior mitral valve leaflet, severe pulmonary hypertension and dilatation of the aortic root along with fistula between the right coronary aortic cusp and the right ventricular (RV) outflow tract. Blood cultures grew Abiotrophia Defectiva (AD) sensitive to vancomycin. Patient underwent emergent surgical closure of aorto RV fistula and aortic root replacement along with pulmonary and mitral valve replacement. Endocarditis caused by AD has been reported to result in heart failure, septic embolization and destruction of the valve despite use of appropriate antibiotics. To our knowledge, this is the only case of AD endocarditis without any identified entrance route; requiring replacement of pulmonary, mitral and aortic valve due to extensive valvular damage and large vegetations.'}, {'qas': [{'id': '32685375:Gordonia sputi', 'question': 'Can Gordonia sputi cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Gordonia sputi related multiple brain abscesses, an AIDS-presenting illness: Thinking outside the box.\\nABSTRACT:HIV/AIDS has been recognized as a global health issue with significant burden on healthcare services worldwide. Diagnostic and therapeutic challenges include wide range of difficult to identify and treat infections. Gordonia sputi is known to cause multi-system infections in setting of HIV/AIDS. It is often difficult to isolate this organism requiring high suspicion index and special testing techniques. While there is no guidelines-recommended antibacterials regimen for Gordonia sputi infection, extended combined broad spectrum antibacterials have been successfully used. Our patient in this report is a 50-year-old male with no past history who presented with progressive weakness on the right side of the body and urinary incontinence over the duration of one month. MRI scan of the brain showed bilateral ring-enhancing lesions. Gordonia sputi was identified from a tissue biopsy using 16S ribosomal RNA sequencing technique. HIV test for antibodies came to be reactive and a CD4 cell count of 7/muL. The patient was treated with combination of antibacterials and had remarkable radiological interval changes and relatively slower yet apparent clinical improvement. Unfortunately, and despite initial recovery, patient has later developed multi-drug resistant hospital acquired pneumonia leading to his death in ICU during course of hospitalization. Treatment of Gordonia sputi in setting of HIV infection with a combination of antibacterials over extended period appears to be safe and effective. To our knowledge, this is the first report of Gordonia sputi related multiple brain abscesses as AIDS-presenting illness.'}, {'qas': [{'id': '25463697:non-tuberculous mycobacteria', 'question': 'Can non-tuberculous mycobacteria cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Identification and characterization of non-tuberculous mycobacteria isolated from tuberculosis suspects in Southern-central China.\\nABSTRACT:The incidence of non-tuberculous mycobacteria (NTM)-related death has increased globally recently. To obtain information of the species and characterization of pathogens involved in NTM pulmonary infection in Southern-central China, we identified 160 non-tuberculous infection cases from 3995 acid-fast bacilli (AFB)-positive tuberculous suspects. We then randomly selected 101 non-tuberculous patients, isolated bacteria from their sputa and genotyped the pathogens using the 16S rRNA gene and 16S-23S rRNA internal transcribed spacer sequences. M. intracellulare (32.67%, 33/101), M. abscessus (32.67%, 33/101) and M. fortuitum (7.92%, 8/101) are identified in these isolates. Surprisingly, non-mycobacteria including Gordonia (8.91%, 9/101), Nocardia (5.94%, 6/101) and Tsukamurella (0.99%, 1/101) are also discovered, and the case of Tsukamurella pulmonis infection is first discovered in Southern-central China. Moreover, species of M. mucogenicum group, M. chubuense, M. kansasii, M. gastri, M. avium, M. porcinum and M. smegmatis are identified. In addition, nine immune compromised cases (8.91%, 9/101), including type two diabetes mellitus and HIV/AIDS are found to be infected with non-tuberculous bacteria. This study revealed the distribution and characteristics of non-tuberculous AFB pathogen infection occurred in Southern-central China, and suggested that physicians should be alert of the emerging of NTM and non-mycobacteria infection in AFB positive cases and take caution when choosing chemotherapy for tuberculosis-like pulmonary infections. Generally, this study may help with the development of new strategy for the diagnosis and treatment of mycobacterial infection. '}, {'qas': [{'id': '32690600:Collinsella stercoris', 'question': 'Can Collinsella stercoris cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Depicting SARS-CoV-2 faecal viral activity in association with gut microbiota composition in patients with COVID-19.\\nABSTRACT:\\n'}, {'qas': [{'id': '18173068:Mycobacterium intermedium', 'question': 'Can Mycobacterium intermedium cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:[Identification of atypical mycobacteria isolated from clinical specimens by line probe assay (LIPA)].\\nABSTRACT:The aim of this study was the identification of atypical mycobacteria isolated from various samples of patients prediagnosed as tuberculosis between November 2004 and June 2006 by a commercial polymerase chain reaction (PCR) based reverse hybridization kit (INNO-LiPA Mycobacteria v2, Innogenetics NV, Belgium). A total of 21,060 samples obtained from 9660 patients were included to the study. After decontamination and homogenization processes, the samples were cultivated in automated MGIT Bactec 960 system and the diagnosis of atypical mycobacteria was performed in 4532 (21.5%) culture positive samples with NAP test by using Bactec 460 TB system. After DNA isolation, PCR method was performed by using the primers specific for mycobacterial 16S-23S spacer region. PCR products were then hybridized with the probes specific for Mycobacterium species on nitrocellulose strips according to the recommendations of the manufacturer and evaluated. Additionally, two different versions of another commercial Line Probe Assay (LiPA) kit [GenoType Mycobacterium and GenoType Mycobacterium AS (Additional Species), Hain Lifescience, Germany] were used for the detection of unidentified mycobacterial strains. In our study period, 10 different Mycobacterium species were identified from 44 (1%) respiratory tract samples (sputum, bronchial aspiration fluid, bronchoalveolar lavage) belonging to 30 patients. While repeated atypical mycobacterial growth was found in 13 patients on different days, 17 patients showed atypical mycobacterial growth only in one sample or in separate multiple samples taken within the same day. The species distribution among patients were as follows; M. fortuitum-M. peregrinum complex (n=5), M. intracellulare (n=4), M. avium complex (n=4), M. chelonae complex (n=4), M. gordonae (n=4), M. kansasii (n=3), M. szulgailintermedium (n=2), M. simiae (n=1) and M. scrofulaceum (n=1). Two of four samples which were unidentified by INNO-LiPA and GenoType MTBC were identified as M. szulgai/intermedium by GenoType Mycobacterium AS and the other two were found as unidentified atypical mycobacteria (Mycobacterium spp.). As a result, the frequency of atypical mycobacteria isolated in our hospital was thought to be low, however, species-level identification might be useful for the planning of therapy in such patients. In addition, after NAP test, INNO-LiPA and GenoType Mycobacterium were useful tests in microbiological identification of atypical mycobacteria, and GenoType Mycobacterium AS test could be applied in mycobacterial strains which were not identified by the former assays.'}, {'qas': [{'id': '32890724:Streptococcus pneumoniae', 'question': 'Can Streptococcus pneumoniae cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Risk factors for symptoms of infection and microbial carriage among French medical students abroad.\\nABSTRACT:OBJECTIVES: To investigate symptoms of infections and their risk factors among French medical students undertaking an internship abroad. \\n'}, {'qas': [{'id': '31504641:Rickettsia rickettsii', 'question': 'Can Rickettsia rickettsii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Rickettsia rickettsii (Rickettsiales: Rickettsiaceae) Infecting Amblyomma sculptum (Acari: Ixodidae) Ticks and Capybaras in a Brazilian Spotted Fever-Endemic Area of Brazil.\\nABSTRACT:The bacterium Rickettsia rickettsii is the agent of Brazilian spotted fever (BSF), a highly fatal disease that is transmitted in Brazil mainly by the tick Amblyomma sculptum Berlese, which uses capybaras (Hydrochoerus hydrochaeris Linnaeus) as major hosts. In 2015, we captured nine capybaras in a BSF-endemic area of southeastern Brazil. From each capybara, we collected blood sera that were tested through the immunofluorescence assay using Rickettsia spp. antigens, and A. sculptum ticks, processed for isolation of R. rickettsii through guinea pig inoculation. All capybaras (100%) were seroreactive to Rickettsia spp., with highest titers to R. rickettsii. A total of 166 A. sculptum ticks were macerated and inoculated into nine guinea pigs, from which only one presented high fever and seroconverted to R. rickettsii. Blood from this febrile animal was inoculated into additional guinea pigs (guinea pig passages), which also became febrile and seroconverted, or when euthanized during the fever period, their internal organs (spleen, lung) were shown to contain R. rickettsii DNA. The present rickettsial isolate has been maintained cryopreserved as infected guinea pig organs. There was at least one R. rickettsii-infected tick among the 166 macerated ticks, giving a minimal infection rate of 0.6% (1/166). This infection rate is within the range of previous studies, which reported that only 0.05% to at most 1.28% A. sculptum ticks were infected by R. rickettsii in other BSF-endemic areas. These low infection rates support the low incidence of BSF, despite of A. sculptum being the most frequent human-biting tick in southeastern Brazil.'}, {'qas': [{'id': '32642742:Mycobacterium orygis', 'question': 'Can Mycobacterium orygis cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Reconsidering Mycobacterium bovis as a proxy for zoonotic tuberculosis: a molecular epidemiological surveillance study.\\nABSTRACT:Background: Zoonotic tuberculosis is defined as human infection with Mycobacterium bovis. Although globally, India has the largest number of human tuberculosis cases and the largest cattle population, in which bovine tuberculosis is endemic, the burden of zoonotic tuberculosis is unknown. The aim of this study was to obtain estimates of the human prevalence of animal-associated members of the Mycobacterium tuberculosis complex (MTBC) at a large referral hospital in India. Methods: We did a molecular epidemiological surveillance study of 940 positive mycobacteria growth indicator tube (MGIT) cultures, collected from patients visiting the outpatient department at Christian Medical College (Vellore, India) with suspected tuberculosis between Oct 1, 2018, and March 31, 2019. A PCR-based approach was applied to subspeciate cultures. Isolates identified as MTBC other than M tuberculosis or as inconclusive on PCR were subject to whole-genome sequencing (WGS), and phylogenetically compared with publicly available MTBC sequences from south Asia. Sequences from WGS were deposited in the National Center for Biotechnology Information Sequence Read Archive, accession number SRP226525 (BioProject database number PRJNA575883). Findings: The 940 MGIT cultures were from 548 pulmonary and 392 extrapulmonary samples. A conclusive identification was obtained for all 940 isolates; wild-type M bovis was not identified. The isolates consisted of M tuberculosis (913 [97 1%] isolates), Mycobacterium orygis (seven [0 7%]), M bovis BCG (five [0 5%]), and non-tuberculous mycobacteria (15 [1 6%]). Subspecies were assigned for 25 isolates by WGS, which were analysed against 715 MTBC sequences from south Asia. Among the 715 genomes, no M bovis was identified. Four isolates of cattle origin were dispersed among human sequences within M tuberculosis lineage 1, and the seven M orygis isolates from human MGIT cultures were dispersed among sequences from cattle. Interpretation: M bovis prevalence in humans is an inadequate proxy of zoonotic tuberculosis. The recovery of M orygis from humans highlights the need to use a broadened definition, including MTBC subspecies such as M orygis, to investigate zoonotic tuberculosis. The identification of M tuberculosis in cattle also reinforces the need for One Health investigations in countries with endemic bovine tuberculosis. Funding: Bill & Melinda Gates Foundation, Canadian Institutes for Health Research.'}, {'qas': [{'id': '16836564:Staphylococcus microti', 'question': 'Can Staphylococcus microti cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Photochemical treatment of plasma with amotosalen and long-wavelength ultraviolet light inactivates pathogens while retaining coagulation function.\\nABSTRACT:\\n'}, {'qas': [{'id': '17039274:Actinomadura cremea', 'question': 'Can Actinomadura cremea cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Majority of Actinomadura clinical isolates from sputa or bronchoalveolar lavage fluid in Japan belongs to the cluster of Actinomadura cremea and Actinomadura nitritigenes, and the description of Actinomadura chibensis sp. nov.\\nABSTRACT:In Japan during 1996-2004, 21 actinomycete strains that have madurose as the diagnostic cell-wall sugar and show true branching in their substrate and aerial mycelia were isolated from sputa or bronchoalveolar lavage (BAL) fluid of patients with pulmonary infections or who were suspected of having related infections. Chemotaxonomic studies showed that all the isolates belong to the genus Actinomadura. Among them, six and seven strains were classified respectively into clusters of Actinomadura nitritigenes and Actinomadura cremea based on 16S rDNA analyses because their 16S rDNA similarities to those respective species were greater than 99.5%. To our knowledge, this is first report that strains of above two species were isolated from clinical specimens. Neither Actinomadura madurae nor Actinomadura pelletieri strain was isolated, and one new species, Actinomadura chibensis, was proposed; the remaining seven strains were not assigned into any known species, suggesting the presence of another new Actinomadura species.'}, {'qas': [{'id': '32023994:Alcaligenes faecalis', 'question': 'Can Alcaligenes faecalis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Microbiological Air Quality and Drug Resistance in Airborne Bacteria Isolated from a Waste Sorting Plant Located in Poland-A Case Study.\\nABSTRACT:International interests in biological air pollutants have increased rapidly to broaden the pool of knowledge on their identification and health impacts (e.g., infectious, respiratory diseases and allergies). Antibiotic resistance and its wider implications present us with a growing healthcare crisis, and an increased understanding of antibiotic-resistant bacteria populations should enable better interpretation of bioaerosol exposure found in the air. Waste sorting plant (WSP) activities are a source of occupational bacterial exposures that are associated with many health disorders. The objectives of this study were (a) to assess bacterial air quality (BAQ) in two cabins of a WSP: preliminary manual sorting cabin (PSP) and purification manual sorting cabin (quality control) (QCSP), (b) determine the particle size distribution (PSD) of bacterial aerosol (BA) in PSP, QCSP, and in the outdoor air (OUT), and (c) determine the antibiotic resistance of isolated strains of bacteria. Bacterial strains were identified on a Biolog GEN III (Biolog, Hayward, CA, USA), and disc diffusion method for antimicrobial susceptibility testing was carried out according to the Kirby-Bauer Disk Diffusion Susceptibility Test Protocol. A large share of fecal bacteria, Enterococcus faecalis and Alcaligenes faecalis spp. feacalis, was found in the tested indoor air, which is a potential health hazard to the workers of the monitored WSP. Our results demonstrate the necessity to take into account fecal air pollution levels to avoid making erroneous assumptions regarding the environmental selection of antibiotic resistance. Total elimination of many anthropogenic sources is not possible, but important findings of this study can be used to develop realistic management policies methods to improve BAQ.'}, {'qas': [{'id': '26432408:Slackia exigua', 'question': 'Can Slackia exigua cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:A large and unusual lung abscess with positive culture to Slackia exigua.\\nABSTRACT:'}, {'qas': [{'id': '24155565:Serratia odorifera', 'question': 'Can Serratia odorifera cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Bacterial pleuritis with thickened mesothelial hyperplasia in a young beagle dog.\\nABSTRACT:A five-month-old male beagle dog suddenly became moribund. Bloody fluid accumulated in the thoracic and abdominal cavities, and soft yellow flecks were floating in the thoracic fluid. The mediastinum and pericardium became dark reddish with villous thickening. Other parietal and pulmonary pleurae were rough, and the organs adhered to each other. Histologically, most mediastinal pleura formed papillary projections covered by a single layer of mesothelial cells. Many macrophages and neutrophils infiltrated the submesothelial connective tissue. At the mediastinum adjacent to the pericardium, cuboidal mesothelial cells proliferated solidly and formed a thick surface stratum. The flecks consisted of gram-negative filamentous or small bacillary (coccoid) bacteria. In the right posterior lobe of the lung, neutrophilic infiltration and a large encapsulated abscess including a bacterial colony were present. We diagnosed this case as \"bacterial pleuritis with thickened mesothelial hyperplasia\". The cause of the pleuritis might be a chronic pleural infection spread via the lung abscess. '}, {'qas': [{'id': '28847524:Desulfovibrio desulfuricans', 'question': 'Can Desulfovibrio desulfuricans cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:Understanding the response of Desulfovibrio desulfuricans ATCC 27774 to the electron acceptors nitrate and sulfate - biosynthetic costs modulate substrate selection.\\nABSTRACT:Sulfate-reducing bacteria (SRB) are a diverse group of anaerobic microorganisms that obtain their energy from dissimilatory sulfate reduction. Some SRB species have high respiratory versatility due to the possible use of alternative electron acceptors. A good example is Desulfovibrio desulfuricans ATCC 27774, which grows in the presence of nitrate (end product: ammonium) with higher rates and yields to those observed in sulfate containing medium (end product: sulfide). In this work, the mechanisms supporting the respiratory versatility of D. desulfuricans were unraveled through the analysis of the proteome of the bacterium under different experimental conditions. The most remarkable difference in the two-dimensional gel electrophoresis maps is the high number of spots exclusively represented in the nitrate medium. Most of the proteins with increase abundance are involved in the energy metabolism and the biosynthesis of amino acids (or proteins), especially those participating in ammonium assimilation processes. qPCR analysis performed during different stages of the bacterium's growth showed that the genes involved in nitrate and nitrite reduction (napA and nrfA, respectively) have different expressions profiles: while napA did not vary significantly, nrfA was highly expressed at a 6h time point. Nitrite levels measured along the growth curve revealed a peak at 3h. Thus, the initial consumption of nitrate and concomitant production of nitrite must induce nrfA expression. The activation of alternative mechanisms for energy production, aside several N-assimilation metabolisms and detoxification processes, solves potential survival problems in adapting to different environments and contributes to higher bacterial growth rates.\"}, {'qas': [{'id': '25356323:Pandoraea pulmonicola', 'question': 'Can Pandoraea pulmonicola cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Pandoraea pulmonicola chronic colonization in a cystic fibrosis patient, France.\\nABSTRACT:Pandoraea are considered emerging multidrug resistant pathogens in the context of cystic fibrosis. We report herein for the first time the case of a 30-year-old woman with cystic fibrosis, living in France, who was chronically infected with Pandoraea pulmonicola and who died of Pseudomonas aeruginosa sepsis 3 weeks after bilateral lung transplantation. '}, {'qas': [{'id': '17625172:Parascardovia', 'question': 'Can Parascardovia cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Alloscardovia omnicolens gen. nov., sp. nov., from human clinical samples.\\nABSTRACT:The taxonomic position of 12 isolates tentatively assigned to the genus Bifidobacterium on the basis of a limited phenotypic characterization was examined. The isolates were collected between 1978 and 2005 in Belgium, Sweden and Norway, and originated from various human clinical samples, including urine, blood, urethra, oral cavity, tonsil, and abscesses of lung and aortic valve. On the basis of band number and clustering analysis, repetitive DNA element-based PCR fingerprinting using the BOXA1R and (GTG)(5) primers indicated that the clinical isolates represented a taxon probably not belonging to the genus Bifidobacterium. Analysis of 16S rRNA gene sequence similarities revealed that the isolates were most closely affiliated to Parascardovia denticolens LMG 18312(T) (93.0-93.2 %), Scardovia inopinata LMG 18313(T) (92.9-93.1 %) and other members of the Bifidobacteriaceae, indicating that the isolates belong to a novel genus within that family. This observation was further substantiated by the results of partial sequencing of the heat-shock protein 60 gene (hsp60) and determination of the DNA G+C contents (47.3-48.3 mol%). Members of the novel taxon can be phenotypically distinguished from S. inopinata, P. denticolens and Gardnerella vaginalis by the ability to grow on agar under aerobic conditions and on the basis of positive reactions for acid production from L-arabinose, raffinose, salicin and D-xylose. Unambiguous phenotypic differentiation from Aeriscardovia aeriphila and Bifidobacterium species may be difficult, so phenotypic analyses should be complemented by molecular methods. The values for DNA-DNA binding among four members of the novel genus were in the range of 89-100 %, indicating that the strains should be considered as a single novel species of a novel genus, for which the name Alloscardovia omnicolens gen. nov., sp. nov. is proposed. The type strain of Alloscardovia omnicolens is CCUG 31649(T) (=LMG 23792(T)).'}, {'qas': [{'id': '33125314:Turicibacter', 'question': 'Can Turicibacter cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Culicoidibacter larvae gen. nov., sp. nov., from the gastrointestinal tract of the biting midge (Culicoides sonorensis) larva, belongs to a novel lineage Culicoidibacteraceae fam. nov., Culicoidibacterales ord. nov. and Culicoidibacteria classis nov. of the phylum Firmicutes.\\nABSTRACT:Strain CS-1T, a novel facultative anaerobic bacterium, was isolated from the larval gastrointestinal tract of the biting midge, Culicoides sonorensis, a vector of the epizootic haemorrhagic disease virus and the bluetongue virus. Cells were Gram-stain-positive, non-motile, non-spore-forming, pleomorphic rods. Optimal growth occurred at pH 7.5 and 37  C. The G+C content of the genomic DNA was 38.3 mol%, estimated by using HPLC. The dominant cellular fatty acids were C14 : 0 (45.9 %) and C16 : 0 (26.6 %). The polar lipid profile comprised glycolipids, diphosphatidylglycerol, phospholipids and phosphoglycolipids. Respiratory quinones were not detected. Strain CS-1T had very low 16S rRNA gene similarity to members of the phylum Firmicutes: Macrococcus canis KM45013T (85 % similarity) and Turicibacter sanguinis MOL361T (88 % similarity). Phylogenetic analysis based on 16S rRNA, rpoB, gyrB genes, and conserved protein sequences of the whole genome revealed that strain CS-1T was related to members of the classes Bacilli and Erysipelotrichia within the phylum Firmicutes. Furthermore, average nucleotide identity and digital DNA-DNA hybridization analyses of the whole genome revealed very low sequence similarity to species of Bacilli and Erysipelotrichaceae (Macrococcus canis KM45013T and Turicibacter sp. H121). These results indicate that strain CS-1T belongs to the phylum Firmicutes and represents a new species of a novel genus, family, order and class. Based on the phenotypic, chemotaxonomic, phylogenetic and genomic characteristics, we propose the novel taxon Culicoidibacter larvae gen. nov., sp. nov. with the type strain CS-1T (=CCUG 71726T=DSM 106607T) within the hereby new proposed novel family Culicoidibacteraceae fam. nov., new order Culicoidibacaterales ord. nov. and new class Culicoidibacteria classis nov. in the phylum Firmicutes.'}, {'qas': [{'id': '31274617:Gardnerella vaginalis', 'question': 'Can Gardnerella vaginalis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:FDG PET/CT of Gardnerella vaginalis Infection.\\nABSTRACT:We report the case of a 23-year-old woman with a history of cystic fibrosis and bilung transplantation, who presented clinically cervical swollen lymph nodes with alteration of her general state. F-FDG PET/CT was performed because of lymphoma suspicion and showed cervical and pelvic hypermetabolic lymphadenopathies, with linear vaginal hypermetabolism. There was an increase of lactate dehydrogenase, and Epstein-Barr virus detection was negative. A right cervical lymph node biopsy was performed, with no lymphoma involvement. Complementary microbiological investigations showed positive results for Gardnerella vaginalis. F-FDG PET/CT lymphatic node hypermetabolism is not specific to lymphoma, particularly in immunocompromised patients.'}, {'qas': [{'id': '6618987:Brucella ovis', 'question': 'Can Brucella ovis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Brucella ovis-induced abortion in ewes.\\nABSTRACT:Aborted fetuses and placental tissue from ewes in 2 flocks were submitted for diagnostic examinations. Brucella ovis was isolated from fetal lung or abomasal contents, or both. Placentitis was a consistent gross lesion. Microscopic lesions included suppurative pneumonia and placentitis.'}, {'qas': [{'id': '27506070:Streptococcus defectivus', 'question': 'Can Streptococcus defectivus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:[Abiotrophia defectiva: an unusual cause of endocarditis].\\nABSTRACT:This article reports one of the rare cases of Abiotrophia defectiva endocarditis with no underlying valvular condition. A sixty-three years old man was hospitalized because of complicated respiratory sepsis with acute heart failure. Hemocultures and echocardiogram enabled the diagnosis of A. defectiva endocarditis. The clinical course was favorable under combined aminoglycoside and cephalosporin. The patient ultimately required valvular replacement. A. defectiva is a micro-organism part of the Nutritionnaly Variant Streptococci (NVS) associated with a high mortality rate and often resistant to antibiotics. Although A. defectiva is a rare cause of endocarditis, prompt recognition and appropriate antibiotic treatment are essential to clinical course.'}, {'qas': [{'id': '30575735:Desulfovibrio vulgaris', 'question': 'Can Desulfovibrio vulgaris cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:An electrogenic redox loop in sulfate reduction reveals a likely widespread mechanism of energy conservation.\\nABSTRACT:The bioenergetics of anaerobic metabolism frequently relies on redox loops performed by membrane complexes with substrate- and quinone-binding sites on opposite sides of the membrane. However, in sulfate respiration (a key process in the biogeochemical sulfur cycle), the substrate- and quinone-binding sites of the QrcABCD complex are periplasmic, and their role in energy conservation has not been elucidated. Here we show that the QrcABCD complex of Desulfovibrio vulgaris is electrogenic, as protons and electrons required for quinone reduction are extracted from opposite sides of the membrane, with a H+/e- ratio of 1. Although the complex does not act as a H+-pump, QrcD may include a conserved proton channel leading from the N-side to the P-side menaquinone pocket. Our work provides evidence of how energy is conserved during dissimilatory sulfate reduction, and suggests mechanisms behind the functions of related bacterial respiratory complexes in other bioenergetic contexts.'}, {'qas': [{'id': '31751195:Pseudomonas oryzihabitans', 'question': 'Can Pseudomonas oryzihabitans cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Pseudomonas rhizoryzae sp. nov., isolated from rice.\\nABSTRACT:Two yellow-pigmented, Gram-stain-negative and rod-shaped bacterial strains, designated as RY24T and ZYY160, were isolated from rice. Results of phylogenetic analysis based on 16S rRNA gene sequences showed that strains RY24T and ZYY160 belonged to the genus Pseudomonas, and the 16S rRNA gene sequence similarity was 100 % The DNA homology between the two strains was 99.7 %. The 16S rRNA and rpoD gene sequences of the two strains showed highest similarity values to Pseudomonas oryzihabitans CGMCC 1.3392T and Pseudomonas psychrotolerans DSM 15758T (sharing 99.31 and 94.34 %, respectively). The major fatty acids of two strains were identified as summed feature 8 (C18:1omega7c and/or C18:1omega6c), C16;0 and summed feature 3 (C16:1omega7c and/or C16:1omega6c), and the major respiratory quinone was identified as ubiquinone Q-9, which are typical chemotaxonomic features of members of the genus Pseudomonas. The genomic DNA G+C contents of strains RY24T and ZYY160 were determined to be 64.25 and 64.21 mol%, respectively. The DNA-DNA relatedness and average nucleotide identity values between the two strains and their closely related type strains were below 36 and 90 %, which supported that RY24T and ZYY160 represent a novel species in the genus Pseudomonas. Phylogenetic and chemotaxonomic evidence, together with phenotypic characteristics, showed that the two isolates constitute a novel species of the genus Pseudomonas. The type strain is RY24T (JCM 33201T=ACCC 61555T), for which the name Pseudomonas rhizoryzae sp. nov. is proposed.'}, {'qas': [{'id': '401538:Clostridium hastiforme', 'question': 'Can Clostridium hastiforme cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:A special report: four-year study of a boy with combined immune deficiency maintained in strict reverse isolation from birth.\\nABSTRACT:A 4-year study of a boy with combined immune deficiency is presented, and the impact of this disease on various aspects of his growth and development is examined. There is no evidence of immune deficiency in either parent or in the genetic background on the maternal side. Three children of a brother of the mother's father may have had immune deficiencies but two have grown to be teenagers with no problems. Another died. At autopsy, however, lymph nodes appeared normal. The deceased older brother had severe combined immune deficiency (SCID). The autopsy findings showed Pneumocystis carinii pneumonia to be the direct cause of death and these findings contributed to the diagnosis of SCID. After a successful germ-free birth, the male infant (DV) was placed in the isolator. Laboratory tests were normal except that the x-rays showed no thymic shadow, his absolute lymphocyte count ranged from 399-440/mm and the lymphocytes showed no proliferative response to phytohemagglutinin (PHA). Specific tests showed the antibody-producing immune system and the cell-mediated immune system to be severely defective. The patient's lymphocytes elicited positive responses by lymphocytes from father, mother, and sister. Subsequent search in national and international tissue-typing laboratories has shown four HLA matches but none has been nonreactive in mixed lymphocyte culture (MLC). therefore, this patient has remained in isolation to the present; now he is 4 years old. Approximately 35 species of microorganisms, mostly transient contaminants, have been isolated, taking into account that the same organism may have been identified under different names in different laboratories. Those isolated frequently and in sufficiently high concentration to indicate colonization have been speciated as follows: anaerobes-Propionibacterium acnes, Lactobacillus catenaforme (disappeared spontaneously), Bacteroides oralis ss. elongatus, Clostridium (perenne, hastiforme, bifermentans), Bacteroides clostridiiformis ss. clostridiiformis; aerobes-Alcaligenes faecalis (eradicated by antibiotics), Staphylococcus epidermidis, Enterobacter agglomerans, Microcossu sp. subgroup 1, Bacillus pulvifaciens (disappeared spontaneously); yeasts-Candida (tropicalis, parapsilosis). Seven are considered to be probable components of the current autoflora: P. acnes, C. bifermentans, B. clostridiiformis ss. clostridiiformis, S. epidermidis, Micrococcus sp. subgroup 1, E. agglomerans, C. parapsilosis. No viruses or protozoa have been isolated. At age 3 years, the mean quantitation of anaerobic cells was 7.9 X 10(7) viable cells/g feces; this falls short of the mean anaerobic load from normal children. The mean aerobic concentration was 1.2 X 10(8) viable cells/g feces, not unlike normal children. Qualitatively his flora has abnormally few species and lacks those most common in normal subjects. This child has had no evidence of infection and has always been in excellent health even though some organisms which could be pathogenic under some circumstances have been isolated...\"}, {'qas': [{'id': '32329066:Porphyromonas intermedia', 'question': 'Can Porphyromonas intermedia cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Periodontitis and abdominal aortic aneurysm: Further evidence on the relationship between abdominal aortic aneurysm and periodontitis: a cross-sectional study.\\nABSTRACT:\\n'}, {'qas': [{'id': '16192681:Acinetobacter haemolyticus', 'question': 'Can Acinetobacter haemolyticus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Increasing ciprofloxacin resistance among prevalent urinary tract bacterial isolates in Gaza Strip, Palestine.\\nABSTRACT:This article presents the incidence of ciprofloxacin resistance among 480 clinical isolates obtained from patients with urinary tract infection (UTI) during January to June 2004 in Gaza Strip, Palestine. The resistance rates observed were 15.0% to ciprofloxacin, 82.5% to amoxycillin, 64.4% to cotrimoxazole, 63.1% to doxycycline, 32.5% to cephalexin, 31.9% to nalidixic acid, and 10.0% to amikacin. High resistance to ciprofloxacin was detected among Acinetobacter haemolyticus (28.6%), Staphylococcus saprophyticus (25.0%),Pseudomonas aeruginosa (20.0%), Klebsiella pneumonia (17.6%), and Escherichia coli (12.0%). Minimal inhibitory concentration (MIC) of ciprofloxacin evenly ranged from 4 to 32 mu g/mL with a mean of 25.0 mu g/mL. This study indicates emerging ciprofloxacin resistance among urinary tract infection isolates. Increasing resistance against ciprofloxacin demands coordinated monitoring of its activity and rational use of the antibiotics.'}, {'qas': [{'id': '32739532:Photobacterium damselae', 'question': 'Can Photobacterium damselae cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Effects of aqueous and ethanolic leaf extracts from drumstick tree (Moringa oleifera) on gilthead seabream (Sparus aurata L.) leucocytes, and their cytotoxic, antitumor, bactericidal and antioxidant activities.\\nABSTRACT:Aqueous and ethanolic extracts of drumstick, Moringa oleifera, leaves were evaluated in vitro to ascertain their principal active components and determine their immunostimulant, cytotoxic, antitumoral, bactericidal and antioxidant activities. Phytochemical screening of M. oleifera leaf extracts showed a greater abundance of phenolic and cyanogenic glycosides in aqueous than in ethanolic extracts, characterized by several flavonoids, condensed tannins and saponins. No significant effects on gilthead seabream (Sparus aurata) head-kidney leucocyte activities (phagocytic ability and capacity, respiratory burst and peroxidase) were detected after incubation for 24 h with different concentrations (0.001/1 mg mL-1) of either extract. In addition, the aqueous extract showed a marked cytotoxic effect on both SAF-1 (at doses above 0.01 mg mL-1) and PLHC-1 (at doses above 0.25 mg mL-1) cell lines. The ethanolic extract improved the viability of SAF-1 cells and decreased the viability of PLHC-1 cells when used at higher concentrations. Both the ethanolic and, particularly, the aqueous extracts showed significant bactericidal activity on pathogenic Vibrio anguillarum and Photobacterium damselae strains. The antiradical activity of M. oleifera, as determined by the ABTS assay, increased in a linear dose-response with increasing extract concentrations. The results as a whole for the cytotoxic, bactericidal and antioxidant activities of M. oleifera leaf extracts point to their possible use as additives in functional diets for farmed fish.'}, {'qas': [{'id': '8347507:Moraxella lincolnii', 'question': 'Can Moraxella lincolnii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:Moraxella lincolnii sp. nov., isolated from the human respiratory tract, and reevaluation of the taxonomic position of Moraxella osloensis.\\nABSTRACT:A polyphasic taxonomic study was performed to determine the relationships of 10 Moraxella-like strains isolated mainly from the human respiratory tract in Sweden. Two of the strains formed a separate subgroup on the basis of both their protein contents and their fatty acid contents. However, the overall protein and fatty acid profiles revealed that all 10 strains were highly related. Representative strains of the two subgroups exhibited high DNA binding values (98%) with each other and had an identical DNA base ratio (44 mol% G+C). DNA-rRNA hybridizations revealed that this taxon can be included in the genus Moraxella, which is only distantly related to phenotypically similar genera, such as the genera Neisseria and Kingella. The results of an extensive phenotypic analysis indicated that the general biochemical profile of the 10 strains conforms with the description of the genus Moraxella given in Bergey's Manual of Systematic Bacteriology. We therefore consider these organisms members of a new Moraxella species, for which the name Moraxella lincolnii is proposed. Furthermore, we also conclude that Moraxella osloensis belongs, genotypically as well as phenotypically, to the genus Moraxella.\"}, {'qas': [{'id': '26634165:Citrobacter koseri', 'question': 'Can Citrobacter koseri cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Community-Acquired Pneumonia and Empyema Caused by Citrobacter koseri in an Immunocompetent Patient.\\nABSTRACT:Citrobacter species, belonging to the family Enterobacteriaceae, are environmental organisms commonly found in soil, water, and the intestinal tracts of animals and humans. Citrobacter koseri is known to be an uncommon but serious cause of both sporadic and epidemic septicemia and meningitis in neonates and young infants. Most cases reported have occurred in immunocompromised hosts. The infections caused by Citrobacter are difficult to treat with usual broad spectrum antibiotics owing to rapid generation of mutants and have been associated with high death rates in the past. We believe this is the first case described in the literature of a community-acquired pneumonia and empyema caused by Citrobacter koseri in an immunocompetent adult patient. '}, {'qas': [{'id': '16885494:Dorea', 'question': 'Can Dorea cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Quality control of banked milk in Brasilia, Brazil.\\nABSTRACT:The authors studied quality control procedures at human milk banks and nutritional profiles of 909 milk samples (from 195 donors, aged 15 to 45 years) from banked human milk (BHM) in Brasilia, Brazil. Number of donations per donor ranged from 1 to &gt; 10 that consisted mostly of mature milk (90.9%) with a mean total energy of 529 +/- 85 kcal/L and a mean total lipid of 22.7 g/L +/- 13.2. Microbiological quality (titrable acidity-Dornic, degrees D) was suitable for infant feeding in 99.2% of samples (&lt; 8 degrees D), ranging from 2 degrees D to 8 degrees D (mean 4.8 +/- 1.4 degrees D). Most BHM (98.1%) samples were dispensed to inpatient infants (1-7 days) diagnosed with respiratory distress (30.1%), prematurity (20.7%), metabolic distress (16.0%), jaundice (14.4%), bacterial infection (6.0%), pneumonia (3.3%), congenital cardiac distress (2.2%), or other conditions (6.2%). Well-motivated mothers and trained staff are serving and sustaining an important life-saving network with long-lasting impact on public health.'}, {'qas': [{'id': '29377633:Desulfovibrio vulgaris', 'question': 'Can Desulfovibrio vulgaris cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Constraint-based modelling captures the metabolic versatility of Desulfovibrio vulgaris.\\nABSTRACT:A refined Desulfovibrio vulgaris Hildenborough flux balance analysis (FBA) model (iJF744) was developed, incorporating 1016 reactions that include 744 genes and 951 metabolites. A draft model was first developed through automatic model reconstruction using the ModelSeed Server and then curated based on existing literature. The curated model was further refined by incorporating three recently proposed redox reactions involving the Hdr-Flx and Qmo complexes and a lactate dehydrogenase (LdhAB, DVU 3027-3028) indicated by mutation and transcript analyses to serve electron transfer reactions central to syntrophic and respiratory growth. Eight different variations of this model were evaluated by comparing model predictions to experimental data determined for four different growth conditions - three for sulfate respiration (with lactate, pyruvate or H2 /CO2 -acetate) and one for fermentation in syntrophic coculture. The final general model supports (i) a role for Hdr-Flx in the oxidation of DsrC and ferredoxin, and reduction of NAD+ in a flavin-based electron confurcating reaction sequence, (ii) a function of the Qmo complex in receiving electrons from the menaquinone pool and potentially from ferredoxin to reduce APS and (iii) a reduction of the soluble DsrC by LdhAB and a function of DsrC in electron transfer reactions other than sulfite reduction.'}, {'qas': [{'id': '30356723:Klebsiella variicola', 'question': 'Can Klebsiella variicola cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Complete Genomic Analysis of a Kingdom-Crossing Klebsiella variicola Isolate.\\nABSTRACT:Bacterial isolate X39 was isolated from a community-acquired pneumonia patient in Beijing, China. A phylogenetic tree based on rpoB genes and average nucleotide identity data confirmed that isolate X39 belonged to Klebsiella variicola. The genome of K. variicola X39 contained one circular chromosome and nine plasmids. Comparative genomic analyses with other K. variicola isolates revealed that K. variicola X39 contained the most unique genes. Of these unique genes, many were prophages and transposases. Many virulence factors were shared between K. variicola X39 and Klebsiella pneumoniae F1. The pathogenicity of K. variicola X39 was compared with that of K. pneumoniae F1 in an abdominal infection model. The results indicated that K. variicola X39 was less virulent than typical clinical K. pneumoniae F1. The genome of K. variicola X39 also contained some genes involved in plant colonization, nitrogen fixation, and defense against oxidative stress. GFP-labeled K. variicola X39 could colonize maize as an endophytic bacterium. We concluded that K. variicola X39 was a kingdom-crossing strain.'}, {'qas': [{'id': '24120155:Bartonella koehlerae', 'question': 'Can Bartonella koehlerae cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Experimental infection of dogs with Bartonella henselae and Bartonella vinsonii subsp. berkhoffii.\\nABSTRACT:The lack of a suitable infection model remains an important obstacle for the pathophysiological understanding of Bartonella spp. The following pilot study was designed to determine whether cell culture-grown Bartonella henselae SA2 and Bartonella vinsonii subsp. berkhoffii genotype III would cause persistent bacteremia in dogs. Pre-inoculation screening established that two laboratory-raised Golden retrievers were naturally-infected with Bartonella koehlerae. Despite prior infection, one dog each was inoculated subcutaneously with 5 x 10(4)B. henselae (SA2 strain) or 3 x 10(4)B. vinsonii subsp. berkhoffii genotype III. Dogs were bled weekly for serological testing and culture using Bartonella alpha proteobacteria growth medium (BAPGM) diagnostic platform. Dog 1 seroconverted to B. henselae and Dog 2 seroconverted to B. vinsonii subsp. berkhoffii genotype III. Throughout the study period, Bartonella spp. DNA was neither amplified nor isolated in ante-mortem BAPGM enrichment blood cultures. B. henselae SA2 was isolated from a postmortem bone marrow from Dog 1 and B. koehlerae DNA was amplified from postmortem lung from Dog 2 following BAPGM enrichment culture. Limitations include lack of uninfected controls, a potentially suboptimal B. vinsonii subsp. berkhoffii inoculum and a relatively short duration of study. We conclude that following intradermal infection, sequestration of Bartonella spp. in tissues may limit diagnostic detection of these bacteria in dog blood samples.'}, {'qas': [{'id': '29719162:Achromobacter mucicolens', 'question': 'Can Achromobacter mucicolens cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Molecular and eco-physiological characterization of arsenic (As)-transforming Achromobacter sp. KAs 3-5T from As-contaminated groundwater of West Bengal, India.\\nABSTRACT:Molecular and eco-physiological characterization of arsenic (As)-transforming and hydrocarbon-utilizing Achromobacter type strain KAs 3-5T has been investigated in order to gain an insight into As-geomicrobiology in the contaminated groundwater. The bacterium is isolated from As-rich groundwater of West Bengal, India. Comparative 16S rRNA gene sequence phylogenetic analysis confirmed that the strain KAs 3-5T is closely related to Achromobacter mucicolens LMG 26685T (99.17%) and Achromobacter animicus LMG 26690T (99.17%), thus affiliated to the genus Achromobacter. Strain KAs 3-5T is nonflagellated, mesophilic, facultative anaerobe, having a broad metabolic repertoire of using various sugars, sugar-/fatty acids, hydrocarbons as principal carbon substrates, and O2, NO3-, NO2-, and Fe3+ as terminal electron acceptors. Growth with hydrocarbons led to cellular aggregation and adherence of the cells to the hydrocarbon particles confirmed through electron microscopic observations. The strain KAs 3-5T showed high As resistance (MIC of 5 mM for As3+, 25 mM for As5+) and reductive transformation of As5+ under aerobic conditions while utilizing both sugars and hydrocarbons. Molecular taxonomy specified a high genomic GC content (65.5 mol %), ubiquinone 8 (UQ-8) as respiratory quinone, spermidine as predominant polyamine in the bacterium. The differential presence of C12:0, C14:0 2-OH, C18:1 omega7c, and C 14:0 iso 3-OH/ C16:1 iso fatty acids, phosphatidylglycerol (PG), phosphatidylcholine (PC), two unknown phospholipid (PL1, PL2) as polar lipids, low DNA-DNA relatedness (33.0-41.0%) with the Achromobacter members, and unique metabolic capacities clearly indicated the distinct genomic and physiological properties of strain KAs 3-5T among known species of the genus Achromobacter. These findings lead to improve our understanding on metabolic flexibility of bacteria residing in As-contaminated groundwater and As-bacteria interactions within oligotrophic aquifer system.'}, {'qas': [{'id': '31723136:Geobacillus thermodenitrificans', 'question': 'Can Geobacillus thermodenitrificans cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Homologous bd oxidases share the same architecture but differ in mechanism.\\nABSTRACT:Cytochrome bd oxidases are terminal reductases of bacterial and archaeal respiratory chains. The enzyme couples the oxidation of ubiquinol or menaquinol with the reduction of dioxygen to water, thus contributing to the generation of the protonmotive force. Here, we determine the structure of the Escherichia coli bd oxidase treated with the specific inhibitor aurachin by cryo-electron microscopy (cryo-EM). The major subunits CydA and CydB are related by a pseudo two fold symmetry. The heme b and d cofactors are found in CydA, while ubiquinone-8 is bound at the homologous positions in CydB to stabilize its structure. The architecture of the E. coli enzyme is highly similar to that of Geobacillus thermodenitrificans, however, the positions of heme b595 and d are interchanged, and a common oxygen channel is blocked by a fourth subunit and substituted by a more narrow, alternative channel. Thus, with the same overall fold, the homologous enzymes exhibit a different mechanism.'}, {'qas': [{'id': '30212329:Neorickettsia helminthoeca', 'question': 'Can Neorickettsia helminthoeca cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:IMMUNOHISTOCHEMICAL AND MOLECULAR EVIDENCE OF PUTATIVE NEORICKETTSIA INFECTION IN COATIS ( NASUA NASUA) FROM SOUTHERN BRAZIL.\\nABSTRACT:The pathologic, molecular, and immunohistochemical findings associated with Neorickettsia helminthoeca are described in coatis ( Nasua nasua). Tissue sections (small intestine, lungs, kidney, liver, and spleen) of coatis ( n = 3) that died at the Bela Vista Biological Refuge, Foz do Iguacu, Parana, southern Brazil were routinely processed from histopathology. Selected formalin-fixed paraffin-embedded (FFPE) tissue sections of the small intestine, lungs, and spleen were used in an immunohistochemical (IHC) assay designed to identify the antigens of N. helminthoeca. Additionally, FFPE tissue sections of the small intestine were used to demonstrate antigens of canine parvovirus-2 (CPV-2) by IHC. Histopathology revealed chronic enteritis in all coatis. Parasitic enteritis was diagnosed in two coatis; one of these contained examples of a trematode within the lumen of the small intestine and the ovum of a trematode encysted in the intestinal mucosa. Other significant pathologic findings included interstitial pneumonia ( n = 2) and pyogranulomatous splenitis ( n = 1). Positive immunolabeling for N. helminthoeca was identified within macrophages of the small intestine and reticuloendothelial cells within the germinal centers of the spleen of all coatis; the intestinal trematode was N. helminthoeca IHC-positive. All pulmonary sections revealed negative immunolabeling for N. helminthoeca. Furthermore, the antigens of CPV-2 were not identified in the intestine of any coati. These findings indicate that these coatis were infected by N. helminthoeca, but since clinical and gross pathological findings were not recorded, it is uncertain if this pathogen produced clinical disease in this canid host; therefore, coatis may be asymptomatic or dead-end hosts for this organism.'}, {'qas': [{'id': '32369004:Pusillimonas', 'question': 'Can Pusillimonas cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Pusillimonas maritima sp. nov., isolated from surface seawater.\\nABSTRACT:Two Gram-stain-negative, short rod-shaped and non-flagellated strains, designated 17-4AT and L52-1-41, were isolated from the surface seawater of the Indian Ocean and South China Sea, respectively. The 16S rRNA genes of the two strains shared sequence similarity of 99.45 %. Strain 17-4AT shared the highest 16S rRNA gene similarity of 98.02 % with Pusillimonas caeni EBR-8-1T, followed by Pusillimonas noertemannii BN9T (97.47 %), Pusillimonas soli MJ07T (96.93 %), Parapusillimonas granuli Ch07T (96.68 %), Pusillimonas ginsengisoli DCY25T (96.65 %), Eoetvoesia caeni PB3-7BT (96.63 %), Paracandidimonas caeni 24T (96.34 %), Castellaniella defragrans 54PinT (96.28 %) and Pusillimonas harenae B201T (96.05 %). L52-1-41 shared the highest 16S rRNA gene similarity of 97.74 % with Pusillimonas caeni EBR-8-1T, followed by Pusillimonas noertemannii BN9T (97.47 %), Pusillimonas soli MJ07T (96.65 %), Parapusillimonas granuli Ch07T (96.41 %), Pusillimonas ginsengisoli DCY25T (96.37 %), Eoetvoesia caeni PB3-7BT (96.35 %), Pusillimonas harenae B201T (96.28 %), and Paracandidimonas caeni 24T (96.06 %). The results of phylogenetic analyses indicated that 17-4AT and L52-1-41 formed a stable, distinct and highly supported lineage affiliated to the genus Pusillimonas. The results of the digital DNA-DNA hybridization (dDDH) and average nucleotide identity (ANI) analyses indicated that they represented a single species. They featured similar genomic DNA G+C contents of 53.2-53.4 mol%. Activities of catalase and oxidase were negative for both strains. The fatty acids patterns of 17-4AT and L52-1-41 were most similar, mostly comprised of C16 : 0, C17 : 0cyclo, C18 : 0, C18 : 1omega9c and summed feature 8 (C18 : 1omega7c and/or C18 : 1  omega6c). The major polar lipids of the two strains were phosphatidylethanolamine, phosphatidylglycerol, diphosphatidylglycerol and unidentified aminolipids. The respiratory quinone of the two strains was Q-8. Hence, on the basis of the phenotypic, chemotaxonomic and genotypic data presented in this study, we proposed the classification of both strains as representatives of a novel species named Pusillimonas maritima sp. nov., with the type strain 17-4AT (=MCCC 1A12670T=KCTC 62121T=NBRC 113794T), and another strain L52-1-41 (=MCCC 1A05046=KCTC 52313).'}, {'qas': [{'id': '25230330:Clostridium sordellii', 'question': 'Can Clostridium sordellii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:View from the front lines: an emergency medicine perspective on clostridial infections in injection drug users.\\nABSTRACT:Injection drug use (IDU), specifically non-intravenous \"skin-popping\" of heroin, seems to provide optimal conditions for Clostridial infection and toxin production. IDU is therefore a major risk factor for wound botulism and Clostridial necrotizing soft tissue infections (NSTI) and continues to be linked to cases of tetanus. Case clusters of all 3 diseases have occurred among IDUs in Western U.S. and Europe. Medical personnel who care for the IDU population must be thoroughly familiar with the clinical presentation and management of these diseases. Wound botulism presents with bulbar symptoms and signs that are easily overlooked; rapid acquisition and administration of antitoxin can prevent neuromuscular respiratory failure. In addition to Clostridium perfringens, IDU-related NSTIs can be caused by Clostridium sordellii and Clostridium novyi, which may share a distinct clinical presentation. Early definitive NSTI management, which decreases mortality, requires a low index of suspicion on the part of emergency physicians and low threshold for surgical exploration and debridement on the part of the surgeon. Tetanus should be preventable in the IDU population through careful attention to vaccination status. '}, {'qas': [{'id': '28723007:Rhodococcus fascians', 'question': 'Can Rhodococcus fascians cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Rhodococcus Equi\\nABSTRACT:Human infection due to Rhodococcus equi was first reported in 1967 by a young man on immunosuppressant agents who was working in a stockyard. As the name implies, the infection is closely linked to animals, horses and foal are considered as the natural host. Other species of Rhodococcus similarly described as human pathogens include R. fascians, R. rhodochrous, and R. erythroplis. Rhodococcus belongs to the Norcadiaceae family which also comprises Nocardia, Mycobacterium, Corynebacterium, and Gordonia with some similarities among the group. Immunosuppression, especially defects in cell-mediated immunity, plays a major role in disease and is present in most reported cases even though infection has also been described less commonly in immunocompetent hosts. Common immunocompromised conditions where infection had been described include HIV, solid organ and stem cell transplant recipients, leukemia, lymphoma, lung cancer, and following chemotherapy, monoclonal antibodies, or prolonged steroid use.'}, {'qas': [{'id': '33412232:Vibrio vulnificus', 'question': 'Can Vibrio vulnificus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:A teleost bactericidal permeability-increasing protein-derived peptide that possesses a broad antibacterial spectrum and inhibits bacterial infection as well as human colon cancer cells growth.\\nABSTRACT:The bactericidal permeability-increasing protein (BPI) is a multifunctional cationic protein produced by neutrophils with antibacterial, antitumor, and LPS-neutralizing properties. In teleost, a number of BPIs have been reported, but their functions are very limited. In this study, an N-terminal peptide, BO18 (with 18 amino acids), derived from rock bream (Oplegnathus fasciatus) BPI, was synthesized and investigated for its antibacterial spectrum, action mechanism, immunoregulatory property as well as the inhibition effects on bacterial invasion and human colon cancer cells growth. The results showed that BO18 was active against Gram-positive bacteria Bscillus subiilis, Micrococcus luteus, and Staphylococcus aureus, as well as Gram-negative bacteria Vibrio alginolyticus, Vibrio litoralis, Vibrio parahaemolyticus and Vibrio vulnificus. BO18 treatment facilitated the bactericidal process of erythromycin and rifampicin by enhancing the permeability of the outer membrane. During its interaction with V. alginolyticus, BO18 exerted its antibacterial activity by destroying cell membrane integrity, penetrating into the cytoplasm and binding to genomic DNA and total RNA. In vitro analysis indicated BO18 could enhance the respiratory burst ability and regulate the expression of immune related genes of macrophages. In vivo detection showed the administration of fish with BO18 before bacterial infection significantly reduced pathogen dissemination and replication in tissues. In addition, BO18 exerted a cytotoxic effect on the growth of human colon cancer cells HT-29. Together, these results add new insights into the function of teleost BPIs, and support that BO18 is a novel and broad-spectrum antibacterial peptide with potential to apply in fighting pathogenic infection in aquaculture.'}, {'qas': [{'id': '32487494:Anaerostipes', 'question': 'Can Anaerostipes cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Development of the gut microbiota in early life: The impact of cystic fibrosis and antibiotic treatment.\\nABSTRACT:OBJECTIVES: Patients with Cystic Fibrosis (CF) suffer from pancreatic insufficiency, lipid malabsorption and gastrointestinal complaints, next to progressive pulmonary disease. Altered mucosal homoeostasis due to malfunctioning chloride channels results in an adapted microbial composition of the gastrointestinal and the respiratory tract. Additionally, antibiotic treatment has the potential to distort resident microbial communities dramatically. This study aims to investigate early life development of the gut microbial community composition of children with CF compared to healthy infants and to study the independent effects of antibiotics taking into account other clinical and lifestyle factors. STUDY DESIGN: Faecal samples from 20 infants with CF and 45 healthy infants were collected regularly during the first 18 months of life and microbial composition was determined using 16S rRNA based sequencing. \\n'}, {'qas': [{'id': '20648020:Saccharomonospora', 'question': 'Can Saccharomonospora cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:Genomics and the ancient origins of the daptomycin biosynthetic gene cluster.\\nABSTRACT:Daptomycin is a clinically useful lipopeptide antibiotic produced by Streptomyces roseosporus. The antibiotic is assembled by a nonribosomal peptide synthetase (NRPS) mechanism, and the cyclized tridecapeptide contains three non-proteinogenic-amino acids: ornithine (Orn), 3-methyl-glutamic acid (3mGlu) and kynurinine (Kyn). Daptomycin also has three D-amino acids. The two genes that encode proteins involved in the formation of 3mGlu and Kyn have no known orthologs, and hence they are good probes to search for daptomycin-like gene clusters among newly sequenced actinomycete genomes. A recent search with these genes revealed a daptomycin-like gene cluster in Saccharomonospora viridis, a causative agent for farmer's lung disease. S. viridis has a genome of 4.3 Mb, which is about one half the size of Streptomyces genomes. Searches for other NRPS and polyketide synthase (PKS) genes revealed only one other NRPS gene and no PKS genes in S. viridis. The orthologous lipopeptide biosynthetic genes and gene products in S. viridis and S. roseosporus have diverged extensively from a last common ancestor dating back to a pathway that had evolved over 1 billion years ago.\"}, {'qas': [{'id': '16014473:Nocardia veterana', 'question': 'Can Nocardia veterana cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Nocardia elegans sp. nov., a member of the Nocardia vaccinii clade isolated from sputum.\\nABSTRACT:Two bacterial isolates from the sputa of a patient with a pulmonary infection were subjected to a polyphasic taxonomic study. Chemotaxonomic investigations revealed the presence of cell-wall chemotype IV and mycolic acids consistent with the profile for the genus Nocardia. Comparative 16S rRNA gene sequencing showed that these isolates constitute a distinct subline within the genus Nocardia, displaying 99.6-95.5% sequence similarities with established species. However, DNA-DNA hybridization studies demonstrated unambiguously that the isolates are genealogically distinct from closely related species, namely Nocardia veterana and Nocardia africana, which show high levels of 16S rRNA sequence similarity (99.2 and 99.6% sequence similarity, respectively). On the basis of both phenotypic and phylogenetic evidence, it is proposed that these isolates be classified as a novel species of the genus Nocardia, for which the name Nocardia elegans sp. nov. is proposed. The type strain is IMMIB N-402(T) (=CCUG 50200(T)=CIP 108553(T)).'}, {'qas': [{'id': '8969997:Aeromonas sobria', 'question': 'Can Aeromonas sobria cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Near-drowning-associated Aeromonas pneumonia.\\nABSTRACT:Aeromonas is increasingly recognized as a human pathogen that causes a variety of different infections. Aeromonas has rarely been reported as a cause of respiratory infection, and it has been described in near-drowning-associated pneumonia. This article reviews a case of Aeromonas sobria pneumonia associated with a near drowning and considers the clinical and epidemiological characteristics of 10 previously reported cases. Nearly all of the cases involved young healthy men, a rapid development of pneumonia and sepsis after a brief stable period postimmersion, and bilateral infiltrates on chest radiography. A very high rate of positive blood cultures and mortality was also noted. The epidemiological and clinical data in this review may be helpful to the clinician caring for near-drowning victims. Although prophylactic antibiotics are not recommended for near-drowning victims, broad-spectrum antibiotics should be rapidly instituted with any evidence of infection.'}, {'qas': [{'id': '31330349:Myroides', 'question': 'Can Myroides cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Characteristics of biological particulate matters at urban and rural sites in the North China Plain.\\nABSTRACT:Depending on their concentrations, sizes, and types, particulate matters of biological origins (bioPM) significantly affect human health. However, for different air environments, they are not well characterized and can vary considerably. As an example, we investigated the bioPM differences at an urban (Beijing) site and a rural (Wangdu) site in the North China Plain (NCP) using an online monitoring instrument, an ultraviolet aerodynamic particle sizer (UV-APS), the limulus amebocyte lysate (LAL) assay, and a high-throughput sequencing method. Generally, lower concentrations of viable bioPM (hourly mean: 1.3 x 103 +- 1.6 x 103 m-3) and endotoxin (0.66 +- 0.16 EU/m3) in Beijing were observed compared to viable bioPM (0.79 x 105 +- 1.4 x 105 m-3) and endotoxin (15.1 +- 23.96 EU/m3) at the Wangdu site. The percentage of viable bioPM number concentration in the total PM was 3.1% in Beijing and 6.4% in Wangdu. Approximately 80% of viable bioPM was found to be in the range from 1 to 2.5 mum. Nevertheless, the size distribution patterns for viable bioPM at the Beijing and Wangdu sites differed and were affected by PM pollution, leading to distinct lung deposition profiles. Moreover, the distinct diurnal variations in viable bioPM on clean days were dimmed by the PM pollution at both sites. Distinct bacterial community structures were found in the air from the Beijing and Wangdu sites. The bacterial community in urban Beijing was dominated by genus Lactococcus (49.5%) and Pseudomonas (15.1%), while the rural Wangdu site was dominated by Enterococcus (65%) and Paenibacillus (10%). Human-derived genera, including Myroides, Streptococcus, Propionibacterium, Dietzia, Helcococcus, and Facklamia, were higher in Beijing, suggesting bacterial emission from humans in the urban air environment. Our results show that different air harbors different biological species, and people residing in different environments thus could have very different biological particle exposure.'}, {'qas': [{'id': '33221567:Corynebacterium urealyticum', 'question': 'Can Corynebacterium urealyticum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:The importance of surgical treatment in encrusted cystitis and pyelitis: A case report.\\nABSTRACT:INTRODUCTION: Encrusted cystitis and Encrusted pyelitis are rare chronic inflammatory diseases. Those conditions are commonly caused by the Corynebacterium spp. especially the type D2 which is a gram positive, aerobic, slow-growing, and urea-sliting bacteria with a multi-antibiotic resistant profile. PRESENTATION OF CASE: We report the case of a 62-year-old man with a past history of chronic obstructive pulmonary disease. He was referred to the department of urology for urosepsis. Bacterial culture results were positive to Corynebacterium urealyticum. The diagnosis of encrusted cystitis and pyelitis were highly considered. An adapted antibiotherapy was undertaken using vancomycin during 3 weeks. The patient presented two acute peritonitis : the first was caused by a spontaneous bladder dome rupture which was surgically repaired and the second was caused by a total bladder rupture which required cysto-prostatectomy and bilateral ureterostomy. The post operative outcomes were uneventful. Bacterial urinalysis was negative and total recovery was obtained. DISCUSSION: In the majority of the reported cases, there were no sepsis or peritonitis conditions. Medical treatment by the glycopeptides and urine acidification was sufficient. However in this case, the sepsis condition and the bladder rupture with acute peritonitis made exclusively medical treatment by antibiotics insufficient. Therefore cystectomy associated to conventional antibiotics were able to limit the systemic dissemination of the bacteria and save the patient's life. \\n\"}, {'qas': [{'id': '26276159:Myroides', 'question': 'Can Myroides cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Myroides indicus sp. nov., isolated from garden soil.\\nABSTRACT:A novel aerobic, non-motile, rod-shaped, catalase- and oxidase-positive bacterial strain, designated UKS3T, was isolated from garden soil, and subjected to polyphasic taxonomic analysis. Strain UKS3T formed whitish, viscous colonies on nutrient agar and was Gram-staining negative. Phylogenetic analysis, based on 16S rRNA gene sequence, showed that maximum pairwise similarity occurs with representatives of the genus Myroides. The most closely related species include Myroides marinus JS-08T (92.7 % sequence similarity), Myroides phaeus MY15T (92.7 %), Myroides odoratus DSM 2801T (91.5 %) and Myroides odoratimimus CCUG 39352T (91.4 %). Strain UKS3T contained menaquinone-6 (MK-6) as the major respiratory quinone and iso-C15 : 0 (40.2 %), anteiso-C15 : 0 (9.4 %) and iso-C17 : 0 3-OH (8.5 %) as major fatty acids. Phosphatidylethanolamine, phospholipids and three aminolipids were the major polar lipids. The DNA G+C content of strain UKS3T was 36.8 +- 2.0 mol%. On the basis of phenotypic, chemotaxonomic and molecular analysis, strain UKS3T represents a novel species of the genus Myroides, for which the name Myroides indicus sp. nov., is proposed. The type strain is UKS3T ( = DSM 28213T = NCIM 5555T ).'}, {'qas': [{'id': '30111628:Veillonella atypica', 'question': 'Can Veillonella atypica cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Tongue Microbiota and Oral Health Status in Community-Dwelling Elderly Adults.\\nABSTRACT:Tongue microbiota are a dominant source of oral microbial populations that are ingested with saliva, and therefore careful attention is required for the maintenance of health of elderly adults, who are susceptible to aspiration of oral contents. This study aimed to investigate the variation in tongue microbiota among community-dwelling elderly adults. Following a dental examination, tongue coating was collected from a 15-mm-diameter circular area at the center of the tongue dorsum of 506 elderly adults aged 70 to 80 years inhabiting the town of Hisayama, Japan. The microbial composition and density were determined by a 16S rRNA gene sequencing approach using a next-generation sequencer and quantitative PCR analysis, respectively. Co-occurrence network analysis identified two cohabiting groups of predominant commensals, one of which was primarily composed of Prevotella histicola, Veillonella atypica, Streptococcus salivarius, and Streptococcus parasanguinis; these organisms have been previously associated with an increased risk of mortality due to pneumonia in the frail elderly. This bacterial group was more predominant in the elderly with fewer teeth, a higher plaque index, and more dental caries experience, whereas the total bacterial density was independent of these traits. A higher density of fungi was also observed in the elderly with these traits, as well as in individuals who wore dentures. These results suggest that elderly adults with poorer oral health swallow a more dysbiotic microbiota formed on the tongue.IMPORTANCE Aspiration of oral contents can lead to pneumonia, which is a major cause of death among elderly adults susceptible to swallowing impairments. Tongue microbiota are a dominant source of oral microbial populations that are ingested with saliva. This large-scale population-based study revealed variations in the tongue microbiota among community-dwelling elderly adults. The total bacterial density was independent of the conditions of teeth surrounding the tongue, whereas the microbiota composition, especially the relative abundances of predominant commensals, showed an association with tooth conditions. Our results demonstrate that the elderly with fewer teeth, poorer dental hygiene, and more dental caries experience constantly ingest more dysbiotic microbiota, which could be harmful for their respiratory health.'}, {'qas': [{'id': '17594870:Serratia liquefaciens', 'question': 'Can Serratia liquefaciens cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Chronic granulomatous disease in pediatric patients: 25 years of experience.\\nABSTRACT:INTRODUCTION: Chronic granulomatous disease (CGD) is an uncommon primary immune deficiency (affecting 1/200,000 newborn infants) caused by a defect in phagocyte production of oxygen metabolites, and resulting in bacterial infections produced by catalase-positive microorganisms and fungal diseases that occasionally may prove fatal. \\n'}, {'qas': [{'id': '1921795:Legionella maceachernii', 'question': 'Can Legionella maceachernii cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Isolation of Legionella maceachernii from an immunocompromised patient with severe underlying lung disease.\\nABSTRACT:\\n'}, {'qas': [{'id': '29583112:Gordonibacter', 'question': 'Can Gordonibacter cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Ellagibacter isourolithinifaciens gen. nov., sp. nov., a new member of the family Eggerthellaceae, isolated from human gut.\\nABSTRACT:Urolithins are gut microbial metabolites that exert health benefits in vivo and are generated from ellagic acid (EA) and ellagitannin-containing foods such as strawberries, pomegranates and walnuts. Gordonibacter species produce some intermediary urolithins but the micro-organisms responsible for the transformation of EA into the final and more bioactive urolithins, such as urolithin A and isourolithin A, are unknown. We report here a new bacterium, capable of metabolizing EA into isourolithin A, isolated from healthy human faeces and characterized by determining phenotypic, biochemical and molecular methods. Strain CEBAS 4A belongs to the Eggerthellaceae family and differed from other genera of this family, both phylogenetically and phenotypically. Based on 16S rRNA gene sequence similarity, the strain was related to Enterorhabdus musicola DSM 19490T (92.9 % similarity), Enterorhabdus caecimuris DSM 21839T (92.7 % similarity), Adlercreutzia equolifaciens DSM 19450T (92.5 % similarity), Asaccharobacter celatus DSM 18785T (92.5 % similarity) and Parvibacter caecicola DSM 22242T (91.2 % similarity). This strain was strictly anaerobic and Gram-stain-positive. The whole-cell fatty acids were saturated (98.3 %), a very high percentage that differs from the nearest genera ranging from 62 to 73 %. The major respiratory lipoquinone was menaquinone-7 and the diamino acid in the peptidoglycan was meso-diaminopimelic acid. Diphosphatidylglycerol and phosphatidylglycerol comprised the main polar lipid profile in addition to several phosphoglycolipids (PGL1-2), phospholipids (PL1-4), glycolipids (GL1-6) and lipids. Based on these data, a new genus, Ellagibacter gen. nov. is proposed with one species, Ellagibacter isourolithinifaciens sp. nov. The type strain of Ellagibacter isourolithinifaciens is CEBAS 4AT (=DSM 104140T=CCUG 70284T).'}, {'qas': [{'id': '28348763:Streptococcus adjacens', 'question': 'Can Streptococcus adjacens cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Granulicatella adiacens prosthetic hip joint infection after dental treatment.\\nABSTRACT:INTRODUCTION: Granulicatella adiacens is a Gram-positive bacteria and a normal component of oral flora. It is also found in dental plaques, endodontic abscesses and can rarely cause more serious infections. CASE PRESENTATION: We describe a prosthetic hip joint infection in an 81-year-old fit and healthy man due to Granulicatella adiacens who underwent a prolonged dental intervention two days earlier without antibiotic prophylaxis. The infection was successfully treated with surgical intervention and a combination of antibiotics. The patient eventually succumbed to severe community-acquired pneumonia two months later. \\n'}, {'qas': [{'id': '31765515:Burkholderia ubonensis', 'question': 'Can Burkholderia ubonensis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Discovery of ubonodin, an antimicrobial lasso peptide active against members of the Burkholderia cepacia complex.\\nABSTRACT:We report the heterologous expression, structure, and antimicrobial activity of a lasso peptide, ubonodin, encoded in the genome of Burkholderia ubonensis . The topology of ubonodin is unprecedented amongst lasso peptides with 18 of its 28 amino acids found in the mechanically-bonded loop segment. Ubonodin inhibits RNA polymerase in vitro and has potent antimicrobial activity against several pathogenic members of the Burkholderia genus, most notably B. cepacia and B. multivorans , causative agents of lung infections in cystic fibrosis patients.'}, {'qas': [{'id': '29593062:Corynebacterium kroppenstedtii', 'question': 'Can Corynebacterium kroppenstedtii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Novel selective medium for the isolation of corynebacterium kroppenstedtii from heavily colonised clinical specimens.\\nABSTRACT:AIMS: Granulomatous mastitis due to Corynebacterium kroppenstedtii is an increasingly recognised cause of an indolent and distressing mastitis in non-lactating females. This slow-growing lipophilic organism is not reliably isolated using routine culture methods. A novel selective culture medium (CKSM) is designed to optimise the isolation of this organism from clinical specimens. \\n'}, {'qas': [{'id': '16967423:Mycobacterium marinum', 'question': 'Can Mycobacterium marinum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Reluctant to dive: coelomic effusion in a frog.\\nABSTRACT:An adult female, albino South African Clawed frog (Xenopus laevis) from a research colony at the Biological Resources Facility of the College of Agriculture and Life Sciences at North Carolina State University (NCSU) was presented with depression, lethargy, loss of diving reflex, and a distended abdomen. Cytologic examination of coelomic effusion fluid at the NCSU veterinary teaching hospital revealed a mixed population of inflammatory cells, including heterophils and a predominance of large mononuclear cells (macrophages) that often contained intracytoplasmic, negatively-stained, rod-shaped to filamentous organisms consistent with Mycobacterium sp. Ziehl-Neelsen stain revealed bright pink to red, acid-fast organisms with a beaded appearance. Histopathologic findings in tissues obtained at necropsy included marked, multifocal to coalescing, heterophilic, granulomatous and fibrinous coelomitis as well as severe multifocal heterophilic and granulomatous hepatitis, interstitial pneumonia and sinusitis/rhinitis. Slender gram-positive, acid-fast bacterial rods were identified in sections of coelomic pleura, kidneys, nasal cavities, spleen, liver, and pulmonary interstitium, indicative of systemic mycobacteriosis. Based on mycobacterial culture, the organism was identified as M marinum complex. Mycobacteria are variably gram-positive, often acid-fast, small rods that are ubiquitous in aquatic environments. The clinical and pathologic spectrum of disease in amphibians depends on host and pathogen status. Xenopus sp and several other frogs are good models for studying the pathogenesis of M tuberculosis infection. In addition to culture, polymerase chain reaction assays may be used for definitive identification of the organisms; accurate speciation may require further genetic investigation.'}, {'qas': [{'id': '30429139:Mycobacterium haemophilum', 'question': 'Can Mycobacterium haemophilum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Cutaneous Mycobacterial Infections.\\nABSTRACT:Humans encounter mycobacterial species due to their ubiquity in different environmental niches. In many individuals, pathogenic mycobacterial species may breach our first-line barrier defenses of the innate immune system and modulate the activation of phagocytes to cause disease of the respiratory tract or the skin and soft tissues, sometimes resulting in disseminated infection. Cutaneous mycobacterial infections may cause a wide range of clinical manifestations, which are divided into four main disease categories: (i) cutaneous manifestations of Mycobacterium tuberculosis infection, (ii) Buruli ulcer caused by Mycobacterium ulcerans and other related slowly growing mycobacteria, (iii) leprosy caused by Mycobacterium leprae and Mycobacterium lepromatosis, and (iv) cutaneous infections caused by rapidly growing mycobacteria. Clinically, cutaneous mycobacterial infections present with widely different clinical presentations, including cellulitis, nonhealing ulcers, subacute or chronic nodular lesions, abscesses, superficial lymphadenitis, verrucous lesions, and other types of findings. Mycobacterial infections of the skin and subcutaneous tissue are associated with important stigma, deformity, and disability. Geography-based environmental exposures influence the epidemiology of cutaneous mycobacterial infections. Cutaneous tuberculosis exhibits different clinical phenotypes acquired through different routes, including via extrinsic inoculation of the tuberculous bacilli and dissemination to the skin from other sites, or represents hypersensitivity reactions to M. tuberculosis infection. In many settings, leprosy remains an important cause of neurological impairment, deformity, limb loss, and stigma. Mycobacterium lepromatosis, a mycobacterial species related to M. leprae, is linked to diffuse lepromatous leprosy of Lucio and Latapi. Mycobacterium ulcerans produces a mycolactone toxin that leads to subcutaneous tissue destruction and immunosuppression, resulting in deep ulcerations that often produce substantial disfigurement and disability. Mycobacterium marinum, a close relative of M. ulcerans, is an important cause of cutaneous sporotrichoid nodular lymphangitic lesions. Among patients with advanced immunosuppression, Mycobacterium kansasii, the Mycobacterium avium-intracellulare complex, and Mycobacterium haemophilum may cause cutaneous or disseminated disease. Rapidly growing mycobacteria, including the Mycobacterium abscessus group, Mycobacterium chelonei, and Mycobacterium fortuitum, are increasingly recognized pathogens in cutaneous infections associated particularly with plastic surgery and cosmetic procedures. Skin biopsies of cutaneous lesions to identify acid-fast staining bacilli and cultures represent the cornerstone of diagnosis. Additionally, histopathological evaluation of skin biopsy specimens may be useful in identifying leprosy, Buruli ulcer, and cutaneous tuberculosis. Molecular assays are useful in some cases. The treatment for cutaneous mycobacterial infections depends on the specific pathogen and therefore requires a careful consideration of antimicrobial choices based on official treatment guidelines.'}, {'qas': [{'id': '24092787:Corynebacterium argentoratense', 'question': 'Can Corynebacterium argentoratense cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Whole-Genome Sequence of the Clinical Strain Corynebacterium argentoratense DSM 44202, Isolated from a Human Throat Specimen.\\nABSTRACT:Corynebacterium argentoratense is part of the human skin microbiota and is occasionally detected in the upper respiratory tract of patients suffering from tonsillitis. The complete DNA sequence of the type strain DSM 44202 comprises 2,031,902 bp, yielding the smallest genome sequenced thus far for a corynebacterium associated with humans. '}, {'qas': [{'id': '18554413:Tsukamurella pulmonis', 'question': 'Can Tsukamurella pulmonis cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Coexistence of primary adenocarcinoma of the lung and Tsukamurella infection: a case report and review of the literature.\\nABSTRACT:INTRODUCTION: A major diagnostic challenge in the evaluation of a cavitary lung lesion is to distinguish between infectious and malignant etiologies. CASE PRESENTATION: We present the case of an elderly man presenting with fever, hemoptysis and a left upper lobe cavitary lesion. Serial sputum cultures grew Tsukamurella pulmonis, a rare pathogen associated with cavitary pneumonia in immunocompromised patients. However, despite clinical improvement with antibiotic therapy targeted to the organism, concomitant discovery of a papillary thyroid carcinoma led to a needle biopsy of the cavitary lesion, which showed evidence of primary lung adenocarcinoma. \\n'}, {'qas': [{'id': '26513268:Campylobacter rectus', 'question': 'Can Campylobacter rectus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Periodontal Pathogens and Risk of Incident Cancer in Postmenopausal Females: The Buffalo OsteoPerio Study.\\nABSTRACT:\\n'}, {'qas': [{'id': '2407704:Campylobacter coli', 'question': 'Can Campylobacter coli cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Efficacy of a vaccine to prevent Chlamydia- or Campylobacter-induced abortions in ewes.\\nABSTRACT:In a sheep flock, Chlamydia psittaci, Campylobacter fetus, Ca jejuni, and Salmonella dublin caused abortions. A vaccine that contained C psittaci type I from 2 sources: a cow with pneumonia and an aborted ovine fetus, Ca fetus, Ca jejuni, and 4 strains of K99 Escherichia coli was given to 240 ewes before they were bred. All fetuses, placentas, and lambs, that died within 36 hours of birth were examined for infectious agents. Of 55 abortions, 30 (55%) were caused by Chlamydia or Campylobacter spp; 25 of the 30 (83%) abortions took place in the nonvaccinated group (n = 240). Forty-five more lambs survived in the vaccinated group than in the nonvaccinated group. Abortion rates for Chlamydia and Campylobacter spp (2.1 vs 10.4% in vaccinated and nonvaccinated groups, respectively) were significantly different (P = 0.003). Abortion rates for S dublin were not significantly different between groups. The Salmonella epizootic was controlled quickly by sanitation and treatment procedures. The vaccine was at least 80% efficacious against Chlamydia and Campylobacter spp and appeared to be protective.'}, {'qas': [{'id': '32874341:Pediococcus pentosaceus', 'question': 'Can Pediococcus pentosaceus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Assessment of the application for renewal of the authorisation of Pediococcus pentosaceus DSM 16244 as a feed additive for all animal species.\\nABSTRACT:Following a request from the European Commission, the Panel on Additives and Products or Substances used in Animal Feed (FEEDAP) was asked to deliver a scientific opinion on the assessment of the application for renewal of authorisation of Pediococcus pentosaceus DSM 16244 as a technological additive for all animal species. The applicant has provided evidence that the additive currently in the market complies with the existing conditions of authorisation. There was no new evidence that would lead the FEEDAP Panel to reconsider its previous conclusions. Thus, the Panel concluded that the additive remains safe for all animal species, consumer and the environment under the authorised conditions of use. The additive was not irritant to skin and eyes but considered a skin and respiratory sensitiser. The present application for renewal of the authorisation did not include a proposal for amending or supplementing the conditions of the original authorisation that would have an impact on the efficacy of the additive. Therefore, there was no need for assessing the efficacy of the additive in the context of the renewal of the authorisation.'}, {'qas': [{'id': '27535588:Achromobacter insuavis', 'question': 'Can Achromobacter insuavis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Achromobacter xylosoxidans is the predominant Achromobacter species isolated from diverse non-respiratory samples.\\nABSTRACT:Achromobacter spp. are emerging opportunistic Gram-negative rods responsible for diverse nosocomial or community-acquired infections. We describe, for the first time, the distribution of Achromobacter spp., defined by nrdA gene sequencing, and their antimicrobial susceptibility in a variety of non-respiratory samples recovered from hospitalized patients from 2010 to 2015. Of the 63 isolates studied, A. xylosoxidans was the most prevalent (41 isolates), and with the exception of A. insuavis (four isolates), the remaining 10 species identified were represented by one or two isolates only. All isolates were uniformly susceptible to piperacillin and piperacillin-tazobactam and 97% to meropenem, but 76% showed resistance to ciprofloxacin. This study confirms the diversity of Achromobacter spp. in non-cystic fibrosis (CF) isolates and the predominance of A. xylosoxidans, as previously reported for CF sputum isolates. There was no apparent link between the clinical site of infection and the species of Achromobacter.'}, {'qas': [{'id': '26632328:Legionella hackeliae', 'question': 'Can Legionella hackeliae cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': \"TITLE:Molecular diagnosis of Legionella infections--Clinical utility of front-line screening as part of a pneumonia diagnostic algorithm.\\nABSTRACT:OBJECTIVES: Urinary antigen testing for Legionella pneumophila serogroup 1 is the leading rapid diagnostic test for Legionnaires' Disease (LD); however other Legionella species and serogroups can also cause LD. The aim was to determine the utility of front-line L. pneumophila and Legionella species PCR in a severe respiratory infection algorithm. \\n\"}, {'qas': [{'id': '28924833:Clostridium limosum', 'question': 'Can Clostridium limosum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Rho-inhibiting C2IN-C3 fusion toxin inhibits chemotactic recruitment of human monocytes ex vivo and in mice in vivo.\\nABSTRACT:Bacterial protein toxins became valuable molecular tools for the targeted modulation of cell functions in experimental pharmacology and attractive therapeutics because of their potent and specific mode of action in human cells. C2IN-C3lim, a recombinant fusion toxin (~50 kDa) of the Rho-inhibiting C3lim from Clostridium (C.) limosum and a non-toxic portion of the C. botulinum C2 toxin (C2IN), is selectively internalized into the cytosol of monocytic cells where C3lim specifically ADP-ribosylates Rho A and -B, thereby inhibiting Rho-mediated signaling. Thus, we hypothesized that these unique features make C2IN-C3lim an attractive molecule for the targeted pharmacological down-regulation of Rho-mediated functions in monocytes. The analysis of the actin structure and the Rho ADP-ribosylation status implied that C2IN-C3lim entered the cytosol of primary human monocytes from healthy donors ex vivo within 1 h. Moreover, it inhibited the fMLP-induced chemotaxis of human monocytes in a Boyden chamber model ex vivo. Similarly, in a 3-dimensional ex vivo model of extravasation, single cell analysis revealed that C2IN-C3lim-treated cells were not able to move. In a clinically relevant mouse model of blunt chest trauma, the local application of C2IN-C3lim into the lungs after thorax trauma prevented the trauma-induced recruitment of monocytes into the lungs in vivo. Thus, C2IN-C3lim might be an attractive lead compound for novel pharmacological strategies to avoid the cellular damage response caused by monocytes in damaged tissue after trauma and during systemic inflammation. The results suggest that the pathophysiological role of clostridial C3 toxins might be a down-modulation of the innate immune system.'}, {'qas': [{'id': '30208875:Sutterella wadsworthensis', 'question': 'Can Sutterella wadsworthensis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:A metagenome-wide association study of gut microbiota in asthma in UK adults.\\nABSTRACT:\\n'}, {'qas': [{'id': '28164552:Moraxella atlantae', 'question': 'Can Moraxella atlantae cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Bacteremia and Bone Marrow Infection Caused by Moraxella Atlantae in an Elderly Patient with Pneumonia.\\nABSTRACT:The clinical manifestations of Moraxella Atlantae infection were rarely described. Here we reported an elderly pneumonia patient with Moraxella Atlantae infection and the detailed clinical manifestations were firstly described. A bacterial automatic identification system in combination with phenotypic methods can be routinely used to identify this pathogen. If possible, 16S rDNA gene sequencing is also an alternative and effective method.'}, {'qas': [{'id': '19940053:Nocardia pseudobrasiliensis', 'question': 'Can Nocardia pseudobrasiliensis cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Nocardia pseudobrasiliensis as an emerging cause of opportunistic infection after allogeneic hematopoietic stem cell transplantation.\\nABSTRACT:We report the case of a 55-year-old man who exhibited a nodular pneumonia 4 months after an allogeneic hematopoietic stem cell transplantation. Culture of the bronchoalveolar lavage fluid revealed Nocardia pseudobrasiliensis. This recently described carbapenem-resistant species should be included in the differential diagnosis of fungal infection in this setting.'}, {'qas': [{'id': '30802399:Corynebacterium xerosis', 'question': 'Can Corynebacterium xerosis cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:[The sensitivity to antibiotics of Corynebacterium non diphtheriae isolated in hospitals of Rostov-on-Don and the Rostovskaia oblast.]\\nABSTRACT:The purpose of study is to analyze sensitivity of various species Corynebacterium non diphtheriae to antibacterial pharmaceuticals. The strains of C. non diphtheriae are separated from patients with pathology of respiratory and urogenital tract and also from individuals subjected to preventive examination. The sensitivity to antibacterial pharmaceuticals was determined using technique of serial dilution in fluid growth medium on the basis values of minimal inhibitory concentration (mg per l). It is established that the most efficient antibacterial pharmaceuticals in the case of strains C. non diphtheriae proved to be Vancomycin, Cefazolin and Cefotaxime in general. In case of such particular species as S. pseudodiphtheriticum -- Cefazolin, Cefotaxime and Gentamycin; S.pseudotuberculosis -- Vancomycin, Cefazolin, Cefotaxime and Gentamycin; C. xerosis -- Cefotaxime; S.striatum -- Cefazolin and i Rifampicin. The least efficiency was manifested for strains C. non diphtheriae by Benzylpenicillin and Lincomycin in general. In case of such particular species as S. pseudodiphtheriticum and S. pseudotuberculosis -- Lincomycin and Erythromycin; C. xerosis and S. striatum -- Benzylpenicillin, Lincomycin and Erythromycin. In case of various species of C. non diphtheriae Sephalosporins (Cefotaxime and Cefazolin) can be recommended as pharmaceuticals of choice and Gentamycin and Vancomycin as reserve pharmaceuticals.'}, {'qas': [{'id': '19783614:Micrococcus lylae', 'question': 'Can Micrococcus lylae cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Agrococcus terreus sp. nov. and Micrococcus terreus sp. nov., isolated from forest soil.\\nABSTRACT:Two bacterial strains, DNG5T and V3M1T, isolated from forest soil of the Changbai mountains in China, were characterized using a polyphasic approach. Analysis of their 16S rRNA gene sequences indicated that strains DNG5T and V3M1T were phylogenetically related to members of the genus Agrococcus (96.0-98.4% similarity) and Micrococcus (96.7-98.0% similarity), respectively, within the order Actinomycetales. Strains DNG5T and V3M1T were Gram-stain-positive and strictly aerobic and formed yellow colonies on LB agar. Cells of strain DNG5T were short, non-motile rods, 0.4-0.5x0.8-1.0 microm. Strain DNG5T contained MK-10 and MK-11 as the major respiratory quinones and anteiso-C15:0 (49.2%) and iso-C16:0 (22.4%) as the major fatty acids. The diamino acid in the peptidoglycan of strain DNG5T was 2,4-diaminobutyric acid and the murein was of the acetyl type. Cells of strain V3M1T were cocci, 0.6-0.7 microm in diameter. The cell-wall peptidoglycan of strain V3M1T contained the amino acids lysine, glutamic acid, alanine and glycine. Strain V3M1T contained MK-7, MK-7(H2), MK-8 and MK-8(H2) as respiratory quinones and anteiso-C15:0 (78.2%) and iso-C15:0 (13.1%) as the major cellular fatty acids. The DNA G+C contents of strains DNG5T and V3M1T were 75.9 and 67.2 mol%, respectively. The DNA-DNA relatedness of strain DNG5T to Agrococcus jejuensis DSM 22002T, A. jenensis JCM 9950T, A. baldri JCM 12132T and A. citreus JCM 12398T was 58.3, 43.9, 36.1 and 54.1%, respectively. The DNA-DNA relatedness of strain V3M1T to Micrococcus luteus CGMCC 1.2299T, M. antarcticus CGMCC 1.2373T and M. lylae CGMCC 1.2300T was 57.5, 45.4 and 39.0%, respectively. Combining phenotypic and genotypic traits, strain DNG5T represents a novel species of the genus Agrococcus, for which the name Agrococcus terreus sp. nov. is proposed, with DNG5T (=CGMCC 1.6960T =NBRC 104260T) as the type strain. Strain V3M1T represents a novel species of the genus Micrococcus, for which the name Micrococcus terreus sp. nov. is proposed, with V3M1T (=CGMCC 1.7054T =NBRC 104258T) as the type strain.'}, {'qas': [{'id': '33210609:Mycobacterium pinnipedii', 'question': 'Can Mycobacterium pinnipedii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Tuberculosis in wild South American sea lions Otaria flavescens stranded in Chubut, Argentina.\\nABSTRACT:Pinniped tuberculosis, commonly caused by Mycobacterium pinnipedii, is a zoonotic disease reported in free-living and captive otariid species of the southern hemisphere. Currently, data concerning pinniped tuberculosis in South America are scarce, reinforcing the need for further studies of the disease in free-ranging pinnipeds. In this study, we investigated the presence of tuberculosis in South American sea lions Otaria flavescens (SASLs) stranded along the Chubut coastline (Argentina). Necropsies were performed in 9 SASLs, and tissue samples were collected for histopathology, bacteriology, and molecular diagnosis. Four SASLs showed enlarged tracheobronchial lymph nodes (TBLNs) with multifocal to coalescing granulomas. In these animals, a direct IS6110-PCR amplification confirmed the presence of a Mycobacterium tuberculosis complex member in TBLNs (n = 4) and lungs (n = 2), but the agent could not be further identified. In one SASL, Mycobacterium murale was isolated from lungs without lesions. This study confirms the presence of tuberculosis in SASLs from Chubut, where tourist activities promote close interaction with the animals, generating a potential risk to human health. Further research is currently focusing on addressing the prevalence of tuberculosis in wild SASLs, to assess the risk for public health and develop management strategies to avoid human infection.'}, {'qas': [{'id': '29441164:Mycobacterium nebraskense', 'question': 'Can Mycobacterium nebraskense cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Rare case of Mycobacterium nebraskense presenting as asymptomatic cavitary lung lesion.\\nABSTRACT:Introduction: Mycobacterium Nebraskense is a rare nontuberculous mycobacterial infection. The first isolate of the species was from human sputum at University of Nebraska Medical Center. There are only a few cases have been reported and the exact behavior of the disease is not clearly described. Here, we present a case from New York City incidentally found to have a cavitary lung lesion due to M. nebraskense.Case report: An 82-year-old female with a history of diabetes mellitus, hypertension, and dementia presented with constipation and urinary retention for 1 day. She had no fever, cough, shortness of breath, nausea, vomiting, appetite change, or weight loss. Computed tomography (CT) scan of abdomen and pelvis revealed retained fecal material in the colon, non-obstructing left renal calculus, and bilateral small pleural effusion with right-sided lung infiltrates. Subsequent CT scan of the chest showed 4.5 cm pleural-based opacity in right lung base with a small cavity. Sputum smear for Acid-fast bacilli was positive. Mycobacterial culture reported positive growth of M. nebraskense, while polymerase chain reaction returned negative for Mycobacterium gordonae, Mycobacterium kansasii, Mycobacterium avium complex and Mycobacterium tuberculosis. With the patient asymptomatic and her constipation improved, she was discharged with plans for close follow-up as outpatient. Conclusion: M. nebraskense is a very rare nontuberculous mycobacterial infection. From only a few cases reported in the USA, the exact presentation of infection, disease progression, and treatment have not been described well. Asymptomatic cavitary lung disease caused by M. nebraskense has not been reported before.'}, {'qas': [{'id': '29872152:Corynebacterium jeikeium', 'question': 'Can Corynebacterium jeikeium cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Multidrug resistant pathogens respond differently to the presence of co-pathogen, commensal, probiotic and host cells.\\nABSTRACT:In light of the ongoing antimicrobial resistance crisis, there is a need to understand the role of co-pathogens, commensals, and the local microbiome in modulating virulence and antibiotic resistance. To identify possible interactions that influence the expression of virulence or survival mechanisms in both the multidrug-resistant organisms (MDROs) and human host cells, unique cohorts of clinical isolates were selected for whole genome sequencing with enhanced assembly and full annotation, pairwise co-culturing, and transcriptome profiling. The MDROs were co-cultured in pairwise combinations either with: (1) another MDRO, (2) skin commensals (Staphylococcus epidermidis and Corynebacterium jeikeium), (3) the common probiotic Lactobacillus reuteri, and (4) human fibroblasts. RNA-Seq analysis showed distinct regulation of virulence and antimicrobial resistance gene responses across different combinations of MDROs, commensals, and human cells. Co-culture assays demonstrated that microbial interactions can modulate gene responses of both the target and pathogen/commensal species, and that the responses are specific to the identity of the pathogen/commensal species. In summary, bacteria have mechanisms to distinguish between friends, foe and host cells. These results provide foundational data and insight into the possibility of manipulating the local microbiome when treating complicated polymicrobial wound, intra-abdominal, or respiratory infections.'}, {'qas': [{'id': '30568479:Arcanobacterium haemolyticum', 'question': 'Can Arcanobacterium haemolyticum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Spectrum of bactericidal action of amylmetacresol/2,4-dichlorobenzyl alcohol lozenges against oropharyngeal organisms implicated in pharyngitis.\\nABSTRACT:Purpose: Pharyngitis is commonly caused by a self-limiting upper respiratory tract infection (URTI) and symptoms typically include sore throat. Antibiotics are often inappropriately used for the treatment of pharyngitis, which can contribute to antimicrobial resistance, therefore non-antibiotic treatments which have broad antiseptic effects may be more appropriate. Amylmetacresol (AMC) and 2,4-dichlorobenzyl alcohol (DCBA) are present in some antiseptic lozenges and have established benefits in providing symptomatic relief and some in vitro antiviral action. Methods: Seven bacterial species associated with pharyngitis, namely Streptococcus pyogenes, Fusobacterium necrophorum, Streptococcus dysgalactiae subspecies equisimilis, Moraxella catarrhalis, Haemophilus influenza, Arcanobacterium haemolyticum and Staphylococcus aureus, were exposed to an AMC/DCBA lozenge dissolved in artificial saliva. In vitro bactericidal activity was measured as a log reduction in colony-forming units (CFUs). Results: Bactericidal activity was recorded against all organisms after 1 minute. Greater than 3 log10 reductions in CFUs were observed at 1 minute for S. pyogenes (log10 reduction CFU/mL +- SD, 5.7+-0.1), H. influenza (6.1+-0.1), A. haemolyticum (6.5+-0.0) and F. necrophorum (6.5+-0.0), at 5 minutes for S. dysgalactiae (6.3+-0.0) and M. catarrhalis (5.0+-0.9) and at 10 minutes for S. aureus (3.5+-0.1). Conclusion: An AMC/DCBA lozenge demonstrated a greater than 99.9% reduction in CFUs against all tested species within 10 minutes, which is consistent with the time a lozenge remains in the mouth. Patients with uncomplicated bacterial pharyngitis may benefit from the antibacterial action of antiseptic AMC/DCBA lozenges. Furthermore, AMC/DCBA lozenges may be more relevant and appropriate than antibiotics for pharyngitis associated with a self-limiting viral URTI.'}, {'qas': [{'id': '19860823:Nocardia takedensis ', 'question': 'Can Nocardia takedensis  cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Clinical and microbiological characteristics of Nocardiosis including those caused by emerging Nocardia species in Taiwan, 1998-2008.\\nABSTRACT:The genus of Nocardia is rapidly expanding and the species distribution varies with different geographical locations. We retrospectively reviewed the laboratory records of the bacteriology laboratory at National Taiwan University Hospital from January 1998 to June 2008 to identify patients with nocardiosis. During the study period, 164 isolates of Nocardia spp. were identified from 134 patients but only 113 patients had Nocardia infection. Nocardia brasiliensis (n = 54) was the most common pathogen, followed by N. asteroides (n = 36), N. farcinica (n = 7), N. flavorosea (n = 4), N. otitidiscaviarum (n = 3), N. nova (n = 3), N. beijingensis (n = 2) and one each of N. puris, N. jinanensis and N. takedensis. The major types of infection were cutaneous infection (56.6%), pulmonary infection (33.6%) and disseminated infection (7.1%). Eighty-eight patients received sulfonamide-containing antibiotic and eight of 100 patients with available data on outcomes died during the episode of nocardiosis. In conclusion, the clinical and microbiological manifestations of Nocardiosis vary with the different Nocardia species. Accurate identification of the species is crucial to make the diagnosis.'}, {'qas': [{'id': '33412232:Vibrio alginolyticus', 'question': 'Can Vibrio alginolyticus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:A teleost bactericidal permeability-increasing protein-derived peptide that possesses a broad antibacterial spectrum and inhibits bacterial infection as well as human colon cancer cells growth.\\nABSTRACT:The bactericidal permeability-increasing protein (BPI) is a multifunctional cationic protein produced by neutrophils with antibacterial, antitumor, and LPS-neutralizing properties. In teleost, a number of BPIs have been reported, but their functions are very limited. In this study, an N-terminal peptide, BO18 (with 18 amino acids), derived from rock bream (Oplegnathus fasciatus) BPI, was synthesized and investigated for its antibacterial spectrum, action mechanism, immunoregulatory property as well as the inhibition effects on bacterial invasion and human colon cancer cells growth. The results showed that BO18 was active against Gram-positive bacteria Bscillus subiilis, Micrococcus luteus, and Staphylococcus aureus, as well as Gram-negative bacteria Vibrio alginolyticus, Vibrio litoralis, Vibrio parahaemolyticus and Vibrio vulnificus. BO18 treatment facilitated the bactericidal process of erythromycin and rifampicin by enhancing the permeability of the outer membrane. During its interaction with V. alginolyticus, BO18 exerted its antibacterial activity by destroying cell membrane integrity, penetrating into the cytoplasm and binding to genomic DNA and total RNA. In vitro analysis indicated BO18 could enhance the respiratory burst ability and regulate the expression of immune related genes of macrophages. In vivo detection showed the administration of fish with BO18 before bacterial infection significantly reduced pathogen dissemination and replication in tissues. In addition, BO18 exerted a cytotoxic effect on the growth of human colon cancer cells HT-29. Together, these results add new insights into the function of teleost BPIs, and support that BO18 is a novel and broad-spectrum antibacterial peptide with potential to apply in fighting pathogenic infection in aquaculture.'}, {'qas': [{'id': '6756238:Legionella wadsworthii', 'question': 'Can Legionella wadsworthii cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Legionella wadsworthii species nova: a cause of human pneumonia.\\nABSTRACT:A new species of Legionella that caused pneumonia in a patient with chronic lymphocytic leukemia has been characterized. The Legionella wadsworthii species nova is proposed for this bacterium. The type strain is Wadsworth 81-716A (American Type Culture Collection 33877). The new species differs phenotypically from L. pneumophila in that the predominant cellular fatty acid is methyl-12-methyltetradecanoic acid (a-15:0) rather than methyl-14-methylpentadecanoic acid (i-16:0), and in its failure to react with L. pneumophila antiserum. The clinical illness caused by L. wadsworthii was not apparently different from that seen with other legionella infections, except for prolonged fever and slow resolution of pulmonary infiltrates.'}, {'qas': [{'id': '20720366:Mycobacterium parascrofulaceum', 'question': 'Can Mycobacterium parascrofulaceum cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Pulmonary Mycobacterium parascrofulaceum infection as an immune reconstitution inflammatory syndrome in an AIDS patient.\\nABSTRACT:Nontuberculous mycobacterial (NTM) infection in HIV (human immunodeficiency virus)-infected patients who have started highly active antiretroviral therapy (HAART) is well known to be one scenario of immune reconstitution inflammatory syndrome (IRIS). We encountered the first case in Japan of an HIV-infected patient with pulmonary Mycobacterium parascrofulaceum infection as IRIS. A 34 year-old man with acquired immunodeficiency syndrome (AIDS) was receiving highly active antiretroviral therapy. Lymphadenopathy was observed at the left pulmonary hilum. IRIS was suspected and thoracoscopic surgery was performed to diagnose the cause of lymphadenopathy. Granulomas were observed histologically, and M. parascrofulaceum was cultured. This organism was susceptible to Clarithromycin, rifampicin and levofloxacin. After the operation and without treatment, recurrence of M. parascrofulaceum infection was not observed. M. parascrofulaceum was isolated from several clinical specimens for the first time in 2004. To date, only five cases have been reported.'}, {'qas': [{'id': '33412232:Vibrio alginolyticus', 'question': 'Can Vibrio alginolyticus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:A teleost bactericidal permeability-increasing protein-derived peptide that possesses a broad antibacterial spectrum and inhibits bacterial infection as well as human colon cancer cells growth.\\nABSTRACT:The bactericidal permeability-increasing protein (BPI) is a multifunctional cationic protein produced by neutrophils with antibacterial, antitumor, and LPS-neutralizing properties. In teleost, a number of BPIs have been reported, but their functions are very limited. In this study, an N-terminal peptide, BO18 (with 18 amino acids), derived from rock bream (Oplegnathus fasciatus) BPI, was synthesized and investigated for its antibacterial spectrum, action mechanism, immunoregulatory property as well as the inhibition effects on bacterial invasion and human colon cancer cells growth. The results showed that BO18 was active against Gram-positive bacteria Bscillus subiilis, Micrococcus luteus, and Staphylococcus aureus, as well as Gram-negative bacteria Vibrio alginolyticus, Vibrio litoralis, Vibrio parahaemolyticus and Vibrio vulnificus. BO18 treatment facilitated the bactericidal process of erythromycin and rifampicin by enhancing the permeability of the outer membrane. During its interaction with V. alginolyticus, BO18 exerted its antibacterial activity by destroying cell membrane integrity, penetrating into the cytoplasm and binding to genomic DNA and total RNA. In vitro analysis indicated BO18 could enhance the respiratory burst ability and regulate the expression of immune related genes of macrophages. In vivo detection showed the administration of fish with BO18 before bacterial infection significantly reduced pathogen dissemination and replication in tissues. In addition, BO18 exerted a cytotoxic effect on the growth of human colon cancer cells HT-29. Together, these results add new insights into the function of teleost BPIs, and support that BO18 is a novel and broad-spectrum antibacterial peptide with potential to apply in fighting pathogenic infection in aquaculture.'}, {'qas': [{'id': '22782695:Borrelia persica', 'question': 'Can Borrelia persica cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Aseptic meningitis and adult respiratory distress syndrome caused by Borrelia persica.\\nABSTRACT:INTRODUCTION: The occurrence of some of the clinical complications of tickborne relapsing fever  varies with Borrelia species. For example, adult respiratory distress syndrome (ARDS), a newly reported complication, was described so far only with B. hermsii infection. MATERIALS AND \\n'}, {'qas': [{'id': '9373597:Campylobacter avium', 'question': 'Can Campylobacter avium cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Microbiologically proven bacterial infections in AIDS.\\nABSTRACT:We have reviewed the incidence, type and site of microbiologically proven bacterial infection occurring in 52 patients with the acquired immunodeficiency syndrome (AIDS) who presented to Southmead Hospital, Bristol between 1990 and 1994. A total of 30 (58%) patients had significant bacterial isolates. The majority of infections were community acquired. Overall, more infections were caused by Gram-negative organisms but Gram-positive organisms predominated in bacteraemia. Mycobacterium avium intracellulare (MAI) caused infection in the largest number of patients, followed by Staphylococcus aureus, Streptococcus pneumoniae, Pseudomonas sp, and Campylobacter sp. When individual episodes of infection were considered, after MAI, Haemophilus influenzae, Streptococcus pneumoniae and Pseudomonas sp were the organisms most frequently isolated; often these same organisms caused recurrent chest infection. Bacterial infections in AIDS patients are common and although they generally respond well to antimicrobial chemotherapy there is a high recurrence rate, particularly in the respiratory tract, which is the commonest site of infection.'}, {'qas': [{'id': '29984149:Mycobacterium asiaticum', 'question': 'Can Mycobacterium asiaticum cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:A case of pulmonary infection caused by Mycobacterium asiaticum: Difficulties on diagnostic and therapeutic approaches.\\nABSTRACT:A 56-year-old previously healthy female presented with chronic productive cough and fever. Chest X-ray revealed right middle lung opacities. Sputum acid fast bacilli smear was positive, however polymerase chain reaction was negative for Mycobacterium tuberculosis. Further investigations were pursued, which identified Mycobacterium asiaticum. Appropriate therapy with isoniazid, rifampin and clarithromycin for total 18 months (including pyrazinamide and ethambutol for first 4 months) resulted in clinical and radiographic improvement. Recognition of the possibility of this rarely described pulmonary pathogen is essential for successful treatment.'}, {'qas': [{'id': '31731086:Mycobacterium triplex', 'question': 'Can Mycobacterium triplex cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:The clinical significance of Mycobacterium triplex.\\nABSTRACT:OBJECTIVES: Mycobacterium triplex is a slow-growing nontuberculous mycobacteria (NTM) and is a rare cause of human disease. The pathogenicity, natural history and spectrum of disease is unknown. The aim of this study was to review the clinical features, outcomes and drug susceptibility testing (DST) of all M. triplex isolates in Queensland, Australia to guide management of this rare NTM infection in the future. \\n'}, {'qas': [{'id': '29980746:Streptomyces griseus', 'question': 'Can Streptomyces griseus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Dolyemycins A and B, two novel cyclopeptides isolated from Streptomyces griseus subsp. griseus HYS31.\\nABSTRACT:Two novel cyclopeptides with special skeleton, namely, dolyemycins A (1) and B (2) were isolated from Streptomyces griseus subsp. griseus HYS31 by bio-guided isolation. Their structures were elucidated by detailed analysis of spectroscopic data. These two compounds were cyclopeptides containing eleven amino acids including five unusual amino acids (hydroxyglycine, 3-hydroxyleucine, 3-phenylserine, beta-hydroxy-O-methyltyrosine, 2,3-diaminobutyric acid) in both of them and an extra nonprotein amino acids (3-methylaspartic acid) in Dolyemycin B only. Dolyemycins A and B performed antiproliferative activity against human lung cancer A549 cells with IC50 values of 1.0 and 1.2 microM, respectively.'}, {'qas': [{'id': '22556323:Nocardiopsis dassonvillei', 'question': 'Can Nocardiopsis dassonvillei cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Nasal vestibulitis due to Nocardiopsis dassonvillei in a diabetic patient.\\nABSTRACT:Human infection due to Nocardiopsis, an actinomycete, is rare and the majority of those infections are due to Nocardiopsis dassonvillei. This agent has been implicated in cutaneous, pulmonary, eye and disseminated infections. It has never been isolated from the nose or any nasal infection. We report here a rare case of nasal vestibular abscess due to N. dassonvillei in an adult diabetic patient. The bacterium was identified on the basis of morphological and biochemical characteristics, and confirmed by sequencing the 16S rRNA and hsp65 genes. The patient was successfully treated with clarithromycin and levofloxacin. Though N. dassonvillei infections may be rare, the study highlights that it may cause a wide spectrum of disease manifestations, and laboratories should take care to isolate and identify the easily treatable pathogen.'}, {'qas': [{'id': '19552995:Dialister', 'question': 'Can Dialister cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:[Pneumonia due to Dialister pneumosintes in a patient with chronic obstructive pulmonary disease].\\nABSTRACT:'}, {'qas': [{'id': '24135316:Haematobacter massiliensis', 'question': 'Can Haematobacter massiliensis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Paenirhodobacter enshiensis gen. nov., sp. nov., a non-photosynthetic bacterium isolated from soil, and emended descriptions of the genera Rhodobacter and Haematobacter.\\nABSTRACT:A Gram-reaction-negative, facultatively anaerobic, non-motile, rod-shaped, non-photosynthetic bacterial strain, DW2-9(T), was isolated from soil. The highest 16S rRNA gene sequence similarities were found to Rhodobacter capsulatus ATCC 11166(T) (97.1%), Rhodobacter viridis JA737(T) (96.4%), Rhodobacter maris JA276(T) (96.2%), Rhodobacter veldkampii ATCC 35703(T) (96.0%), Haematobacter massiliensis CCUG 47968(T) (96.0%), Haematobacter missouriensis CCUG 52307(T) (95.9%) and Rhodobacter aestuarii JA296(T) (95.7%). The genomic DNA G+C content was 67.2 mol% and the major respiratory quinone was ubiquinone 10 (Q-10). The major cellular fatty acids (&gt;5%) were C(18 : 1)omega7c, C(16 : 0), C(19 : 0) cyclo omega8c and summed feature 3 (one or more of iso-C(15 : 0) 2-OH, C(16 : 1)omega6c and C(16 : 1)omega7c). However, unlike species of the genus Rhodobacter, strain DW2-9(T) neither formed internal photosynthetic membranes nor produced photosynthetic pigments. DNA-DNA hybridization between strain DW2-9(T) and R. capsulatus JCM 21090(T) showed a relatedness of 33%. Strain DW2-9(T) contained phosphatidylethanolamine, phosphatidylglycerol and an unknown aminophospholipid as major polar lipids, which differed from those of species of the genera Rhodobacter and Haematobacter. In addition to the differences in phylogenetic position and polar lipid types, strain DW2-9(T) could be distinguished from species of the genus Haematobacter by the cultivation conditions. On the basis of our polyphasic taxonomic analysis, strain DW2-9(T) is considered to represent a novel genus and species, for which the name Paenirhodobacter enshiensis gen. nov., sp. nov. is proposed. The type strain of Paenirhodobacter enshiensis is DW2-9(T) ( = CCTCC AB 2011145(T) = KCTC 15169(T)). Emended descriptions of the genera Rhodobacter and Haematobacter are also proposed.'}, {'qas': [{'id': '23646192:Faecalibacterium', 'question': 'Can Faecalibacterium cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:Fecal microbial diversity in pre-weaned dairy calves as described by pyrosequencing of metagenomic 16S rDNA. Associations of Faecalibacterium species with health and growth.\\nABSTRACT:In this study, we use barcoded pyrosequencing of the 16S rRNA gene to characterize the fecal microbiota of neonatal calves and identify possible relationships of certain microbiota profiles with health and weight gain. Fecal samples were obtained weekly from 61 calves from birth until weaning (seventh week of the calves' life). Firmicutes was the most prevalent phylum, with a prevalence ranging from 63.84% to 81.90%, followed by Bacteroidetes (8.36% to 23.93%), Proteobacteria (3.72% to 9.75%), Fusobacteria (0.76% to 5.67%), and Actinobacteria (1.02% to 2.35%). Chao1 index gradually increased from the first to the seventh postnatal week. Chao1 index was lower during the third, fourth, and fifth week of life in calves that suffered from pneumonia and were treated with antibiotics. Diarrhea incidence during the first four weeks of the calves' life was also associated with a reduction of microbial diversity during the third week of life. Increased fecal microbial diversity after the second week of life was associated with higher weight gain. Using discriminant analysis we were able to show differences in the microbiota profiles between different weeks of life, between high and low weight gain groups of calves, and between calves affected and not affected with diarrhea during the first four weeks life. The prevalence of Faecalibacterium spp. in the first week of life was associated with weight gain and the incidence of diarrhea, with higher prevalence being associated with higher weight gain and less diarrhea. Representative sequences from Faecalibacterium spp. were closely affiliated to Faecalibacterium prausnitzii. Results presented here provide new information regarding the intestinal microbiota of neonatal calves and its association with health and growth. Fecal microbial diversity was associated with calf age, disease status and growth rates. Results suggesting a possible beneficial effect of Faecalibacterium spp. on health and growth are promising.\"}, {'qas': [{'id': '18650348:Pandoraea sputorum', 'question': 'Can Pandoraea sputorum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Misidentification of Pandoraea sputorum isolated from sputum of a patient with cystic fibrosis and review of Pandoraea species infections in transplant patients.\\nABSTRACT:Pandoraea species are considered emerging pathogens in cystic fibrosis (CF) patients, but few data exist regarding outcomes of patients colonized with these organisms. We report a case of Pandoraea sputorum colonization in a CF patient under consideration for lung transplantation and review five cases of lung transplantation involving Pandoraea species.'}, {'qas': [{'id': '22622257:Aggregatibacter paraphrophilus', 'question': 'Can Aggregatibacter paraphrophilus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Aggregatibacter aphrophilus pyogenic liver abscess in an immunocompetent young woman.\\nABSTRACT:Aggregatibacter aphrophilus (formerly Haemophilus aphrophilus/paraphrophilus) is a small Gram-negative coccobacillus with fastidious growth requirements. It is a normal commensal of the human oropharynx and upper respiratory tract, and it can infrequently cause invasive human diseases, including bone and joint infections and subacute infective endocarditis. Cases of liver abscess caused by Aggregatibacter aphrophilus have been sparsely recorded in the English-language literature, but have not yet been reported in Taiwan. Here we present a case of Aggregatibacter aphrophilus pyogenic liver abscess in an immunocompetent young woman. She recovered uneventfully after repeated percutaneous abscess aspiration and antibiotic treatment for 5 weeks.'}, {'qas': [{'id': '26400786:Francisella philomiragia', 'question': 'Can Francisella philomiragia cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Francisella philomiragia Bacteremia in a Patient with Acute  Respiratory Insufficiency and Acute-on-Chronic Kidney Disease.\\nABSTRACT:Francisella philomiragia is a very uncommon pathogen of humans. Diseases caused by it are protean and have been reported largely in near-drowning victims and those with chronic granulomatous disease. We present a case of F. philomiragia pneumonia with peripheral edema and bacteremia in a renal transplant patient and review the diverse reports of F. philomiragia infections.'}, {'qas': [{'id': '27235198:Acinetobacter pittii', 'question': 'Can Acinetobacter pittii cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Acinetobacter baumannii and A. pittii clinical isolates lack adherence and cytotoxicity to lung epithelial cells in vitro.\\nABSTRACT:The molecular and genetic basis of Acinetobacter baumannii and Acinetobacter pittii virulence remains poorly understood, and there is still lack of knowledge in host cell response to these bacteria. In this study, we have used eleven clinical Acinetobacter strains (A. baumannii n = 5; A. pittii n = 6) to unravel bacterial adherence, invasion and cytotoxicity to human lung epithelial cells. Our results showed that adherence to epithelial cells by Acinetobacter strains is scarce and cellular invasion was not truly detected. In addition, all Acinetobacter strains failed to induce any cytotoxic effect on A549 cells.'}, {'qas': [{'id': '25614744:Delftia acidovorans', 'question': 'Can Delftia acidovorans cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Polymicrobial Bacteremia Involving Comamonas testosteroni.\\nABSTRACT:Comamonas spp. are uncommon isolates in microbiology laboratories and have been rarely observed as an infectious agent in clinical practice. They have widespread environmental distribution and have been isolated from water, soil, and plants as well as from some hospital devices such as intravenous catheters and water contained in humidifier reservoirs used in respiratory treatment. The genus Comamonas originally contained the following species: acidovorans, testosteroni, kerstersii, terrigena, denitrificans, and nitrativorans. It now contains 17 species, while acidovorans spp. have been reclassified as Delftia acidovorans. In spite of its uncommon human pathogenesis, there are few reports on the aggressive manner of it as an opportunistic pathogen, mostly related to testosteroni spp. We present a case of polymicrobial bacteremia involving Comamonas testosteroni. The aim of this case report is to alert clinicians to the potential diagnosis of bloodstream infections caused by uncommon pathogens. '}, {'qas': [{'id': '32392444:Alistipes', 'question': 'Can Alistipes cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Lycium barbarum Polysaccharide Extracted from Lycium barbarum Leaves Ameliorates Asthma in Mice by Reducing Inflammation and Modulating Gut Microbiota.\\nABSTRACT:This study was designed to explore the impact of Lycium barbarum polysaccharide (LBP) on inflammation and gut microbiota in mice with allergic asthma. Mice were divided into four groups: control group, OVA (ovalbumin) group, Con+LBP group, OVA+LBP group. After 28 days of LBP intervention, mice were euthanized and associated indications were investigated. Histopathological examination demonstrated that LBP reduced lung injury. The results of our current study provide evidence that supplementation with LBP in asthmatic mice decreases TNF, IL-4, IL-6, MCP-1, and IL-17A in plasma and bronchoalveolar lavage fluid (BALF). Sequencing and analysis of gut microbiota indicated that compared with the OVA group, Lactobacillus and Bifidobacterium were increased, but Firmicutes, Actinobacteria, Alistipes, and Clostridiales were decreased in the OVA+LBP group. We also found that gut microbiota were related to inflammation-related factors. Therefore, we speculate that LBP may improve allergic asthma by altering gut microbiota and inhibiting inflammation in mice.'}, {'qas': [{'id': '27025816:Enterococcus canis', 'question': 'Can Enterococcus canis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Pharmacokinetics of Ceftiofur Crystalline-Free Acid in Clinically Healthy Dogs (Canis lupus familiaris).\\nABSTRACT:Economical, injectable antibiotics are beneficial when clinical manifestations of an animal model prevent the use of oral antibiotics. Ceftiofur crystalline-free acid (CCFA) is an injectable, sustained-release form of ceftiofur, a third-generation cephalosporin that is labeled for use in swine, cattle, and horses. Because CCFA is an economical, injectable antibiotic that could be of value for use in research dogs, the objective of this study was to determine the pharmacokinetic properties of CCFA in apparently healthy dogs and to determine the minimal inhibitory concentrations of ceftiofur for veterinary pathogens cultured during 2011 through 2014 from the respiratory system, integumentary system, and urinary system of dogs. The study population comprised of 5 dogs (age, 1 y; weight, 24.7 to 26.9 kg) that were deemed healthy after no abnormalities were found on physical exam, CBC analysis, and clinical chemistry panel. Each dog received CCFA at 5.0 mg/kg SC, and blood samples were collected before administration of CCFA and at 1, 4, 8, 12, 24, 36, 48, 72, 96, 120, 144, 168, 192, 216, and 240 h after injection. The maximal plasma concentration (mean +- 1 SD) of CCFA was 1.98 +- 0.40 mu g/mL, time to reach maximal concentration was 22.3 +- 8.9 h, half-life was 56.6 +- 16.9 h, and AUC0-last was 124.98 +- 18.45 mu g-h/mL. The minimal inhibitory concentrations of ceftiofur ranged from &lt;= 0.25 to &gt;= 8.0 mu g/mL; ceftiofur was most effective against Pasteurella spp., Proteus spp., and Escherichia coli haemolytica and least effective against Bordatella bronchiseptica, Enterococcus spp., and Pseudomonas aeruginosa. '}, {'qas': [{'id': '26997202:Enterococcus casseliflavus', 'question': 'Can Enterococcus casseliflavus cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Host-derived probiotics Enterococcus casseliflavus improves resistance against Streptococcus iniae infection in rainbow trout (Oncorhynchus mykiss) via immunomodulation.\\nABSTRACT:The present study evaluated the benefits of dietary administration of host-derived candidate probiotics Enterococcus casseliflavus in juvenile rainbow trout Oncorhynchus mykiss. Experimental diets were prepared by incorporating the microorganisms in the basal feed at 3 inclusion levels (i.e. 10(7) CFU g(-1) of feed [T1], 10(8) CFU g(-1) of feed [T2], 10(9) CFU g(-1) of feed [T3]). The probiotic feeds were administered for 8 weeks, with a group fed with the basal diet serving as control. The effects on growth performance, gut health, innate immunity and disease resistance were evaluated. Results showed that growth performance parameters were significantly improved in T2 and T3 groups. Activities of digestive enzymes such as trypsin and lipase were significantly higher in these two groups as well. Gut micro-ecology was influenced by probiotic feeding as shown by the significant increase in intestinal lactic acid bacteria and total viable aerobic counts in T2 and T3. Humoral immunity was impacted by dietary probiotics as total serum protein and albumin were significantly elevated in T3. The levels of serum IgM significantly increased in all probiotic fed groups at week 8; with the T3 group registering the highest increment. Respiratory burst activity of blood leukocytes were significantly improved in T2 and T3. Hematological profiling further revealed that neutrophil counts significantly increased in all probiotic fed groups. Challenge test showed that probiotic feeding significantly improved host resistance to Streptococcus iniae infection, specifically in T2 and T3 where a considerable modulation of immune responses was observed. Taken together, this study demonstrated E. casseliflavus as a potential probiotics for rainbow trout with the capability of improving growth performance and enhancing disease resistance by immunomodulation.'}, {'qas': [{'id': '25657265:Pandoraea pnomenusa', 'question': 'Can Pandoraea pnomenusa cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Genome Sequences of Two Pandoraea pnomenusa Isolates Recovered 11 Months Apart from a Cystic Fibrosis Patient.\\nABSTRACT:Pandoraea is an emerging respiratory pathogen capable of causing chronic lung infections in people with cystic fibrosis (CF), but the clinical significance of this infection is ambiguous. We have sequenced and annotated the genomes of two multidrug-resistant Pandoraea pnomenusa isolates recovered 11 months apart from the same CF patient. '}, {'qas': [{'id': '27367327:Nocardia paucivorans', 'question': 'Can Nocardia paucivorans cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': \"TITLE:Diagnosis of Nocardia paucivorans central nervous system infection by DNA sequencing from paraffin-embedded tissue.\\nABSTRACT:Infections by Nocardia spp. are generally regarded as opportunistic diseases in immunocompromised patients, but can also affect immunocompetent subjects. Such infections represent an important diagnostic challenge for clinicians and microbiologists, and diagnosis is frequently delayed or even conducted post mortem. A 54-year-old man was admitted to our hospital because of ventriculitis and relapsing brain abscess. Five months prior, this patient had undergone external ventricular drain and surgery for a cerebellar abscess. Histopathology demonstrated pyogenic inflammatory reaction, microbiologic investigations proved negative and empiric antimicrobial therapy was administered for a total of eight weeks. Six weeks later, the patient developed relapsing neurologic manifestations. On reviewing the patient's clinical history it emerged that the patient had suffered pneumonia two months prior to neurosurgery, treated with amoxicillin/clavulanate 3g a day and levofloxacin 500mg a day for three weeks. On the CNS relapsing manifestations, nocardiosis was suspected and DNA sequencing from the formalin-fixed paraffin-embedded cerebellar tissue collected during neurosurgery allowed diagnosis of Nocardia paucivorans infection. The patient received medical therapy for 11 months. At follow-up, eight months after treatment was discontinued, the patient was aymptomatic. Nocardia spp. infections need to be suspected not only in immunocompromised, but also in immunocompetent patients. Proper samples need to be collected for proper microbiologic investigations. Paraffin-embedded tissue genomic sequencing can be a useful tool for diagnosis of nocardiosis. \"}, {'qas': [{'id': '28348776:Nocardia mexicana', 'question': 'Can Nocardia mexicana cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:First case report of pulmonary nocardiosis caused by Nocardia mexicana.\\nABSTRACT:INTRODUCTION: Nocardia species usually cause opportunistic infections, and the frequency of these infections is increasing owing to the growing population of immunocompromised hosts. However, Nocardia species may sometimes cause an infection disease in immunocompetent hosts. Nocardia mexicana infections are the least common and are very rare. CASE PRESENTATION: Herein, we report the first case of a pulmonary infection with N. mexicana in a 61-year-old Japanese woman with a history of hyperlipidaemia and bronchiectasis and a 6-month history of non-productive hacking cough. A sample of bronchial lavage fluid obtained by bronchofiberscopy showed filamentous branching gram-positive rods and acid-fast filamentous branching rods, and a colony of suspected Nocardia was cultured. Based on 16S rRNA, gyrB,rpoB, secA1 and hsp65 gene sequence analyses and biochemical and physiological properties, the strain was identified as N. mexicana. The strain was resistant to the antimicrobial agents amoxicillin-clavulanic acid, clarithromycin, minocycline, gentamycin, tobramycin, ciprofloxacin and trimethoprim-sulfamethoxazole. The patient was treated with biapenem followed by intravenous amikacin and oral linezolid. \\n'}, {'qas': [{'id': '32307314:Leptospira santarosai', 'question': 'Can Leptospira santarosai cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:First report of human Leptospira santarosai infection in French Guiana.\\nABSTRACT:Leptospirosis is a zoonotic disease with worldwide distribution. In French Guiana, a French overseas department of South America, this bacterial infection is endemic with the increase of human cases since the last 5 years. Nevertheless, the epidemiological data on the circulating infecting strain remains scarce due to the lack of specific symptoms and the used diagnostic approaches. We report a severe case of leptospirosis in a 52-years-old male, working as a street cleaner, hospitalized at the Intensive Care Unit of city capital hospital of French Guiana because of hemodynamic, neurological, renal, and respiratory failure. At ICU admission, the patient was comatose, his temperature was 37.3 C, heart rate 104 beats per minute, blood pressure 84/45mmHg, and oxygen saturation 95% while under mechanical ventilation. Retrospective exploration using molecular and serological approaches from the samples allowed reporting an infection by Leptospira santarosai serogroup Sejroe serovar Hardjo. To our knowledge, this is the first human case infected with this species in French Guiana. Through these analyses, this report provides new epidemiological information about the Leptospira strains circulating in French Guiana. In particular, this emphasizes the importance of accurate diagnosis to characterize with more precision the circulating Leptospira strains in this department.'}, {'qas': [{'id': '28962584:Sphingobacterium spiritivorum', 'question': 'Can Sphingobacterium spiritivorum cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Sphingobacterium spiritivorum bacteremia due to cellulitis in an elderly man with chronic obstructive pulmonary disease and congestive heart failure: a case report.\\nABSTRACT:\\n'}, {'qas': [{'id': '26585900:Brevibacillus parabrevis', 'question': 'Can Brevibacillus parabrevis cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Brevibacillus gelatini sp. nov., isolated from a hot spring.\\nABSTRACT:Two Gram-stain-positive, moderately thermophilic, endospore-forming, rod-shaped, motile bacteria designated PDF4T and PDF10, were isolated from Camkoy hot spring in the provinces of Aydin, Turkey and were characterized in order to determine their phylogenetic position. 16S rRNA gene sequence analysis revealed that the two strains belonged to the genus Brevibacillus. Strain PDF4T showed highest 16S rRNA gene sequence similarity to strain PDF10 (99.5 %), Brevibacillus brevis DSM 30T (98.9 %), Brevibacillus parabrevis DSM 8376T (98.6 %) and Brevibacillus formosus DSM 9885T (98.5 %); similarities to other species of the genus Brevibacillus were less than 98.5 %. The predominant fatty acids of strain PDF4T were anteiso-C15 : 0 (60.0 %) and iso-C15 : 0 (22.3 %). The polar lipids of strain PDF4T consisted of diphosphatidylglycerol, phosphatidylglycerol, phosphatidylethanolamine, phosphatidylmonomethylethanolamine, an unknown phospholipid, two unknown lipid, an unknown aminophospholipid and two unknown aminolipids. MK-7 was detected as a sole respiratory quinone, and the cell wall of strain PDF4T contained meso-diaminopimelic acid. The DNA G+C content of strain PDF4T was 51.7 mol%. DNA-DNA hybridization showed less than 60 % relatedness between strain PDF4T and type strains of the most closely related species given above. Based on these data, the two strains are considered to represent a novel species of the genus Brevibacillus, for which the name Brevibacillus gelatini sp. nov. is proposed. The type strain is PDF4T ( = NCCB 100559T = DSM 100115T).'}, {'qas': [{'id': '15520483:Mycobacterium conspicuum', 'question': 'Can Mycobacterium conspicuum cause Pneumonia?', 'is_impossible': True, 'answers': 'no'}], 'context': 'TITLE:Infections due to non-tuberculous mycobacteria (NTM).\\nABSTRACT:The membership list of genus mycobacterium is ever expanding and it has grown to 95 in year 2003. While leprosy and tuberculosis are specific diseases caused by mycobacteria, other members are usually saprophytes but can be opportunistic and at times deadly pathogens. These other mycobacteria are referred to as atypical mycobacteria, non-tuberculous mycobacteria (NTM) or mycobacteria other than tubercle bacilli (MOTT). These organisms can produce localized disease in the lungs, lymph glands, skin, wounds or bone. Occasionally they may produce disseminated disease. Of the more than 90 known species of NTM, about one third have been associated with disease in humans. The species causing human disease are : Mycobacterium avium, M. intracellulare, M. kansasii, M. paratuberculosis, M. scrofulaceum, M. simiae, M. habana, M. interjectum, M. xenopi, M. heckeshornense, M. szulgai, M. fortuitum, M. immunogenum, M. chelonae, M. marinum, M. genavense, M. haemophilum, M. celatum, M. conspicuum, M. malmoense, M. ulcerans, M. smegmatis, M. wolinskyi, M. goodii, M. thermoresistible, M. neoaurum, M. vaccae, M.palustre, M. elephantis, M. bohemicam and M. septicum. Isolation of these mycobacteria from representative specimens and their rapid identification is very important as the treatment strategy for tuberculosis and other mycobacterioses is different. Several biochemical, chemical (lipid) and molecular techniques have been developed for rapid identification of these species. Along with suggestive clinical features, poor response to antitubercular treatment and repeated isolation of the organisms from the clinical specimens these techniques can help in establishing correct diagnosis. Further, many drugs like rifampicin, rifabutin, ethambutol, clofazimine, amikacin, new generation quinolones and macrolides effective against mycobacterial infections are available that can be used in appropriate combinations and dosage to treat these infections.'}, {'qas': [{'id': '25506444:Kerstersia gyiorum', 'question': 'Can Kerstersia gyiorum cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Kerstersia gyiorum Isolated from a Bronchoalveolar Lavage in a Patient with a Chronic Tracheostomy.\\nABSTRACT:The use of the matrix-assisted laser desorption ionization-time of flight (MALDI-TOF) mass spectrometry (MS) generates rapid microbial identification. We are presenting a case of a 63-year-old woman with a medical history of chronic tracheostomy admitted for hypotension and fevers to illustrate the clinical implication of MALDI-TOF MS on bacterial identification. Kerstersia gyiorum was identified from the bronchoalveolar lavage isolate. Kerstersia gyiorum has been isolated from human sputum samples, and may be a previously unrecognized colonizer of the upper respiratory tract. Thus, patients with long-term tracheotomies or who are chronically aspirating may be at risk of lower respiratory infection with this organism. Increased use of MALDI-TOF MS in the clinical setting may increase reporting of this atypical isolate. '}, {'qas': [{'id': '26887928:Mycobacterium hiberniae', 'question': 'Can Mycobacterium hiberniae cause Pneumonia?', 'is_impossible': False, 'answers': 'yes'}], 'context': 'TITLE:Increased isolation of nontuberculous mycobacteria among TB suspects in Northeastern, Tanzania: public health and diagnostic implications for control programmes.\\nABSTRACT:\\n'}]\n"
     ]
    }
   ],
   "source": [
    "re = 2\n",
    "eval_qa = random.sample(train_qa, int(len(train_qa)/5))\n",
    "\n",
    "ttrain_qa = [ele for ele in train_qa if ele not in eval_qa]\n",
    "ttrain_set = {\"data\":[{'paragraphs':ttrain_qa, 'title':\"LungB\"}], \"version\":\"LungB\"}\n",
    "json.dump(ttrain_set, open(\"eval/input_%s/ttrain_set.json\" % re, \"w\"))\n",
    "\n",
    "eval_q = []\n",
    "eval_a = {}\n",
    "for i in eval_qa:\n",
    "    qq = {'qas': [{\n",
    "            'id': i['qas'][0]['id'] ,\n",
    "            'question': i['qas'][0]['question']}],\n",
    "            'context': i['context']}\n",
    "    eval_q.append(qq)\n",
    "    eval_a[i['qas'][0]['id']] = i['qas'][0]['answers']\n",
    "    \n",
    "eval_qset = {\"data\":[{'paragraphs':eval_q, 'title':\"LungB\"}], \"version\":\"LungB\"}\n",
    "json.dump(eval_qset, open(\"eval/input_%s/eval_qset.json\" % re, \"w\"))\n",
    "json.dump(eval_a, open(\"eval/input_%s/eval_aset.json\" % re, \"w\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 309,
   "metadata": {},
   "outputs": [],
   "source": [
    "re = 1\n",
    "eval_a = json.load(open(\"eval/input_%s/eval_aset.json\" % re, \"r\"))\n",
    "standard = pd.DataFrame(eval_a ,index = [\"golden\"]).T\n",
    "prediction = json.load(open(\"eval/output_%s/predictions_.json\" % re, 'r'))\n",
    "prediction = pd.DataFrame(prediction, index = [\"prediction\", \"rate\"]).T\n",
    "prediction[\"cut\"] = 1\n",
    "for i in prediction.index:\n",
    "    if prediction.loc[i, \"rate\"][0] > 0.95:\n",
    "        prediction.loc[i, \"cut\"] = \"yes\"\n",
    "    else:\n",
    "        prediction.loc[i, \"cut\"] = \"no\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 310,
   "metadata": {},
   "outputs": [],
   "source": [
    "from sklearn.metrics import accuracy_score, f1_score, precision_score, recall_score\n",
    "accuracy = accuracy_score(standard[\"golden\"], prediction[\"cut\"], normalize=True, sample_weight=None)\n",
    "precision = precision_score(standard[\"golden\"], prediction[\"cut\"], labels=None, pos_label='yes', average='binary', sample_weight=None, zero_division='warn')\n",
    "recall = recall_score(standard[\"golden\"], prediction[\"cut\"], labels=None, pos_label='yes', average='binary', sample_weight=None, zero_division='warn')\n",
    "f1 = f1_score(standard[\"golden\"], prediction[\"cut\"], labels=None, pos_label='yes', average='binary', sample_weight=None, zero_division='warn')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 311,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.8716216216216216 0.7083333333333334 0.8717948717948718 0.7816091954022988\n"
     ]
    }
   ],
   "source": [
    "print(accuracy, precision, recall, f1)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# plot roc"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 258,
   "metadata": {},
   "outputs": [],
   "source": [
    "standard.loc[standard[\"golden\"] == \"yes\", \"golden\"] = 1\n",
    "standard.loc[standard[\"golden\"] == \"no\", \"golden\"] = 0\n",
    "standard[\"minus\"] = 1 - standard[\"golden\"]\n",
    "standard_v  = np.array(standard.values, dtype=\"float\")\n",
    "\n",
    "prediction[\"minus\"] = 1\n",
    "for i in prediction.index:\n",
    "    prediction.loc[i, \"rate\"] = prediction.loc[i, \"rate\"]\n",
    "    prediction.loc[i, \"minus\"] = 1 - prediction.loc[i, \"rate\"]\n",
    "prediction_v  = np.array(prediction.iloc[:, 1:3].values, dtype=\"float\")\n",
    "\n",
    "\n",
    "# Binarize the output\n",
    "y_test = standard_v\n",
    "y_score = prediction_v\n",
    "n_classes = y_score.shape[1]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 246,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Automatically created module for IPython interactive environment\n"
     ]
    }
   ],
   "source": [
    "print(__doc__)\n",
    "\n",
    "import numpy as np\n",
    "import matplotlib.pyplot as plt\n",
    "from itertools import cycle\n",
    "\n",
    "from sklearn import svm, datasets\n",
    "from sklearn.metrics import roc_curve, auc\n",
    "from sklearn.model_selection import train_test_split\n",
    "from sklearn.preprocessing import label_binarize\n",
    "from sklearn.multiclass import OneVsRestClassifier\n",
    "from scipy import interp\n",
    "from sklearn.metrics import roc_auc_score"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 247,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAYoAAAEWCAYAAAB42tAoAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjMuNCwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy8QVMy6AAAACXBIWXMAAAsTAAALEwEAmpwYAAA7FUlEQVR4nO3dd3gU1frA8e+bBFLoXaSLCKGJGgFFAUGKBOFeRbGB+uOKSFHBC6iocFGsiIJUvSqWawNRESkXuCgqorTQFVARgvQOISHl/f0xk7CGkCyQ3c1u3s/z7MPOzJmZd4ck755zZs4RVcUYY4w5k7BAB2CMMaZgs0RhjDEmV5YojDHG5MoShTHGmFxZojDGGJMrSxTGGGNyZYnCGGNMrixRGJ8Ska0ickJEjonILhGZKiLFPbZPFZGT7vbM12p3W00RUY/1W0XkUY99VUSOZ9t3SLbz3+OW6+4uX+tR9ni24x8TkeoicquILBGRJBH5Oo/P11pEMtx9j4rILyJyb7YyIiKDRWSzey22ichzIhKZrVxTEZktIodE5ICI/JT9WMYEgiUK4w83qmpxoAlwGfBYtu0vqmpxj9el2baXdve/HXhKRDp6bLs0274vZtv3buAA0BNAVb/NLAs08Dy++9rmln8VeN7Lz/ene7ySwEDgDRGp67F9HNDbjaEEcAPQFvgks4CIXAX8D/gGuBgoBzzglvUZEYnw5fFNaLBEYfxGVXcB83ASxrns/wOwHmjoTXkRqQG0wvkj3UFELvDyPAtU9RPgz7OMT1V1Nk6iaezGUAfoC9ypqj+oapqqrgduBjqKSBt395eAd1T1BVXd5x5rharemsvnu09ENro1mQ0icrm7XkXkYo9yU0XkGfd9axFJFJGhIrILeNs9RmeP8hEistfjeM3dGtYhEVktIq3P5rqY4GeJwviNiFTF+Ya85Rz2FRFpgVMLWOXlbj2B5ar6KbARuPNsz3s2RCRMRLoA5Tn1GdsCiar6k2dZVd0OLAXaiUgMcBUw/SzOdQswAuczlgS6APu93P0CoCxQAyeJfohTW8vUAdinqitFpArwFfCMu88/gU9FpIK3sZrgZ4nC+MPnInIU2A7sAYZn2/5P99tq5uudbNv34XxL/zfwqKou9Ni2Mtu+HTy29QQ+cN9/4C77woUicgg4AXwGDFLVzGRWHth5hv12utvL4PwunqlcTv6B02S3zK19bFHVP7zcNwMYrqopqnoC59p0cRMWwB04yQPgLmC2qs5W1QxVnQ8sBzqdRawmyFmiMP7wN1UtAbQG6uH8cfQ0WlVLe7zuzra9vKqWUdVYVR2Xbdvl2fadB+DWPmoBH7nlPgAaiUiT/Pxgrj9VtTTON/txQBuPbfuAymfYr7K7/SDOH+8zlctJNeDXs47UsVdVkzMXVHULTo3rRjdZdOFUgq0B3OKZjIFrzjJWE+QsURi/UdVvgKnAaD+c7m5AgAS3Lf5Hj/U+oaopwFCchPQ3d/X/gGoi0tSzrIhUA5oDC1U1CfgBp9/CW9uB2mfYlgTEeCxn75vJacjozOanrsAGN3lknue9bMm4mKp629FvQoAlCuNvr+K0y2e/synfiEgUcCtO+3sTj9cA4I687vQRkXD3GBFAmIhEiUgRb86tqieBl4Gn3OVNwGTgP26ncLiINAA+BRao6gJ31yHAPe5ttOXcOC4VkY9OPwvgNMP9U0SucPtvLnY77wES3M8Z7t4h1sqL0D8C2uPcafWBx/r3cWoaHTKvi9shXtWb62FChKray14+ewFbgeuzrZsEfOq+nwqcBI55vPa522rifPuNOMOxFTiebd9Xgdtw2vuLZCsfjdPh2zm34wP3uOs9X1PPEENrnM5qz3UxOE1KN7rLYTg1jS04/RjbgReBqGz7NQXmAIdx+mR+BHrmcm37AL+4n3sdcJm7Pg7n7rCjwHs4tYVnzhSvx/EWAmnABdnWN8O5bfcAsBenc7t6oH+27OW/l7g/CMYYY0yOrOnJGGNMrixRGGOMyZUlCmOMMbmyRGGMMSZXQTcgWPny5bVmzZqBDsMYY4LKihUr9qnqOQ29EnSJombNmixfvjzQYRhjTFAREW+HeDmNNT0ZY4zJlSUKY4wxubJEYYwxJleWKIwxxuTKEoUxxphcWaIwxhiTK58lChF5S0T2iMi6M2wXERknIltEZE3m/LzGGGMKFl/WKKYCHXPZfgNQx331xhl62hhjTD47eTL9vPb32QN3qrpYRGrmUqQr8K4645wvFZHSIlJZVc9m3mBjAm9GPPw+O9BRGJOjwV+2Y9Wf5zdzbSCfzK6CM4FLpkR33WmJQkR649Q6qF69ul+CM+YvLBmYINXwgj2M+67ZeR0jKIbwUNXXgdcB4uLibKYl4395JYlaneCmr/wTizG52LBhLytX7uSuuxoD0FOVVs8fplatZ875mIFMFDuAah7LVd11pjAKlm/sj9j3FFMwJSWl8swzi3nppSWEhwvNm1fl4ovLIiLUrFn6vI4dyEQxE+jvTh7fDDhs/ROFWDAkiVqdAh2BMTmaM2cz/frN5vffDwHQq9cVlCsXnW/H91miEJEPcSZyLy8iicBwoAiAqk4GZgOdcCacTwLu9VUspgDwtsZg39iN8dqOHUd4+OF5TJ++AYDGjSsxeXI8V11VLY89z44v73q6PY/tCvTz1flNAeNNkrBv7MaclX79ZvPFF78QE1OEkSNb89BDzYmIyP+nHoKiM9sUUOfSr2A1BmPOS1paRlYyeOGF6ylSJJyXX25P9eqlfHZOG8LDnLuzTRJWYzDmnB0+nMyAAbOJj/8Ap0EG6tYtz7Rpt/g0SYDVKMzZyqkWYbUEY3xGVZk2bQMPPzyXnTuPER4uJCTs4rLLzu8hurNhicLkLbcmJqslGOMzv/56gP795zB37hYArrqqKpMnd6Zx40p+jcMShclb9iRhD5cZ43OjRy/hyScXkZycRunSUbzwwvX84x+XExYmfo/FEoXxnjUxGeM3SUmpJCen0aNHY0aPbk/FisUCFoslCmOMKQD27j3OL7/s55prnPHshg5tQevWNWnZskaAI7NEUTgEy/AYxhRCGRnKW2+tYsiQ+UREhPHzz/0pWzaayMiIApEkwBJF4ZAfScI6rY3Jd+vW7aFPn1l8/70zkHa7dheRlJRK2bL5N/xGfrBEEWj+/LZvfQzGFAjHj59k5MhvGDNmKWlpGVSqVIxXX+1I9+4NEPF/Z3VeLFEEmr+ShNUIjCkwunWbxty5WxCBvn3jGDWqLaVLRwU6rDOyRFFQ2Ld9YwqNoUNbsHv3MSZNiqdZs6qBDidPligCaUZ8oCMwxvhYWloGr732I1u3HmLs2BsAaN26JsuX9w7IMxHnwhJFIGU2O1mzkDEh6aefdnD//bNISNgFQO/eV9CgQUWAoEkSYInCf3LrtLannI0JKYcOJfP44wuZPHk5qlCjRinGj++UlSSCjSUKf7GxkowpFD76aB0PPzyX3buPExERxiOPXMWTT7akWLGigQ7tnFmi8AfPvgjrtDYmpP33v7+ye/dxWrSoxqRJ8TRq5N8B/HzBEoUvZW9ustqDMSEnJSWNHTuOctFFZQB48cV2XHttde6+u0lQ9UPkxiYu8qXsScL6IowJKf/73+80bjyZ+PgPOHkyHYDy5WO4997LQiZJgNUo8p9N7GNMyNu9+xj//Od83n9/DQD16pUnMfFIVq0i1FiiyG85zd1gjAkJGRnKG2+s4NFHF3LoUDJRURE88cS1DB7cgqJFwwMdns9YovAVq0UYE3L+/vePmTnzFwA6dKjNhAmdqF27bICj8j3rozDGGC/ddFM9LrigOB9/3I05c+4sFEkCrEaRP2y+B2NC0syZv5CYeIS+fa8EoGfPS7npplhKlIgMcGT+ZYkiP1i/hDEhZdu2wzz44By++OIXIiPD6djxYi66qAwiUuiSBFiiOH/2MJ0xISM1NZ1x435k+PCvOX48lRIlivLMM22oUaNUoEMLKEsU58sG9jMmJCxdmsj9989izZrdANxyS31eeaUDVaqUDHBkgWeJIr/Yw3TGBLUnn1zEmjW7qVWrNOPHd6JTpzqBDqnAsERxtqzj2piQoKocPXqSkiWdPofx42/g3XdXM2xYS2JiigQ4uoLFbo89WzklCWt2Miao/PLLPq6//j1uuuljVJ2+xbp1yzNqVFtLEjmwGsW5so5rY4JOcnIazz33Lc8//z0nT6ZTrlw0W7ceolat0Bx6I79YojDGFArz5/9K376z2bLlAAD/939NePHFdpQrFxPgyAo+nzY9iUhHEflFRLaIyKM5bK8uIotEZJWIrBGRgtuGMyMeXg6d0SCNKSxUlf/7vy9o3/59tmw5QP36FVi8+B7efLOrJQkv+axGISLhwASgHZAILBORmaq6waPYE8AnqjpJROoDs4GavorpvNi8EsYEJRGhZs3SREdH8NRTrRg06KqQHsDPF3zZ9NQU2KKqvwGIyEdAV8AzUSiQeZNyKeBPH8aTP6xvwpgCLyFhFzt3HuWGG5xbXIcObUGPHo2tL+Ic+bLpqQqw3WM50V3naQRwl4gk4tQmBuR0IBHpLSLLRWT53r17fRGrMSYEHD2awqBB87jiite5++7POXDgBACRkRGWJM5DoG+PvR2YqqpVgU7AeyJyWkyq+rqqxqlqXIUKFfwepDGmYFNVPvtsI/XrT+SVV5YCcMcdjShSJNB/4kKDL5uedgDVPJaruus89QI6AqjqDyISBZQH9vgwLmNMCPnjj0P07z+HWbM2ARAXdyFTpnTm8ssrBziy0OHLdLsMqCMitUSkKHAbMDNbmW1AWwARiQWiAGtbMsZ4RVW5+eZPmDVrEyVLRjJ+/A0sXdrLkkQ+81mNQlXTRKQ/MA8IB95S1fUiMhJYrqozgUeAN0RkIE7H9j2a+ZhkQeI5QqwxJuAyMpSwMEFEGD26PZMnL+eVVzpQuXKJQIcWkqQg/l3OTVxcnC5fvty/J818fqJWJxv8z5gA2r8/iUcfXQDAG290CXA0wUVEVqhq3Lnsa09m5+RMA/9ZkjAmIFSVd99dzT//OZ99+5IoWjSc4cNbU7WqDQHuD5YosjtTkrCH7IwJiI0b9/LAA1/xzTd/ANC6dU0mTYq3JOFHliiy85yIyGoQxgSMqvLUU4t44YXvSU3NoHz5GF5+uT09ejRGxIbT8SdLFGdiScKYgBIRduw4SmpqBvfddznPP389ZctGBzqsQskShTGmwPjzz6Ps25dE48aVAHjxxXb06nUZLVpUD3BkhZs9tmiMCbj09AzGj/+J2NgJ3HbbdE6eTAegfPkYSxIFgNUojDEBtXLlTu6/fxbLlztjgrZsWYMjR1IoX96GAC8oLFEYYwLiyJEUnnzyf4wfv4yMDKVq1ZKMG9eRv/2tnnVWFzBeJwoRiVHVJF8GY4wpHFSVli3fZvXq3YSHC4MGNWfEiNaUKBEZ6NBMDvLsoxCRq0VkA/Czu3ypiEz0eWTGmJAlIgwc2JymTauwfHlvXn65gyWJAsybGsUrQAfcAf1UdbWItPRpVMaYkHLyZDpjxvxAeLgweHALAHr2vJS77mpMeLjdU1PQedX0pKrbs7UZpvsmHGNMqPn22z/o0+crNmzYS2RkOD17XkqlSsUREcLDrS8iGHiTKLaLyNWAikgR4CFgo2/DMsYEu337khgyZD5vv50AQJ06ZZk4MZ5KlYoHNjBz1rxJFH2AsTjTmO4A/gv09WVQxpjgpapMnZrA4MHz2b//BEWLhvPYY9fw6KPXEBVlN1oGI2/+1+qq6p2eK0SkBfC9b0IyxgS7999fy/79J2jTphYTJ3aibt3ygQ7JnAdvEsVrwOVerAt+NkGRMeckKSmVw4eTqVy5BCLCxImdWLbsT+68s5E9ExECzpgoROQq4GqggogM8thUEmfGutDjOXKsMcYrc+Zspl+/2Vx0URnmz++BiFC3bnmrRYSQ3GoURYHibhnP+QWPAN18GVTA2cixxuRpx44jPPzwPKZP3wBAiRKR7N9/wobeCEFnTBSq+g3wjYhMVdU//BiTMaYAS0/PYMKEZTzxxP84evQkxYoVYeTI63jwwWZERNgzEaHImz6KJBF5CWgARGWuVNU2PovKGFMgZWQorVpN5fvvtwPwt7/VY+zYjlSvXirAkRlf8ib9/wdn+I5awL+ArcAyH8ZkjCmgwsKE9u1rU61aSb744jY++6y7JYlCwJsaRTlVfVNEHvJojrJEYUwhoKp88sl6IiLCuPnm+gAMHdqCQYOuonjxogGOzviLN4ki1f13p4jEA38CZX0XkjGmIPj11wP07Tub//73VypUiKFNm1qUKRNNZGQEkTZ+X6HiTaJ4RkRKAY/gPD9REnjYl0EZYwInJSWNl15awqhR35KcnEaZMlGMGtWGUqWi8t7ZhKQ8E4WqznLfHgaug6wns40xIebrr7fywANf8fPP+wDo0aMxo0e3p2LFYgGOzARSbg/chQO34ozxNFdV14lIZ+BxIBq4zD8hGmP8IT09g759nSRRt245Jk2K57rragU6LFMA5FajeBOoBvwEjBORP4E44FFV/dwPsRljfCwjQ0lOTiMmpgjh4WFMmhTP4sV/MGRICyIjbQA/48jtJyEOaKyqGSISBewCaqvqfv+EZozxpbVrd9Onz1fUq1eON9/sCkCrVjVp1apmYAMzBU5uieKkqmYAqGqyiPxmScKY4Hf8+ElGjvyGMWOWkpaWwe+/H+TgwROUKRMd6NBMAZVboqgnImvc9wLUdpcFUFVt7PPojDH56ssvf6F//zls23YYEejbN45Ro9pSurTd0WTOLLdEEeu3KAoCG2LchLC0tAy6d5/OjBnO5JRNmlzAlCmdadq0SoAjM8Egt0EBC9dAgDbEuAlhERFhlCoVSfHiRXn66evo37+pDeBnvObTnxQR6Sgiv4jIFhF59AxlbhWRDSKyXkQ+8GU8XrEhxk2I+PHHRH78MTFr+aWX2rFxYz8efri5JQlzVnx2/5v7HMYEoB2QCCwTkZmqusGjTB3gMaCFqh4UkYq+iueMZsSfqk0YEwIOHUrmsccWMGXKCurVK09CQh+KFg2nXDmbJ8KcG68ShYhEA9VV9ZezOHZTYIuq/uYe4yOgK7DBo8x9wARVPQigqnvO4vj5wzNJWLOTCWKqyocfrmPQoHns3n2ciIgwunSpS3p6BqE6KaXxjzwThYjcCIzGmfGulog0AUaqapc8dq0CbPdYTgSaZStziXuO73F+kkeo6lzvQs9nj2hATmtMfti8eT99+85mwYLfAGjRohqTJ3emYUP/V9JN6PGmRjECp3bwNYCqJohIfj3XHwHUAVoDVYHFItJIVQ95FhKR3kBvgOrVq+fTqY0JDamp6bRp8y6JiUcoWzaaF1+8nnvvvYywMAl0aCZEeDXMuKoeFvnLD503X7934AwBkqmqu85TIvCjqqYCv4vIJpzE8Zf5LlT1deB1gLi4OPvqbwxOU5OIUKRIOKNGtWHRoq28+OL1VKhgA/iZ/OXNrQ/rReQOIFxE6ojIa8ASL/ZbBtQRkVoiUhS4DZiZrcznOLUJRKQ8TlPUb17GbkyhtHv3MXr0+Ixnnlmcta5nz0t5++2uliSMT3iTKAbgzJedAnyAM9z4w3ntpKppQH9gHrAR+ERV14vISBHJ7N+YB+wXkQ3AImCwDRNiTM4yMpQpU5ZTr94E3n9/DWPGLOXo0ZRAh2UKAW+anuqp6jBg2NkeXFVnA7OzrXvK470Cg9yXMeYMVq/eRZ8+X7F0qfNcRMeOFzNhQidKlLCp5ozveZMoXhaRC4DpwMequs7HMRljXKmp6Tz22EJefXUp6elK5crFGTu2I9261Sdbv6ExPpNn05OqXoczs91eYIqIrBWRJ3wemT/Y+E6mgIuICGPVql1kZCgDBjRl48Z+3HJLA0sSxq+8euBOVXfhTF60CBgCPAU848vA/MLGdzIF0LZth0lPz6BWrTKICJMnx3P4cApxcRcGOjRTSOVZoxCRWBEZISJrgcw7nqr6PDJ/svGdTAGQmprO6NFLiI2dwH33fYnThQd16pSzJGECypsaxVvAx0AHVf3Tx/EYUyj98MN2+vT5ijVrdgNQtmw0SUmpFCtWNMCRGeNFolDVq/wRiDGF0cGDJ3j00QW8/vpKAGrVKs2ECZ244YY6AY7MmFPOmChE5BNVvdVtcvJ8GtpmuDMmH6SkpNGkyRS2bTtMkSJhDB58NcOGtSQmpkigQzPmL3KrUTzk/tvZH4EYU9hERkbQq9dlLFz4O5MmxVO/foVAh2RMjs7Yma2qO923fVX1D88X0Nc/4fmQ3Rpr/Cw5OY3hwxfxwQdrs9Y9/vi1fP313ZYkTIHmzRAe7XJYd0N+B+J3dmus8aP583+lUaNJjBy5mIED53HiRCrgPCdhz0SYgi63PooHcGoOF4nIGo9NJYDvfR2Y39itscaHdu06xqBB8/jwQ2dAgwYNKjB5cmeio60fwgSP3PooPgDmAM8BnvNdH1XVAz6Nypggl56ewZQpK3j88YUcPpxCdHQEw4e3YuDAqyha1GabM8Elt0ShqrpVRPpl3yAiZS1ZGHNm6enKa6/9xOHDKXTqVIfx42+gVq0ygQ7LmHOSV42iM7AC5/ZYz4ZUBS7yYVzGBJ2jR1NIT1dKl46iaNFw3njjRnbvPsZNN8VaP4QJamdMFKra2f03v6Y9NSYkqSqfffYzDz44hw4davPmm10BuOYam7bXhAZvxnpqISLF3Pd3icgYEbHfAGOArVsP0aXLR9x88yfs2HGUdev2kpycFuiwjMlX3tweOwlIEpFLgUeAX4H3fBqVMQVcamo6L7zwHfXrT2DWrE2ULBnJ+PE3sGTJ/xEV5dWgzMYEDW9+otNUVUWkKzBeVd8UkV6+DsxnZsSfeobCmHOQlJRK8+b/Zu3aPQDcdltDxoxpT+XKJQIcmTG+4U2iOCoijwE9gGtFJAwI3pvAPZOEPWxnzkFMTBHi4i4kKSmViRPjad++dqBDMsanvEkU3YE7gP9T1V1u/8RLvg3LDx7RvMsYg9NZ/e67q6ldu2xWB/Urr3SgaNFwe3DOFAreTIW6C/gPUEpEOgPJqvquzyMzpgDYuHEv1133Dvfc8wW9e3/JyZPpAJQqFWVJwhQa3tz1dCvwE3ALcCvwo4h083VgxgTSiROpPPHE/7j00sl8880fVKgQw2OPXUORIt7c/2FMaPGm6WkYcKWq7gEQkQrAAmC6LwPzCRsx1nhh7twt9Os3m99+OwjAffddzvPPX0/ZstEBjsyYwPAmUYRlJgnXfry7rbbgsRFjTR6OHTtJjx6fsW9fEg0bVmTy5HhatLDHhkzh5k2imCsi84AP3eXuQHDfX2ojxhoP6ekZZGQoRYqEU7x4UcaO7Uhi4hEGDmxOkSI2gJ8x3syZPVhEbgKucVe9rqqf+TYsY/xjxYo/uf/+WXTtWpcnn2wFwB13NApwVMYULLnNR1EHGA3UBtYC/1TVHf4KzBhfOnIkhSef/B/jxy8jI0M5ciSFRx+9xmoQxuQgt76Gt4BZwM04I8i+5peIjPEhVWXatPXUqzeeceN+QgQGDWrOypX3W5Iw5gxya3oqoapvuO9/EZGV/gjIGF85ejSF7t2nM2fOFgCaNavC5MmdadLkggBHZkzBlluiiBKRyzg1D0W057KqBlfisFtjC73ixYuSkpJOqVKRPP/89fTufQVhYTZPhDF5yS1R7ATGeCzv8lhWoI2vgvIJuzW2UFq8+A8qVy5OnTrlEBHeeqsLUVERVKpUPNChGRM0cpu46Dp/BuJTnrUJuzW2UNi3L4khQ+bz9tsJtG1bi/nzeyAi1KhROtChGRN0CsfA+VabKDQyMpSpUxMYPHg+Bw6coGjRcK69tjrp6UpEhDUzGXMufPqEtYh0FJFfRGSLiDyaS7mbRURFJM6X8VhtIrStX7+H1q2n0qvXTA4cOEHbtrVYu/YBhg9vTUREcA4mYExB4LMahYiEAxOAdkAisExEZqrqhmzlSgAPAT/6KhYT+g4fTqZ58zc5duwkFSsWY8yY9txxRyNErBZhzPnKM1GI85t2J3CRqo5056O4QFV/ymPXpsAWVf3NPc5HQFdgQ7ZyTwMvAIPPNnhjVBURoVSpKIYObcGOHUd49tm2lCljA/gZk1+8qY9PBK4CbneXj+LUFPJSBdjusZzorssiIpcD1VQ11zYhEektIstFZPnevXu9OLUJdTt2HKFbt094//01WeuGDbuWSZM6W5IwJp95kyiaqWo/IBlAVQ8CRc/3xO6UqmOAR/Iqq6qvq2qcqsZVqFDhfE9tglhaWgZjxy6lXr0JfPrpRoYP/5r09AwAa2Yyxke86aNIdfsbFLLmo8jwYr8dQDWP5aruukwlgIbA1+4v+AXATBHpoqrLvTh+3mbE/3WObBPUli3bQZ8+X7Fy5U4A/va3eowb15HwcOuoNsaXvEkU44DPgIoiMgroBjzhxX7LgDoiUgsnQdyGM/c2AKp6GCifuSwiX+MMPJg/SQL+miTs1tigdfz4SYYOXcDEictQherVS/HaazfQpUvdQIdmTKHgzTDj/xGRFUBbnOE7/qaqG73YL01E+gPzgHDgLVVdLyIjgeWqOvM8Y/feI+q3U5n8FxERxoIFvxEWJgwadBXDh7eiWLHzbv00xnjJm7ueqgNJwJee61R1W177qupssk1ypKpPnaFs67yOZwqPX389QOnSUZQrF0NkZATvvfd3oqIiaNSoUqBDM6bQ8abp6Suc/gkBooBawC9AAx/Gdf5sEMCglJKSxksvLWHUqG+5885G/PvfXQC48soqeexpjPEVb5qe/jLdl3tLa1+fRZRfbNiOoPP111t54IGv+PnnfYBzh1N6eoZ1VhsTYGf9ZLaqrhSRZr4Ixids2I4Cb8+e4wwePJ93310NQN265Zg0KZ7rrqsV4MiMMeBdH8Ugj8Uw4HLgT59FZAqVffuSiI2dwIEDJ4iMDGfYsGsZMqQFkZGFY7xKY4KBN7+NJTzep+H0WXzqm3BMYVO+fAxdu9YlMfEIEyfGc/HFZQMdkjEmm1wThfugXQlV/aef4skf1pFdYB0/fpKRI78hPv4SWrasAcDEifFERobbk9XGFFBnTBQiEuE+C9HCnwHlC+vILpC+/PIX+vefw7Zth/nqq82sWfMAYWFCVJQ1MxlTkOX2G/oTTn9EgojMBKYBxzM3quoMH8d2/qwju0DYvv0wDz00l88++xmAyy67gClTOtt81cYECW++ykUB+3HmyM58nkKBgp8oTEClpWUwbtyPPPXUIo4fT6V48aI888x19OvX1CYSMiaI5JYoKrp3PK3jVILIZGNimDwdOZLCc899x/Hjqdx8cyyvvtqRqlVLBjosY8xZyi1RhAPF+WuCyGSJwuTo0KFkoqMjiIyMoGzZaKZM6UxkZDjx8ZcEOjRjzDnKLVHsVNWRfovEBDVV5cMP1zFw4Dz697+SJ59sBcBNN8UGODJjzPnKLVFYT6PxyqZN++nb9ysWLvwdgMWLt2VNUWqMCX65JYq2fovCBKXk5DReeOE7nn32O06eTKds2Wheeqkd99zTxJKEMSHkjIlCVQ/4M5B8YTPa+c2uXcdo2fJtNm92fkzuuacJL73UjvLlYwIcmTEmv4XWk042o53fVKpUjGrVShEREcakSfG0alUz0CEZY3wktBJFJpvRLt9lZChvvLGC666rxSWXlENE+OCDmyhTJpqiRcMDHZ4xxofsqSeTp9Wrd9GixVv06fMVfft+haqTiCtVKm5JwphCIDRrFCZfHDt2khEjvubVV5eSnq5ceGEJ+vSJC3RYxhg/s0RhcvT55z8zYMAcEhOPEBYmDBjQlGeeaUPJkpGBDs0Y42eWKMxpduw4wm23TSclJZ0rrqjM5MmdiYu7MNBhGWMCxBKFASA1NZ2IiDBEhCpVSjJqVBuKFg2nb98rbc5qYwo5+wtgWLJkO1dc8Trvv78ma90jj1zNgAHNLEkYYyxRFGYHDpzg/vu/pEWLt1i7dg8TJy7PuqPJGGMyWdNTIaSqvP/+Gh555L/s3ZtEkSJhDBnSgmHDrrWhN4wxp7FEUcjs3n2M22//lEWLtgLQqlUNJk2KJza2QmADM8YUWJYoCpnSpaPYufMY5cvHMHp0O3r2vNRqEcaYXFmiKATmz/+Vyy+vTLlyMURGRjBt2i1UrlyccuVsAD9jTN5CpzN7RnygIyhwdu48yu23f0r79u8zdOiCrPUNG1a0JGGM8Vro1CgyR461UWNJT89gypQVPPbYQo4cSSE6OoK6dcvZZELGmHMSOoki001fBTqCgFq5cid9+sxi2bI/AYiPr8P48Z2oWbN0YAMzxgSt0EsUhdjWrYdo2vQN0tOVKlVKMG7cDfz97/WsFmGMOS8+TRQi0hEYC4QD/1bV57NtHwT8A0gD9gL/p6p/+DKmUFazZmnuvbcJJUpE8q9/taZECRvAzxhz/nzWmS0i4cAE4AagPnC7iNTPVmwVEKeqjYHpwIvndLJC2pG9deshbrzxQ775ZmvWutdfv5ExYzpYkjDG5Btf1iiaAltU9TcAEfkI6ApsyCygqos8yi8F7jqnMxWyjuzU1HTGjPmBf/3rG06cSGPfviR++KEXgDUzGWPynS8TRRVgu8dyItAsl/K9gDk5bRCR3kBvgOrVq5/5CIWgI/u777bRp88s1q/fC8BttzVkzJj2AY7KGBPKCkRntojcBcQBrXLarqqvA68DxMXFFcpR6w4ePMHgwfN5881VANSuXYaJE+Np3752gCMzxoQ6XyaKHUA1j+Wq7rq/EJHrgWFAK1VN8WE8QS0jQ/nii18oUiSMRx+9hsceu4bo6CKBDssYUwj4MlEsA+qISC2cBHEbcIdnARG5DJgCdFTVPT6MJSj9/PM+atUqTWRkBOXKxfCf/9xE9eqlqFevfKBDM8YUIj6760lV04D+wDxgI/CJqq4XkZEi0sUt9hJQHJgmIgkiMtNX8QSTpKRUhg1bSOPGk3jxxe+z1rdvX9uShDHG73zaR6Gqs4HZ2dY95fH+el+ePxjNnbuFvn2/4vffDwGwb19SYAMyxhR6BaIz28Cffx7l4YfnMm2ac/dwo0YVmTy5M1dfXS2PPY0xxreCO1HMiD/1DEUQ27RpP3Fxr3P06EliYoowYkQrHn64OUWKhAc6NGOMCfJE4Zkkgvhhuzp1ynLllVUoVqwIr712AzVqlA50SMYYkyW4E0WmR4Lr0YojR1J46qlF9O17JZdcUg4RYebM2yhWrGigQzPGmNOERqIIEqrK9OkbeOihuezceYyff97H3LnOqCWWJIwxBZUlCj/57beD9O8/mzlztgDQvHlVXnjBbvoyxhR8lih87OTJdEaPXsLTTy8mOTmN0qWjeP75ttx33xWEhdkAfsaYgs8ShY9t336YkSO/ISUlnTvvbMTLL7enUqXigQ7LGGO8FryJogDPQXHw4AlKl45CRKhduyxjx3bk4ovL0rbtRYEOzRhjzprPhvDwuQI4B0VGhvLWW6u4+OLXeP/9NVnr778/zpKEMSZoBW+iyFRA5qBYv34PrVtPpVevmRw4cCKr09oYY4Jd8DY9FRBJSak8/fQ3jB79A2lpGVSsWIxXXunA7bc3DHRoxhiTL4IzURSQ/olNm/bTocP7bN16CBHo0+cKnn22LWXKRAc6NGOMyTfBmSgKSP9EjRqliIqK4NJLKzF5cmeaN68a0HhMwZKamkpiYiLJycmBDsUUIlFRUVStWpUiRfJvYrPgTBSZ/Nw/kZaWweTJy7n99oaUKxdDZGQEc+feSZUqJYmICP7uHpO/EhMTKVGiBDVr1kTEnpkxvqeq7N+/n8TERGrVqpVvx7W/bl766acdNG36BgMGzGHo0AVZ62vUKG1JwuQoOTmZcuXKWZIwfiMilCtXLt9rscFdo/CDw4eTGTbsf0ycuAxVqF69FF271g10WCZIWJIw/uaLn7ngSxSHNvvlNKrKxx+vZ+DAeezadYyIiDAGDWrOU0+1sgH8jDGFSvC1maQccf71cUf26tW7uf32T9m16xhXX12NlSt788IL7SxJmKASHh5OkyZNaNiwITfeeCOHDh3K2rZ+/XratGlD3bp1qVOnDk8//TSqp4bsnzNnDnFxcdSvX5/LLruMRx55JACfIHerVq2iV69egQ7jjFJSUujevTsXX3wxzZo1Y+vWrTmWGzt2LA0bNqRBgwa8+uqrWeuffPJJGjduTJMmTWjfvj1//vknALNmzeKpp57K8Vg+oapB9bqiKqqjUV9IS0v/y/LAgXP1jTdWaHp6hk/OZ0Lbhg0bAh2CFitWLOt9z5499ZlnnlFV1aSkJL3ooot03rx5qqp6/Phx7dixo44fP15VVdeuXasXXXSRbty4UVVV09LSdOLEifkaW2pq6nkfo1u3bpqQkODXc56NCRMm6P3336+qqh9++KHeeuutp5VZu3atNmjQQI8fP66pqanatm1b3bx5s6qqHj58OKvc2LFjs46VkZGhTZo00ePHj+d43px+9oDleo5/d4Ov6clHFi36nb59ZzNlSmdatqwBwJgxHQIclQkZL/uor+IsJu266qqrWLPGGVrmgw8+oEWLFrRv3x6AmJgYxo8fT+vWrenXrx8vvvgiw4YNo169eoBTM3nggQdOO+axY8cYMGAAy5cvR0QYPnw4N998M8WLF+fYsWMATJ8+nVmzZjF16lTuueceoqKiWLVqFS1atGDGjBkkJCRQunRpAOrUqcN3331HWFgYffr0Ydu2bQC8+uqrtGjR4i/nPnr0KGvWrOHSSy8F4KeffuKhhx4iOTmZ6Oho3n77berWrcvUqVOZMWMGx44dIz09ndmzZzNgwADWrVtHamoqI0aMoGvXrmzdupUePXpw/PhxAMaPH8/VV1/t9fXNyRdffMGIESMA6NatG/3790dV/9KPsHHjRpo1a0ZMTAwArVq1YsaMGQwZMoSSJUtmlTt+/HjWfiJC69atmTVrFrfeeut5xeiNQp8o9uw5zuDB83n33dUAjBnzQ1aiMCZUpKens3DhwqxmmvXr13PFFVf8pUzt2rU5duwYR44cYd26dV41NT399NOUKlWKtWvXAnDw4ME890lMTGTJkiWEh4eTnp7OZ599xr333suPP/5IjRo1qFSpEnfccQcDBw7kmmuuYdu2bXTo0IGNGzf+5TjLly+nYcNTIyDUq1ePb7/9loiICBYsWMDjjz/Op59+CsDKlStZs2YNZcuW5fHHH6dNmza89dZbHDp0iKZNm3L99ddTsWJF5s+fT1RUFJs3b+b2229n+fLlp8V/7bXXcvTo0dPWjx49muuv/+scMzt27KBatWoAREREUKpUKfbv30/58uWzyjRs2JBhw4axf/9+oqOjmT17NnFxcVnbhw0bxrvvvkupUqVYtGhR1vq4uDi+/fZbSxRnlA/9ExkZyptvrmTo0AUcPJhMZGQ4TzzRksGDz+8bhDE5CtB0vSdOnKBJkybs2LGD2NhY2rVrl6/HX7BgAR999FHWcpkyZfLc55ZbbiE8PByA7t27M3LkSO69914++ugjunfvnnXcDRs2ZO1z5MgRjh07RvHip4bo37lzJxUqVMhaPnz4MHfffTebN29GREhNTc3a1q5dO8qWLQvAf//7X2bOnMno0aMB5zbmbdu2ceGFF9K/f38SEhIIDw9n06ZNOcb/7bff5vkZz0ZsbCxDhw6lffv2FCtWjCZNmmRdH4BRo0YxatQonnvuOcaPH8+//vUvACpWrJjVZ+FrwdeZDef9oN3vvx/k2mvfpnfvWRw8mEz79rVZt64vTzzRksjI4MydxuQkOjqahIQE/vjjD1SVCRMmAFC/fn1WrFjxl7K//fYbxYsXp2TJkjRo0OC07WfDs2kl+z39xYoVy3p/1VVXsWXLFvbu3cvnn3/OTTfdBEBGRgZLly4lISGBhIQEduzY8ZckkfnZPI/95JNPct1117Fu3Tq+/PLLv2zzPKeq8umnn2Yde9u2bcTGxvLKK69QqVIlVq9ezfLlyzl58mSOn+3aa6+lSZMmp70WLFhwWtkqVaqwfft2ANLS0jh8+DDlypU7rVyvXr1YsWIFixcvpkyZMlxyySWnlbnzzjuzakiZ1zU62j/DBQVnojhPJUtGsmnTfi64oDgffXQzc+feycUXlw10WMb4TExMDOPGjePll18mLS2NO++8k++++y7rj9uJEyd48MEHGTJkCACDBw/m2WefzfpWnZGRweTJk087brt27bKSD5xqeqpUqRIbN24kIyODzz777IxxiQh///vfGTRoELGxsVl/RNu3b89rr72WVS4hIeG0fWNjY9my5dQozYcPH6ZKlSoATJ069Yzn7NChA6+99lrWHV6rVq3K2r9y5cqEhYXx3nvvkZ6enuP+3377bVaS8Xxlb3YC6NKlC++88w7g9NW0adMmx+cc9uzZA8C2bduYMWMGd9xxBwCbN596HOCLL77I6jMC2LRp01+a3nzqXHvBA/W6ouq53fE0d+5mTU4+dcfDkiXb9NChE+d0LGO8UdDuelJV7dy5s7777ruqqrpmzRpt1aqVXnLJJVq7dm0dMWKEZmScusPvyy+/1Msvv1zr1aunsbGxOnjw4NOOf/ToUe3Zs6c2aNBAGzdurJ9++qmqqk6bNk0vuugibdasmfbr10/vvvtuVVW9++67ddq0aX85xrJlyxTQqVOnZq3bu3ev3nrrrdqoUSONjY3Nutsnu4YNG+qRI0dUVXXJkiVap04dbdKkiQ4bNkxr1Kihqqpvv/229uvXL2ufpKQk7d27tzZs2FDr16+v8fHxqqq6adMmbdSokTZu3FiHDBly2rU7FydOnNBu3bpp7dq19corr9Rff/1VVVV37NihN9xwQ1a5a665RmNjY7Vx48a6YMGCrPU33XSTNmjQQBs1aqSdO3fWxMTErG3x8fG6Zs2aHM+b33c9iWpg2k7PVVw10eXbvY95+/bDPPjgXD7//Geefvo6nniipQ+jM+aUjRs3EhsbG+gwQtorr7xCiRIl+Mc//hHoUPxq9+7d3HHHHSxcuDDH7Tn97InIClWNy3GHPIRs01NaWgZjxvxAbOwEPv/8Z4oXL0rZsjb8tzGh5IEHHiAyMjLQYfjdtm3bePnll/12vpDsuV26NJE+fWaxevVuAG6+OZaxYztSpUrJPPY0xgSTqKgoevToEegw/O7KK6/06/lCLlH8+GMiV1/9JqpQs2Zpxo+/gfj40+8gMMYfNNvDVcb4mi+6E0IuUTRtWoUOHS7msssu4IknWhITk3+TdxhzNqKioti/f78NNW78RtWZjyIqKipfjxv0ndmbN+9n4MB5jBnTgUsucW6ty8hQwsLsF9MEls1wZwLhTDPcnU9ndtDWKFJS0nj++e947rnvSElJJyoqgunTnUfZLUmYgqBIkSL5OsuYMYHi07ueRKSjiPwiIltE5NEctkeKyMfu9h9FpKY3x1248DcaN57MiBHfkJKSzr33NmHy5M75Hr8xxhgf1ihEJByYALQDEoFlIjJTVTd4FOsFHFTVi0XkNuAFoHtux/39QGmuv/49AGJjyzN5cmcbxM8YY3zIlzWKpsAWVf1NVU8CHwFds5XpCrzjvp8OtJU8ev0OJkUTFRXBs8+2ISGhjyUJY4zxMZ91ZotIN6Cjqv7DXe4BNFPV/h5l1rllEt3lX90y+7IdqzfQ211sCKzzSdDBpzywL89ShYNdi1PsWpxi1+KUuqpa4lx2DIrObFV9HXgdQESWn2vPfaixa3GKXYtT7FqcYtfiFBE5fXINL/my6WkHUM1juaq7LscyIhIBlAL2+zAmY4wxZ8mXiWIZUEdEaolIUeA2YGa2MjOBu9333YD/abA92GGMMSHOZ01PqpomIv2BeUA48JaqrheRkTjD3c4E3gTeE5EtwAGcZJKX130VcxCya3GKXYtT7FqcYtfilHO+FkH3ZLYxxhj/Ctlhxo0xxuQPSxTGGGNyVWATha+G/whGXlyLQSKyQUTWiMhCEQnZpxDzuhYe5W4WERWRkL010ptrISK3uj8b60XkA3/H6C9e/I5UF5FFIrLK/T3pFIg4fU1E3hKRPe4zajltFxEZ516nNSJyuVcHPtc5VH35wun8/hW4CCgKrAbqZyvTF5jsvr8N+DjQcQfwWlwHxLjvHyjM18ItVwJYDCwF4gIddwB/LuoAq4Ay7nLFQMcdwGvxOvCA+74+sDXQcfvoWrQELgfWnWF7J2AOIEBz4EdvjltQaxQ+Gf4jSOV5LVR1kaomuYtLcZ5ZCUXe/FwAPI0zblgoj+/tzbW4D5igqgcBVHWPn2P0F2+uhQKZU1yWAv70Y3x+o6qLce4gPZOuwLvqWAqUFpHKeR23oCaKKsB2j+VEd12OZVQ1DTgMlPNLdP7lzbXw1AvnG0MoyvNauFXpaqr6lT8DCwBvfi4uAS4Rke9FZKmIdPRbdP7lzbUYAdwlIonAbGCAf0IrcM727wkQJEN4GO+IyF1AHNAq0LEEgoiEAWOAewIcSkERgdP81BqnlrlYRBqp6qFABhUgtwNTVfVlEbkK5/mthqqaEejAgkFBrVHY8B+neHMtEJHrgWFAF1VN8VNs/pbXtSiBM2jk1yKyFacNdmaIdmh783ORCMxU1VRV/R3YhJM4Qo0316IX8AmAqv4AROEMGFjYePX3JLuCmihs+I9T8rwWInIZMAUnSYRqOzTkcS1U9bCqllfVmqpaE6e/pouqnvNgaAWYN78jn+PUJhCR8jhNUb/5MUZ/8eZabAPaAohILE6i2OvXKAuGmUBP9+6n5sBhVd2Z104FsulJfTf8R9Dx8lq8BBQHprn9+dtUtUvAgvYRL69FoeDltZgHtBeRDUA6MFhVQ67W7eW1eAR4Q0QG4nRs3xOKXyxF5EOcLwfl3f6Y4UARAFWdjNM/0wnYAiQB93p13BC8VsYYY/JRQW16MsYYU0BYojDGGJMrSxTGGGNyZYnCGGNMrixRGGOMyZUlClMgiUi6iCR4vGrmUvZYPpxvqoj87p5rpfv07tke498iUt99/3i2bUvON0b3OJnXZZ2IfCkipfMo3yRUR0o1/mO3x5oCSUSOqWrx/C6byzGmArNUdbqItAdGq2rj8zjeeceU13FF5B1gk6qOyqX8PTgj6PbP71hM4WE1ChMURKS4O9fGShFZKyKnjRorIpVFZLHHN+5r3fXtReQHd99pIpLXH/DFwMXuvoPcY60TkYfddcVE5CsRWe2u7+6u/1pE4kTkeSDajeM/7rZj7r8fiUi8R8xTRaSbiISLyEsissydJ+B+Ly7LD7gDuolIU/czrhKRJSJS131KeSTQ3Y2luxv7WyLyk1s2p9F3jfmrQI+fbi975fTCeZI4wX19hjOKQEl3W3mcJ0sza8TH3H8fAYa578Nxxn4qj/OHv5i7fijwVA7nmwp0c9/fAvwIXAGsBYrhPPm+HrgMuBl4w2PfUu6/X+POf5EZk0eZzBj/Drzjvi+KM5JnNNAbeMJdHwksB2rlEOcxj883DejoLpcEItz31wOfuu/vAcZ77P8scJf7vjTO+E/FAv3/ba+C/SqQQ3gYA5xQ1SaZCyJSBHhWRFoCGTjfpCsBuzz2WQa85Zb9XFUTRKQVzkQ137vDmxTF+Saek5dE5AmcMYB64YwN9JmqHndjmAFcC8wFXhaRF3Caq749i881BxgrIpFAR2Cxqp5wm7sai0g3t1wpnAH8fs+2f7SIJLiffyMw36P8OyJSB2eIiiJnOH97oIuI/NNdjgKqu8cyJkeWKEywuBOoAFyhqqnijA4b5VlAVRe7iSQemCoiY4CDwHxVvd2LcwxW1emZCyLSNqdCqrpJnHkvOgHPiMhCVR3pzYdQ1WQR+RroAHTHmWQHnBnHBqjqvDwOcUJVm4hIDM7YRv2AcTiTNS1S1b+7Hf9fn2F/AW5W1V+8idcYsD4KEzxKAXvcJHEdcNq84OLMFb5bVd8A/o0zJeRSoIWIZPY5FBORS7w857fA30QkRkSK4TQbfSsiFwJJqvo+zoCMOc07nOrWbHLyMc5gbJm1E3D+6D+QuY+IXOKeM0fqzGj4IPCInBpmP3O46Hs8ih7FaYLLNA8YIG71SpyRh43JlSUKEyz+A8SJyFqgJ/BzDmVaA6tFZBXOt/WxqroX5w/nhyKyBqfZqZ43J1TVlTh9Fz/h9Fn8W1VXAY2An9wmoOHAMzns/jqwJrMzO5v/4kwutUCdqTvBSWwbgJUisg5n2Phca/xuLGtwJuV5EXjO/eye+y0C6md2ZuPUPIq4sa13l43Jld0ea4wxJldWozDGGJMrSxTGGGNyZYnCGGNMrixRGGOMyZUlCmOMMbmyRGGMMSZXliiMMcbk6v8BRciXY/AqOq4AAAAASUVORK5CYII=\n",
      "text/plain": [
       "<Figure size 432x288 with 1 Axes>"
      ]
     },
     "metadata": {
      "needs_background": "light"
     },
     "output_type": "display_data"
    }
   ],
   "source": [
    "# Compute ROC curve and ROC area for each class\n",
    "fpr = dict()\n",
    "tpr = dict()\n",
    "roc_auc = dict()\n",
    "for i in range(n_classes):\n",
    "    fpr[i], tpr[i], _ = roc_curve(y_test[:, i], y_score[:, i])\n",
    "    roc_auc[i] = auc(fpr[i], tpr[i])\n",
    "# Compute micro-average ROC curve and ROC area\n",
    "fpr[\"micro\"], tpr[\"micro\"], _ = roc_curve(y_test.ravel(), y_score.ravel())\n",
    "roc_auc[\"micro\"] = auc(fpr[\"micro\"], tpr[\"micro\"])\n",
    "fig  = plt.figure()\n",
    "\n",
    "plt.plot(fpr[0], tpr[0], color='darkorange',\n",
    "         lw=2, label='ROC curve (area = %0.2f)' % roc_auc[0])\n",
    "plt.plot([0, 1], [0, 1], color='navy', lw=lw, linestyle='--')\n",
    "plt.xlim([0.0, 1.0])\n",
    "plt.ylim([0.0, 1.05])\n",
    "plt.xlabel('False Positive Rate')\n",
    "plt.ylabel('True Positive Rate')\n",
    "plt.title('REPEAT%s ROC curve' % re)\n",
    "plt.legend(loc=\"lower right\")\n",
    "plt.show()\n",
    "fig.savefig(\"eval/output_%s/roc_curve.png\" % re)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 预测"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 388,
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Anaerobic\n",
      "Curved\n",
      "GramN\n",
      "GramPc\n",
      "GramPr\n",
      "Intracellular\n"
     ]
    }
   ],
   "source": [
    "sped_path = \"../4.Label_prepare/paper_md\"\n",
    "sped_l = [ele for ele in os.listdir(sped_path) if '.' not in ele]\n",
    "\n",
    "test_set = {\"data\":[{'paragraphs':[], 'title':\"LungB\"}], \"version\":\"LungB\"}\n",
    "test_qa = []\n",
    "\n",
    "for sped in sped_l:\n",
    "    \n",
    "    print(sped)\n",
    "    df = pd.DataFrame(np.zeros((0,4)), columns = [\"SPECIES\", \"Title\", \"Ab\", \"Full\"])\n",
    "    \n",
    "    id_l = [ele for ele in os.listdir(\"%s/%s\" % (sped_path, sped))if 'txt' in ele]\n",
    "    \n",
    "    for idx in id_l:\n",
    "        \n",
    "        txt = open(\"%s/%s/%s\" % (sped_path, sped, idx)).readlines()\n",
    "        idx = idx.split(\".\")[0]\n",
    "        df.loc[idx] = [txt[0].split(\":\")[1], txt[2], txt[3], txt[2] + txt[3]]\n",
    "        \n",
    "        qa = {'qas': [{\n",
    "            'id': '%s:%s' % (idx, df.loc[idx, \"SPECIES\"]),\n",
    "            'question': 'Can %s cause Pneumonia?' % df.loc[idx, \"SPECIES\"]}],\n",
    "            'context': '%s' % df.loc[idx, \"Full\"]}\n",
    "        test_qa.append(qa)\n",
    "    df.to_csv(\"input/test/%s.csv\" % sped, header=True, index=True)\n",
    "\n",
    "test_set['data'][0][\"paragraphs\"] = test_qa\n",
    "json.dump(test_set, open(\"input/test_set.json\", \"w\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "prediction = json.load(open(\"output/predictions_.json\", 'r'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {
    "tags": []
   },
   "outputs": [],
   "source": [
    "result = {}\n",
    "for i in prediction.keys():\n",
    "    toke = i.split(\":\")[0]\n",
    "    species = \"\"\n",
    "    for j in toke.split(\"_\")[0:-1]:\n",
    "        species = species + \" \" + j\n",
    "    ncbi = \"https://pubmed.ncbi.nlm.nih.gov/%s/\" % toke.split(\"_\")[-1]\n",
    "\n",
    "    result[toke] = [toke.split(\"_\")[-1], species, prediction[i][1][0], ncbi]\n",
    "    \n",
    "result = pd.DataFrame(result, index = [\"ID\", \"SPECIES\", \"rate\", \"link\"]).T"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {
    "tags": []
   },
   "outputs": [],
   "source": [
    "a = result.loc[result[\"rate\"]>0.95, :]\n",
    "a.to_csv(\"output/prediction.csv\", header=True, index=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\Users\\xinzh\\anaconda3\\lib\\site-packages\\numpy\\core\\_asarray.py:136: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this, you must specify 'dtype=object' when creating the ndarray\n",
      "  return array(a, dtype, copy=False, order=order, subok=True)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Anaerobic 27 26\n",
      "Curved 9 8\n",
      "GramN 99 94\n",
      "GramPc 46 42\n",
      "GramPr 56 53\n",
      "Intracellular 17 15\n"
     ]
    }
   ],
   "source": [
    "sped_l = os.listdir(\"../4.Label_prepare/paper_md/\")\n",
    "for sped in sped_l:\n",
    "    spec_l = os.listdir(\"../4.Label_prepare/paper_md/%s\" % sped)\n",
    "    spec_l = [spec.replace(\".txt\", \"\") for spec in spec_l]\n",
    "    spec_tl = [spec for spec in result.index if spec in spec_l]\n",
    "    spec_yl = [spec for spec in a.index if spec in spec_l]\n",
    "#     print(sped, len(np.unique(spec_tl)), len(np.unique(spec_yl)))\n",
    "    specc_tl = [spec.split(\"_\")[0:-1] for spec in spec_tl]\n",
    "    specc_yl = [spec.split(\"_\")[0:-1] for spec in spec_yl]\n",
    "    print(sped, len(np.unique(specc_tl)), len(np.unique(specc_yl)))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "collapsed": true,
    "jupyter": {
     "outputs_hidden": true
    },
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " Actinobacteria\n",
      " Actinobacteria\n",
      " Actinobacteria\n",
      " Actinobacteria\n",
      " Actinobacteria\n",
      " Actinobacteria\n",
      " Actinobacteria\n",
      " Actinobacteria\n",
      " Actinobacteria\n",
      " Actinobacteria\n",
      " Actinobacteria\n",
      " Actinobacteria\n",
      " Actinobacteria\n",
      " Actinobacteria\n",
      " Actinobacteria\n",
      " Actinobacteria\n",
      " Actinobacteria\n",
      " Actinobacteria\n",
      " Actinobacteria\n",
      " Actinobacteria\n",
      " Actinobacteria\n",
      " Actinobacteria\n",
      " Actinobacteria\n",
      " Actinobacteria\n",
      " Actinobacteria\n",
      " Actinobacteria\n",
      " Actinobacteria\n",
      " Actinobacteria\n",
      " Actinobacteria\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces\n",
      " Actinomyces israelii\n",
      " Actinomyces\n",
      " Actinomyces israelii\n",
      " Actinomyces\n",
      " Actinomyces israelii\n",
      " Actinomyces\n",
      " Actinomyces israelii\n",
      " Actinomyces\n",
      " Actinomyces israelii\n",
      " Actinomyces\n",
      " Actinomyces israelii\n",
      " Actinomyces\n",
      " Actinomyces israelii\n",
      " Actinomyces\n",
      " Actinomyces israelii\n",
      " Actinomyces\n",
      " Actinomyces israelii\n",
      " Actinomyces\n",
      " Actinomyces israelii\n",
      " Actinomyces\n",
      " Actinomyces israelii\n",
      " Actinomyces\n",
      " Actinomyces israelii\n",
      " Actinomyces\n",
      " Actinomyces israelii\n",
      " Actinomyces\n",
      " Actinomyces israelii\n",
      " Actinomyces\n",
      " Actinomyces israelii\n",
      " Actinomyces\n",
      " Actinomyces israelii\n",
      " Actinomyces\n",
      " Actinomyces israelii\n",
      " Actinomyces\n",
      " Actinomyces israelii\n",
      " Actinomyces\n",
      " Actinomyces israelii\n",
      " Actinomyces\n",
      " Actinomyces israelii\n",
      " Actinomyces\n",
      " Actinomyces meyeri\n",
      " Actinomyces\n",
      " Actinomyces meyeri\n",
      " Actinomyces\n",
      " Actinomyces meyeri\n",
      " Actinomyces\n",
      " Actinomyces meyeri\n",
      " Actinomyces\n",
      " Actinomyces meyeri\n",
      " Actinomyces\n",
      " Actinomyces meyeri\n",
      " Actinomyces\n",
      " Actinomyces meyeri\n",
      " Actinomyces\n",
      " Actinomyces meyeri\n",
      " Actinomyces\n",
      " Actinomyces meyeri\n",
      " Actinomyces\n",
      " Actinomyces meyeri\n",
      " Actinomyces\n",
      " Actinomyces meyeri\n",
      " Actinomyces\n",
      " Actinomyces meyeri\n",
      " Actinomyces\n",
      " Actinomyces meyeri\n",
      " Actinomyces\n",
      " Actinomyces meyeri\n",
      " Actinomyces\n",
      " Actinomyces meyeri\n",
      " Actinomyces\n",
      " Actinomyces meyeri\n",
      " Actinomyces\n",
      " Actinomyces meyeri\n",
      " Actinomyces\n",
      " Actinomyces meyeri\n",
      " Actinomyces\n",
      " Actinomyces meyeri\n",
      " Actinomyces\n",
      " Actinomyces meyeri\n",
      " Actinomyces\n",
      " Actinomyces meyeri\n",
      " Actinomyces\n",
      " Actinomyces meyeri\n",
      " Actinomyces\n",
      " Actinomyces meyeri\n",
      " Actinomyces\n",
      " Actinomyces meyeri\n",
      " Actinomyces\n",
      " Actinomyces odontolyticus\n",
      " Actinomyces\n",
      " Actinomyces odontolyticus\n",
      " Actinomyces\n",
      " Actinomyces odontolyticus\n",
      " Actinomyces\n",
      " Actinomyces odontolyticus\n",
      " Actinomyces\n",
      " Actinomyces odontolyticus\n",
      " Actinomyces\n",
      " Actinomyces odontolyticus\n",
      " Actinomyces\n",
      " Actinomyces odontolyticus\n",
      " Actinomyces\n",
      " Actinomyces odontolyticus\n",
      " Actinomyces\n",
      " Actinomyces odontolyticus\n",
      " Actinomyces\n",
      " Actinomyces odontolyticus\n",
      " Actinomyces\n",
      " Actinomyces odontolyticus\n",
      " Actinomyces\n",
      " Actinomyces odontolyticus\n",
      " Actinomyces\n",
      " Actinomyces odontolyticus\n",
      " Bacteroides\n",
      " Bacteroides\n",
      " Bacteroides\n",
      " Bacteroides\n",
      " Bacteroides\n",
      " Bacteroides\n",
      " Bacteroides\n",
      " Bacteroides\n",
      " Bacteroides\n",
      " Bacteroides\n",
      " Bacteroides\n",
      " Bacteroides\n",
      " Bacteroides\n",
      " Bacteroides\n",
      " Bacteroides\n",
      " Bacteroides\n",
      " Bacteroides\n",
      " Bacteroides\n",
      " Bacteroides\n",
      " Bacteroides\n",
      " Bacteroides\n",
      " Bacteroides\n",
      " Bacteroides\n",
      " Bacteroides\n",
      " Bacteroides\n",
      " Bacteroides\n",
      " Bacteroides\n",
      " Bacteroides\n",
      " Bacteroides\n",
      " Bacteroides\n",
      " Bacteroides\n",
      " Bacteroides fragilis\n",
      " Bacteroides\n",
      " Bacteroides fragilis\n",
      " Bacteroides\n",
      " Bacteroides fragilis\n",
      " Bacteroides\n",
      " Bacteroides fragilis\n",
      " Bacteroides\n",
      " Bacteroides fragilis\n",
      " Bacteroides\n",
      " Bacteroides fragilis\n",
      " Bacteroides\n",
      " Bacteroides fragilis\n",
      " Bacteroides\n",
      " Bacteroides fragilis\n",
      " Bacteroides\n",
      " Bacteroides fragilis\n",
      " Bacteroides\n",
      " Bacteroides fragilis\n",
      " Bacteroides\n",
      " Bacteroides fragilis\n",
      " Bacteroides\n",
      " Bacteroides fragilis\n",
      " Bacteroides\n",
      " Bacteroides fragilis\n",
      " Bacteroides\n",
      " Bacteroides fragilis\n",
      " Bacteroides\n",
      " Bacteroides fragilis\n",
      " Bacteroides\n",
      " Bacteroides fragilis\n",
      " Bacteroides\n",
      " Bacteroides fragilis\n",
      " Bacteroides\n",
      " Bacteroides fragilis\n",
      " Bacteroides\n",
      " Bacteroides fragilis\n",
      " Bifidobacterium\n",
      " Clostridium\n",
      " Clostridium botulinum\n",
      " Clostridium\n",
      " Clostridium botulinum\n",
      " Clostridium\n",
      " Clostridium botulinum\n",
      " Clostridium\n",
      " Clostridium botulinum\n",
      " Clostridium\n",
      " Clostridium botulinum\n",
      " Clostridium\n",
      " Clostridium difficile\n",
      " Clostridium\n",
      " Clostridium difficile\n",
      " Clostridium\n",
      " Clostridium difficile\n",
      " Clostridium\n",
      " Clostridium difficile\n",
      " Clostridium\n",
      " Clostridium difficile\n",
      " Clostridium\n",
      " Clostridium difficile\n",
      " Clostridium\n",
      " Clostridium difficile\n",
      " Clostridium\n",
      " Clostridium difficile\n",
      " Clostridium\n",
      " Clostridium difficile\n",
      " Clostridium\n",
      " Clostridium difficile\n",
      " Clostridium\n",
      " Clostridium difficile\n",
      " Clostridium\n",
      " Clostridium difficile\n",
      " Clostridium\n",
      " Clostridium difficile\n",
      " Clostridium\n",
      " Clostridium difficile\n",
      " Clostridium\n",
      " Clostridium difficile\n",
      " Clostridium\n",
      " Clostridium difficile\n",
      " Clostridium\n",
      " Clostridium difficile\n",
      " Clostridium\n",
      " Clostridium difficile\n",
      " Clostridium\n",
      " Clostridium difficile\n",
      " Clostridium\n",
      " Clostridium difficile\n",
      " Clostridium\n",
      " Clostridium difficile\n",
      " Clostridium\n",
      " Clostridium perfringens\n",
      " Clostridium\n",
      " Clostridium perfringens\n",
      " Clostridium\n",
      " Clostridium perfringens\n",
      " Clostridium\n",
      " Clostridium perfringens\n",
      " Clostridium\n",
      " Clostridium perfringens\n",
      " Clostridium\n",
      " Clostridium perfringens\n",
      " Clostridium\n",
      " Clostridium perfringens\n",
      " Clostridium\n",
      " Clostridium perfringens\n",
      " Clostridium\n",
      " Clostridium perfringens\n",
      " Clostridium\n",
      " Clostridium perfringens\n",
      " Clostridium\n",
      " Clostridium perfringens\n",
      " Clostridium\n",
      " Clostridium perfringens\n",
      " Clostridium\n",
      " Clostridium perfringens\n",
      " Clostridium\n",
      " Clostridium perfringens\n",
      " Clostridium\n",
      " Clostridium perfringens\n",
      " Clostridium\n",
      " Clostridium perfringens\n",
      " Clostridium\n",
      " Clostridium putrificum\n",
      " Clostridium\n",
      " Clostridium putrificum\n",
      " Clostridium\n",
      " Clostridium putrificum\n",
      " Clostridium\n",
      " Clostridium putrificum\n",
      " Clostridium\n",
      " Clostridium tetani\n",
      " Clostridium\n",
      " Clostridium tetani\n",
      " Clostridium\n",
      " Clostridium tetani\n",
      " Clostridium\n",
      " Clostridium tetani\n",
      " Clostridium\n",
      " Clostridium tetani\n",
      " Eubacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium necrophorum\n",
      " Fusobacterium\n",
      " Fusobacterium nucleatum\n",
      " Fusobacterium\n",
      " Fusobacterium nucleatum\n",
      " Fusobacterium\n",
      " Fusobacterium nucleatum\n",
      " Fusobacterium\n",
      " Fusobacterium nucleatum\n",
      " Fusobacterium\n",
      " Fusobacterium nucleatum\n",
      " Fusobacterium\n",
      " Fusobacterium nucleatum\n",
      " Fusobacterium\n",
      " Fusobacterium nucleatum\n",
      " Fusobacterium\n",
      " Fusobacterium nucleatum\n",
      " Fusobacterium\n",
      " Fusobacterium nucleatum\n",
      " Fusobacterium\n",
      " Fusobacterium nucleatum\n",
      " Fusobacterium\n",
      " Fusobacterium nucleatum\n",
      " Fusobacterium\n",
      " Fusobacterium nucleatum\n",
      " Fusobacterium\n",
      " Fusobacterium nucleatum\n",
      " Lactobacillus\n",
      " Lactobacillus\n",
      " Lactobacillus\n",
      " Lactobacillus\n",
      " Lactobacillus\n",
      " Lactobacillus\n",
      " Lactobacillus\n",
      " Lactobacillus\n",
      " Lactobacillus\n",
      " Leptotrichia\n",
      " Leptotrichia\n",
      " Leptotrichia\n",
      " Parvimonas\n",
      " Parvimonas micra\n",
      " Parvimonas\n",
      " Parvimonas micra\n",
      " Parvimonas\n",
      " Parvimonas micra\n",
      " Parvimonas\n",
      " Parvimonas micra\n",
      " Parvimonas\n",
      " Parvimonas micra\n",
      " Peptostreptococcus\n",
      " Peptostreptococcus\n",
      " Peptostreptococcus\n",
      " Peptostreptococcus\n",
      " Peptostreptococcus\n",
      " Peptostreptococcus\n",
      " Peptostreptococcus\n",
      " Peptostreptococcus\n",
      " Peptostreptococcus\n",
      " Peptostreptococcus\n",
      " Peptostreptococcus\n",
      " Peptostreptococcus\n",
      " Peptostreptococcus\n",
      " Peptostreptococcus\n",
      " Peptostreptococcus\n",
      " Peptostreptococcus\n",
      " Peptostreptococcus\n",
      " Peptostreptococcus\n",
      " Peptostreptococcus\n",
      " Peptostreptococcus\n",
      " Peptostreptococcus\n",
      " Peptostreptococcus\n",
      " Peptostreptococcus\n",
      " Peptostreptococcus\n",
      " Peptostreptococcus\n",
      " Peptostreptococcus\n",
      " Peptostreptococcus\n",
      " Porphyromonas\n",
      " Porphyromonas\n",
      " Porphyromonas\n",
      " Porphyromonas\n",
      " Porphyromonas\n",
      " Porphyromonas\n",
      " Prevotella\n",
      " Prevotella\n",
      " Prevotella\n",
      " Prevotella\n",
      " Prevotella\n",
      " Prevotella\n",
      " Prevotella\n",
      " Prevotella\n",
      " Prevotella\n",
      " Prevotella\n",
      " Prevotella\n",
      " Prevotella\n",
      " Prevotella\n",
      " Prevotella\n",
      " Prevotella\n",
      " Prevotella\n",
      " Prevotella\n",
      " Prevotella\n",
      " Prevotella\n",
      " Prevotella\n",
      " Prevotella\n",
      " Propionibacterium\n",
      " Propionibacterium\n",
      " Propionibacterium\n",
      " Propionibacterium\n",
      " Propionibacterium\n",
      " Propionibacterium\n",
      " Propionibacterium\n",
      " Propionibacterium acnes\n",
      " Propionibacterium\n",
      " Propionibacterium acnes\n",
      " Propionibacterium\n",
      " Propionibacterium acnes\n",
      " Propionibacterium\n",
      " Propionibacterium acnes\n",
      " Propionibacterium\n",
      " Propionibacterium acnes\n",
      " Propionibacterium\n",
      " Propionibacterium acnes\n",
      " Staphylococcus\n",
      " Staphylococcus saccharolyticu\n",
      " Staphylococcus\n",
      " Staphylococcus saccharolyticu\n",
      " Staphylococcus\n",
      " Staphylococcus saccharolyticu\n",
      " Staphylococcus\n",
      " Staphylococcus saccharolyticu\n",
      " Staphylococcus\n",
      " Staphylococcus saccharolyticu\n",
      " Staphylococcus\n",
      " Staphylococcus saccharolyticu\n",
      " Staphylococcus\n",
      " Staphylococcus saccharolyticu\n",
      " Staphylococcus\n",
      " Staphylococcus saccharolyticu\n",
      " Staphylococcus\n",
      " Staphylococcus saccharolyticu\n",
      " Staphylococcus\n",
      " Staphylococcus saccharolyticu\n",
      " Staphylococcus\n",
      " Staphylococcus saccharolyticu\n",
      " Staphylococcus\n",
      " Staphylococcus saccharolyticu\n",
      " Staphylococcus\n",
      " Staphylococcus saccharolyticu\n",
      " Staphylococcus\n",
      " Staphylococcus saccharolyticu\n",
      " Staphylococcus\n",
      " Staphylococcus saccharolyticu\n",
      " Staphylococcus\n",
      " Staphylococcus saccharolyticu\n",
      " Staphylococcus\n",
      " Staphylococcus saccharolyticu\n",
      " Staphylococcus\n",
      " Staphylococcus saccharolyticu\n",
      " Staphylococcus\n",
      " Staphylococcus saccharolyticu\n",
      " Staphylococcus\n",
      " Staphylococcus saccharolyticu\n",
      " Staphylococcus\n",
      " Staphylococcus saccharolyticu\n",
      " Staphylococcus\n",
      " Staphylococcus saccharolyticu\n",
      " Staphylococcus\n",
      " Staphylococcus saccharolyticu\n",
      " Staphylococcus\n",
      " Staphylococcus saccharolyticu\n",
      " Staphylococcus\n",
      " Staphylococcus saccharolyticu\n",
      " Staphylococcus\n",
      " Staphylococcus saccharolyticu\n",
      " Staphylococcus\n",
      " Staphylococcus saccharolyticu\n",
      " Staphylococcus\n",
      " Staphylococcus saccharolyticu\n",
      " Staphylococcus\n",
      " Staphylococcus saccharolyticu\n",
      " Staphylococcus\n",
      " Staphylococcus saccharolyticu\n",
      " Staphylococcus\n",
      " Staphylococcus saccharolyticu\n",
      " Staphylococcus\n",
      " Staphylococcus saccharolyticu\n",
      " Anaerobes\n",
      " Anaerobes\n",
      " Anaerobes\n",
      " Anaerobes\n",
      " Anaerobes\n",
      " Anaerobes\n",
      " Anaerobes\n",
      " Anaerobes\n",
      " Anaerobes\n",
      " Anaerobes\n",
      " Anaerobes\n",
      " Anaerobes\n",
      " Anaerobes\n",
      " Anaerobes\n",
      " Anaerobes\n",
      " Anaerobes\n",
      " Anaerobes\n",
      " Anaerobes\n",
      " Anaerobes\n",
      " Anaerobes\n",
      " Anaerobes\n",
      " Anaerobes\n",
      " Anaerobes\n",
      " Anaerobes\n",
      " Anaerobes\n",
      " Anaerobes\n",
      " Anaerobes\n",
      " Anaerobes\n",
      " Anaerobes\n",
      " Anaerobes\n",
      " Anaerobes\n",
      " Anaerobes\n",
      " Anaerobes\n",
      " Borrelia\n",
      " Borrelia burgdorferi\n",
      " Borrelia\n",
      " Borrelia burgdorferi\n",
      " Borrelia\n",
      " Borrelia burgdorferi\n",
      " Borrelia\n",
      " Borrelia burgdorferi\n",
      " Borrelia\n",
      " Borrelia burgdorferi\n",
      " Campylobacter\n",
      " Campylobacter fetus\n",
      " Campylobacter\n",
      " Campylobacter fetus\n",
      " Campylobacter\n",
      " Campylobacter fetus\n",
      " Campylobacter\n",
      " Campylobacter fetus\n",
      " Campylobacter\n",
      " Campylobacter fetus\n",
      " Campylobacter\n",
      " Campylobacter jejuni\n",
      " Campylobacter\n",
      " Campylobacter jejuni\n",
      " Campylobacter\n",
      " Campylobacter jejuni\n",
      " Campylobacter\n",
      " Campylobacter jejuni\n",
      " Campylobacter\n",
      " Campylobacter jejuni\n",
      " Helicobacter\n",
      " Helicobacter pylori\n",
      " Helicobacter\n",
      " Helicobacter pylori\n",
      " Helicobacter\n",
      " Helicobacter pylori\n",
      " Helicobacter\n",
      " Helicobacter pylori\n",
      " Helicobacter\n",
      " Helicobacter pylori\n",
      " Helicobacter\n",
      " Helicobacter pylori\n",
      " Helicobacter\n",
      " Helicobacter pylori\n",
      " Leptospira\n",
      " Leptospira interrogans\n",
      " Leptospira\n",
      " Leptospira interrogans\n",
      " Leptospira\n",
      " Leptospira interrogans\n",
      " Leptospira\n",
      " Leptospira interrogans\n",
      " Leptospira\n",
      " Leptospira interrogans\n",
      " Leptospira\n",
      " Leptospira interrogans\n",
      " Leptospira\n",
      " Leptospira interrogans\n",
      " Treponema\n",
      " Treponema pallidum\n",
      " Treponema\n",
      " Treponema pallidum\n",
      " Treponema\n",
      " Treponema pallidum\n",
      " Treponema\n",
      " Treponema pallidum\n",
      " Vibrio\n",
      " Vibrio cholerae\n",
      " Vibrio\n",
      " Vibrio cholerae\n",
      " Vibrio\n",
      " Vibrio cholerae\n",
      " Achromobacter\n",
      " Achromobacter denitrificans\n",
      " Achromobacter\n",
      " Achromobacter denitrificans\n",
      " Achromobacter\n",
      " Achromobacter denitrificans\n",
      " Achromobacter\n",
      " Achromobacter denitrificans\n",
      " Achromobacter\n",
      " Achromobacter denitrificans\n",
      " Achromobacter\n",
      " Achromobacter denitrificans\n",
      " Achromobacter\n",
      " Achromobacter denitrificans\n",
      " Achromobacter\n",
      " Achromobacter denitrificans\n",
      " Achromobacter\n",
      " Achromobacter denitrificans\n",
      " Achromobacter\n",
      " Achromobacter denitrificans\n",
      " Achromobacter\n",
      " Achromobacter denitrificans\n",
      " Achromobacter\n",
      " Achromobacter xylosoxidans\n",
      " Achromobacter\n",
      " Achromobacter xylosoxidans\n",
      " Achromobacter\n",
      " Achromobacter xylosoxidans\n",
      " Achromobacter\n",
      " Achromobacter xylosoxidans\n",
      " Achromobacter\n",
      " Achromobacter xylosoxidans\n",
      " Achromobacter\n",
      " Achromobacter xylosoxidans\n",
      " Achromobacter\n",
      " Achromobacter xylosoxidans\n",
      " Achromobacter\n",
      " Achromobacter xylosoxidans\n",
      " Achromobacter\n",
      " Achromobacter xylosoxidans\n",
      " Achromobacter\n",
      " Achromobacter xylosoxidans\n",
      " Achromobacter\n",
      " Achromobacter xylosoxidans\n",
      " Achromobacter\n",
      " Achromobacter xylosoxidans\n",
      " Achromobacter\n",
      " Achromobacter xylosoxidans\n",
      " Achromobacter\n",
      " Achromobacter xylosoxidans\n",
      " Achromobacter\n",
      " Achromobacter xylosoxidans\n",
      " Achromobacter\n",
      " Achromobacter xylosoxidans\n",
      " Achromobacter\n",
      " Achromobacter xylosoxidans\n",
      " Achromobacter\n",
      " Achromobacter xylosoxidans\n",
      " Achromobacter\n",
      " Achromobacter xylosoxidans\n",
      " Achromobacter\n",
      " Achromobacter xylosoxidans\n",
      " Achromobacter\n",
      " Achromobacter xylosoxidans\n",
      " Achromobacter\n",
      " Achromobacter xylosoxidans\n",
      " Achromobacter\n",
      " Achromobacter xylosoxidans\n",
      " Achromobacter\n",
      " Achromobacter xylosoxidans\n",
      " Achromobacter\n",
      " Achromobacter xylosoxidans\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter baumannii\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter calcoaceticus\n",
      " Acinetobacter\n",
      " Acinetobacter lwoffii\n",
      " Acinetobacter\n",
      " Acinetobacter lwoffii\n",
      " Acinetobacter\n",
      " Acinetobacter lwoffii\n",
      " Acinetobacter\n",
      " Acinetobacter lwoffii\n",
      " Acinetobacter\n",
      " Acinetobacter nosocomialis\n",
      " Acinetobacter\n",
      " Acinetobacter nosocomialis\n",
      " Acinetobacter\n",
      " Acinetobacter nosocomialis\n",
      " Acinetobacter\n",
      " Acinetobacter nosocomialis\n",
      " Acinetobacter\n",
      " Acinetobacter nosocomialis\n",
      " Acinetobacter\n",
      " Acinetobacter nosocomialis\n",
      " Acinetobacter\n",
      " Acinetobacter nosocomialis\n",
      " Actinobacillus\n",
      " Actinobacillus actinomycetemcomitans\n",
      " Actinobacillus\n",
      " Actinobacillus actinomycetemcomitans\n",
      " Actinobacillus\n",
      " Actinobacillus actinomycetemcomitans\n",
      " Actinobacillus\n",
      " Actinobacillus actinomycetemcomitans\n",
      " Actinobacillus\n",
      " Actinobacillus actinomycetemcomitans\n",
      " Actinobacillus\n",
      " Actinobacillus actinomycetemcomitans\n",
      " Actinobacillus\n",
      " Actinobacillus actinomycetemcomitans\n",
      " Actinobacillus\n",
      " Actinobacillus actinomycetemcomitans\n",
      " Actinobacillus\n",
      " Actinobacillus actinomycetemcomitans\n",
      " Actinobacillus\n",
      " Actinobacillus actinomycetemcomitans\n",
      " Actinobacillus\n",
      " Actinobacillus actinomycetemcomitans\n",
      " Actinobacillus\n",
      " Actinobacillus equuli\n",
      " Actinobacillus\n",
      " Actinobacillus equuli\n",
      " Actinobacillus\n",
      " Actinobacillus hominis\n",
      " Actinobacillus\n",
      " Actinobacillus hominis\n",
      " Actinobacillus\n",
      " Actinobacillus hominis\n",
      " Actinobacillus\n",
      " Actinobacillus hominis\n",
      " Actinobacillus\n",
      " Actinobacillus hominis\n",
      " Actinobacillus\n",
      " Actinobacillus hominis\n",
      " Actinobacillus\n",
      " Actinobacillus hominis\n",
      " Actinobacillus\n",
      " Actinobacillus lignieresii\n",
      " Aeromonas\n",
      " Aeromonas hydrophila\n",
      " Aeromonas\n",
      " Aeromonas hydrophila\n",
      " Aeromonas\n",
      " Aeromonas hydrophila\n",
      " Aeromonas\n",
      " Aeromonas hydrophila\n",
      " Aeromonas\n",
      " Aeromonas hydrophila\n",
      " Aeromonas\n",
      " Aeromonas hydrophila\n",
      " Aeromonas\n",
      " Aeromonas hydrophila\n",
      " Aeromonas\n",
      " Aeromonas hydrophila\n",
      " Aeromonas\n",
      " Aeromonas hydrophila\n",
      " Aeromonas\n",
      " Aeromonas hydrophila\n",
      " Aeromonas\n",
      " Aeromonas hydrophila\n",
      " Aeromonas\n",
      " Aeromonas hydrophila\n",
      " Aeromonas\n",
      " Aeromonas hydrophila\n",
      " Aeromonas\n",
      " Aeromonas hydrophila\n",
      " Aeromonas\n",
      " Aeromonas hydrophila\n",
      " Aeromonas\n",
      " Aeromonas hydrophila\n",
      " Aeromonas\n",
      " Aeromonas hydrophila\n",
      " Bordetella\n",
      " Bordetella avium\n",
      " Bordetella\n",
      " Bordetella avium\n",
      " Bordetella\n",
      " Bordetella avium\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella bronchiseptica\n",
      " Bordetella\n",
      " Bordetella hinzii\n",
      " Bordetella\n",
      " Bordetella hinzii\n",
      " Bordetella\n",
      " Bordetella hinzii\n",
      " Bordetella\n",
      " Bordetella hinzii\n",
      " Bordetella\n",
      " Bordetella hinzii\n",
      " Bordetella\n",
      " Bordetella holmesii\n",
      " Bordetella\n",
      " Bordetella holmesii\n",
      " Bordetella\n",
      " Bordetella holmesii\n",
      " Bordetella\n",
      " Bordetella holmesii\n",
      " Bordetella\n",
      " Bordetella holmesii\n",
      " Bordetella\n",
      " Bordetella holmesii\n",
      " Bordetella\n",
      " Bordetella holmesii\n",
      " Bordetella\n",
      " Bordetella holmesii\n",
      " Bordetella\n",
      " Bordetella holmesii\n",
      " Bordetella\n",
      " Bordetella holmesii\n",
      " Bordetella\n",
      " Bordetella holmesii\n",
      " Bordetella\n",
      " Bordetella holmesii\n",
      " Bordetella\n",
      " Bordetella holmesii\n",
      " Bordetella\n",
      " Bordetella holmesii\n",
      " Bordetella\n",
      " Bordetella holmesii\n",
      " Bordetella\n",
      " Bordetella holmesii\n",
      " Bordetella\n",
      " Bordetella holmesii\n",
      " Bordetella\n",
      " Bordetella holmesii\n",
      " Bordetella\n",
      " Bordetella holmesii\n",
      " Bordetella\n",
      " Bordetella holmesii\n",
      " Bordetella\n",
      " Bordetella holmesii\n",
      " Bordetella\n",
      " Bordetella holmesii\n",
      " Bordetella\n",
      " Bordetella parapertussis\n",
      " Bordetella\n",
      " Bordetella parapertussis\n",
      " Bordetella\n",
      " Bordetella parapertussis\n",
      " Bordetella\n",
      " Bordetella parapertussis\n",
      " Bordetella\n",
      " Bordetella parapertussis\n",
      " Bordetella\n",
      " Bordetella parapertussis\n",
      " Bordetella\n",
      " Bordetella parapertussis\n",
      " Bordetella\n",
      " Bordetella parapertussis\n",
      " Bordetella\n",
      " Bordetella parapertussis\n",
      " Bordetella\n",
      " Bordetella parapertussis\n",
      " Bordetella\n",
      " Bordetella parapertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Bordetella\n",
      " Bordetella pertussis\n",
      " Brucella\n",
      " Brucella abortus\n",
      " Brucella\n",
      " Brucella melitensis\n",
      " Brucella\n",
      " Brucella melitensis\n",
      " Brucella\n",
      " Brucella melitensis\n",
      " Brucella\n",
      " Brucella melitensis\n",
      " Brucella\n",
      " Brucella melitensis\n",
      " Brucella\n",
      " Brucella melitensis\n",
      " Brucella\n",
      " Brucella melitensis\n",
      " Brucella\n",
      " Brucella suis\n",
      " Burkholderia\n",
      " Burkholderia ambifaria\n",
      " Burkholderia\n",
      " Burkholderia ambifaria\n",
      " Burkholderia\n",
      " Burkholderia cenocepacia\n",
      " Burkholderia\n",
      " Burkholderia cenocepacia\n",
      " Burkholderia\n",
      " Burkholderia cenocepacia\n",
      " Burkholderia\n",
      " Burkholderia cenocepacia\n",
      " Burkholderia\n",
      " Burkholderia cenocepacia\n",
      " Burkholderia\n",
      " Burkholderia cenocepacia\n",
      " Burkholderia\n",
      " Burkholderia cenocepacia\n",
      " Burkholderia\n",
      " Burkholderia cenocepacia\n",
      " Burkholderia\n",
      " Burkholderia cenocepacia\n",
      " Burkholderia\n",
      " Burkholderia cenocepacia\n",
      " Burkholderia\n",
      " Burkholderia cenocepacia\n",
      " Burkholderia\n",
      " Burkholderia cenocepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia cepacia\n",
      " Burkholderia\n",
      " Burkholderia contaminans\n",
      " Burkholderia\n",
      " Burkholderia contaminans\n",
      " Burkholderia\n",
      " Burkholderia contaminans\n",
      " Burkholderia\n",
      " Burkholderia contaminans\n",
      " Burkholderia\n",
      " Burkholderia contaminans\n",
      " Burkholderia\n",
      " Burkholderia contaminans\n",
      " Burkholderia\n",
      " Burkholderia dolosa\n",
      " Burkholderia\n",
      " Burkholderia gladioli\n",
      " Burkholderia\n",
      " Burkholderia gladioli\n",
      " Burkholderia\n",
      " Burkholderia gladioli\n",
      " Burkholderia\n",
      " Burkholderia gladioli\n",
      " Burkholderia\n",
      " Burkholderia mallei\n",
      " Burkholderia\n",
      " Burkholderia mallei\n",
      " Burkholderia\n",
      " Burkholderia mallei\n",
      " Burkholderia\n",
      " Burkholderia multivorans\n",
      " Burkholderia\n",
      " Burkholderia multivorans\n",
      " Burkholderia\n",
      " Burkholderia multivorans\n",
      " Burkholderia\n",
      " Burkholderia multivorans\n",
      " Burkholderia\n",
      " Burkholderia multivorans\n",
      " Burkholderia\n",
      " Burkholderia multivorans\n",
      " Burkholderia\n",
      " Burkholderia multivorans\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia pseudomallei\n",
      " Burkholderia\n",
      " Burkholderia stabilis\n",
      " Burkholderia\n",
      " Burkholderia stabilis\n",
      " Burkholderia\n",
      " Burkholderia stabilis\n",
      " Burkholderia\n",
      " Burkholderia stabilis\n",
      " Burkholderia\n",
      " Burkholderia stabilis\n",
      " Burkholderia\n",
      " Burkholderia stabilis\n",
      " Burkholderia\n",
      " Burkholderia stabilis\n",
      " Burkholderia\n",
      " Burkholderia thailandensis\n",
      " Burkholderia\n",
      " Burkholderia vietnamiensis\n",
      " Burkholderia\n",
      " Burkholderia vietnamiensis\n",
      " Burkholderia\n",
      " Burkholderia vietnamiensis\n",
      " Burkholderia\n",
      " Burkholderia vietnamiensis\n",
      " Burkholderia\n",
      " Burkholderia vietnamiensis\n",
      " Burkholderia\n",
      " Burkholderia vietnamiensis\n",
      " Burkholderia\n",
      " Burkholderia vietnamiensis\n",
      " Cardiobacterium\n",
      " Cardiobacterium hominis\n",
      " Cardiobacterium\n",
      " Cardiobacterium hominis\n",
      " Cardiobacterium\n",
      " Cardiobacterium hominis\n",
      " Cardiobacterium\n",
      " Cardiobacterium hominis\n",
      " Cardiobacterium\n",
      " Cardiobacterium hominis\n",
      " Cardiobacterium\n",
      " Cardiobacterium hominis\n",
      " Cardiobacterium\n",
      " Cardiobacterium hominis\n",
      " Cardiobacterium\n",
      " Cardiobacterium hominis\n",
      " Cardiobacterium\n",
      " Cardiobacterium hominis\n",
      " Chromobacterium\n",
      " Chromobacterium violaceum\n",
      " Chromobacterium\n",
      " Chromobacterium violaceum\n",
      " Chromobacterium\n",
      " Chromobacterium violaceum\n",
      " Chromobacterium\n",
      " Chromobacterium violaceum\n",
      " Chryseobacterium\n",
      " Chryseobacterium anthropib\n",
      " Chryseobacterium\n",
      " Chryseobacterium anthropib\n",
      " Chryseobacterium\n",
      " Chryseobacterium anthropib\n",
      " Chryseobacterium\n",
      " Chryseobacterium anthropib\n",
      " Chryseobacterium\n",
      " Chryseobacterium anthropib\n",
      " Chryseobacterium\n",
      " Chryseobacterium anthropib\n",
      " Chryseobacterium\n",
      " Chryseobacterium anthropib\n",
      " Chryseobacterium\n",
      " Chryseobacterium anthropib\n",
      " Chryseobacterium\n",
      " Chryseobacterium anthropib\n",
      " Chryseobacterium\n",
      " Chryseobacterium anthropib\n",
      " Chryseobacterium\n",
      " Chryseobacterium anthropib\n",
      " Chryseobacterium\n",
      " Chryseobacterium anthropib\n",
      " Chryseobacterium\n",
      " Chryseobacterium anthropib\n",
      " Chryseobacterium\n",
      " Chryseobacterium anthropib\n",
      " Chryseobacterium\n",
      " Chryseobacterium anthropib\n",
      " Chryseobacterium\n",
      " Chryseobacterium anthropib\n",
      " Chryseobacterium\n",
      " Chryseobacterium anthropib\n",
      " Chryseobacterium\n",
      " Chryseobacterium anthropib\n",
      " Chryseobacterium\n",
      " Chryseobacterium anthropib\n",
      " Chryseobacterium\n",
      " Chryseobacterium anthropib\n",
      " Chryseobacterium\n",
      " Chryseobacterium anthropib\n",
      " Chryseobacterium\n",
      " Chryseobacterium anthropib\n",
      " Chryseobacterium\n",
      " Chryseobacterium anthropib\n",
      " Chryseobacterium\n",
      " Chryseobacterium anthropib\n",
      " Chryseobacterium\n",
      " Chryseobacterium anthropib\n",
      " Chryseobacterium\n",
      " Chryseobacterium anthropib\n",
      " Chryseobacterium\n",
      " Chryseobacterium anthropib\n",
      " Chryseobacterium\n",
      " Chryseobacterium anthropib\n",
      " Chryseobacterium\n",
      " Chryseobacterium anthropib\n",
      " Chryseobacterium\n",
      " Chryseobacterium anthropib\n",
      " Chryseobacterium\n",
      " Chryseobacterium anthropib\n",
      " Chryseobacterium\n",
      " Chryseobacterium anthropib\n",
      " Chryseobacterium\n",
      " Chryseobacterium anthropib\n",
      " Chryseobacterium\n",
      " Chryseobacterium anthropib\n",
      " Chryseobacterium\n",
      " Chryseobacterium anthropib\n",
      " Chryseobacterium\n",
      " Chryseobacterium gleum\n",
      " Chryseobacterium\n",
      " Chryseobacterium gleum\n",
      " Chryseobacterium\n",
      " Chryseobacterium gleum\n",
      " Chryseobacterium\n",
      " Chryseobacterium gleum\n",
      " Chryseobacterium\n",
      " Chryseobacterium gleum\n",
      " Chryseobacterium\n",
      " Chryseobacterium indologenes\n",
      " Chryseobacterium\n",
      " Chryseobacterium indologenes\n",
      " Chryseobacterium\n",
      " Chryseobacterium indologenes\n",
      " Chryseobacterium\n",
      " Chryseobacterium indologenes\n",
      " Chryseobacterium\n",
      " Chryseobacterium indologenes\n",
      " Chryseobacterium\n",
      " Chryseobacterium indologenes\n",
      " Chryseobacterium\n",
      " Chryseobacterium indologenes\n",
      " Chryseobacterium\n",
      " Chryseobacterium indologenes\n",
      " Chryseobacterium\n",
      " Chryseobacterium indologenes\n",
      " Chryseobacterium\n",
      " Chryseobacterium indologenes\n",
      " Chryseobacterium\n",
      " Chryseobacterium indologenes\n",
      " Chryseobacterium\n",
      " Chryseobacterium indologenes\n",
      " Chryseobacterium\n",
      " Chryseobacterium indologenes\n",
      " Chryseobacterium\n",
      " Chryseobacterium indologenes\n",
      " Chryseobacterium\n",
      " Chryseobacterium indologenes\n",
      " Chryseobacterium\n",
      " Chryseobacterium indologenes\n",
      " Chryseobacterium\n",
      " Chryseobacterium indologenes\n",
      " Chryseobacterium\n",
      " Chryseobacterium indologenes\n",
      " Chryseobacterium\n",
      " Chryseobacterium indologenes\n",
      " Chryseobacterium\n",
      " Chryseobacterium indologenes\n",
      " Chryseobacterium\n",
      " Chryseobacterium indologenes\n",
      " Chryseobacterium\n",
      " Chryseobacterium indologenes\n",
      " Chryseobacterium\n",
      " Chryseobacterium indologenes\n",
      " Chryseobacterium\n",
      " Chryseobacterium indologenes\n",
      " Citrobacter\n",
      " Citrobacter freundii\n",
      " Citrobacter\n",
      " Citrobacter freundii\n",
      " Citrobacter\n",
      " Citrobacter freundii\n",
      " Citrobacter\n",
      " Citrobacter freundii\n",
      " Citrobacter\n",
      " Citrobacter freundii\n",
      " Citrobacter\n",
      " Citrobacter freundii\n",
      " Citrobacter\n",
      " Citrobacter freundii\n",
      " Citrobacter\n",
      " Citrobacter freundii\n",
      " Citrobacter\n",
      " Citrobacter freundii\n",
      " Citrobacter\n",
      " Citrobacter freundii\n",
      " Citrobacter\n",
      " Citrobacter freundii\n",
      " Citrobacter\n",
      " Citrobacter freundii\n",
      " Citrobacter\n",
      " Citrobacter freundii\n",
      " Citrobacter\n",
      " Citrobacter freundii\n",
      " Citrobacter\n",
      " Citrobacter freundii\n",
      " Citrobacter\n",
      " Citrobacter freundii\n",
      " Citrobacter\n",
      " Citrobacter freundii\n",
      " Citrobacter\n",
      " Citrobacter freundii\n",
      " Eikenella\n",
      " Eikenella corrodens\n",
      " Eikenella\n",
      " Eikenella corrodens\n",
      " Eikenella\n",
      " Eikenella corrodens\n",
      " Eikenella\n",
      " Eikenella corrodens\n",
      " Eikenella\n",
      " Eikenella corrodens\n",
      " Eikenella\n",
      " Eikenella corrodens\n",
      " Eikenella\n",
      " Eikenella corrodens\n",
      " Eikenella\n",
      " Eikenella corrodens\n",
      " Eikenella\n",
      " Eikenella corrodens\n",
      " Eikenella\n",
      " Eikenella corrodens\n",
      " Eikenella\n",
      " Eikenella corrodens\n",
      " Eikenella\n",
      " Eikenella corrodens\n",
      " Eikenella\n",
      " Eikenella corrodens\n",
      " Eikenella\n",
      " Eikenella corrodens\n",
      " Eikenella\n",
      " Eikenella corrodens\n",
      " Eikenella\n",
      " Eikenella corrodens\n",
      " Eikenella\n",
      " Eikenella corrodens\n",
      " Eikenella\n",
      " Eikenella corrodens\n",
      " Eikenella\n",
      " Eikenella corrodens\n",
      " Eikenella\n",
      " Eikenella corrodens\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Elizabethkingia\n",
      " Elizabethkingia meningoseptica\n",
      " Enterobacter\n",
      " Enterobacter aerogenes\n",
      " Enterobacter\n",
      " Enterobacter aerogenes\n",
      " Enterobacter\n",
      " Enterobacter aerogenes\n",
      " Enterobacter\n",
      " Enterobacter aerogenes\n",
      " Enterobacter\n",
      " Enterobacter aerogenes\n",
      " Enterobacter\n",
      " Enterobacter aerogenes\n",
      " Enterobacter\n",
      " Enterobacter aerogenes\n",
      " Enterobacter\n",
      " Enterobacter aerogenes\n",
      " Enterobacter\n",
      " Enterobacter aerogenes\n",
      " Enterobacter\n",
      " Enterobacter aerogenes\n",
      " Enterobacter\n",
      " Enterobacter aerogenes\n",
      " Enterobacter\n",
      " Enterobacter aerogenes\n",
      " Enterobacter\n",
      " Enterobacter aerogenes\n",
      " Enterobacter\n",
      " Enterobacter aerogenes\n",
      " Enterobacter\n",
      " Enterobacter aerogenes\n",
      " Enterobacter\n",
      " Enterobacter aerogenes\n",
      " Enterobacter\n",
      " Enterobacter aerogenes\n",
      " Enterobacter\n",
      " Enterobacter aerogenes\n",
      " Enterobacter\n",
      " Enterobacter aerogenes\n",
      " Enterobacter\n",
      " Enterobacter aerogenes\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Enterobacter\n",
      " Enterobacter cloacae\n",
      " Escherichia\n",
      " Escherichia coli\n",
      " Escherichia\n",
      " Escherichia coli\n",
      " Escherichia\n",
      " Escherichia coli\n",
      " Escherichia\n",
      " Escherichia coli\n",
      " Escherichia\n",
      " Escherichia coli\n",
      " Escherichia\n",
      " Escherichia coli\n",
      " Escherichia\n",
      " Escherichia coli\n",
      " Escherichia\n",
      " Escherichia coli\n",
      " Escherichia\n",
      " Escherichia coli\n",
      " Escherichia\n",
      " Escherichia coli\n",
      " Escherichia\n",
      " Escherichia coli\n",
      " Escherichia\n",
      " Escherichia coli\n",
      " Escherichia\n",
      " Escherichia coli\n",
      " Escherichia\n",
      " Escherichia coli\n",
      " Escherichia\n",
      " Escherichia coli\n",
      " Escherichia\n",
      " Escherichia coli\n",
      " Escherichia\n",
      " Escherichia coli\n",
      " Escherichia\n",
      " Escherichia coli\n",
      " Escherichia\n",
      " Escherichia coli\n",
      " Escherichia\n",
      " Escherichia coli\n",
      " Escherichia\n",
      " Escherichia coli\n",
      " Escherichia\n",
      " Escherichia coli\n",
      " Francisella\n",
      " Francisella hispanensis\n",
      " Francisella\n",
      " Francisella hispanensis\n",
      " Francisella\n",
      " Francisella hispanensis\n",
      " Francisella\n",
      " Francisella hispanensis\n",
      " Francisella\n",
      " Francisella hispanensis\n",
      " Francisella\n",
      " Francisella hispanensis\n",
      " Francisella\n",
      " Francisella hispanensis\n",
      " Francisella\n",
      " Francisella hispanensis\n",
      " Francisella\n",
      " Francisella hispanensis\n",
      " Francisella\n",
      " Francisella hispanensis\n",
      " Francisella\n",
      " Francisella hispanensis\n",
      " Francisella\n",
      " Francisella hispanensis\n",
      " Francisella\n",
      " Francisella hispanensis\n",
      " Francisella\n",
      " Francisella hispanensis\n",
      " Francisella\n",
      " Francisella hispanensis\n",
      " Francisella\n",
      " Francisella hispanensis\n",
      " Francisella\n",
      " Francisella hispanensis\n",
      " Francisella\n",
      " Francisella hispanensis\n",
      " Francisella\n",
      " Francisella hispanensis\n",
      " Francisella\n",
      " Francisella hispanensis\n",
      " Francisella\n",
      " Francisella tularensis\n",
      " Francisella\n",
      " Francisella tularensis\n",
      " Francisella\n",
      " Francisella tularensis\n",
      " Francisella\n",
      " Francisella tularensis\n",
      " Francisella\n",
      " Francisella tularensis\n",
      " Francisella\n",
      " Francisella tularensis\n",
      " Francisella\n",
      " Francisella tularensis\n",
      " Francisella\n",
      " Francisella tularensis\n",
      " Francisella\n",
      " Francisella tularensis\n",
      " Francisella\n",
      " Francisella tularensis\n",
      " Francisella\n",
      " Francisella tularensis\n",
      " Francisella\n",
      " Francisella tularensis\n",
      " Francisella\n",
      " Francisella tularensis\n",
      " Francisella\n",
      " Francisella tularensis\n",
      " Francisella\n",
      " Francisella tularensis\n",
      " Francisella\n",
      " Francisella tularensis\n",
      " Francisella\n",
      " Francisella tularensis\n",
      " Francisella\n",
      " Francisella tularensis\n",
      " Francisella\n",
      " Francisella tularensis\n",
      " Francisella\n",
      " Francisella tularensis\n",
      " Francisella\n",
      " Francisella tularensis\n",
      " Francisella\n",
      " Francisella tularensis\n",
      " Francisella\n",
      " Francisella tularensis\n",
      " Francisella\n",
      " Francisella tularensis\n",
      " Francisella\n",
      " Francisella tularensis\n",
      " Francisella\n",
      " Francisella tularensis\n",
      " Francisella\n",
      " Francisella tularensis\n",
      " Haemophilus\n",
      " Haemophilus aegyptius\n",
      " Haemophilus\n",
      " Haemophilus aegyptius\n",
      " Haemophilus\n",
      " Haemophilus aphrophilus\n",
      " Haemophilus\n",
      " Haemophilus aphrophilus\n",
      " Haemophilus\n",
      " Haemophilus aphrophilus\n",
      " Haemophilus\n",
      " Haemophilus aphrophilus\n",
      " Haemophilus\n",
      " Haemophilus aphrophilus\n",
      " Haemophilus\n",
      " Haemophilus aphrophilus\n",
      " Haemophilus\n",
      " Haemophilus haemolyticus\n",
      " Haemophilus\n",
      " Haemophilus haemolyticus\n",
      " Haemophilus\n",
      " Haemophilus haemolyticus\n",
      " Haemophilus\n",
      " Haemophilus haemolyticus\n",
      " Haemophilus\n",
      " Haemophilus haemolyticus\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus influenzae\n",
      " Haemophilus\n",
      " Haemophilus parainfluenzae\n",
      " Haemophilus\n",
      " Haemophilus parainfluenzae\n",
      " Haemophilus\n",
      " Haemophilus parainfluenzae\n",
      " Haemophilus\n",
      " Haemophilus parainfluenzae\n",
      " Haemophilus\n",
      " Haemophilus parainfluenzae\n",
      " Haemophilus\n",
      " Haemophilus parainfluenzae\n",
      " Haemophilus\n",
      " Haemophilus parainfluenzae\n",
      " Haemophilus\n",
      " Haemophilus parainfluenzae\n",
      " Haemophilus\n",
      " Haemophilus parainfluenzae\n",
      " Haemophilus\n",
      " Haemophilus parainfluenzae\n",
      " Haemophilus\n",
      " Haemophilus parainfluenzae\n",
      " Haemophilus\n",
      " Haemophilus parainfluenzae\n",
      " Haemophilus\n",
      " Haemophilus parainfluenzae\n",
      " Haemophilus\n",
      " Haemophilus parainfluenzae\n",
      " Haemophilus\n",
      " Haemophilus parainfluenzae\n",
      " Haemophilus\n",
      " Haemophilus parainfluenzae\n",
      " Haemophilus\n",
      " Haemophilus parainfluenzae\n",
      " Haemophilus\n",
      " Haemophilus parainfluenzae\n",
      " Haemophilus\n",
      " Haemophilus parainfluenzae\n",
      " Haemophilus\n",
      " Haemophilus parainfluenzae\n",
      " Haemophilus\n",
      " Haemophilus parainfluenzae\n",
      " Inquilinus\n",
      " Inquilinus limosus\n",
      " Inquilinus\n",
      " Inquilinus limosus\n",
      " Inquilinus\n",
      " Inquilinus limosus\n",
      " Inquilinus\n",
      " Inquilinus limosus\n",
      " Inquilinus\n",
      " Inquilinus limosus\n",
      " Kingella\n",
      " Kingella kingae\n",
      " Kingella\n",
      " Kingella kingae\n",
      " Kingella\n",
      " Kingella kingae\n",
      " Kingella\n",
      " Kingella kingae\n",
      " Kingella\n",
      " Kingella kingae\n",
      " Kingella\n",
      " Kingella kingae\n",
      " Klebsiella\n",
      " Klebsiella oxytoca\n",
      " Klebsiella\n",
      " Klebsiella oxytoca\n",
      " Klebsiella\n",
      " Klebsiella oxytoca\n",
      " Klebsiella\n",
      " Klebsiella oxytoca\n",
      " Klebsiella\n",
      " Klebsiella oxytoca\n",
      " Klebsiella\n",
      " Klebsiella oxytoca\n",
      " Klebsiella\n",
      " Klebsiella oxytoca\n",
      " Klebsiella\n",
      " Klebsiella oxytoca\n",
      " Klebsiella\n",
      " Klebsiella oxytoca\n",
      " Klebsiella\n",
      " Klebsiella oxytoca\n",
      " Klebsiella\n",
      " Klebsiella oxytoca\n",
      " Klebsiella\n",
      " Klebsiella oxytoca\n",
      " Klebsiella\n",
      " Klebsiella oxytoca\n",
      " Klebsiella\n",
      " Klebsiella oxytoca\n",
      " Klebsiella\n",
      " Klebsiella oxytoca\n",
      " Klebsiella\n",
      " Klebsiella oxytoca\n",
      " Klebsiella\n",
      " Klebsiella oxytoca\n",
      " Klebsiella\n",
      " Klebsiella oxytoca\n",
      " Klebsiella\n",
      " Klebsiella oxytoca\n",
      " Klebsiella\n",
      " Klebsiella ozaenae\n",
      " Klebsiella\n",
      " Klebsiella ozaenae\n",
      " Klebsiella\n",
      " Klebsiella ozaenae\n",
      " Klebsiella\n",
      " Klebsiella ozaenae\n",
      " Klebsiella\n",
      " Klebsiella ozaenae\n",
      " Klebsiella\n",
      " Klebsiella ozaenae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella pneumoniae\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Klebsiella\n",
      " Klebsiella rhinoscleromatis\n",
      " Legionella\n",
      " Legionella anisa\n",
      " Legionella\n",
      " Legionella anisa\n",
      " Legionella\n",
      " Legionella anisa\n",
      " Legionella\n",
      " Legionella anisa\n",
      " Legionella\n",
      " Legionella anisa\n",
      " Legionella\n",
      " Legionella anisa\n",
      " Legionella\n",
      " Legionella anisa\n",
      " Legionella\n",
      " Legionella anisa\n",
      " Legionella\n",
      " Legionella anisa\n",
      " Legionella\n",
      " Legionella anisa\n",
      " Legionella\n",
      " Legionella anisa\n",
      " Legionella\n",
      " Legionella anisa\n",
      " Legionella\n",
      " Legionella dumoffii\n",
      " Legionella\n",
      " Legionella dumoffii\n",
      " Legionella\n",
      " Legionella dumoffii\n",
      " Legionella\n",
      " Legionella dumoffii\n",
      " Legionella\n",
      " Legionella dumoffii\n",
      " Legionella\n",
      " Legionella dumoffii\n",
      " Legionella\n",
      " Legionella dumoffii\n",
      " Legionella\n",
      " Legionella dumoffii\n",
      " Legionella\n",
      " Legionella dumoffii\n",
      " Legionella\n",
      " Legionella dumoffii\n",
      " Legionella\n",
      " Legionella dumoffii\n",
      " Legionella\n",
      " Legionella dumoffii\n",
      " Legionella\n",
      " Legionella dumoffii\n",
      " Legionella\n",
      " Legionella dumoffii\n",
      " Legionella\n",
      " Legionella dumoffii\n",
      " Legionella\n",
      " Legionella dumoffii\n",
      " Legionella\n",
      " Legionella dumoffii\n",
      " Legionella\n",
      " Legionella dumoffii\n",
      " Legionella\n",
      " Legionella dumoffii\n",
      " Legionella\n",
      " Legionella dumoffii\n",
      " Legionella\n",
      " Legionella dumoffii\n",
      " Legionella\n",
      " Legionella dumoffii\n",
      " Legionella\n",
      " Legionella feeleii\n",
      " Legionella\n",
      " Legionella feeleii\n",
      " Legionella\n",
      " Legionella feeleii\n",
      " Legionella\n",
      " Legionella feeleii\n",
      " Legionella\n",
      " Legionella feeleii\n",
      " Legionella\n",
      " Legionella feeleii\n",
      " Legionella\n",
      " Legionella feeleii\n",
      " Legionella\n",
      " Legionella feeleii\n",
      " Legionella\n",
      " Legionella gormanii\n",
      " Legionella\n",
      " Legionella gormanii\n",
      " Legionella\n",
      " Legionella gormanii\n",
      " Legionella\n",
      " Legionella gormanii\n",
      " Legionella\n",
      " Legionella gormanii\n",
      " Legionella\n",
      " Legionella gormanii\n",
      " Legionella\n",
      " Legionella gormanii\n",
      " Legionella\n",
      " Legionella gormanii\n",
      " Legionella\n",
      " Legionella gormanii\n",
      " Legionella\n",
      " Legionella gormanii\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella gresiliensis\n",
      " Legionella\n",
      " Legionella jordanis\n",
      " Legionella\n",
      " Legionella jordanis\n",
      " Legionella\n",
      " Legionella jordanis\n",
      " Legionella\n",
      " Legionella jordanis\n",
      " Legionella\n",
      " Legionella jordanis\n",
      " Legionella\n",
      " Legionella jordanis\n",
      " Legionella\n",
      " Legionella jordanis\n",
      " Legionella\n",
      " Legionella jordanis\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella longbeachae\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella micdadei\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Legionella\n",
      " Legionella pneumophila\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella catarrhalis\n",
      " Moraxella\n",
      " Moraxella nonliquefaciens\n",
      " Moraxella\n",
      " Moraxella nonliquefaciens\n",
      " Moraxella\n",
      " Moraxella nonliquefaciens\n",
      " Moraxella\n",
      " Moraxella nonliquefaciens\n",
      " Moraxella\n",
      " Moraxella nonliquefaciens\n",
      " Moraxella\n",
      " Moraxella nonliquefaciens\n",
      " Moraxella\n",
      " Moraxella nonliquefaciens\n",
      " Moraxella\n",
      " Moraxella nonliquefaciens\n",
      " Morganella\n",
      " Morganella morganii\n",
      " Morganella\n",
      " Morganella morganii\n",
      " Morganella\n",
      " Morganella morganii\n",
      " Morganella\n",
      " Morganella morganii\n",
      " Morganella\n",
      " Morganella morganii\n",
      " Neisseria\n",
      " Neisseria cinerea\n",
      " Neisseria\n",
      " Neisseria cinerea\n",
      " Neisseria\n",
      " Neisseria cinerea\n",
      " Neisseria\n",
      " Neisseria cinerea\n",
      " Neisseria\n",
      " Neisseria cinerea\n",
      " Neisseria\n",
      " Neisseria cinerea\n",
      " Neisseria\n",
      " Neisseria cinerea\n",
      " Neisseria\n",
      " Neisseria cinerea\n",
      " Neisseria\n",
      " Neisseria cinerea\n",
      " Neisseria\n",
      " Neisseria cinerea\n",
      " Neisseria\n",
      " Neisseria gonorrhoeae\n",
      " Neisseria\n",
      " Neisseria gonorrhoeae\n",
      " Neisseria\n",
      " Neisseria gonorrhoeae\n",
      " Neisseria\n",
      " Neisseria gonorrhoeae\n",
      " Neisseria\n",
      " Neisseria gonorrhoeae\n",
      " Neisseria\n",
      " Neisseria gonorrhoeae\n",
      " Neisseria\n",
      " Neisseria gonorrhoeae\n",
      " Neisseria\n",
      " Neisseria gonorrhoeae\n",
      " Neisseria\n",
      " Neisseria gonorrhoeae\n",
      " Neisseria\n",
      " Neisseria gonorrhoeae\n",
      " Neisseria\n",
      " Neisseria gonorrhoeae\n",
      " Neisseria\n",
      " Neisseria gonorrhoeae\n",
      " Neisseria\n",
      " Neisseria lactamica\n",
      " Neisseria\n",
      " Neisseria lactamica\n",
      " Neisseria\n",
      " Neisseria lactamica\n",
      " Neisseria\n",
      " Neisseria lactamica\n",
      " Neisseria\n",
      " Neisseria lactamica\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria meningitidis\n",
      " Neisseria\n",
      " Neisseria mucosa\n",
      " Neisseria\n",
      " Neisseria mucosa\n",
      " Neisseria\n",
      " Neisseria sicca\n",
      " Neisseria\n",
      " Neisseria sicca\n",
      " Neisseria\n",
      " Neisseria sicca\n",
      " Neisseria\n",
      " Neisseria sicca\n",
      " Neisseria\n",
      " Neisseria sicca\n",
      " Neisseria\n",
      " Neisseria sicca\n",
      " Neisseria\n",
      " Neisseria sicca\n",
      " Neisseria\n",
      " Neisseria sicca\n",
      " Neisseria\n",
      " Neisseria sicca\n",
      " Neisseria\n",
      " Neisseria sicca\n",
      " Neisseria\n",
      " Neisseria subflava\n",
      " Neisseria\n",
      " Neisseria subflava\n",
      " Neisseria\n",
      " Neisseria subflava\n",
      " Neisseria\n",
      " Neisseria subflava\n",
      " Neisseria\n",
      " Neisseria subflava\n",
      " Pandoraea\n",
      " Pandoraea apista\n",
      " Pandoraea\n",
      " Pandoraea apista\n",
      " Pandoraea\n",
      " Pandoraea apista\n",
      " Pandoraea\n",
      " Pandoraea apista\n",
      " Pasteurella\n",
      " Pasteurella canis\n",
      " Pasteurella\n",
      " Pasteurella canis\n",
      " Pasteurella\n",
      " Pasteurella canis\n",
      " Pasteurella\n",
      " Pasteurella canis\n",
      " Pasteurella\n",
      " Pasteurella canis\n",
      " Pasteurella\n",
      " Pasteurella multocida\n",
      " Pasteurella\n",
      " Pasteurella multocida\n",
      " Pasteurella\n",
      " Pasteurella multocida\n",
      " Pasteurella\n",
      " Pasteurella multocida\n",
      " Pasteurella\n",
      " Pasteurella multocida\n",
      " Pasteurella\n",
      " Pasteurella multocida\n",
      " Pasteurella\n",
      " Pasteurella multocida\n",
      " Pasteurella\n",
      " Pasteurella multocida\n",
      " Pasteurella\n",
      " Pasteurella multocida\n",
      " Pasteurella\n",
      " Pasteurella multocida\n",
      " Pasteurella\n",
      " Pasteurella multocida\n",
      " Pasteurella\n",
      " Pasteurella multocida\n",
      " Pasteurella\n",
      " Pasteurella multocida\n",
      " Pasteurella\n",
      " Pasteurella multocida\n",
      " Pasteurella\n",
      " Pasteurella multocida\n",
      " Pasteurella\n",
      " Pasteurella multocida\n",
      " Proteus\n",
      " Proteus mirabilis\n",
      " Proteus\n",
      " Proteus mirabilis\n",
      " Proteus\n",
      " Proteus mirabilis\n",
      " Proteus\n",
      " Proteus mirabilis\n",
      " Proteus\n",
      " Proteus mirabilis\n",
      " Proteus\n",
      " Proteus mirabilis\n",
      " Proteus\n",
      " Proteus mirabilis\n",
      " Proteus\n",
      " Proteus mirabilis\n",
      " Proteus\n",
      " Proteus mirabilis\n",
      " Proteus\n",
      " Proteus mirabilis\n",
      " Proteus\n",
      " Proteus mirabilis\n",
      " Proteus\n",
      " Proteus mirabilis\n",
      " Proteus\n",
      " Proteus mirabilis\n",
      " Proteus\n",
      " Proteus mirabilis\n",
      " Proteus\n",
      " Proteus mirabilis\n",
      " Proteus\n",
      " Proteus mirabilis\n",
      " Proteus\n",
      " Proteus mirabilis\n",
      " Proteus\n",
      " Proteus mirabilis\n",
      " Proteus\n",
      " Proteus mirabilis\n",
      " Proteus\n",
      " Proteus mirabilis\n",
      " Proteus\n",
      " Proteus mirabilis\n",
      " Proteus\n",
      " Proteus mirabilis\n",
      " Proteus\n",
      " Proteus mirabilis\n",
      " Proteus\n",
      " Proteus mirabilis\n",
      " Proteus\n",
      " Proteus mirabilis\n",
      " Proteus\n",
      " Proteus mirabilis\n",
      " Proteus\n",
      " Proteus mirabilis\n",
      " Proteus\n",
      " Proteus mirabilis\n",
      " Proteus\n",
      " Proteus mirabilis\n",
      " Pseudomonas\n",
      " Pseudomonas aeruginosa\n",
      " Pseudomonas\n",
      " Pseudomonas aeruginosa\n",
      " Pseudomonas\n",
      " Pseudomonas aeruginosa\n",
      " Pseudomonas\n",
      " Pseudomonas aeruginosa\n",
      " Pseudomonas\n",
      " Pseudomonas aeruginosa\n",
      " Pseudomonas\n",
      " Pseudomonas aeruginosa\n",
      " Pseudomonas\n",
      " Pseudomonas aeruginosa\n",
      " Pseudomonas\n",
      " Pseudomonas aeruginosa\n",
      " Pseudomonas\n",
      " Pseudomonas aeruginosa\n",
      " Pseudomonas\n",
      " Pseudomonas aeruginosa\n",
      " Pseudomonas\n",
      " Pseudomonas aeruginosa\n",
      " Pseudomonas\n",
      " Pseudomonas aeruginosa\n",
      " Pseudomonas\n",
      " Pseudomonas aeruginosa\n",
      " Pseudomonas\n",
      " Pseudomonas aeruginosa\n",
      " Pseudomonas\n",
      " Pseudomonas aeruginosa\n",
      " Pseudomonas\n",
      " Pseudomonas aeruginosa\n",
      " Pseudomonas\n",
      " Pseudomonas aeruginosa\n",
      " Pseudomonas\n",
      " Pseudomonas aeruginosa\n",
      " Pseudomonas\n",
      " Pseudomonas aeruginosa\n",
      " Pseudomonas\n",
      " Pseudomonas aeruginosa\n",
      " Pseudomonas\n",
      " Pseudomonas aeruginosa\n",
      " Pseudomonas\n",
      " Pseudomonas aeruginosa\n",
      " Pseudomonas\n",
      " Pseudomonas aeruginosa\n",
      " Pseudomonas\n",
      " Pseudomonas aeruginosa\n",
      " Pseudomonas\n",
      " Pseudomonas aeruginosa\n",
      " Pseudomonas\n",
      " Pseudomonas aeruginosa\n",
      " Pseudomonas\n",
      " Pseudomonas aeruginosa\n",
      " Pseudomonas\n",
      " Pseudomonas aeruginosa\n",
      " Pseudomonas\n",
      " Pseudomonas fluorescens\n",
      " Pseudomonas\n",
      " Pseudomonas fluorescens\n",
      " Pseudomonas\n",
      " Pseudomonas fluorescens\n",
      " Pseudomonas\n",
      " Pseudomonas fluorescens\n",
      " Pseudomonas\n",
      " Pseudomonas putida\n",
      " Pseudomonas\n",
      " Pseudomonas putida\n",
      " Pseudomonas\n",
      " Pseudomonas putida\n",
      " Pseudomonas\n",
      " Pseudomonas putida\n",
      " Pseudomonas\n",
      " Pseudomonas putida\n",
      " Pseudomonas\n",
      " Pseudomonas putida\n",
      " Pseudomonas\n",
      " Pseudomonas putida\n",
      " Raoultella\n",
      " Raoultella planticola\n",
      " Raoultella\n",
      " Raoultella planticola\n",
      " Raoultella\n",
      " Raoultella planticola\n",
      " Raoultella\n",
      " Raoultella planticola\n",
      " Raoultella\n",
      " Raoultella planticola\n",
      " Raoultella\n",
      " Raoultella planticola\n",
      " Raoultella\n",
      " Raoultella planticola\n",
      " Raoultella\n",
      " Raoultella planticola\n",
      " Raoultella\n",
      " Raoultella planticola\n",
      " Raoultella\n",
      " Raoultella planticola\n",
      " Raoultella\n",
      " Raoultella planticola\n",
      " Raoultella\n",
      " Raoultella planticola\n",
      " Raoultella\n",
      " Raoultella planticola\n",
      " Rhizobium\n",
      " Rhizobium radiobacter\n",
      " Rhizobium\n",
      " Rhizobium radiobacter\n",
      " Rhizobium\n",
      " Rhizobium radiobacter\n",
      " Rhizobium\n",
      " Rhizobium radiobacter\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Salmonella\n",
      " Salmonella enterica\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Serratia\n",
      " Serratia marcescens\n",
      " Shewanella\n",
      " Shewanella putrefaciens\n",
      " Shewanella\n",
      " Shewanella putrefaciens\n",
      " Shewanella\n",
      " Shewanella putrefaciens\n",
      " Shewanella\n",
      " Shewanella putrefaciens\n",
      " Shewanella\n",
      " Shewanella putrefaciens\n",
      " Shewanella\n",
      " Shewanella putrefaciens\n",
      " Shewanella\n",
      " Shewanella putrefaciens\n",
      " Shewanella\n",
      " Shewanella putrefaciens\n",
      " Shigella\n",
      " Shigella flexneri\n",
      " Shigella\n",
      " Shigella sonnei\n",
      " Shigella\n",
      " Shigella sonnei\n",
      " Shigella\n",
      " Shigella sonnei\n",
      " Shigella\n",
      " Shigella sonnei\n",
      " Shigella\n",
      " Shigella sonnei\n",
      " Sphingomonas\n",
      " Sphingomonas\n",
      " Sphingomonas\n",
      " Sphingomonas\n",
      " Sphingomonas\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Stenotrophomonas\n",
      " Stenotrophomonas maltophilia\n",
      " Yersinia\n",
      " Yersinia enterocolitica\n",
      " Yersinia\n",
      " Yersinia enterocolitica\n",
      " Yersinia\n",
      " Yersinia enterocolitica\n",
      " Yersinia\n",
      " Yersinia enterocolitica\n",
      " Yersinia\n",
      " Yersinia enterocolitica\n",
      " Yersinia\n",
      " Yersinia enterocolitica\n",
      " Yersinia\n",
      " Yersinia enterocolitica\n",
      " Yersinia\n",
      " Yersinia enterocolitica\n",
      " Yersinia\n",
      " Yersinia enterocolitica\n",
      " Yersinia\n",
      " Yersinia enterocolitica\n",
      " Yersinia\n",
      " Yersinia enterocolitica\n",
      " Yersinia\n",
      " Yersinia enterocolitica\n",
      " Yersinia\n",
      " Yersinia pestis\n",
      " Yersinia\n",
      " Yersinia pestis\n",
      " Yersinia\n",
      " Yersinia pestis\n",
      " Yersinia\n",
      " Yersinia pestis\n",
      " Yersinia\n",
      " Yersinia pestis\n",
      " Yersinia\n",
      " Yersinia pestis\n",
      " Yersinia\n",
      " Yersinia pestis\n",
      " Yersinia\n",
      " Yersinia pestis\n",
      " Yersinia\n",
      " Yersinia pestis\n",
      " Aerococcus\n",
      " Aerococcus viridans\n",
      " Aerococcus\n",
      " Aerococcus viridans\n",
      " Aerococcus\n",
      " Aerococcus viridans\n",
      " Aerococcus\n",
      " Aerococcus viridans\n",
      " Aerococcus\n",
      " Aerococcus viridans\n",
      " Aerococcus\n",
      " Aerococcus viridans\n",
      " Aerococcus\n",
      " Aerococcus viridans\n",
      " Aerococcus\n",
      " Aerococcus viridans\n",
      " Aerococcus\n",
      " Aerococcus viridans\n",
      " Aerococcus\n",
      " Aerococcus viridans\n",
      " Aerococcus\n",
      " Aerococcus viridans\n",
      " Aerococcus\n",
      " Aerococcus viridans\n",
      " Aerococcus\n",
      " Aerococcus viridans\n",
      " Aerococcus\n",
      " Aerococcus viridans\n",
      " Aerococcus\n",
      " Aerococcus viridans\n",
      " Aerococcus\n",
      " Aerococcus viridans\n",
      " Aerococcus\n",
      " Aerococcus viridans\n",
      " Aerococcus\n",
      " Aerococcus viridans\n",
      " Aerococcus\n",
      " Aerococcus viridans\n",
      " Aerococcus\n",
      " Aerococcus viridans\n",
      " Aerococcus\n",
      " Aerococcus viridans\n",
      " Aerococcus\n",
      " Aerococcus viridans\n",
      " Aerococcus\n",
      " Aerococcus viridans\n",
      " Aerococcus\n",
      " Aerococcus viridans\n",
      " Aerococcus\n",
      " Aerococcus viridans\n",
      " Alloiococcus\n",
      " Alloiococcus\n",
      " Dolosigranulum\n",
      " Dolosigranulum pigrum\n",
      " Dolosigranulum\n",
      " Dolosigranulum pigrum\n",
      " Dolosigranulum\n",
      " Dolosigranulum pigrum\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus CDCPNS-E3\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecalis\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus faecium\n",
      " Enterococcus\n",
      " Enterococcus hirae\n",
      " Enterococcus\n",
      " Enterococcus hirae\n",
      " Gemella\n",
      " Gemella morbillorum\n",
      " Gemella\n",
      " Gemella morbillorum\n",
      " Gemella\n",
      " Gemella morbillorum\n",
      " Gemella\n",
      " Gemella morbillorum\n",
      " Gemella\n",
      " Gemella morbillorum\n",
      " Gemella\n",
      " Gemella morbillorum\n",
      " Gemella\n",
      " Gemella morbillorum\n",
      " Gemella\n",
      " Gemella morbillorum\n",
      " Gemella\n",
      " Gemella morbillorum\n",
      " Gemella\n",
      " Gemella morbillorum\n",
      " Gemella\n",
      " Gemella morbillorum\n",
      " Gemella\n",
      " Gemella morbillorum\n",
      " Gemella\n",
      " Gemella morbillorum\n",
      " Gemella\n",
      " Gemella morbillorum\n",
      " Gemella\n",
      " Gemella morbillorum\n",
      " Gemella\n",
      " Gemella morbillorum\n",
      " Gemella\n",
      " Gemella morbillorum\n",
      " Lactococcus\n",
      " Lactococcus lactis\n",
      " Micrococcus\n",
      " Micrococcus luteus\n",
      " Micrococcus\n",
      " Micrococcus luteus\n",
      " Micrococcus\n",
      " Micrococcus luteus\n",
      " Rothia\n",
      " Rothia mucilaginosa\n",
      " Rothia\n",
      " Rothia mucilaginosa\n",
      " Rothia\n",
      " Rothia mucilaginosa\n",
      " Rothia\n",
      " Rothia mucilaginosa\n",
      " Rothia\n",
      " Rothia mucilaginosa\n",
      " Rothia\n",
      " Rothia mucilaginosa\n",
      " Rothia\n",
      " Rothia mucilaginosa\n",
      " Rothia\n",
      " Rothia mucilaginosa\n",
      " Rothia\n",
      " Rothia mucilaginosa\n",
      " Rothia\n",
      " Rothia mucilaginosa\n",
      " Rothia\n",
      " Rothia mucilaginosa\n",
      " Rothia\n",
      " Rothia mucilaginosa\n",
      " Rothia\n",
      " Rothia mucilaginosa\n",
      " Rothia\n",
      " Rothia mucilaginosa\n",
      " Rothia\n",
      " Rothia mucilaginosa\n",
      " Rothia\n",
      " Rothia mucilaginosa\n",
      " Rothia\n",
      " Rothia mucilaginosa\n",
      " Rothia\n",
      " Rothia mucilaginosa\n",
      " Rothia\n",
      " Rothia mucilaginosa\n",
      " Rothia\n",
      " Rothia mucilaginosa\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus aureus\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus epidermidis\n",
      " Staphylococcus\n",
      " Staphylococcus felis\n",
      " Staphylococcus\n",
      " Staphylococcus haemolyticus\n",
      " Staphylococcus\n",
      " Staphylococcus haemolyticus\n",
      " Staphylococcus\n",
      " Staphylococcus haemolyticus\n",
      " Staphylococcus\n",
      " Staphylococcus haemolyticus\n",
      " Staphylococcus\n",
      " Staphylococcus haemolyticus\n",
      " Staphylococcus\n",
      " Staphylococcus haemolyticus\n",
      " Staphylococcus\n",
      " Staphylococcus haemolyticus\n",
      " Staphylococcus\n",
      " Staphylococcus haemolyticus\n",
      " Staphylococcus\n",
      " Staphylococcus haemolyticus\n",
      " Staphylococcus\n",
      " Staphylococcus haemolyticus\n",
      " Staphylococcus\n",
      " Staphylococcus haemolyticus\n",
      " Staphylococcus\n",
      " Staphylococcus haemolyticus\n",
      " Staphylococcus\n",
      " Staphylococcus haemolyticus\n",
      " Staphylococcus\n",
      " Staphylococcus haemolyticus\n",
      " Staphylococcus\n",
      " Staphylococcus haemolyticus\n",
      " Staphylococcus\n",
      " Staphylococcus haemolyticus\n",
      " Staphylococcus\n",
      " Staphylococcus hominis\n",
      " Staphylococcus\n",
      " Staphylococcus hominis\n",
      " Staphylococcus\n",
      " Staphylococcus hominis\n",
      " Staphylococcus\n",
      " Staphylococcus hominis\n",
      " Staphylococcus\n",
      " Staphylococcus hominis\n",
      " Staphylococcus\n",
      " Staphylococcus hominis\n",
      " Staphylococcus\n",
      " Staphylococcus hominis\n",
      " Staphylococcus\n",
      " Staphylococcus hominis\n",
      " Staphylococcus\n",
      " Staphylococcus hominis\n",
      " Staphylococcus\n",
      " Staphylococcus intermedius\n",
      " Staphylococcus\n",
      " Staphylococcus intermedius\n",
      " Staphylococcus\n",
      " Staphylococcus intermedius\n",
      " Staphylococcus\n",
      " Staphylococcus intermedius\n",
      " Staphylococcus\n",
      " Staphylococcus intermedius\n",
      " Staphylococcus\n",
      " Staphylococcus intermedius\n",
      " Staphylococcus\n",
      " Staphylococcus intermedius\n",
      " Staphylococcus\n",
      " Staphylococcus lugdunensis\n",
      " Staphylococcus\n",
      " Staphylococcus lugdunensis\n",
      " Staphylococcus\n",
      " Staphylococcus lugdunensis\n",
      " Staphylococcus\n",
      " Staphylococcus lugdunensis\n",
      " Staphylococcus\n",
      " Staphylococcus lugdunensis\n",
      " Staphylococcus\n",
      " Staphylococcus lugdunensis\n",
      " Staphylococcus\n",
      " Staphylococcus lugdunensis\n",
      " Staphylococcus\n",
      " Staphylococcus lugdunensis\n",
      " Staphylococcus\n",
      " Staphylococcus lugdunensis\n",
      " Staphylococcus\n",
      " Staphylococcus lugdunensis\n",
      " Staphylococcus\n",
      " Staphylococcus lugdunensis\n",
      " Staphylococcus\n",
      " Staphylococcus lugdunensis\n",
      " Staphylococcus\n",
      " Staphylococcus lugdunensis\n",
      " Staphylococcus\n",
      " Staphylococcus saprophyticus\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus agalactiae\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus anginosus\n",
      " Streptococcus\n",
      " Streptococcus bovis\n",
      " Streptococcus\n",
      " Streptococcus bovis\n",
      " Streptococcus\n",
      " Streptococcus bovis\n",
      " Streptococcus\n",
      " Streptococcus bovis\n",
      " Streptococcus\n",
      " Streptococcus bovis\n",
      " Streptococcus\n",
      " Streptococcus bovis\n",
      " Streptococcus\n",
      " Streptococcus bovis\n",
      " Streptococcus\n",
      " Streptococcus bovis\n",
      " Streptococcus\n",
      " Streptococcus bovis\n",
      " Streptococcus\n",
      " Streptococcus bovis\n",
      " Streptococcus\n",
      " Streptococcus bovis\n",
      " Streptococcus\n",
      " Streptococcus bovis\n",
      " Streptococcus\n",
      " Streptococcus bovis\n",
      " Streptococcus\n",
      " Streptococcus bovis\n",
      " Streptococcus\n",
      " Streptococcus bovis\n",
      " Streptococcus\n",
      " Streptococcus bovis\n",
      " Streptococcus\n",
      " Streptococcus bovis\n",
      " Streptococcus\n",
      " Streptococcus bovis\n",
      " Streptococcus\n",
      " Streptococcus bovis\n",
      " Streptococcus\n",
      " Streptococcus bovis\n",
      " Streptococcus\n",
      " Streptococcus bovis\n",
      " Streptococcus\n",
      " Streptococcus bovis\n",
      " Streptococcus\n",
      " Streptococcus bovis\n",
      " Streptococcus\n",
      " Streptococcus canis\n",
      " Streptococcus\n",
      " Streptococcus constellatus\n",
      " Streptococcus\n",
      " Streptococcus constellatus\n",
      " Streptococcus\n",
      " Streptococcus constellatus\n",
      " Streptococcus\n",
      " Streptococcus constellatus\n",
      " Streptococcus\n",
      " Streptococcus constellatus\n",
      " Streptococcus\n",
      " Streptococcus constellatus\n",
      " Streptococcus\n",
      " Streptococcus constellatus\n",
      " Streptococcus\n",
      " Streptococcus constellatus\n",
      " Streptococcus\n",
      " Streptococcus constellatus\n",
      " Streptococcus\n",
      " Streptococcus constellatus\n",
      " Streptococcus\n",
      " Streptococcus constellatus\n",
      " Streptococcus\n",
      " Streptococcus constellatus\n",
      " Streptococcus\n",
      " Streptococcus constellatus\n",
      " Streptococcus\n",
      " Streptococcus constellatus\n",
      " Streptococcus\n",
      " Streptococcus constellatus\n",
      " Streptococcus\n",
      " Streptococcus constellatus\n",
      " Streptococcus\n",
      " Streptococcus constellatus\n",
      " Streptococcus\n",
      " Streptococcus constellatus\n",
      " Streptococcus\n",
      " Streptococcus constellatus\n",
      " Streptococcus\n",
      " Streptococcus constellatus\n",
      " Streptococcus\n",
      " Streptococcus constellatus\n",
      " Streptococcus\n",
      " Streptococcus constellatus\n",
      " Streptococcus\n",
      " Streptococcus constellatus\n",
      " Streptococcus\n",
      " Streptococcus constellatus\n",
      " Streptococcus\n",
      " Streptococcus constellatus\n",
      " Streptococcus\n",
      " Streptococcus constellatus\n",
      " Streptococcus\n",
      " Streptococcus constellatus\n",
      " Streptococcus\n",
      " Streptococcus constellatus\n",
      " Streptococcus\n",
      " Streptococcus constellatus\n",
      " Streptococcus\n",
      " Streptococcus constellatus\n",
      " Streptococcus\n",
      " Streptococcus constellatus\n",
      " Streptococcus\n",
      " Streptococcus constellatus\n",
      " Streptococcus\n",
      " Streptococcus constellatus\n",
      " Streptococcus\n",
      " Streptococcus constellatus\n",
      " Streptococcus\n",
      " Streptococcus constellatus\n",
      " Streptococcus\n",
      " Streptococcus dysgalactiae\n",
      " Streptococcus\n",
      " Streptococcus dysgalactiae\n",
      " Streptococcus\n",
      " Streptococcus dysgalactiae\n",
      " Streptococcus\n",
      " Streptococcus dysgalactiae\n",
      " Streptococcus\n",
      " Streptococcus dysgalactiae\n",
      " Streptococcus\n",
      " Streptococcus dysgalactiae\n",
      " Streptococcus\n",
      " Streptococcus dysgalactiae\n",
      " Streptococcus\n",
      " Streptococcus dysgalactiae\n",
      " Streptococcus\n",
      " Streptococcus dysgalactiae\n",
      " Streptococcus\n",
      " Streptococcus dysgalactiae\n",
      " Streptococcus\n",
      " Streptococcus equinus\n",
      " Streptococcus\n",
      " Streptococcus equinus\n",
      " Streptococcus\n",
      " Streptococcus equinus\n",
      " Streptococcus\n",
      " Streptococcus equinus\n",
      " Streptococcus\n",
      " Streptococcus equinus\n",
      " Streptococcus\n",
      " Streptococcus equinus\n",
      " Streptococcus\n",
      " Streptococcus equinus\n",
      " Streptococcus\n",
      " Streptococcus equinus\n",
      " Streptococcus\n",
      " Streptococcus equinus\n",
      " Streptococcus\n",
      " Streptococcus equinus\n",
      " Streptococcus\n",
      " Streptococcus equinus\n",
      " Streptococcus\n",
      " Streptococcus equinus\n",
      " Streptococcus\n",
      " Streptococcus equinus\n",
      " Streptococcus\n",
      " Streptococcus equinus\n",
      " Streptococcus\n",
      " Streptococcus equinus\n",
      " Streptococcus\n",
      " Streptococcus equinus\n",
      " Streptococcus\n",
      " Streptococcus equinus\n",
      " Streptococcus\n",
      " Streptococcus equinus\n",
      " Streptococcus\n",
      " Streptococcus equinus\n",
      " Streptococcus\n",
      " Streptococcus equinus\n",
      " Streptococcus\n",
      " Streptococcus equinus\n",
      " Streptococcus\n",
      " Streptococcus equinus\n",
      " Streptococcus\n",
      " Streptococcus gallolyticus\n",
      " Streptococcus\n",
      " Streptococcus gallolyticus\n",
      " Streptococcus\n",
      " Streptococcus gallolyticus\n",
      " Streptococcus\n",
      " Streptococcus gallolyticus\n",
      " Streptococcus\n",
      " Streptococcus gallolyticus\n",
      " Streptococcus\n",
      " Streptococcus gordonii\n",
      " Streptococcus\n",
      " Streptococcus gordonii\n",
      " Streptococcus\n",
      " Streptococcus gordonii\n",
      " Streptococcus\n",
      " Streptococcus gordonii\n",
      " Streptococcus\n",
      " Streptococcus gordonii\n",
      " Streptococcus\n",
      " Streptococcus gordonii\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus intermedius\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus milleri\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus mitis\n",
      " Streptococcus\n",
      " Streptococcus morbillorum\n",
      " Streptococcus\n",
      " Streptococcus morbillorum\n",
      " Streptococcus\n",
      " Streptococcus morbillorum\n",
      " Streptococcus\n",
      " Streptococcus morbillorum\n",
      " Streptococcus\n",
      " Streptococcus morbillorum\n",
      " Streptococcus\n",
      " Streptococcus morbillorum\n",
      " Streptococcus\n",
      " Streptococcus morbillorum\n",
      " Streptococcus\n",
      " Streptococcus morbillorum\n",
      " Streptococcus\n",
      " Streptococcus morbillorum\n",
      " Streptococcus\n",
      " Streptococcus morbillorum\n",
      " Streptococcus\n",
      " Streptococcus morbillorum\n",
      " Streptococcus\n",
      " Streptococcus morbillorum\n",
      " Streptococcus\n",
      " Streptococcus morbillorum\n",
      " Streptococcus\n",
      " Streptococcus morbillorum\n",
      " Streptococcus\n",
      " Streptococcus morbillorum\n",
      " Streptococcus\n",
      " Streptococcus morbillorum\n",
      " Streptococcus\n",
      " Streptococcus morbillorum\n",
      " Streptococcus\n",
      " Streptococcus morbillorum\n",
      " Streptococcus\n",
      " Streptococcus morbillorum\n",
      " Streptococcus\n",
      " Streptococcus morbillorum\n",
      " Streptococcus\n",
      " Streptococcus morbillorum\n",
      " Streptococcus\n",
      " Streptococcus morbillorum\n",
      " Streptococcus\n",
      " Streptococcus morbillorum\n",
      " Streptococcus\n",
      " Streptococcus morbillorum\n",
      " Streptococcus\n",
      " Streptococcus mutans\n",
      " Streptococcus\n",
      " Streptococcus mutans\n",
      " Streptococcus\n",
      " Streptococcus mutans\n",
      " Streptococcus\n",
      " Streptococcus oralis\n",
      " Streptococcus\n",
      " Streptococcus oralis\n",
      " Streptococcus\n",
      " Streptococcus oralis\n",
      " Streptococcus\n",
      " Streptococcus oralis\n",
      " Streptococcus\n",
      " Streptococcus oralis\n",
      " Streptococcus\n",
      " Streptococcus oralis\n",
      " Streptococcus\n",
      " Streptococcus parasanguinis\n",
      " Streptococcus\n",
      " Streptococcus parasanguinis\n",
      " Streptococcus\n",
      " Streptococcus parasanguinis\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pseudopneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pseudopneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pseudopneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pseudopneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pseudopneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pseudopneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pseudopneumoniae\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus pyogenes\n",
      " Streptococcus\n",
      " Streptococcus salivarius\n",
      " Streptococcus\n",
      " Streptococcus salivarius\n",
      " Streptococcus\n",
      " Streptococcus salivarius\n",
      " Streptococcus\n",
      " Streptococcus salivarius\n",
      " Streptococcus\n",
      " Streptococcus salivarius\n",
      " Streptococcus\n",
      " Streptococcus salivarius\n",
      " Streptococcus\n",
      " Streptococcus salivarius\n",
      " Streptococcus\n",
      " Streptococcus sanguinis\n",
      " Streptococcus\n",
      " Streptococcus sanguinis\n",
      " Streptococcus\n",
      " Streptococcus sanguinis\n",
      " Streptococcus\n",
      " Streptococcus sanguinis\n",
      " Streptococcus\n",
      " Streptococcus sanguinis\n",
      " Streptococcus\n",
      " Streptococcus sanguinis\n",
      " Streptococcus\n",
      " Streptococcus sanguinis\n",
      " Streptococcus\n",
      " Streptococcus sanguinis\n",
      " Streptococcus\n",
      " Streptococcus sanguinis\n",
      " Streptococcus\n",
      " Streptococcus sanguinis\n",
      " Streptococcus\n",
      " Streptococcus sanguinis\n",
      " Streptococcus\n",
      " Streptococcus sanguinis\n",
      " Streptococcus\n",
      " Streptococcus sanguinis\n",
      " Streptococcus\n",
      " Streptococcus sanguinis\n",
      " Streptococcus\n",
      " Streptococcus sanguinis\n",
      " Streptococcus\n",
      " Streptococcus sanguinis\n",
      " Streptococcus\n",
      " Streptococcus sanguinis\n",
      " Streptococcus\n",
      " Streptococcus sanguinis\n",
      " Streptococcus\n",
      " Streptococcus suis\n",
      " Streptococcus\n",
      " Streptococcus suis\n",
      " Streptococcus\n",
      " Streptococcus suis\n",
      " Streptococcus\n",
      " Streptococcus suis\n",
      " Streptococcus\n",
      " Streptococcus suis\n",
      " Streptococcus\n",
      " Streptococcus suis\n",
      " Streptococcus\n",
      " Streptococcus suis\n",
      " Tetragenococcus\n",
      " Tetragenococcus\n",
      " Tetragenococcus\n",
      " Tetragenococcus\n",
      " Tetragenococcus\n",
      " Tetragenococcus\n",
      " Tetragenococcus\n",
      " Tetragenococcus\n",
      " Tetragenococcus\n",
      " Tetragenococcus\n",
      " Tetragenococcus\n",
      " Tetragenococcus\n",
      " Tetragenococcus\n",
      " Tetragenococcus\n",
      " Tetragenococcus\n",
      " Tetragenococcus\n",
      " Tetragenococcus\n",
      " Tetragenococcus\n",
      " Tetragenococcus\n",
      " Tetragenococcus\n",
      " Tetragenococcus\n",
      " Tetragenococcus\n",
      " Tetragenococcus\n",
      " Tetragenococcus\n",
      " Bacillus\n",
      " Bacillus anthracis\n",
      " Bacillus\n",
      " Bacillus anthracis\n",
      " Bacillus\n",
      " Bacillus anthracis\n",
      " Bacillus\n",
      " Bacillus anthracis\n",
      " Bacillus\n",
      " Bacillus anthracis\n",
      " Bacillus\n",
      " Bacillus anthracis\n",
      " Bacillus\n",
      " Bacillus anthracis\n",
      " Bacillus\n",
      " Bacillus anthracis\n",
      " Bacillus\n",
      " Bacillus anthracis\n",
      " Bacillus\n",
      " Bacillus anthracis\n",
      " Bacillus\n",
      " Bacillus anthracis\n",
      " Bacillus\n",
      " Bacillus anthracis\n",
      " Bacillus\n",
      " Bacillus anthracis\n",
      " Bacillus\n",
      " Bacillus cereus\n",
      " Bacillus\n",
      " Bacillus cereus\n",
      " Bacillus\n",
      " Bacillus cereus\n",
      " Bacillus\n",
      " Bacillus cereus\n",
      " Bacillus\n",
      " Bacillus cereus\n",
      " Bacillus\n",
      " Bacillus cereus\n",
      " Bacillus\n",
      " Bacillus cereus\n",
      " Bacillus\n",
      " Bacillus cereus\n",
      " Bacillus\n",
      " Bacillus cereus\n",
      " Bacillus\n",
      " Bacillus cereus\n",
      " Bacillus\n",
      " Bacillus cereus\n",
      " Bacillus\n",
      " Bacillus cereus\n",
      " Bacillus\n",
      " Bacillus cereus\n",
      " Bacillus\n",
      " Bacillus cereus\n",
      " Bacillus\n",
      " Bacillus cereus\n",
      " Bacillus\n",
      " Bacillus cereus\n",
      " Bacillus\n",
      " Bacillus cereus\n",
      " Bacillus\n",
      " Bacillus cereus\n",
      " Bacillus\n",
      " Bacillus cereus\n",
      " Bacillus\n",
      " Bacillus cereus\n",
      " Bacillus\n",
      " Bacillus cereus\n",
      " Bacillus\n",
      " Bacillus cereus\n",
      " Bacillus\n",
      " Bacillus cereus\n",
      " Bacillus\n",
      " Bacillus cereus\n",
      " Bacillus\n",
      " Bacillus cereus\n",
      " Bacillus\n",
      " Bacillus cereus\n",
      " Corynebacterium\n",
      " Corynebacterium diphtheriae\n",
      " Corynebacterium\n",
      " Corynebacterium diphtheriae\n",
      " Corynebacterium\n",
      " Corynebacterium diphtheriae\n",
      " Corynebacterium\n",
      " Corynebacterium diphtheriae\n",
      " Corynebacterium\n",
      " Corynebacterium diphtheriae\n",
      " Corynebacterium\n",
      " Corynebacterium diphtheriae\n",
      " Corynebacterium\n",
      " Corynebacterium diphtheriae\n",
      " Corynebacterium\n",
      " Corynebacterium diphtheriae\n",
      " Corynebacterium\n",
      " Corynebacterium diphtheriae\n",
      " Corynebacterium\n",
      " Corynebacterium diphtheriae\n",
      " Corynebacterium\n",
      " Corynebacterium diphtheriae\n",
      " Corynebacterium\n",
      " Corynebacterium diphtheriae\n",
      " Corynebacterium\n",
      " Corynebacterium diphtheriae\n",
      " Corynebacterium\n",
      " Corynebacterium diphtheriae\n",
      " Corynebacterium\n",
      " Corynebacterium diphtheriae\n",
      " Corynebacterium\n",
      " Corynebacterium diphtheriae\n",
      " Corynebacterium\n",
      " Corynebacterium diphtheriae\n",
      " Corynebacterium\n",
      " Corynebacterium diphtheriae\n",
      " Corynebacterium\n",
      " Corynebacterium diphtheriae\n",
      " Corynebacterium\n",
      " Corynebacterium diphtheriae\n",
      " Corynebacterium\n",
      " Corynebacterium diphtheriae\n",
      " Corynebacterium\n",
      " Corynebacterium diphtheriae\n",
      " Corynebacterium\n",
      " Corynebacterium mastitidis-like\n",
      " Corynebacterium mastitidis-like organism\n",
      " Corynebacterium\n",
      " Corynebacterium mastitidis-like\n",
      " Corynebacterium mastitidis-like organism\n",
      " Corynebacterium\n",
      " Corynebacterium mastitidis-like\n",
      " Corynebacterium mastitidis-like organism\n",
      " Corynebacterium\n",
      " Corynebacterium mastitidis-like\n",
      " Corynebacterium mastitidis-like organism\n",
      " Corynebacterium\n",
      " Corynebacterium mastitidis-like\n",
      " Corynebacterium mastitidis-like organism\n",
      " Corynebacterium\n",
      " Corynebacterium mastitidis-like\n",
      " Corynebacterium mastitidis-like organism\n",
      " Corynebacterium\n",
      " Corynebacterium mastitidis-like\n",
      " Corynebacterium mastitidis-like organism\n",
      " Corynebacterium\n",
      " Corynebacterium mastitidis-like\n",
      " Corynebacterium mastitidis-like organism\n",
      " Corynebacterium\n",
      " Corynebacterium mastitidis-like\n",
      " Corynebacterium mastitidis-like organism\n",
      " Corynebacterium\n",
      " Corynebacterium mastitidis-like\n",
      " Corynebacterium mastitidis-like organism\n",
      " Corynebacterium\n",
      " Corynebacterium propinquum\n",
      " Corynebacterium\n",
      " Corynebacterium propinquum\n",
      " Corynebacterium\n",
      " Corynebacterium propinquum\n",
      " Corynebacterium\n",
      " Corynebacterium propinquum\n",
      " Corynebacterium\n",
      " Corynebacterium propinquum\n",
      " Corynebacterium\n",
      " Corynebacterium propinquum\n",
      " Corynebacterium\n",
      " Corynebacterium propinquum\n",
      " Corynebacterium\n",
      " Corynebacterium propinquum\n",
      " Corynebacterium\n",
      " Corynebacterium pseudodiphtheriticum\n",
      " Corynebacterium\n",
      " Corynebacterium pseudodiphtheriticum\n",
      " Corynebacterium\n",
      " Corynebacterium pseudodiphtheriticum\n",
      " Corynebacterium\n",
      " Corynebacterium pseudodiphtheriticum\n",
      " Corynebacterium\n",
      " Corynebacterium pseudodiphtheriticum\n",
      " Corynebacterium\n",
      " Corynebacterium pseudodiphtheriticum\n",
      " Corynebacterium\n",
      " Corynebacterium pseudodiphtheriticum\n",
      " Corynebacterium\n",
      " Corynebacterium pseudodiphtheriticum\n",
      " Corynebacterium\n",
      " Corynebacterium pseudodiphtheriticum\n",
      " Corynebacterium\n",
      " Corynebacterium pseudodiphtheriticum\n",
      " Corynebacterium\n",
      " Corynebacterium pseudodiphtheriticum\n",
      " Corynebacterium\n",
      " Corynebacterium pseudodiphtheriticum\n",
      " Corynebacterium\n",
      " Corynebacterium pseudodiphtheriticum\n",
      " Corynebacterium\n",
      " Corynebacterium pseudodiphtheriticum\n",
      " Corynebacterium\n",
      " Corynebacterium pseudodiphtheriticum\n",
      " Corynebacterium\n",
      " Corynebacterium pseudodiphtheriticum\n",
      " Corynebacterium\n",
      " Corynebacterium pseudodiphtheriticum\n",
      " Corynebacterium\n",
      " Corynebacterium pseudodiphtheriticum\n",
      " Corynebacterium\n",
      " Corynebacterium pseudodiphtheriticum\n",
      " Corynebacterium\n",
      " Corynebacterium pseudodiphtheriticum\n",
      " Corynebacterium\n",
      " Corynebacterium pseudodiphtheriticum\n",
      " Corynebacterium\n",
      " Corynebacterium pseudodiphtheriticum\n",
      " Corynebacterium\n",
      " Corynebacterium pseudodiphtheriticum\n",
      " Corynebacterium\n",
      " Corynebacterium pseudodiphtheriticum\n",
      " Corynebacterium\n",
      " Corynebacterium pseudodiphtheriticum\n",
      " Corynebacterium\n",
      " Corynebacterium pseudodiphtheriticum\n",
      " Corynebacterium\n",
      " Corynebacterium pseudodiphtheriticum\n",
      " Corynebacterium\n",
      " Corynebacterium pseudodiphtheriticum\n",
      " Corynebacterium\n",
      " Corynebacterium pseudodiphtheriticum\n",
      " Corynebacterium\n",
      " Corynebacterium pseudodiphtheriticum\n",
      " Corynebacterium\n",
      " Corynebacterium pseudodiphtheriticum\n",
      " Corynebacterium\n",
      " Corynebacterium pseudodiphtheriticum\n",
      " Corynebacterium\n",
      " Corynebacterium pseudotuberculosis\n",
      " Corynebacterium\n",
      " Corynebacterium pseudotuberculosis\n",
      " Corynebacterium\n",
      " Corynebacterium pseudotuberculosis\n",
      " Corynebacterium\n",
      " Corynebacterium pseudotuberculosis\n",
      " Corynebacterium\n",
      " Corynebacterium striatum\n",
      " Corynebacterium\n",
      " Corynebacterium striatum\n",
      " Corynebacterium\n",
      " Corynebacterium striatum\n",
      " Corynebacterium\n",
      " Corynebacterium striatum\n",
      " Corynebacterium\n",
      " Corynebacterium striatum\n",
      " Corynebacterium\n",
      " Corynebacterium striatum\n",
      " Corynebacterium\n",
      " Corynebacterium striatum\n",
      " Corynebacterium\n",
      " Corynebacterium striatum\n",
      " Corynebacterium\n",
      " Corynebacterium striatum\n",
      " Corynebacterium\n",
      " Corynebacterium striatum\n",
      " Corynebacterium\n",
      " Corynebacterium striatum\n",
      " Corynebacterium\n",
      " Corynebacterium striatum\n",
      " Corynebacterium\n",
      " Corynebacterium striatum\n",
      " Corynebacterium\n",
      " Corynebacterium striatum\n",
      " Corynebacterium\n",
      " Corynebacterium striatum\n",
      " Corynebacterium\n",
      " Corynebacterium ulcerans\n",
      " Corynebacterium\n",
      " Corynebacterium ulcerans\n",
      " Corynebacterium\n",
      " Corynebacterium ulcerans\n",
      " Corynebacterium\n",
      " Corynebacterium ulcerans\n",
      " Corynebacterium\n",
      " Corynebacterium ulcerans\n",
      " Corynebacterium\n",
      " Corynebacterium ulcerans\n",
      " Corynebacterium\n",
      " Corynebacterium ulcerans\n",
      " Corynebacterium\n",
      " Corynebacterium ulcerans\n",
      " Corynebacterium\n",
      " Corynebacterium ulcerans\n",
      " Corynebacterium\n",
      " Corynebacterium ulcerans\n",
      " Corynebacterium\n",
      " Corynebacterium ulcerans\n",
      " Erysipelothrix\n",
      " Erysipelothrix rhusiopathiae\n",
      " Erysipelothrix\n",
      " Erysipelothrix rhusiopathiae\n",
      " Erysipelothrix\n",
      " Erysipelothrix rhusiopathiae\n",
      " Erysipelothrix\n",
      " Erysipelothrix rhusiopathiae\n",
      " Listeria\n",
      " Listeria monocytogenes\n",
      " Listeria\n",
      " Listeria monocytogenes\n",
      " Listeria\n",
      " Listeria monocytogenes\n",
      " Listeria\n",
      " Listeria monocytogenes\n",
      " Listeria\n",
      " Listeria monocytogenes\n",
      " Listeria\n",
      " Listeria monocytogenes\n",
      " Listeria\n",
      " Listeria monocytogenes\n",
      " Listeria\n",
      " Listeria monocytogenes\n",
      " Listeria\n",
      " Listeria monocytogenes\n",
      " Listeria\n",
      " Listeria monocytogenes\n",
      " Listeria\n",
      " Listeria monocytogenes\n",
      " Listeria\n",
      " Listeria monocytogenes\n",
      " Listeria\n",
      " Listeria monocytogenes\n",
      " Listeria\n",
      " Listeria monocytogenes\n",
      " Listeria\n",
      " Listeria monocytogenes\n",
      " Listeria\n",
      " Listeria monocytogenes\n",
      " Listeria\n",
      " Listeria monocytogenes\n",
      " Listeria\n",
      " Listeria monocytogenes\n",
      " Listeria\n",
      " Listeria monocytogenes\n",
      " Listeria\n",
      " Listeria monocytogenes\n",
      " Listeria\n",
      " Listeria monocytogenes\n",
      " Listeria\n",
      " Listeria monocytogenes\n",
      " Listeria\n",
      " Listeria monocytogenes\n",
      " Listeria\n",
      " Listeria monocytogenes\n",
      " Listeria\n",
      " Listeria monocytogenes\n",
      " Listeria\n",
      " Listeria monocytogenes\n",
      " Listeria\n",
      " Listeria monocytogenes\n",
      " Listeria\n",
      " Listeria monocytogenes\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium abscessus\n",
      " Mycobacterium\n",
      " Mycobacterium africanum\n",
      " Mycobacterium\n",
      " Mycobacterium africanum\n",
      " Mycobacterium\n",
      " Mycobacterium africanum\n",
      " Mycobacterium\n",
      " Mycobacterium africanum\n",
      " Mycobacterium\n",
      " Mycobacterium africanum\n",
      " Mycobacterium\n",
      " Mycobacterium africanum\n",
      " Mycobacterium\n",
      " Mycobacterium africanum\n",
      " Mycobacterium\n",
      " Mycobacterium africanum\n",
      " Mycobacterium\n",
      " Mycobacterium africanum\n",
      " Mycobacterium\n",
      " Mycobacterium africanum\n",
      " Mycobacterium\n",
      " Mycobacterium africanum\n",
      " Mycobacterium\n",
      " Mycobacterium africanum\n",
      " Mycobacterium\n",
      " Mycobacterium africanum\n",
      " Mycobacterium\n",
      " Mycobacterium arupense\n",
      " Mycobacterium\n",
      " Mycobacterium arupense\n",
      " Mycobacterium\n",
      " Mycobacterium arupense\n",
      " Mycobacterium\n",
      " Mycobacterium arupense\n",
      " Mycobacterium\n",
      " Mycobacterium arupense\n",
      " Mycobacterium\n",
      " Mycobacterium arupense\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium avium\n",
      " Mycobacterium\n",
      " Mycobacterium bovis\n",
      " Mycobacterium\n",
      " Mycobacterium bovis\n",
      " Mycobacterium\n",
      " Mycobacterium bovis\n",
      " Mycobacterium\n",
      " Mycobacterium bovis\n",
      " Mycobacterium\n",
      " Mycobacterium bovis\n",
      " Mycobacterium\n",
      " Mycobacterium bovis\n",
      " Mycobacterium\n",
      " Mycobacterium bovis\n",
      " Mycobacterium\n",
      " Mycobacterium bovis\n",
      " Mycobacterium\n",
      " Mycobacterium bovis\n",
      " Mycobacterium\n",
      " Mycobacterium bovis\n",
      " Mycobacterium bovis BCG\n",
      " Mycobacterium\n",
      " Mycobacterium bovis\n",
      " Mycobacterium bovis BCG\n",
      " Mycobacterium\n",
      " Mycobacterium bovis\n",
      " Mycobacterium bovis BCG\n",
      " Mycobacterium\n",
      " Mycobacterium bovis\n",
      " Mycobacterium bovis BCG\n",
      " Mycobacterium\n",
      " Mycobacterium bovis\n",
      " Mycobacterium bovis BCG\n",
      " Mycobacterium\n",
      " Mycobacterium bovis\n",
      " Mycobacterium bovis BCG\n",
      " Mycobacterium\n",
      " Mycobacterium bovis\n",
      " Mycobacterium bovis BCG\n",
      " Mycobacterium\n",
      " Mycobacterium bovis\n",
      " Mycobacterium bovis BCG\n",
      " Mycobacterium\n",
      " Mycobacterium bovis\n",
      " Mycobacterium bovis BCG\n",
      " Mycobacterium\n",
      " Mycobacterium bovis\n",
      " Mycobacterium bovis BCG\n",
      " Mycobacterium\n",
      " Mycobacterium bovis\n",
      " Mycobacterium bovis BCG\n",
      " Mycobacterium\n",
      " Mycobacterium bovis\n",
      " Mycobacterium bovis BCG\n",
      " Mycobacterium\n",
      " Mycobacterium canettii\n",
      " Mycobacterium\n",
      " Mycobacterium canettii\n",
      " Mycobacterium\n",
      " Mycobacterium canettii\n",
      " Mycobacterium\n",
      " Mycobacterium canettii\n",
      " Mycobacterium\n",
      " Mycobacterium canettii\n",
      " Mycobacterium\n",
      " Mycobacterium canettii\n",
      " Mycobacterium\n",
      " Mycobacterium canettii\n",
      " Mycobacterium\n",
      " Mycobacterium celatum\n",
      " Mycobacterium\n",
      " Mycobacterium celatum\n",
      " Mycobacterium\n",
      " Mycobacterium celatum\n",
      " Mycobacterium\n",
      " Mycobacterium celatum\n",
      " Mycobacterium\n",
      " Mycobacterium celatum\n",
      " Mycobacterium\n",
      " Mycobacterium celatum\n",
      " Mycobacterium\n",
      " Mycobacterium celatum\n",
      " Mycobacterium\n",
      " Mycobacterium celatum\n",
      " Mycobacterium\n",
      " Mycobacterium celatum\n",
      " Mycobacterium\n",
      " Mycobacterium celatum\n",
      " Mycobacterium\n",
      " Mycobacterium celatum\n",
      " Mycobacterium\n",
      " Mycobacterium celatum\n",
      " Mycobacterium\n",
      " Mycobacterium celatum\n",
      " Mycobacterium\n",
      " Mycobacterium celatum\n",
      " Mycobacterium\n",
      " Mycobacterium celatum\n",
      " Mycobacterium\n",
      " Mycobacterium celatum\n",
      " Mycobacterium\n",
      " Mycobacterium celatum\n",
      " Mycobacterium\n",
      " Mycobacterium celatum\n",
      " Mycobacterium\n",
      " Mycobacterium chimaera\n",
      " Mycobacterium\n",
      " Mycobacterium chimaera\n",
      " Mycobacterium\n",
      " Mycobacterium chimaera\n",
      " Mycobacterium\n",
      " Mycobacterium chimaera\n",
      " Mycobacterium\n",
      " Mycobacterium chimaera\n",
      " Mycobacterium\n",
      " Mycobacterium chimaera\n",
      " Mycobacterium\n",
      " Mycobacterium chimaera\n",
      " Mycobacterium\n",
      " Mycobacterium chimaera\n",
      " Mycobacterium\n",
      " Mycobacterium chimaera\n",
      " Mycobacterium\n",
      " Mycobacterium chimaera\n",
      " Mycobacterium\n",
      " Mycobacterium chimaera\n",
      " Mycobacterium\n",
      " Mycobacterium chimaera\n",
      " Mycobacterium\n",
      " Mycobacterium chimaera\n",
      " Mycobacterium\n",
      " Mycobacterium chimaera\n",
      " Mycobacterium\n",
      " Mycobacterium chimaera\n",
      " Mycobacterium\n",
      " Mycobacterium chimaera\n",
      " Mycobacterium\n",
      " Mycobacterium chimaera\n",
      " Mycobacterium\n",
      " Mycobacterium chimaera\n",
      " Mycobacterium\n",
      " Mycobacterium chimaera\n",
      " Mycobacterium\n",
      " Mycobacterium chimaera\n",
      " Mycobacterium\n",
      " Mycobacterium chimaera\n",
      " Mycobacterium\n",
      " Mycobacterium chimaera\n",
      " Mycobacterium\n",
      " Mycobacterium chimaera\n",
      " Mycobacterium\n",
      " Mycobacterium chimaera\n",
      " Mycobacterium\n",
      " Mycobacterium chimaera\n",
      " Mycobacterium\n",
      " Mycobacterium chimaera\n",
      " Mycobacterium\n",
      " Mycobacterium chimaera\n",
      " Mycobacterium\n",
      " Mycobacterium chimaera\n",
      " Mycobacterium\n",
      " Mycobacterium chimaera\n",
      " Mycobacterium\n",
      " Mycobacterium genavense\n",
      " Mycobacterium\n",
      " Mycobacterium genavense\n",
      " Mycobacterium\n",
      " Mycobacterium genavense\n",
      " Mycobacterium\n",
      " Mycobacterium genavense\n",
      " Mycobacterium\n",
      " Mycobacterium genavense\n",
      " Mycobacterium\n",
      " Mycobacterium genavense\n",
      " Mycobacterium\n",
      " Mycobacterium genavense\n",
      " Mycobacterium\n",
      " Mycobacterium genavense\n",
      " Mycobacterium\n",
      " Mycobacterium genavense\n",
      " Mycobacterium\n",
      " Mycobacterium genavense\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium gordonae\n",
      " Mycobacterium\n",
      " Mycobacterium heckeshornense\n",
      " Mycobacterium\n",
      " Mycobacterium heckeshornense\n",
      " Mycobacterium\n",
      " Mycobacterium heckeshornense\n",
      " Mycobacterium\n",
      " Mycobacterium heckeshornense\n",
      " Mycobacterium\n",
      " Mycobacterium heckeshornense\n",
      " Mycobacterium\n",
      " Mycobacterium heckeshornense\n",
      " Mycobacterium\n",
      " Mycobacterium heckeshornense\n",
      " Mycobacterium\n",
      " Mycobacterium heckeshornense\n",
      " Mycobacterium\n",
      " Mycobacterium heckeshornense\n",
      " Mycobacterium\n",
      " Mycobacterium heckeshornense\n",
      " Mycobacterium\n",
      " Mycobacterium heckeshornense\n",
      " Mycobacterium\n",
      " Mycobacterium interjectum\n",
      " Mycobacterium\n",
      " Mycobacterium interjectum\n",
      " Mycobacterium\n",
      " Mycobacterium interjectum\n",
      " Mycobacterium\n",
      " Mycobacterium interjectum\n",
      " Mycobacterium\n",
      " Mycobacterium interjectum\n",
      " Mycobacterium\n",
      " Mycobacterium interjectum\n",
      " Mycobacterium\n",
      " Mycobacterium interjectum\n",
      " Mycobacterium\n",
      " Mycobacterium interjectum\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium intracellulare\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium kansasii\n",
      " Mycobacterium\n",
      " Mycobacterium lentiflavum\n",
      " Mycobacterium\n",
      " Mycobacterium lentiflavum\n",
      " Mycobacterium\n",
      " Mycobacterium lentiflavum\n",
      " Mycobacterium\n",
      " Mycobacterium lentiflavum\n",
      " Mycobacterium\n",
      " Mycobacterium lentiflavum\n",
      " Mycobacterium\n",
      " Mycobacterium lentiflavum\n",
      " Mycobacterium\n",
      " Mycobacterium lentiflavum\n",
      " Mycobacterium\n",
      " Mycobacterium lentiflavum\n",
      " Mycobacterium\n",
      " Mycobacterium lentiflavum\n",
      " Mycobacterium\n",
      " Mycobacterium lentiflavum\n",
      " Mycobacterium\n",
      " Mycobacterium lentiflavum\n",
      " Mycobacterium\n",
      " Mycobacterium lentiflavum\n",
      " Mycobacterium\n",
      " Mycobacterium lentiflavum\n",
      " Mycobacterium\n",
      " Mycobacterium lentiflavum\n",
      " Mycobacterium\n",
      " Mycobacterium lentiflavum\n",
      " Mycobacterium\n",
      " Mycobacterium lentiflavum\n",
      " Mycobacterium\n",
      " Mycobacterium lentiflavum\n",
      " Mycobacterium\n",
      " Mycobacterium lentiflavum\n",
      " Mycobacterium\n",
      " Mycobacterium leprae\n",
      " Mycobacterium\n",
      " Mycobacterium leprae\n",
      " Mycobacterium\n",
      " Mycobacterium leprae\n",
      " Mycobacterium\n",
      " Mycobacterium leprae\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium malmoense\n",
      " Mycobacterium\n",
      " Mycobacterium microti\n",
      " Mycobacterium\n",
      " Mycobacterium microti\n",
      " Mycobacterium\n",
      " Mycobacterium microti\n",
      " Mycobacterium\n",
      " Mycobacterium microti\n",
      " Mycobacterium\n",
      " Mycobacterium microti\n",
      " Mycobacterium\n",
      " Mycobacterium scrofulaceum\n",
      " Mycobacterium\n",
      " Mycobacterium scrofulaceum\n",
      " Mycobacterium\n",
      " Mycobacterium scrofulaceum\n",
      " Mycobacterium\n",
      " Mycobacterium scrofulaceum\n",
      " Mycobacterium\n",
      " Mycobacterium scrofulaceum\n",
      " Mycobacterium\n",
      " Mycobacterium scrofulaceum\n",
      " Mycobacterium\n",
      " Mycobacterium scrofulaceum\n",
      " Mycobacterium\n",
      " Mycobacterium scrofulaceum\n",
      " Mycobacterium\n",
      " Mycobacterium scrofulaceum\n",
      " Mycobacterium\n",
      " Mycobacterium scrofulaceum\n",
      " Mycobacterium\n",
      " Mycobacterium scrofulaceum\n",
      " Mycobacterium\n",
      " Mycobacterium scrofulaceum\n",
      " Mycobacterium\n",
      " Mycobacterium scrofulaceum\n",
      " Mycobacterium\n",
      " Mycobacterium scrofulaceum\n",
      " Mycobacterium\n",
      " Mycobacterium scrofulaceum\n",
      " Mycobacterium\n",
      " Mycobacterium scrofulaceum\n",
      " Mycobacterium\n",
      " Mycobacterium scrofulaceum\n",
      " Mycobacterium\n",
      " Mycobacterium scrofulaceum\n",
      " Mycobacterium\n",
      " Mycobacterium scrofulaceum\n",
      " Mycobacterium\n",
      " Mycobacterium scrofulaceum\n",
      " Mycobacterium\n",
      " Mycobacterium scrofulaceum\n",
      " Mycobacterium\n",
      " Mycobacterium shimoidei\n",
      " Mycobacterium\n",
      " Mycobacterium shimoidei\n",
      " Mycobacterium\n",
      " Mycobacterium shimoidei\n",
      " Mycobacterium\n",
      " Mycobacterium shimoidei\n",
      " Mycobacterium\n",
      " Mycobacterium shimoidei\n",
      " Mycobacterium\n",
      " Mycobacterium shimoidei\n",
      " Mycobacterium\n",
      " Mycobacterium shimoidei\n",
      " Mycobacterium\n",
      " Mycobacterium shimoidei\n",
      " Mycobacterium\n",
      " Mycobacterium shimoidei\n",
      " Mycobacterium\n",
      " Mycobacterium shimoidei\n",
      " Mycobacterium\n",
      " Mycobacterium shimoidei\n",
      " Mycobacterium\n",
      " Mycobacterium shimoidei\n",
      " Mycobacterium\n",
      " Mycobacterium shimoidei\n",
      " Mycobacterium\n",
      " Mycobacterium shimoidei\n",
      " Mycobacterium\n",
      " Mycobacterium shimoidei\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium simiae\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium szulgai\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium terrae\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium tuberculosis\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Mycobacterium\n",
      " Mycobacterium xenopi\n",
      " Nocardia\n",
      " Nocardia abscessus\n",
      " Nocardia\n",
      " Nocardia abscessus\n",
      " Nocardia\n",
      " Nocardia abscessus\n",
      " Nocardia\n",
      " Nocardia abscessus\n",
      " Nocardia\n",
      " Nocardia abscessus\n",
      " Nocardia\n",
      " Nocardia abscessus\n",
      " Nocardia\n",
      " Nocardia abscessus\n",
      " Nocardia\n",
      " Nocardia abscessus\n",
      " Nocardia\n",
      " Nocardia abscessus\n",
      " Nocardia\n",
      " Nocardia abscessus\n",
      " Nocardia\n",
      " Nocardia abscessus\n",
      " Nocardia\n",
      " Nocardia abscessus\n",
      " Nocardia\n",
      " Nocardia asiatica\n",
      " Nocardia\n",
      " Nocardia asiatica\n",
      " Nocardia\n",
      " Nocardia asiatica\n",
      " Nocardia\n",
      " Nocardia asiatica\n",
      " Nocardia\n",
      " Nocardia asiatica\n",
      " Nocardia\n",
      " Nocardia asiatica\n",
      " Nocardia\n",
      " Nocardia asiatica\n",
      " Nocardia\n",
      " Nocardia asiatica\n",
      " Nocardia\n",
      " Nocardia asiatica\n",
      " Nocardia\n",
      " Nocardia asiatica\n",
      " Nocardia\n",
      " Nocardia asiatica\n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia asteroides\n",
      " Nocardia asteroides \n",
      " Nocardia\n",
      " Nocardia beijingensis\n",
      " Nocardia beijingensis \n",
      " Nocardia\n",
      " Nocardia beijingensis\n",
      " Nocardia beijingensis \n",
      " Nocardia\n",
      " Nocardia beijingensis\n",
      " Nocardia beijingensis \n",
      " Nocardia\n",
      " Nocardia beijingensis\n",
      " Nocardia beijingensis \n",
      " Nocardia\n",
      " Nocardia beijingensis\n",
      " Nocardia beijingensis \n",
      " Nocardia\n",
      " Nocardia beijingensis\n",
      " Nocardia beijingensis \n",
      " Nocardia\n",
      " Nocardia beijingensis\n",
      " Nocardia beijingensis \n",
      " Nocardia\n",
      " Nocardia beijingensis\n",
      " Nocardia beijingensis \n",
      " Nocardia\n",
      " Nocardia beijingensis\n",
      " Nocardia beijingensis \n",
      " Nocardia\n",
      " Nocardia beijingensis\n",
      " Nocardia beijingensis \n",
      " Nocardia\n",
      " Nocardia beijingensis\n",
      " Nocardia beijingensis \n",
      " Nocardia\n",
      " Nocardia beijingensis\n",
      " Nocardia beijingensis \n",
      " Nocardia\n",
      " Nocardia beijingensis\n",
      " Nocardia beijingensis \n",
      " Nocardia\n",
      " Nocardia beijingensis\n",
      " Nocardia beijingensis \n",
      " Nocardia\n",
      " Nocardia beijingensis\n",
      " Nocardia beijingensis \n",
      " Nocardia\n",
      " Nocardia beijingensis\n",
      " Nocardia beijingensis \n",
      " Nocardia\n",
      " Nocardia beijingensis\n",
      " Nocardia beijingensis \n",
      " Nocardia\n",
      " Nocardia brasiliensis\n",
      " Nocardia brasiliensis \n",
      " Nocardia\n",
      " Nocardia brasiliensis\n",
      " Nocardia brasiliensis \n",
      " Nocardia\n",
      " Nocardia brasiliensis\n",
      " Nocardia brasiliensis \n",
      " Nocardia\n",
      " Nocardia brasiliensis\n",
      " Nocardia brasiliensis \n",
      " Nocardia\n",
      " Nocardia brasiliensis\n",
      " Nocardia brasiliensis \n",
      " Nocardia\n",
      " Nocardia brasiliensis\n",
      " Nocardia brasiliensis \n",
      " Nocardia\n",
      " Nocardia brasiliensis\n",
      " Nocardia brasiliensis \n",
      " Nocardia\n",
      " Nocardia brasiliensis\n",
      " Nocardia brasiliensis \n",
      " Nocardia\n",
      " Nocardia brasiliensis\n",
      " Nocardia brasiliensis \n",
      " Nocardia\n",
      " Nocardia brasiliensis\n",
      " Nocardia brasiliensis \n",
      " Nocardia\n",
      " Nocardia brasiliensis\n",
      " Nocardia brasiliensis \n",
      " Nocardia\n",
      " Nocardia brasiliensis\n",
      " Nocardia brasiliensis \n",
      " Nocardia\n",
      " Nocardia brasiliensis\n",
      " Nocardia brasiliensis \n",
      " Nocardia\n",
      " Nocardia brasiliensis\n",
      " Nocardia brasiliensis \n",
      " Nocardia\n",
      " Nocardia brasiliensis\n",
      " Nocardia brasiliensis \n",
      " Nocardia\n",
      " Nocardia brasiliensis\n",
      " Nocardia brasiliensis \n",
      " Nocardia\n",
      " Nocardia brasiliensis\n",
      " Nocardia brasiliensis \n",
      " Nocardia\n",
      " Nocardia brasiliensis\n",
      " Nocardia brasiliensis \n",
      " Nocardia\n",
      " Nocardia brasiliensis\n",
      " Nocardia brasiliensis \n",
      " Nocardia\n",
      " Nocardia brasiliensis\n",
      " Nocardia brasiliensis \n",
      " Nocardia\n",
      " Nocardia brasiliensis\n",
      " Nocardia brasiliensis \n",
      " Nocardia\n",
      " Nocardia caviae\n",
      " Nocardia caviae \n",
      " Nocardia\n",
      " Nocardia caviae\n",
      " Nocardia caviae \n",
      " Nocardia\n",
      " Nocardia caviae\n",
      " Nocardia caviae \n",
      " Nocardia\n",
      " Nocardia caviae\n",
      " Nocardia caviae \n",
      " Nocardia\n",
      " Nocardia caviae\n",
      " Nocardia caviae \n",
      " Nocardia\n",
      " Nocardia caviae\n",
      " Nocardia caviae \n",
      " Nocardia\n",
      " Nocardia caviae\n",
      " Nocardia caviae \n",
      " Nocardia\n",
      " Nocardia caviae\n",
      " Nocardia caviae \n",
      " Nocardia\n",
      " Nocardia caviae\n",
      " Nocardia caviae \n",
      " Nocardia\n",
      " Nocardia caviae\n",
      " Nocardia caviae \n",
      " Nocardia\n",
      " Nocardia caviae\n",
      " Nocardia caviae \n",
      " Nocardia\n",
      " Nocardia caviae\n",
      " Nocardia caviae \n",
      " Nocardia\n",
      " Nocardia caviae\n",
      " Nocardia caviae \n",
      " Nocardia\n",
      " Nocardia caviae\n",
      " Nocardia caviae \n",
      " Nocardia\n",
      " Nocardia caviae\n",
      " Nocardia caviae \n",
      " Nocardia\n",
      " Nocardia caviae\n",
      " Nocardia caviae \n",
      " Nocardia\n",
      " Nocardia caviae\n",
      " Nocardia caviae \n",
      " Nocardia\n",
      " Nocardia caviae\n",
      " Nocardia caviae \n",
      " Nocardia\n",
      " Nocardia caviae\n",
      " Nocardia caviae \n",
      " Nocardia\n",
      " Nocardia caviae\n",
      " Nocardia caviae \n",
      " Nocardia\n",
      " Nocardia caviae\n",
      " Nocardia caviae \n",
      " Nocardia\n",
      " Nocardia caviae\n",
      " Nocardia caviae \n",
      " Nocardia\n",
      " Nocardia caviae\n",
      " Nocardia caviae \n",
      " Nocardia\n",
      " Nocardia caviae\n",
      " Nocardia caviae \n",
      " Nocardia\n",
      " Nocardia caviae\n",
      " Nocardia caviae \n",
      " Nocardia\n",
      " Nocardia caviae\n",
      " Nocardia caviae \n",
      " Nocardia\n",
      " Nocardia caviae\n",
      " Nocardia caviae \n",
      " Nocardia\n",
      " Nocardia caviae\n",
      " Nocardia caviae \n",
      " Nocardia\n",
      " Nocardia caviae\n",
      " Nocardia caviae \n",
      " Nocardia\n",
      " Nocardia caviae\n",
      " Nocardia caviae \n",
      " Nocardia\n",
      " Nocardia caviae\n",
      " Nocardia caviae \n",
      " Nocardia\n",
      " Nocardia caviae\n",
      " Nocardia caviae \n",
      " Nocardia\n",
      " Nocardia caviae\n",
      " Nocardia caviae \n",
      " Nocardia\n",
      " Nocardia caviae\n",
      " Nocardia caviae \n",
      " Nocardia\n",
      " Nocardia caviae\n",
      " Nocardia caviae \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia cyriacigeorgica\n",
      " Nocardia cyriacigeorgica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia farcinica\n",
      " Nocardia farcinica \n",
      " Nocardia\n",
      " Nocardia nova\n",
      " Nocardia nova \n",
      " Nocardia\n",
      " Nocardia nova\n",
      " Nocardia nova \n",
      " Nocardia\n",
      " Nocardia nova\n",
      " Nocardia nova \n",
      " Nocardia\n",
      " Nocardia nova\n",
      " Nocardia nova \n",
      " Nocardia\n",
      " Nocardia nova\n",
      " Nocardia nova \n",
      " Nocardia\n",
      " Nocardia nova\n",
      " Nocardia nova \n",
      " Nocardia\n",
      " Nocardia nova\n",
      " Nocardia nova \n",
      " Nocardia\n",
      " Nocardia nova\n",
      " Nocardia nova \n",
      " Nocardia\n",
      " Nocardia nova\n",
      " Nocardia nova \n",
      " Nocardia\n",
      " Nocardia nova\n",
      " Nocardia nova \n",
      " Nocardia\n",
      " Nocardia nova\n",
      " Nocardia nova \n",
      " Nocardia\n",
      " Nocardia nova\n",
      " Nocardia nova \n",
      " Nocardia\n",
      " Nocardia nova\n",
      " Nocardia nova \n",
      " Nocardia\n",
      " Nocardia nova\n",
      " Nocardia nova \n",
      " Nocardia\n",
      " Nocardia nova\n",
      " Nocardia nova \n",
      " Nocardia\n",
      " Nocardia nova\n",
      " Nocardia nova \n",
      " Nocardia\n",
      " Nocardia nova\n",
      " Nocardia nova \n",
      " Nocardia\n",
      " Nocardia nova\n",
      " Nocardia nova \n",
      " Nocardia\n",
      " Nocardia nova\n",
      " Nocardia nova \n",
      " Nocardia\n",
      " Nocardia nova\n",
      " Nocardia nova \n",
      " Nocardia\n",
      " Nocardia nova\n",
      " Nocardia nova \n",
      " Nocardia\n",
      " Nocardia nova\n",
      " Nocardia nova \n",
      " Nocardia\n",
      " Nocardia nova\n",
      " Nocardia nova \n",
      " Nocardia\n",
      " Nocardia nova\n",
      " Nocardia nova \n",
      " Nocardia\n",
      " Nocardia nova\n",
      " Nocardia nova \n",
      " Nocardia\n",
      " Nocardia nova\n",
      " Nocardia nova \n",
      " Nocardia\n",
      " Nocardia nova\n",
      " Nocardia nova \n",
      " Nocardia\n",
      " Nocardia nova\n",
      " Nocardia nova \n",
      " Nocardia\n",
      " Nocardia nova\n",
      " Nocardia nova \n",
      " Nocardia\n",
      " Nocardia nova\n",
      " Nocardia nova \n",
      " Nocardia\n",
      " Nocardia nova\n",
      " Nocardia nova \n",
      " Nocardia\n",
      " Nocardia nova\n",
      " Nocardia nova \n",
      " Nocardia\n",
      " Nocardia nova\n",
      " Nocardia nova \n",
      " Nocardia\n",
      " Nocardia nova\n",
      " Nocardia nova \n",
      " Nocardia\n",
      " Nocardia nova\n",
      " Nocardia nova \n",
      " Nocardia\n",
      " Nocardia otitidiscaviarum\n",
      " Nocardia otitidiscaviarum \n",
      " Nocardia\n",
      " Nocardia otitidiscaviarum\n",
      " Nocardia otitidiscaviarum \n",
      " Nocardia\n",
      " Nocardia otitidiscaviarum\n",
      " Nocardia otitidiscaviarum \n",
      " Nocardia\n",
      " Nocardia otitidiscaviarum\n",
      " Nocardia otitidiscaviarum \n",
      " Nocardia\n",
      " Nocardia otitidiscaviarum\n",
      " Nocardia otitidiscaviarum \n",
      " Nocardia\n",
      " Nocardia otitidiscaviarum\n",
      " Nocardia otitidiscaviarum \n",
      " Nocardia\n",
      " Nocardia otitidiscaviarum\n",
      " Nocardia otitidiscaviarum \n",
      " Nocardia\n",
      " Nocardia otitidiscaviarum\n",
      " Nocardia otitidiscaviarum \n",
      " Nocardia\n",
      " Nocardia otitidiscaviarum\n",
      " Nocardia otitidiscaviarum \n",
      " Nocardia\n",
      " Nocardia otitidiscaviarum\n",
      " Nocardia otitidiscaviarum \n",
      " Nocardia\n",
      " Nocardia otitidiscaviarum\n",
      " Nocardia otitidiscaviarum \n",
      " Nocardia\n",
      " Nocardia otitidiscaviarum\n",
      " Nocardia otitidiscaviarum \n",
      " Nocardia\n",
      " Nocardia otitidiscaviarum\n",
      " Nocardia otitidiscaviarum \n",
      " Nocardia\n",
      " Nocardia otitidiscaviarum\n",
      " Nocardia otitidiscaviarum \n",
      " Nocardia\n",
      " Nocardia otitidiscaviarum\n",
      " Nocardia otitidiscaviarum \n",
      " Nocardia\n",
      " Nocardia otitidiscaviarum\n",
      " Nocardia otitidiscaviarum \n",
      " Nocardia\n",
      " Nocardia otitidiscaviarum\n",
      " Nocardia otitidiscaviarum \n",
      " Nocardia\n",
      " Nocardia otitidiscaviarum\n",
      " Nocardia otitidiscaviarum \n",
      " Nocardia\n",
      " Nocardia otitidiscaviarum\n",
      " Nocardia otitidiscaviarum \n",
      " Nocardia\n",
      " Nocardia otitidiscaviarum\n",
      " Nocardia otitidiscaviarum \n",
      " Nocardia\n",
      " Nocardia otitidiscaviarum\n",
      " Nocardia otitidiscaviarum \n",
      " Nocardia\n",
      " Nocardia otitidiscaviarum\n",
      " Nocardia otitidiscaviarum \n",
      " Nocardia\n",
      " Nocardia otitidiscaviarum\n",
      " Nocardia otitidiscaviarum \n",
      " Nocardia\n",
      " Nocardia otitidiscaviarum\n",
      " Nocardia otitidiscaviarum \n",
      " Nocardia\n",
      " Nocardia otitidiscaviarum\n",
      " Nocardia otitidiscaviarum \n",
      " Nocardia\n",
      " Nocardia otitidiscaviarum\n",
      " Nocardia otitidiscaviarum \n",
      " Nocardia\n",
      " Nocardia otitidiscaviarum\n",
      " Nocardia otitidiscaviarum \n",
      " Nocardia\n",
      " Nocardia otitidiscaviarum\n",
      " Nocardia otitidiscaviarum \n",
      " Nocardia\n",
      " Nocardia otitidiscaviarum\n",
      " Nocardia otitidiscaviarum \n",
      " Nocardia\n",
      " Nocardia otitidiscaviarum\n",
      " Nocardia otitidiscaviarum \n",
      " Nocardia\n",
      " Nocardia otitidiscaviarum\n",
      " Nocardia otitidiscaviarum \n",
      " Nocardia\n",
      " Nocardia otitidiscaviarum\n",
      " Nocardia otitidiscaviarum \n",
      " Nocardia\n",
      " Nocardia otitidiscaviarum\n",
      " Nocardia otitidiscaviarum \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia pneumoniae\n",
      " Nocardia pneumoniae \n",
      " Nocardia\n",
      " Nocardia transvalensis\n",
      " Nocardia transvalensis \n",
      " Nocardia\n",
      " Nocardia transvalensis\n",
      " Nocardia transvalensis \n",
      " Nocardia\n",
      " Nocardia transvalensis\n",
      " Nocardia transvalensis \n",
      " Nocardia\n",
      " Nocardia transvalensis\n",
      " Nocardia transvalensis \n",
      " Nocardia\n",
      " Nocardia transvalensis\n",
      " Nocardia transvalensis \n",
      " Nocardia\n",
      " Nocardia transvalensis\n",
      " Nocardia transvalensis \n",
      " Nocardia\n",
      " Nocardia transvalensis\n",
      " Nocardia transvalensis \n",
      " Nocardia\n",
      " Nocardia transvalensis\n",
      " Nocardia transvalensis \n",
      " Nocardia\n",
      " Nocardia transvalensis\n",
      " Nocardia transvalensis \n",
      " Nocardia\n",
      " Nocardia transvalensis\n",
      " Nocardia transvalensis \n",
      " Nocardia\n",
      " Nocardia transvalensis\n",
      " Nocardia transvalensis \n",
      " Nocardia\n",
      " Nocardia transvalensis\n",
      " Nocardia transvalensis \n",
      " Nocardia\n",
      " Nocardia transvalensis\n",
      " Nocardia transvalensis \n",
      " Nocardia\n",
      " Nocardia transvalensis\n",
      " Nocardia transvalensis \n",
      " Nocardia\n",
      " Nocardia transvalensis\n",
      " Nocardia transvalensis \n",
      " Nocardia\n",
      " Nocardia transvalensis\n",
      " Nocardia transvalensis \n",
      " Nocardia\n",
      " Nocardia transvalensis\n",
      " Nocardia transvalensis \n",
      " Nocardia\n",
      " Nocardia transvalensis\n",
      " Nocardia transvalensis \n",
      " Nocardia\n",
      " Nocardia transvalensis\n",
      " Nocardia transvalensis \n",
      " Nocardia\n",
      " Nocardia transvalensis\n",
      " Nocardia transvalensis \n",
      " Nocardia\n",
      " Nocardia transvalensis\n",
      " Nocardia transvalensis \n",
      " Nocardia\n",
      " Nocardia transvalensis\n",
      " Nocardia transvalensis \n",
      " Nocardia\n",
      " Nocardia transvalensis\n",
      " Nocardia transvalensis \n",
      " Nocardia\n",
      " Nocardia transvalensis\n",
      " Nocardia transvalensis \n",
      " Nocardia\n",
      " Nocardia transvalensis\n",
      " Nocardia transvalensis \n",
      " Rhodococcus\n",
      " Rhodococcus equi\n",
      " Rhodococcus\n",
      " Rhodococcus equi\n",
      " Rhodococcus\n",
      " Rhodococcus equi\n",
      " Rhodococcus\n",
      " Rhodococcus equi\n",
      " Rhodococcus\n",
      " Rhodococcus equi\n",
      " Rhodococcus\n",
      " Rhodococcus equi\n",
      " Rhodococcus\n",
      " Rhodococcus equi\n",
      " Rhodococcus\n",
      " Rhodococcus equi\n",
      " Rhodococcus\n",
      " Rhodococcus equi\n",
      " Rhodococcus\n",
      " Rhodococcus equi\n",
      " Rhodococcus\n",
      " Rhodococcus equi\n",
      " Rhodococcus\n",
      " Rhodococcus equi\n",
      " Rhodococcus\n",
      " Rhodococcus equi\n",
      " Rhodococcus\n",
      " Rhodococcus equi\n",
      " Rhodococcus\n",
      " Rhodococcus equi\n",
      " Rhodococcus\n",
      " Rhodococcus equi\n",
      " Rhodococcus\n",
      " Rhodococcus equi\n",
      " Rhodococcus\n",
      " Rhodococcus equi\n",
      " Rhodococcus\n",
      " Rhodococcus equi\n",
      " Rhodococcus\n",
      " Rhodococcus equi\n",
      " Rhodococcus\n",
      " Rhodococcus equi\n",
      " Rhodococcus\n",
      " Rhodococcus equi\n",
      " Rhodococcus\n",
      " Rhodococcus equi\n",
      " Rhodococcus\n",
      " Rhodococcus equi\n",
      " Rhodococcus\n",
      " Rhodococcus equi\n",
      " Rhodococcus\n",
      " Rhodococcus equi\n",
      " Rhodococcus\n",
      " Rhodococcus equi\n",
      " Rhodococcus\n",
      " Rhodococcus equi\n",
      " Rhodococcus\n",
      " Rhodococcus equi\n",
      " Rhodococcus\n",
      " Rhodococcus equi\n",
      " Rhodococcus\n",
      " Rhodococcus equi\n",
      " Rhodococcus\n",
      " Rhodococcus equi\n",
      " Rhodococcus\n",
      " Rhodococcus equi\n",
      " Rhodococcus\n",
      " Rhodococcus equi\n",
      " Thermoactinomyces\n",
      " Thermoactinomyces\n",
      " Thermoactinomyces\n",
      " Thermoactinomyces\n",
      " Trueperella\n",
      " Trueperella pyogenes\n",
      " Tsukamurella\n",
      " Tsukamurella strandjordi\n",
      " Tsukamurella\n",
      " Tsukamurella strandjordi\n",
      " Tsukamurella\n",
      " Tsukamurella strandjordi\n",
      " Tsukamurella\n",
      " Tsukamurella strandjordi\n",
      " Tsukamurella\n",
      " Tsukamurella strandjordi\n",
      " Tsukamurella\n",
      " Tsukamurella strandjordi\n",
      " Tsukamurella\n",
      " Tsukamurella strandjordi\n",
      " Tsukamurella\n",
      " Tsukamurella strandjordi\n",
      " Tsukamurella\n",
      " Tsukamurella strandjordi\n",
      " Tsukamurella\n",
      " Tsukamurella strandjordi\n",
      " Tsukamurella\n",
      " Tsukamurella strandjordi\n",
      " Tsukamurella\n",
      " Tsukamurella strandjordi\n",
      " Tsukamurella\n",
      " Tsukamurella strandjordi\n",
      " Tsukamurella\n",
      " Tsukamurella strandjordi\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia pneumoniae\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia psittaci\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Chlamydia\n",
      " Chlamydia trachomatis\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Coxiella\n",
      " Coxiella burnetii\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma amphoriformeb\n",
      " Mycoplasma\n",
      " Mycoplasma fermentans\n",
      " Mycoplasma\n",
      " Mycoplasma fermentans\n",
      " Mycoplasma\n",
      " Mycoplasma fermentans\n",
      " Mycoplasma\n",
      " Mycoplasma fermentans\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma genitaliumc\n",
      " Mycoplasma\n",
      " Mycoplasma hominis\n",
      " Mycoplasma\n",
      " Mycoplasma hominis\n",
      " Mycoplasma\n",
      " Mycoplasma hominis\n",
      " Mycoplasma\n",
      " Mycoplasma hominis\n",
      " Mycoplasma\n",
      " Mycoplasma hominis\n",
      " Mycoplasma\n",
      " Mycoplasma hominis\n",
      " Mycoplasma\n",
      " Mycoplasma hominis\n",
      " Mycoplasma\n",
      " Mycoplasma hominis\n",
      " Mycoplasma\n",
      " Mycoplasma hominis\n",
      " Mycoplasma\n",
      " Mycoplasma hominis\n",
      " Mycoplasma\n",
      " Mycoplasma hominis\n",
      " Mycoplasma\n",
      " Mycoplasma hominis\n",
      " Mycoplasma\n",
      " Mycoplasma hominis\n",
      " Mycoplasma\n",
      " Mycoplasma hominis\n",
      " Mycoplasma\n",
      " Mycoplasma hominis\n",
      " Mycoplasma\n",
      " Mycoplasma hominis\n",
      " Mycoplasma\n",
      " Mycoplasma hominis\n",
      " Mycoplasma\n",
      " Mycoplasma hominis\n",
      " Mycoplasma\n",
      " Mycoplasma hominis\n",
      " Mycoplasma\n",
      " Mycoplasma hominis\n",
      " Mycoplasma\n",
      " Mycoplasma hominis\n",
      " Mycoplasma\n",
      " Mycoplasma hominis\n",
      " Mycoplasma\n",
      " Mycoplasma hominis\n",
      " Mycoplasma\n",
      " Mycoplasma hominis\n",
      " Mycoplasma\n",
      " Mycoplasma hominis\n",
      " Mycoplasma\n",
      " Mycoplasma hominis\n",
      " Mycoplasma\n",
      " Mycoplasma hominis\n",
      " Mycoplasma\n",
      " Mycoplasma hominis\n",
      " Mycoplasma\n",
      " Mycoplasma hominis\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma pneumoniae\n",
      " Mycoplasma\n",
      " Mycoplasma salivarium\n",
      " Mycoplasma\n",
      " Mycoplasma salivarium\n",
      " Mycoplasma\n",
      " Mycoplasma salivarium\n",
      " Mycoplasma\n",
      " Mycoplasma salivarium\n",
      " Orientia\n",
      " Orientia tsutsugamushi\n",
      " Orientia\n",
      " Orientia tsutsugamushi\n",
      " Orientia\n",
      " Orientia tsutsugamushi\n",
      " Orientia\n",
      " Orientia tsutsugamushi\n",
      " Orientia\n",
      " Orientia tsutsugamushi\n",
      " Orientia\n",
      " Orientia tsutsugamushi\n",
      " Orientia\n",
      " Orientia tsutsugamushi\n",
      " Orientia\n",
      " Orientia tsutsugamushi\n",
      " Orientia\n",
      " Orientia tsutsugamushi\n",
      " Orientia\n",
      " Orientia tsutsugamushi\n",
      " Orientia\n",
      " Orientia tsutsugamushi\n",
      " Orientia\n",
      " Orientia tsutsugamushi\n",
      " Orientia\n",
      " Orientia tsutsugamushi\n",
      " Orientia\n",
      " Orientia tsutsugamushi\n",
      " Orientia\n",
      " Orientia tsutsugamushi\n",
      " Orientia\n",
      " Orientia tsutsugamushi\n",
      " Rickettsia\n",
      " Rickettsia conorii\n",
      " Rickettsia\n",
      " Rickettsia conorii\n",
      " Rickettsia\n",
      " Rickettsia conorii\n",
      " Rickettsia\n",
      " Rickettsia conorii\n",
      " Rickettsia\n",
      " Rickettsia typhi\n",
      " Rickettsia\n",
      " Rickettsia typhi\n",
      " Rickettsia\n",
      " Rickettsia typhi\n",
      " Rickettsia\n",
      " Rickettsia typhi\n",
      " Rickettsia\n",
      " Rickettsia typhi\n",
      " Rickettsia\n",
      " Rickettsia typhi\n",
      " Rickettsia\n",
      " Rickettsia typhi\n",
      " Rickettsia\n",
      " Rickettsia typhi\n",
      " Tropheryma\n",
      " Tropheryma whipplei\n",
      " Tropheryma\n",
      " Tropheryma whipplei\n",
      " Tropheryma\n",
      " Tropheryma whipplei\n",
      " Tropheryma\n",
      " Tropheryma whipplei\n",
      " Tropheryma\n",
      " Tropheryma whipplei\n",
      " Tropheryma\n",
      " Tropheryma whipplei\n",
      " Tropheryma\n",
      " Tropheryma whipplei\n",
      " Tropheryma\n",
      " Tropheryma whipplei\n",
      " Tropheryma\n",
      " Tropheryma whipplei\n",
      " Tropheryma\n",
      " Tropheryma whipplei\n",
      " Ureaplasma\n",
      " Ureaplasma\n",
      " Ureaplasma\n",
      " Ureaplasma\n",
      " Ureaplasma\n",
      " Ureaplasma\n",
      " Ureaplasma\n",
      " Ureaplasma\n",
      " Ureaplasma\n",
      " Ureaplasma\n",
      " Ureaplasma\n",
      " Ureaplasma\n",
      " Ureaplasma\n",
      " Ureaplasma\n",
      " Ureaplasma\n",
      " Ureaplasma\n",
      " Ureaplasma\n",
      " Ureaplasma\n",
      " Ureaplasma\n",
      " Ureaplasma\n",
      " Ureaplasma\n",
      " Ureaplasma\n",
      " Ureaplasma\n",
      " Ureaplasma\n",
      " Ureaplasma\n",
      " Ureaplasma\n",
      " Ureaplasma\n",
      " Ureaplasma\n",
      " Ureaplasma\n",
      " Ureaplasma\n",
      " Ureaplasma\n",
      " Ureaplasma\n",
      " Ureaplasma\n",
      " Ureaplasma\n",
      " Ureaplasma\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "<ipython-input-16-45d15c401fbb>:10: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
      "  a[\"SPECIES\"] = c.append(spec)\n"
     ]
    }
   ],
   "source": [
    "b = a[\"SPECIES\"]\n",
    "\n",
    "c = []\n",
    "for i in b:\n",
    "    spec = \"\"\n",
    "    for j in i:\n",
    "        spec = spec + \" \" + j\n",
    "        print(spec)\n",
    "        c.append(spec)\n",
    "a[\"SPECIES\"] = c.append(spec)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.8"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
